{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Semantic Scholar API\n",
    "100 requests per 5 minutes\n",
    "The API allows up to 100 requests per 5 minutes. \n",
    "\n",
    "To access a higher rate limit, complete the form to request authentication for your project.\n",
    "\n",
    "Max limit for each request is 100, so every 5 min we can gather (100*100)* amount of people\n",
    "\n",
    "Issue/Drawback of semantic scholar: it needs a search term, meaning, the topics/industry/niche needs to be chosen "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "import json\n",
    "import pandas as pd\n",
    "import time\n",
    "import string\n",
    "import json\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Parameters for retrieving data\n",
    "amount_of_data = 100  # Total amount of data to retrieve\n",
    "year_of_publication = 2015  # Year of publication\n",
    "journal = [\"Science\"]\n",
    "# research_field = ['Computer Science'] # Research field(s) to query (as a list)\n",
    "alphabet_letters = ['A','B','C']#list(string.ascii_uppercase) # Research field(s) to query (as a list)\n",
    "initial_offset = 0  # Initial start offset\n",
    "number_of_publications_per_request = 100\n",
    "\n",
    "# Gentle fetching\n",
    "sleep_duration = 5  # Sleep duration in seconds between requests\n",
    "max_retries = 3  # Maximum number of retries for failed requests\n",
    "\n",
    "# Define the API endpoint\n",
    "api_url = 'https://api.semanticscholar.org/graph/v1/paper/search'\n",
    "length_df = 0"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [],
   "source": [
    "params = {\n",
    "    'query': alphabet_letters[0],  # Use the first research field from the list\n",
    "    'venue': journal[0],\n",
    "    'year': year_of_publication,\n",
    "    'limit': 1,  # Number of results per request\n",
    "    'offset': 0,  # Offset for pagination\n",
    "    'fields': 'title,authors,abstract,citationCount,year',\n",
    "}\n",
    "\n",
    "# Make the GET request\n",
    "response = requests.get(api_url, params=params)\n",
    "\n",
    "# Check if the request was successful\n",
    "if response.status_code == 200:\n",
    "    data = json.loads(response.text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "For the letter:  A  batches:  [100, 100]\n",
      "total amount {'total': 250, 'offset': 0, 'next': 100, 'data': [{'paperId': 'ef160aee837c415ee3168efb0fe515ec2df5b2e8', 'title': 'Estimating the reproducibility of psychological science', 'abstract': 'Empirically analyzing empirical evidence One of the central goals in any scientific endeavor is to understand causality. Experiments that seek to demonstrate a cause/effect relation most often manipulate the postulated causal factor. Aarts et al. describe the replication of 100 experiments reported in papers published in 2008 in three high-ranking psychology journals. Assessing whether the replication and the original experiment yielded the same result according to several criteria, they find that about one-third to one-half of the original findings were also observed in the replication study. Science, this issue 10.1126/science.aac4716 A large-scale assessment suggests that experimental reproducibility in psychology leaves a lot to be desired. INTRODUCTION Reproducibility is a defining feature of science, but the extent to which it characterizes current research is unknown. Scientific claims should not gain credence because of the status or authority of their originator but by the replicability of their supporting evidence. Even research of exemplary quality may have irreproducible empirical findings because of random or systematic error. RATIONALE There is concern about the rate and predictors of reproducibility, but limited evidence. Potentially problematic practices include selective reporting, selective analysis, and insufficient specification of the conditions necessary or sufficient to obtain the results. Direct replication is the attempt to recreate the conditions believed sufficient for obtaining a previously observed finding and is the means of establishing reproducibility of a finding with new data. We conducted a large-scale, collaborative effort to obtain an initial estimate of the reproducibility of psychological science. RESULTS We conducted replications of 100 experimental and correlational studies published in three psychology journals using high-powered designs and original materials when available. There is no single standard for evaluating replication success. Here, we evaluated reproducibility using significance and P values, effect sizes, subjective assessments of replication teams, and meta-analysis of effect sizes. The mean effect size (r) of the replication effects (Mr = 0.197, SD = 0.257) was half the magnitude of the mean effect size of the original effects (Mr = 0.403, SD = 0.188), representing a substantial decline. Ninety-seven percent of original studies had significant results (P < .05). Thirty-six percent of replications had significant results; 47% of original effect sizes were in the 95% confidence interval of the replication effect size; 39% of effects were subjectively rated to have replicated the original result; and if no bias in original results is assumed, combining original and replication results left 68% with statistically significant effects. Correlational tests suggest that replication success was better predicted by the strength of original evidence than by characteristics of the original and replication teams. CONCLUSION No single indicator sufficiently describes replication success, and the five indicators examined here are not the only ways to evaluate reproducibility. Nonetheless, collectively these results offer a clear conclusion: A large portion of replications produced weaker evidence for the original findings despite using materials provided by the original authors, review in advance for methodological fidelity, and high statistical power to detect the original effect sizes. Moreover, correlational evidence is consistent with the conclusion that variation in the strength of initial evidence (such as original P value) was more predictive of replication success than variation in the characteristics of the teams conducting the research (such as experience and expertise). The latter factors certainly can influence replication success, but they did not appear to do so here. Reproducibility is not well understood because the incentives for individual scientists prioritize novelty over replication. Innovation is the engine of discovery and is vital for a productive, effective scientific enterprise. However, innovative ideas become old news fast. Journal reviewers and editors may dismiss a new test of a published idea as unoriginal. The claim that “we already know this” belies the uncertainty of scientific evidence. Innovation points out paths that are possible; replication points out paths that are likely; progress relies on both. Replication can increase certainty when findings are reproduced and promote innovation when they are not. This project provides accumulating evidence for many findings in psychological research and suggests that there is still more work to do to verify whether we know what we think we know. Original study effect size versus replication effect size (correlation coefficients). Diagonal line represents replication effect size equal to original effect size. Dotted line represents replication effect size of 0. Points below the dotted line were effects in the opposite direction of the original. Density plots are separated by significant (blue) and nonsignificant (red) effects. Reproducibility is a defining feature of science, but the extent to which it characterizes current research is unknown. We conducted replications of 100 experimental and correlational studies published in three psychology journals using high-powered designs and original materials when available. Replication effects were half the magnitude of original effects, representing a substantial decline. Ninety-seven percent of original studies had statistically significant results. Thirty-six percent of replications had statistically significant results; 47% of original effect sizes were in the 95% confidence interval of the replication effect size; 39% of effects were subjectively rated to have replicated the original result; and if no bias in original results is assumed, combining original and replication results left 68% with statistically significant effects. Correlational tests suggest that replication success was better predicted by the strength of original evidence than by characteristics of the original and replication teams.', 'year': 2015, 'citationCount': 5746, 'authors': [{'authorId': '2231439', 'name': 'Alexander A. Aarts'}, {'authorId': '1400206933', 'name': 'Joanna E. Anderson'}, {'authorId': '96413924', 'name': 'Christopher J. Anderson'}, {'authorId': '113650771', 'name': 'Peter Raymond Attridge'}, {'authorId': '1788796', 'name': 'A. Attwood'}, {'authorId': '152159926', 'name': 'Jordan R. Axt'}, {'authorId': '36417745', 'name': 'Molly Babel'}, {'authorId': '5252377', 'name': 'Š. Bahník'}, {'authorId': '6174975', 'name': 'E. Baranski'}, {'authorId': '1387458734', 'name': 'M. Barnett-Cowan'}, {'authorId': '4867423', 'name': 'E. Bartmess'}, {'authorId': '3039399', 'name': 'Jennifer S. Beer'}, {'authorId': '47431109', 'name': 'Raoul Bell'}, {'authorId': '2074992746', 'name': 'Heather Bentley'}, {'authorId': '114670541', 'name': 'Leah Beyan'}, {'authorId': '30661345', 'name': 'Grace Binion'}, {'authorId': '3237553', 'name': 'D. Borsboom'}, {'authorId': '116191440', 'name': 'Annick Bosch'}, {'authorId': '8597004', 'name': 'Frank Bosco'}, {'authorId': '72356050', 'name': 'Sara D. Bowman'}, {'authorId': '145470466', 'name': 'M. Brandt'}, {'authorId': '66335862', 'name': 'E. L. Braswell'}, {'authorId': '51904698', 'name': 'Hilmar Brohmer'}, {'authorId': '48120787', 'name': 'B. T. Brown'}, {'authorId': '4197188', 'name': 'Kristina Brown'}, {'authorId': '31412979', 'name': 'Jovita Brüning'}, {'authorId': '1400158480', 'name': 'Ann Calhoun-Sauls'}, {'authorId': '36027478', 'name': 'Shannon P. Callahan'}, {'authorId': '95354096', 'name': 'E. Chagnon'}, {'authorId': '48719411', 'name': 'Jesse J. Chandler'}, {'authorId': '38049428', 'name': 'Christopher R. Chartier'}, {'authorId': '145787104', 'name': 'F. Cheung'}, {'authorId': '51055585', 'name': 'Cody Daniel Christopherson'}, {'authorId': '4704887', 'name': 'Linda Cillessen'}, {'authorId': '48987469', 'name': 'R. Clay'}, {'authorId': '46550612', 'name': 'Hayley M. D. Cleary'}, {'authorId': '117283612', 'name': 'M. Cloud'}, {'authorId': '143947719', 'name': 'Michael Conn'}, {'authorId': '1795487', 'name': 'J. Cohoon'}, {'authorId': '7293849', 'name': 'Simon Columbus'}, {'authorId': '2052289610', 'name': 'A. Cordes'}, {'authorId': '49251371', 'name': 'G. Costantini'}, {'authorId': '15675192', 'name': 'Leslie D. Cramblet Alvarez'}, {'authorId': '51034966', 'name': 'E. Cremata'}, {'authorId': '104792844', 'name': 'J. Crusius'}, {'authorId': '2480852', 'name': 'J. DeCoster'}, {'authorId': '122378013', 'name': 'M. DeGaetano'}, {'authorId': '1665535543', 'name': 'Nicolás Delia Penna'}, {'authorId': '1665493156', 'name': 'Bobby Den Bezemer'}, {'authorId': '32368039', 'name': 'M. Deserno'}, {'authorId': '113076847', 'name': 'Olivia Devitt'}, {'authorId': '6629103', 'name': 'L. Dewitte'}, {'authorId': '6958962', 'name': 'David G. Dobolyi'}, {'authorId': '16313368', 'name': 'Geneva T. Dodson'}, {'authorId': '2467192', 'name': 'M. Donnellan'}, {'authorId': '2094220350', 'name': 'Ryan Donohue'}, {'authorId': '5663433', 'name': 'Rebecca A. Dore'}, {'authorId': '5021006', 'name': 'A. Dorrough'}, {'authorId': '3488772', 'name': 'Anna Dreber'}, {'authorId': '5818783', 'name': 'M. Dugas'}, {'authorId': '3100527', 'name': 'Elizabeth W. Dunn'}, {'authorId': '14421983', 'name': 'Kayleigh E. Easey'}, {'authorId': '117807982', 'name': 'Sylvia Eboigbe'}, {'authorId': '34959578', 'name': 'C. Eggleston'}, {'authorId': '123302634', 'name': 'Jo Embley'}, {'authorId': '3018477', 'name': 'S. Epskamp'}, {'authorId': '52004348', 'name': 'Timothy M. Errington'}, {'authorId': '117698833', 'name': 'Vivien Estel'}, {'authorId': '3392676', 'name': 'Frank J. Farach'}, {'authorId': '25452274', 'name': 'J. Feather'}, {'authorId': '2322314', 'name': 'A. Fedor'}, {'authorId': '1400158404', 'name': 'B. Fernández-Castilla'}, {'authorId': '38548340', 'name': 'S. Fiedler'}, {'authorId': '32576544', 'name': 'James G. Field'}, {'authorId': '2237526', 'name': 'Stanka A. Fitneva'}, {'authorId': '2387162', 'name': 'T. Flagan'}, {'authorId': '6543598', 'name': 'Amanda L. Forest'}, {'authorId': '4193992', 'name': 'Eskil Forsell'}, {'authorId': '153664010', 'name': 'J. D. Foster'}, {'authorId': '2536223', 'name': 'Michael C. Frank'}, {'authorId': '144106517', 'name': 'Rebecca S. Frazier'}, {'authorId': '2057921617', 'name': 'Heather M. Fuchs'}, {'authorId': '5807132', 'name': 'P. Gable'}, {'authorId': '4908329', 'name': 'Jeff Galak'}, {'authorId': '52169965', 'name': 'E. Galliani'}, {'authorId': '39007608', 'name': 'Anup Gampa'}, {'authorId': '2072604774', 'name': 'Sara Garcia'}, {'authorId': '4034033', 'name': 'Douglas Gazarian'}, {'authorId': '152595201', 'name': 'E. Gilbert'}, {'authorId': '1400843054', 'name': 'Roger Giner-Sorolla'}, {'authorId': '2098939', 'name': 'A. Glöckner'}, {'authorId': '35467332', 'name': 'Lars Goellner'}, {'authorId': '6760964', 'name': 'Jin X. Goh'}, {'authorId': '1402974706', 'name': 'R. Goldberg'}, {'authorId': '6439793', 'name': 'Patrick T. Goodbourn'}, {'authorId': '1400158332', 'name': 'S. Gordon-McKeon'}, {'authorId': '103288020', 'name': 'Bryan H. Gorges'}, {'authorId': '1677019678', 'name': 'Jessie Gorges'}, {'authorId': '113021959', 'name': 'J. Goss'}, {'authorId': '40433926', 'name': 'J. Graham'}, {'authorId': '50590791', 'name': 'J. Grange'}, {'authorId': '2556954', 'name': 'J. Gray'}, {'authorId': '72271933', 'name': 'C.H.J. Hartgerink'}, {'authorId': '27816717', 'name': 'Joshua K. Hartshorne'}, {'authorId': '3390722', 'name': 'F. Hasselman'}, {'authorId': '144616859', 'name': 'Timothy Hayes'}, {'authorId': '3650446', 'name': 'Emma Heikensten'}, {'authorId': '2196843', 'name': 'Felix Henninger'}, {'authorId': '6941720', 'name': 'J. Hodsoll'}, {'authorId': '3388157', 'name': 'Taylor Holubar'}, {'authorId': '92341957', 'name': 'G. Hoogendoorn'}, {'authorId': '1665486287', 'name': 'D. Humphries'}, {'authorId': '70247182', 'name': 'C. O. Hung'}, {'authorId': '87889800', 'name': 'N. Immelman'}, {'authorId': '6169028', 'name': 'Vanessa C. Irsik'}, {'authorId': '36524750', 'name': 'G. Jahn'}, {'authorId': '80488489', 'name': 'F. Jäkel'}, {'authorId': '2485703', 'name': 'Marc Jekel'}, {'authorId': '145281248', 'name': 'M. Johannesson'}, {'authorId': '117496518', 'name': 'L. Johnson'}, {'authorId': '2150444569', 'name': 'David J. Johnson'}, {'authorId': '10776195', 'name': 'Kate M. Johnson'}, {'authorId': '2071756595', 'name': 'William Johnston'}, {'authorId': '2640241', 'name': 'K. Jonas'}, {'authorId': '1400158286', 'name': 'Jennifer A. Joy-Gaba'}, {'authorId': '4049176', 'name': 'H. Kappes'}, {'authorId': '120498542', 'name': 'Kim Kelso'}, {'authorId': '8633058', 'name': 'Mallory C. Kidwell'}, {'authorId': '2109708345', 'name': 'Seung K. Kim'}, {'authorId': '6395769', 'name': 'M. Kirkhart'}, {'authorId': '6032930', 'name': 'Bennett Kleinberg'}, {'authorId': '50012037', 'name': 'G. Knežević'}, {'authorId': '38197330', 'name': 'F. Kolorz'}, {'authorId': '39916640', 'name': 'J. Kossakowski'}, {'authorId': '153875794', 'name': 'R. Krause'}, {'authorId': '123143083', 'name': 'J.M.T. Krijnen'}, {'authorId': '5199064', 'name': 'T. Kuhlmann'}, {'authorId': '90301659', 'name': 'Y. Kunkels'}, {'authorId': '123804984', 'name': 'Megan M. Kyc'}, {'authorId': '2148764895', 'name': 'Calvin K. Lai'}, {'authorId': '1665495033', 'name': 'Aamir Laique'}, {'authorId': '87792747', 'name': 'D. Lakens'}, {'authorId': '47951763', 'name': 'Kristin A. Lane'}, {'authorId': '6028786', 'name': 'Bethany Lassetter'}, {'authorId': '40526243', 'name': 'L. Lazarević'}, {'authorId': '1665487375', 'name': 'E. Bel'}, {'authorId': '2110236533', 'name': 'Key Jung Lee'}, {'authorId': '46293822', 'name': 'Minha Lee'}, {'authorId': '6560078', 'name': 'K. Lemm'}, {'authorId': '2248921', 'name': 'C. Levitan'}, {'authorId': '48575845', 'name': 'Melissa Lewis'}, {'authorId': '2117106185', 'name': 'Lin Lin'}, {'authorId': '2145198693', 'name': 'Stephanie C. Lin'}, {'authorId': '35780819', 'name': 'Matthias Lippold'}, {'authorId': '6767826', 'name': 'Darren Loureiro'}, {'authorId': '116957928', 'name': 'Ilse Luteijn'}, {'authorId': '35093146', 'name': 'S. Mackinnon'}, {'authorId': '120566087', 'name': 'Heather N. Mainard'}, {'authorId': '4526963', 'name': 'Denise C. Marigold'}, {'authorId': '150078826', 'name': 'D. P. Martin'}, {'authorId': '123396296', 'name': 'Tylar Martinez'}, {'authorId': '4362855', 'name': 'E. Masicampo'}, {'authorId': '84358116', 'name': 'Joshua J. Matacotta'}, {'authorId': '32716879', 'name': 'Maya B. Mathur'}, {'authorId': '2055522260', 'name': 'Michael May'}, {'authorId': '5739306', 'name': 'Nicole C Mechin'}, {'authorId': '2796734', 'name': 'P. Mehta'}, {'authorId': '3505739', 'name': 'Johannes M. Meixner'}, {'authorId': '2292034', 'name': 'Alissa Melinger'}, {'authorId': '2109507581', 'name': 'Jeremy K. Miller'}, {'authorId': '1664699522', 'name': 'Mallorie Miller'}, {'authorId': '3260369', 'name': 'K. Moore'}, {'authorId': '8173711', 'name': 'Marcus Möschl'}, {'authorId': '40632770', 'name': 'Matt Motyl'}, {'authorId': '49338794', 'name': 'S. Muller'}, {'authorId': '3057626', 'name': 'M. Munafo'}, {'authorId': '6699090', 'name': 'K. Neijenhuijs'}, {'authorId': '122669704', 'name': 'Taylor Nervi'}, {'authorId': '28866192', 'name': 'Gandalf Nicolas'}, {'authorId': '5120903', 'name': 'G. Nilsonne'}, {'authorId': '2862527', 'name': 'Brian A. Nosek'}, {'authorId': '4222695', 'name': 'Michèle B. Nuijten'}, {'authorId': '2061321863', 'name': 'Catherine Olsson'}, {'authorId': '21707288', 'name': 'C. Osborne'}, {'authorId': '15808420', 'name': 'Lutz Ostkamp'}, {'authorId': '1800603', 'name': 'M. Pavel'}, {'authorId': '1398036394', 'name': 'I. Penton-Voak'}, {'authorId': '123910286', 'name': 'O. Perna'}, {'authorId': '1733092', 'name': 'C. Pernet'}, {'authorId': '5070565', 'name': 'M. Perugini'}, {'authorId': '145798446', 'name': 'R. Pipitone'}, {'authorId': '7731005', 'name': 'M. Pitts'}, {'authorId': '3086561', 'name': 'F. Plessow'}, {'authorId': '5666302', 'name': 'J. Prenoveau'}, {'authorId': '114309502', 'name': 'R. Rahal'}, {'authorId': '4412299', 'name': 'Kate A. Ratliff'}, {'authorId': '46557202', 'name': 'David A. Reinhard'}, {'authorId': '4137233', 'name': 'Frank Renkewitz'}, {'authorId': '4977822', 'name': 'Ashley A. Ricker'}, {'authorId': '3911950', 'name': 'A. Rigney'}, {'authorId': '145836378', 'name': 'Andrew M Rivers'}, {'authorId': '51055550', 'name': 'Mark A. Roebke'}, {'authorId': '6431015', 'name': 'Abraham M. Rutchick'}, {'authorId': '2055552083', 'name': 'Robert S. Ryan'}, {'authorId': '48176355', 'name': 'O. Şahin'}, {'authorId': '7144751', 'name': 'Anondah Saide'}, {'authorId': '4225466', 'name': 'Gillian M. Sandstrom'}, {'authorId': '2113520139', 'name': 'David Santos'}, {'authorId': '2055715460', 'name': 'R. Saxe'}, {'authorId': '40063306', 'name': 'R. Schlegelmilch'}, {'authorId': '49371070', 'name': 'Kathleen Schmidt'}, {'authorId': '49715936', 'name': 'Sabine Scholz'}, {'authorId': '123478162', 'name': 'L. Seibel'}, {'authorId': '4067481', 'name': 'Dylan Selterman'}, {'authorId': '3397505', 'name': 'S. Shaki'}, {'authorId': '31665223', 'name': 'W. B. Simpson'}, {'authorId': '144232281', 'name': 'H. Sinclair'}, {'authorId': '143804040', 'name': 'Jeanine L. M. Skorinko'}, {'authorId': '2111978802', 'name': 'A. Slowik'}, {'authorId': '2500175', 'name': 'J. Snyder'}, {'authorId': '4110370', 'name': 'C. Soderberg'}, {'authorId': '32218314', 'name': 'Carina M. Sonnleitner'}, {'authorId': '2056814613', 'name': 'N. Spencer'}, {'authorId': '34782684', 'name': 'Jeffrey R. Spies'}, {'authorId': '2889085', 'name': 'S. Steegen'}, {'authorId': '3308834', 'name': 'S. Stieger'}, {'authorId': '3918197', 'name': 'Nina Strohminger'}, {'authorId': '3349937', 'name': 'G. Sullivan'}, {'authorId': '6998374', 'name': 'T. Talhelm'}, {'authorId': '123526524', 'name': 'M. Tapia'}, {'authorId': '120986210', 'name': 'A. T. Dorsthorst'}, {'authorId': '17886332', 'name': 'M. Thomae'}, {'authorId': '2108534776', 'name': 'Sarah L. Thomas'}, {'authorId': '5407737', 'name': 'Pia Tio'}, {'authorId': '26384240', 'name': 'Frits Traets'}, {'authorId': '50343359', 'name': 'Steve Tsang'}, {'authorId': '2109141', 'name': 'F. Tuerlinckx'}, {'authorId': '118528873', 'name': 'Paul Turchan'}, {'authorId': '3738838', 'name': 'Milan Valášek'}, {'authorId': '143861972', 'name': 'A. V. Veer'}, {'authorId': '81322766', 'name': 'R. V. Aert'}, {'authorId': '13333804', 'name': 'M. V. Assen'}, {'authorId': '41197124', 'name': 'R. V. Bork'}, {'authorId': '122303122', 'name': 'Mathijs van de Ven'}, {'authorId': '103097236', 'name': 'D. V. D. Bergh'}, {'authorId': '50093142', 'name': 'M. Hulst'}, {'authorId': '51240054', 'name': 'R. V. Dooren'}, {'authorId': '50233092', 'name': 'J. Doorn'}, {'authorId': '4155110', 'name': 'D. V. Renswoude'}, {'authorId': '2392419', 'name': 'H. Rijn'}, {'authorId': '2409652', 'name': 'W. Vanpaemel'}, {'authorId': '2078595605', 'name': 'Alejandro Echeverría'}, {'authorId': '2070303231', 'name': 'Melissa Vazquez'}, {'authorId': '6122250', 'name': 'Natalia Vélez'}, {'authorId': '35039893', 'name': 'Marieke Vermue'}, {'authorId': '88801524', 'name': 'M. Verschoor'}, {'authorId': '46989769', 'name': 'M. Vianello'}, {'authorId': '2176390', 'name': 'M. Voracek'}, {'authorId': '113898223', 'name': 'Gina Vuu'}, {'authorId': '2505272', 'name': 'E. Wagenmakers'}, {'authorId': '6348079', 'name': 'Joanneke Weerdmeester'}, {'authorId': '2058927069', 'name': 'A. Welsh'}, {'authorId': '5900056', 'name': 'Erin C. Westgate'}, {'authorId': '67239528', 'name': 'Joeri Wissink'}, {'authorId': '145322160', 'name': 'M. Wood'}, {'authorId': '48197426', 'name': 'A. Woods'}, {'authorId': '2058458367', 'name': 'Emily M. Wright'}, {'authorId': '151482749', 'name': 'Sining Wu'}, {'authorId': '145525838', 'name': 'M. Zeelenberg'}, {'authorId': '117597026', 'name': 'Kellylynn Zuni'}]}, {'paperId': 'b5f109c8d49a95f72313aff41cf8a0b2e8e1407a', 'title': 'The Genotype-Tissue Expression (GTEx) pilot analysis: Multitissue gene regulation in humans', 'abstract': 'Expression, genetic variation, and tissues Human genomes show extensive genetic variation across individuals, but we have only just started documenting the effects of this variation on the regulation of gene expression. Furthermore, only a few tissues have been examined per genetic variant. In order to examine how genetic expression varies among tissues within individuals, the Genotype-Tissue Expression (GTEx) Consortium collected 1641 postmortem samples covering 54 body sites from 175 individuals. They identified quantitative genetic traits that affect gene expression and determined which of these exhibit tissue-specific expression patterns. Melé et al. measured how transcription varies among tissues, and Rivas et al. looked at how truncated protein variants affect expression across tissues. Science, this issue p. 648, p. 660, p. 666; see also p. 640 Sampling RNA from multiple tissues across the human body illuminates how genetic variation affects gene expression. [Also see Perspective by Gibson] Understanding the functional consequences of genetic variation, and how it affects complex human disease and quantitative traits, remains a critical challenge for biomedicine. We present an analysis of RNA sequencing data from 1641 samples across 43 tissues from 175 individuals, generated as part of the pilot phase of the Genotype-Tissue Expression (GTEx) project. We describe the landscape of gene expression across tissues, catalog thousands of tissue-specific and shared regulatory expression quantitative trait loci (eQTL) variants, describe complex network relationships, and identify signals from genome-wide association studies explained by eQTLs. These findings provide a systematic understanding of the cellular and biological consequences of human genetic variation and of the heterogeneity of such effects among a diverse set of human tissues.', 'year': 2015, 'citationCount': 3980, 'authors': [{'authorId': '4351471', 'name': 'K. Ardlie'}, {'authorId': '46980374', 'name': 'D. DeLuca'}, {'authorId': '4155963', 'name': 'A. Segrè'}, {'authorId': '2068615525', 'name': 'T. Sullivan'}, {'authorId': '31774477', 'name': 'Taylor R. Young'}, {'authorId': '16172097', 'name': 'Ellen T. Gelfand'}, {'authorId': '11133741', 'name': 'Casandra A. Trowbridge'}, {'authorId': '3254016', 'name': 'J. Maller'}, {'authorId': '5173577', 'name': 'T. Tukiainen'}, {'authorId': '3015823', 'name': 'M. Lek'}, {'authorId': '36987543', 'name': 'L. D. Ward'}, {'authorId': '1957116', 'name': 'P. Kheradpour'}, {'authorId': '3768568', 'name': 'B. Iriarte'}, {'authorId': '2114020679', 'name': 'Y. Meng'}, {'authorId': '4128156', 'name': 'Cameron Palmer'}, {'authorId': '52200875', 'name': 'T. Esko'}, {'authorId': '2060379', 'name': 'W. Winckler'}, {'authorId': '3294928', 'name': 'J. Hirschhorn'}, {'authorId': '2809250', 'name': 'Manolis Kellis'}, {'authorId': '2029438', 'name': 'D. MacArthur'}, {'authorId': '2110594', 'name': 'G. Getz'}, {'authorId': '4407885', 'name': 'A. Shabalin'}, {'authorId': '2108549769', 'name': 'Gen Li'}, {'authorId': '34661060', 'name': 'Yi‐Hui Zhou'}, {'authorId': '1794435', 'name': 'A. Nobel'}, {'authorId': '2430148', 'name': 'I. Rusyn'}, {'authorId': '40250286', 'name': 'F. Wright'}, {'authorId': '144946488', 'name': 'T. Lappalainen'}, {'authorId': '145209580', 'name': 'Pedro G. Ferreira'}, {'authorId': '3410962', 'name': 'H. Ongen'}, {'authorId': '145764939', 'name': 'M. Rivas'}, {'authorId': '145086335', 'name': 'A. Battle'}, {'authorId': '47278080', 'name': 'S. Mostafavi'}, {'authorId': '6596913', 'name': 'Jean Monlong'}, {'authorId': '1971387', 'name': 'M. Sammeth'}, {'authorId': '2326247', 'name': 'Marta Melé'}, {'authorId': '1704215264', 'name': 'F. Reverter'}, {'authorId': '47824049', 'name': 'J. Goldmann'}, {'authorId': '1736370', 'name': 'D. Koller'}, {'authorId': '1853610', 'name': 'R. Guigó'}, {'authorId': '2235633839', 'name': 'M. McCarthy'}, {'authorId': '2939073', 'name': 'E. Dermitzakis'}, {'authorId': '3051203', 'name': 'E. Gamazon'}, {'authorId': '2427364', 'name': 'H. Im'}, {'authorId': '1784291', 'name': 'A. Konkashbaev'}, {'authorId': '1783950', 'name': 'D. Nicolae'}, {'authorId': '2173670631', 'name': 'N. Cox'}, {'authorId': '50045364', 'name': 'T. Flutre'}, {'authorId': '36836750', 'name': 'X. Wen'}, {'authorId': '145761702', 'name': 'M. Stephens'}, {'authorId': '50528060', 'name': 'J. Pritchard'}, {'authorId': '40501864', 'name': 'Zhidong Tu'}, {'authorId': '49846439', 'name': 'Bin Zhang'}, {'authorId': '2110812174', 'name': 'Tao Huang'}, {'authorId': '48548974', 'name': 'Q. Long'}, {'authorId': '34999215', 'name': 'Luan Lin'}, {'authorId': '2144559662', 'name': 'Jialiang Yang'}, {'authorId': '145254054', 'name': 'Jun Zhu'}, {'authorId': '2234974904', 'name': 'Jun S. Liu'}, {'authorId': '2110644715', 'name': 'A. Brown'}, {'authorId': '4721474', 'name': 'B. Mestichelli'}, {'authorId': '5632758', 'name': 'D. Tidwell'}, {'authorId': '4412904', 'name': 'E. Lo'}, {'authorId': '6900442', 'name': 'Mike Salvatore'}, {'authorId': '40353616', 'name': 'Saboor Shad'}, {'authorId': '2115544487', 'name': 'Jeffrey A. Thomas'}, {'authorId': '103811085', 'name': 'J. Lonsdale'}, {'authorId': '28868673', 'name': 'M. T. Moser'}, {'authorId': '13378769', 'name': 'Bryan M. Gillard'}, {'authorId': '5353188', 'name': 'E. Karasik'}, {'authorId': '40652641', 'name': 'K. Ramsey'}, {'authorId': '40155820', 'name': 'Christopher Choi'}, {'authorId': '1844728', 'name': 'B. Foster'}, {'authorId': '6752960', 'name': 'John Syron'}, {'authorId': '3717383', 'name': 'Johnelle Fleming'}, {'authorId': '48365408', 'name': 'Harold Magazine'}, {'authorId': '10396153', 'name': 'R. Hasz'}, {'authorId': '19099263', 'name': 'Gary Walters'}, {'authorId': '52213800', 'name': 'J. Bridge'}, {'authorId': '50062134', 'name': 'Mark Miklos'}, {'authorId': '2074770062', 'name': 'Susan Sullivan'}, {'authorId': '31821652', 'name': 'Laura K. Barker'}, {'authorId': '3633762', 'name': 'H. Traino'}, {'authorId': '3507395', 'name': 'M. Mosavel'}, {'authorId': '1581155106', 'name': 'L. Siminoff'}, {'authorId': '6011362', 'name': 'Dana R. Valley'}, {'authorId': '15828822', 'name': 'D. Rohrer'}, {'authorId': '4317180', 'name': 'S. Jewell'}, {'authorId': '39668238', 'name': 'P. Branton'}, {'authorId': '5477494', 'name': 'L. Sobin'}, {'authorId': '37365786', 'name': 'M. Barcus'}, {'authorId': '50741911', 'name': 'L. Qi'}, {'authorId': '46808203', 'name': 'J. McLean'}, {'authorId': '49119953', 'name': 'P. Hariharan'}, {'authorId': '40341965', 'name': 'K. S. Um'}, {'authorId': '2199287', 'name': 'Shenpei Wu'}, {'authorId': '4007205', 'name': 'D. Tabor'}, {'authorId': '6456063', 'name': 'C. Shive'}, {'authorId': '2116583787', 'name': 'Anna M. Smith'}, {'authorId': '6399876', 'name': 'Stephen A. Buia'}, {'authorId': '5664172', 'name': 'Anita H. Undale'}, {'authorId': '4735590', 'name': 'Karna L. Robinson'}, {'authorId': '39656869', 'name': 'Nancy V. Roche'}, {'authorId': '31606684', 'name': 'Kimberly M. Valentino'}, {'authorId': '49117155', 'name': 'A. Britton'}, {'authorId': '47562897', 'name': 'R. Burges'}, {'authorId': '33915352', 'name': 'Debra Bradbury'}, {'authorId': '3801585', 'name': 'Kenneth W. Hambright'}, {'authorId': '4354568', 'name': 'John Seleski'}, {'authorId': '4575963', 'name': 'G. Korzeniewski'}, {'authorId': '38190914', 'name': 'Kenyon Erickson'}, {'authorId': '40408502', 'name': 'Yvonne Marcus'}, {'authorId': '2006434778', 'name': 'J. Tejada'}, {'authorId': '1934316', 'name': 'M. Taherian'}, {'authorId': '3567450', 'name': 'Chunrong Lu'}, {'authorId': '40512923', 'name': 'Margaret J. Basile'}, {'authorId': '8055139', 'name': 'D. Mash'}, {'authorId': '35338611', 'name': 'S. Volpi'}, {'authorId': '143755016', 'name': 'J. Struewing'}, {'authorId': '6264745', 'name': 'Gary F. Temple'}, {'authorId': '40591554', 'name': 'Joy T. Boyer'}, {'authorId': '4618842', 'name': 'Deborah Colantuoni'}, {'authorId': '120055212', 'name': 'R. Little'}, {'authorId': '32043266', 'name': 'S. Koester'}, {'authorId': '6407402', 'name': 'Latarsha J. Carithers'}, {'authorId': '48294660', 'name': 'Helen M. Moore'}, {'authorId': '46344169', 'name': 'P. Guan'}, {'authorId': '3862532', 'name': 'C. Compton'}, {'authorId': '32536451', 'name': 'Sherilyn J. Sawyer'}, {'authorId': '4915811', 'name': 'J. Demchok'}, {'authorId': '48219890', 'name': 'J. Vaught'}, {'authorId': '47626774', 'name': 'C. Rabiner'}, {'authorId': '90929184', 'name': 'Lockhart'}]}, {'paperId': 'a798f8e9c58258bdbb996b8518b313fcc449be2c', 'title': 'Plastic waste inputs from land into the ocean', 'abstract': 'Dumping lots of plastics into our oceans Considerable progress has been made in determining the amount and location of plastic debris in our seas, but how much plastic actually enters them in the first place is more uncertain. Jambeck et al. combine available data on solid waste with a model that uses population density and economic status to estimate the amount of land-based plastic waste entering the ocean. Unless waste management practices are improved, the flux of plastics to the oceans could increase by an order of magnitude within the next decade. Science, this issue p. 768 Millions of tons of plastic waste end up in our oceans every year. Plastic debris in the marine environment is widely documented, but the quantity of plastic entering the ocean from waste generated on land is unknown. By linking worldwide data on solid waste, population density, and economic status, we estimated the mass of land-based plastic waste entering the ocean. We calculate that 275 million metric tons (MT) of plastic waste was generated in 192 coastal countries in 2010, with 4.8 to 12.7 million MT entering the ocean. Population size and the quality of waste management systems largely determine which countries contribute the greatest mass of uncaptured waste available to become plastic marine debris. Without waste management infrastructure improvements, the cumulative quantity of plastic waste available to enter the ocean from land is predicted to increase by an order of magnitude by 2025.', 'year': 2015, 'citationCount': 7170, 'authors': [{'authorId': '2105939', 'name': 'J. Jambeck'}, {'authorId': '47383913', 'name': 'R. Geyer'}, {'authorId': '143924375', 'name': 'C. Wilcox'}, {'authorId': '118664577', 'name': 'T. R. Siegler'}, {'authorId': '47308651', 'name': 'M. Perryman'}, {'authorId': '8417546', 'name': 'A. Andrady'}, {'authorId': '7745972', 'name': 'R. Narayan'}, {'authorId': '34861621', 'name': 'K. L. Law'}]}, {'paperId': 'ad032c039418ccfa07a9e3449a4007a6781c554a', 'title': 'Tissue-based map of the human proteome', 'abstract': 'Protein expression across human tissues Sequencing the human genome gave new insights into human biology and disease. However, the ultimate goal is to understand the dynamic expression of each of the approximately 20,000 protein-coding genes and the function of each protein. Uhlén et al. now present a map of protein expression across 32 human tissues. They not only measured expression at an RNA level, but also used antibody profiling to precisely localize the corresponding proteins. An interactive website allows exploration of expression patterns across the human body. Science, this issue 10.1126/science.1260419 Transcriptomics and immunohistochemistry map protein expression across 32 human tissues. INTRODUCTION Resolving the molecular details of proteome variation in the different tissues and organs of the human body would greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on quantitative transcriptomics on a tissue and organ level combined with protein profiling using microarray-based immunohistochemistry to achieve spatial localization of proteins down to the single-cell level. We provide a global analysis of the secreted and membrane proteins, as well as an analysis of the expression profiles for all proteins targeted by pharmaceutical drugs and proteins implicated in cancer. RATIONALE We have used an integrative omics approach to study the spatial human proteome. Samples representing all major tissues and organs (n = 44) in the human body have been analyzed based on 24,028 antibodies corresponding to 16,975 protein-encoding genes, complemented with RNA-sequencing data for 32 of the tissues. The antibodies have been used to produce more than 13 million tissue-based immunohistochemistry images, each annotated by pathologists for all sampled tissues. To facilitate integration with other biological resources, all data are available for download and cross-referencing. RESULTS We report a genome-wide analysis of the tissue specificity of RNA and protein expression covering more than 90% of the putative protein-coding genes, complemented with analyses of various subproteomes, such as predicted secreted proteins (n = 3171) and membrane-bound proteins (n = 5570). The analysis shows that almost half of the genes are expressed in all analyzed tissues, which suggests that the gene products are needed in all cells to maintain “housekeeping” functions such as cell growth, energy generation, and basic metabolism. Furthermore, there is enrichment in metabolism among these genes, as 60% of all metabolic enzymes are expressed in all analyzed tissues. The largest number of tissue-enriched genes is found in the testis, followed by the brain and the liver. Analysis of the 618 proteins targeted by clinically approved drugs unexpectedly showed that 30% are expressed in all analyzed tissues. An analysis of metabolic activity based on genome-scale metabolic models (GEMS) revealed liver as the most metabolically active tissue, followed by adipose tissue and skeletal muscle. CONCLUSIONS A freely available interactive resource is presented as part of the Human Protein Atlas portal (www.proteinatlas.org), offering the possibility to explore the tissue-elevated proteomes in tissues and organs and to analyze tissue profiles for specific protein classes. Comprehensive lists of proteins expressed at elevated levels in the different tissues have been compiled to provide a spatial context with localization of the proteins in the subcompartments of each tissue and organ down to the single-cell level. The human tissue–enriched proteins. All tissue-enriched proteins are shown for 13 representative tissues or groups of tissues, stratified according to their predicted subcellular localization. Enriched proteins are mainly intracellular in testis, mainly membrane bound in brain and kidney, and mainly secreted in pancreas and liver. Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.', 'year': 2015, 'citationCount': 9684, 'authors': [{'authorId': '2781789', 'name': 'M. Uhlén'}, {'authorId': '3841801', 'name': 'L. Fagerberg'}, {'authorId': '4787480', 'name': 'B. Hallström'}, {'authorId': '5710895', 'name': 'C. Lindskog'}, {'authorId': '3781249', 'name': 'P. Oksvold'}, {'authorId': '2567446', 'name': 'A. Mardinoğlu'}, {'authorId': '3521812', 'name': 'Åsa Sivertsson'}, {'authorId': '3919959', 'name': 'C. Kampf'}, {'authorId': '6961788', 'name': 'E. Sjöstedt'}, {'authorId': '5177460', 'name': 'A. Asplund'}, {'authorId': '49850546', 'name': 'Ingmarie Olsson'}, {'authorId': '2483844', 'name': 'K. Edlund'}, {'authorId': '89824313', 'name': 'E. Lundberg'}, {'authorId': '6505855', 'name': 'S. Navani'}, {'authorId': '50704448', 'name': 'C. Szigyarto'}, {'authorId': '1950314', 'name': 'J. Odeberg'}, {'authorId': '7484504', 'name': 'D. Djureinovic'}, {'authorId': '8300411', 'name': 'Jenny Ottosson Takanen'}, {'authorId': '6483536', 'name': 'S. Hober'}, {'authorId': '32692536', 'name': 'T. Alm'}, {'authorId': '6542432', 'name': 'P. Edqvist'}, {'authorId': '8049983', 'name': 'H. Berling'}, {'authorId': '3948842', 'name': 'Hanna Tegel'}, {'authorId': '144464470', 'name': 'J. Mulder'}, {'authorId': '4420951', 'name': 'Johan Rockberg'}, {'authorId': '48586383', 'name': 'P. Nilsson'}, {'authorId': '4136538', 'name': 'J. Schwenk'}, {'authorId': '8465156', 'name': 'M. Hamsten'}, {'authorId': '134273659', 'name': 'K. von Feilitzen'}, {'authorId': '2068107910', 'name': 'Mattias Forsberg'}, {'authorId': '46852754', 'name': 'L. Persson'}, {'authorId': '7369919', 'name': 'Fredric Johansson'}, {'authorId': '20374706', 'name': 'M. Zwahlen'}, {'authorId': '3300090', 'name': 'G. von Heijne'}, {'authorId': '145583597', 'name': 'J. Nielsen'}, {'authorId': '143810507', 'name': 'F. Pontén'}]}, {'paperId': 'd0a5bc5ff2c33c640e97009377304357fe8d2838', 'title': 'Mutational landscape determines sensitivity to PD-1 blockade in non–small cell lung cancer', 'abstract': 'Immune checkpoint inhibitors, which unleash a patient’s own T cells to kill tumors, are revolutionizing cancer treatment. To unravel the genomic determinants of response to this therapy, we used whole-exome sequencing of non–small cell lung cancers treated with pembrolizumab, an antibody targeting programmed cell death-1 (PD-1). In two independent cohorts, higher nonsynonymous mutation burden in tumors was associated with improved objective response, durable clinical benefit, and progression-free survival. Efficacy also correlated with the molecular smoking signature, higher neoantigen burden, and DNA repair pathway mutations; each factor was also associated with mutation burden. In one responder, neoantigen-specific CD8+ T cell responses paralleled tumor regression, suggesting that anti–PD-1 therapy enhances neoantigen-specific T cell reactivity. Our results suggest that the genomic landscape of lung cancers shapes response to anti–PD-1 therapy. An anticancer drug is more effective against tumors that carry more mutations. More mutations predict better efficacy Despite the remarkable success of cancer immunotherapies, many patients do not respond to treatment. Rizvi et al. studied the tumors of patients with non–small-cell lung cancer undergoing immunotherapy. In two independent cohorts, treatment efficacy was associated with a higher number of mutations in the tumors. In one patient, a tumor-specific T cell response paralleled tumor regression. Science, this issue p. 124', 'year': 2015, 'citationCount': 5728, 'authors': [{'authorId': '5226159', 'name': 'N. Rizvi'}, {'authorId': '4727109', 'name': 'M. Hellmann'}, {'authorId': '144304933', 'name': 'A. Snyder'}, {'authorId': '5411400', 'name': 'P. Kvistborg'}, {'authorId': '144252051', 'name': 'V. Makarov'}, {'authorId': '6057547', 'name': 'J. Havel'}, {'authorId': '15006569', 'name': 'William Lee'}, {'authorId': '39171288', 'name': 'Jianda Yuan'}, {'authorId': '47181234', 'name': 'P. Wong'}, {'authorId': '153046030', 'name': 'Teresa S. Ho'}, {'authorId': '31957119', 'name': 'Martin L. Miller'}, {'authorId': '5514453', 'name': 'N. Rekhtman'}, {'authorId': '144849915', 'name': 'A. Moreira'}, {'authorId': '46673012', 'name': 'Fawzia Ibrahim'}, {'authorId': '152679701', 'name': 'Cameron Bruggeman'}, {'authorId': '8452615', 'name': 'Billel Gasmi'}, {'authorId': '6951535', 'name': 'R. Zappasodi'}, {'authorId': '48187107', 'name': 'Y. Maeda'}, {'authorId': '144882390', 'name': 'C. Sander'}, {'authorId': '2570491', 'name': 'E. Garon'}, {'authorId': '6949756', 'name': 'T. Merghoub'}, {'authorId': '6892960', 'name': 'J. Wolchok'}, {'authorId': '144394800', 'name': 'T. Schumacher'}, {'authorId': '2234300666', 'name': 'T. Chan'}]}, {'paperId': '36d7fd7b34bb276f35928a4cef30940564b2e276', 'title': 'Cell types in the mouse cortex and hippocampus revealed by single-cell RNA-seq', 'abstract': 'Cellular diversity in the brain revealed The mammalian brain has an extraordinarily large number of cells. Although there are quite a few different cell types, many cells in any one category tend to look alike. Zeisel et al. analyzed the transcriptomes of mouse brain cells to reveal more than meets the eye. Interneurons of similar type were found in dissimilar regions of the brain. Oligodendrocytes that seemed to be all of one class were differentiated by their molecular signatures into a half-dozen classes. Microglia associated with blood vessels were distinguished from look-alike perivascular macrophages. Thus, the complex microanatomy of the brain can be revealed by the RNAs expressed in its cells. Science, this issue p. 1138 A close look at the genes expressed by cells in the brain reveals hidden and coordinated cellular complexity. The mammalian cerebral cortex supports cognitive functions such as sensorimotor integration, memory, and social behaviors. Normal brain function relies on a diverse set of differentiated cell types, including neurons, glia, and vasculature. Here, we have used large-scale single-cell RNA sequencing (RNA-seq) to classify cells in the mouse somatosensory cortex and hippocampal CA1 region. We found 47 molecularly distinct subclasses, comprising all known major cell types in the cortex. We identified numerous marker genes, which allowed alignment with known cell types, morphology, and location. We found a layer I interneuron expressing Pax6 and a distinct postmitotic oligodendrocyte subclass marked by Itpr2. Across the diversity of cortical cell types, transcription factors formed a complex, layered regulatory code, suggesting a mechanism for the maintenance of adult cell type identity.', 'year': 2015, 'citationCount': 2354, 'authors': [{'authorId': '2565412', 'name': 'Amit Zeisel'}, {'authorId': '1399254562', 'name': 'A. B. Muñoz-Manchado'}, {'authorId': '6427264', 'name': 'S. Codeluppi'}, {'authorId': '5211583', 'name': 'P. Lönnerberg'}, {'authorId': '134014902', 'name': 'Gioele La Manno'}, {'authorId': '10271546', 'name': 'Anna Juréus'}, {'authorId': '47123084', 'name': 'Sueli C Marques'}, {'authorId': '5124778', 'name': 'H. Munguba'}, {'authorId': '49159823', 'name': 'Liqun He'}, {'authorId': '5077801', 'name': 'C. Betsholtz'}, {'authorId': '6779735', 'name': 'Charlotte Rolny'}, {'authorId': '1384303053', 'name': 'G. Castelo-Branco'}, {'authorId': '1398819597', 'name': 'J. Hjerling-Leffler'}, {'authorId': '2500111', 'name': 'S. Linnarsson'}]}, {'paperId': 'c8665198592b52ba122e5cf84032a9bc1c61eade', 'title': 'Exposure to ideologically diverse news and opinion on Facebook', 'abstract': \"Not getting all sides of the news? People are increasingly turning away from mass media to social media as a way of learning news and civic information. Bakshy et al. examined the news that millions of Facebook users' peers shared, what information these users were presented with, and what they ultimately consumed (see the Perspective by Lazer). Friends shared substantially less cross-cutting news from sources aligned with an opposing ideology. People encountered roughly 15% less cross-cutting content in news feeds due to algorithmic ranking and clicked through to 70% less of this cross-cutting content. Within the domain of political news encountered in social media, selective exposure appears to drive attention. Science, this issue p. 1130; see also p. 1090 Despite the diversity of information available, people still pay attention to a limited range of opinions. [Also see Perspective by Lazer] Exposure to news, opinion, and civic information increasingly occurs through social media. How do these online networks influence exposure to perspectives that cut across ideological lines? Using deidentified data, we examined how 10.1 million U.S. Facebook users interact with socially shared news. We directly measured ideological homophily in friend networks and examined the extent to which heterogeneous friends could potentially expose individuals to cross-cutting content. We then quantified the extent to which individuals encounter comparatively more or less diverse content while interacting via Facebook’s algorithmically ranked News Feed and further studied users’ choices to click through to ideologically discordant content. Compared with algorithmic ranking, individuals’ choices played a stronger role in limiting exposure to cross-cutting content.\", 'year': 2015, 'citationCount': 2099, 'authors': [{'authorId': '3071703', 'name': 'E. Bakshy'}, {'authorId': '50252567', 'name': 'Solomon Messing'}, {'authorId': '1778398', 'name': 'Lada A. Adamic'}]}, {'paperId': '20ace79f735dfa350fa6c5eb0fa431c68093b19a', 'title': 'Genomic correlates of response to CTLA-4 blockade in metastatic melanoma', 'abstract': 'Is cancer immunotherapy a private affair? Immune checkpoint blockade, a relatively new cancer treatment, substantially extends the survival of a subset of patients. Previous work has shown that patients whose tumors harbor the largest number of mutations—and thus produce a large number of “neoantigens” recognized as foreign by the immune system—are most likely to benefit. Expanding on these earlier studies, Van Allen et al. studied over 100 patients with melanoma and found a similar correlation (see the Perspective by Gubin and Schreiber). There was no evidence, however, that specific neoantigen sequences were shared by patients who responded. Science, this issue p. 207, see also p. 158 Melanoma patients who respond to immunotherapy do not appear to share common tumor neoantigens. [Also see Perspective by Gubin and Schreiber ] Monoclonal antibodies directed against cytotoxic T lymphocyte–associated antigen-4 (CTLA-4), such as ipilimumab, yield considerable clinical benefit for patients with metastatic melanoma by inhibiting immune checkpoint activity, but clinical predictors of response to these therapies remain incompletely characterized. To investigate the roles of tumor-specific neoantigens and alterations in the tumor microenvironment in the response to ipilimumab, we analyzed whole exomes from pretreatment melanoma tumor biopsies and matching germline tissue samples from 110 patients. For 40 of these patients, we also obtained and analyzed transcriptome data from the pretreatment tumor samples. Overall mutational load, neoantigen load, and expression of cytolytic markers in the immune microenvironment were significantly associated with clinical benefit. However, no recurrent neoantigen peptide sequences predicted responder patient populations. Thus, detailed integrated molecular characterization of large patient cohorts may be needed to identify robust determinants of response and resistance to immune checkpoint inhibitors.', 'year': 2015, 'citationCount': 2114, 'authors': [{'authorId': '3459785', 'name': 'E. V. Van Allen'}, {'authorId': '36477474', 'name': 'Diana Miao'}, {'authorId': '145830928', 'name': 'B. Schilling'}, {'authorId': '2150905503', 'name': 'S. Shukla'}, {'authorId': '40616063', 'name': 'C. Blank'}, {'authorId': '32670858', 'name': 'L. Zimmer'}, {'authorId': '3864817', 'name': 'A. Sucker'}, {'authorId': '5005890', 'name': 'U. Hillen'}, {'authorId': '89064804', 'name': 'M. G. Geukes Foppen'}, {'authorId': '1974776', 'name': 'S. Goldinger'}, {'authorId': '6070014', 'name': 'J. Utikal'}, {'authorId': '144237883', 'name': 'J. Hassel'}, {'authorId': '3631833', 'name': 'B. Weide'}, {'authorId': '1390736503', 'name': 'K. Kaehler'}, {'authorId': '6815231', 'name': 'C. Loquai'}, {'authorId': '144577542', 'name': 'P. Mohr'}, {'authorId': '152420720', 'name': 'R. Gutzmer'}, {'authorId': '144139860', 'name': 'R. Dummer'}, {'authorId': '34428249', 'name': 'S. Gabriel'}, {'authorId': '1920712896', 'name': 'Catherine J. Wu'}, {'authorId': '144267265', 'name': 'D. Schadendorf'}, {'authorId': '3003287', 'name': 'L. Garraway'}]}, {'paperId': '514e52838f7513a8eab75a51814d2d06acca1baf', 'title': 'Commensal Bifidobacterium promotes antitumor immunity and facilitates anti–PD-L1 efficacy', 'abstract': 'Gut microbes affect immunotherapy The unleashing of antitumor T cell responses has ushered in a new era of cancer treatment. Although these therapies can cause dramatic tumor regressions in some patients, many patients inexplicably see no benefit. Mice have been used in two studies to investigate what might be happening. Specific members of the gut microbiota influence the efficacy of this type of immunotherapy (see the Perspective by Snyder et al.). Vétizou et al. found that optimal responses to anticytotoxic T lymphocyte antigen blockade required specific Bacteroides spp. Similarly, Sivan et al. discovered that Bifidobacterium spp. enhanced the efficacy of antiprogrammed cell death ligand 1 therapy. Science, this issue, p. 1079 and p. 1084; see also p. 1031 Gut microbes modulate the effectiveness of cancer immunotherapies in mice. [Also see Perspective by Snyder et al.] T cell infiltration of solid tumors is associated with favorable patient outcomes, yet the mechanisms underlying variable immune responses between individuals are not well understood. One possible modulator could be the intestinal microbiota. We compared melanoma growth in mice harboring distinct commensal microbiota and observed differences in spontaneous antitumor immunity, which were eliminated upon cohousing or after fecal transfer. Sequencing of the 16S ribosomal RNA identified Bifidobacterium as associated with the antitumor effects. Oral administration of Bifidobacterium alone improved tumor control to the same degree as programmed cell death protein 1 ligand 1 (PD-L1)–specific antibody therapy (checkpoint blockade), and combination treatment nearly abolished tumor outgrowth. Augmented dendritic cell function leading to enhanced CD8+ T cell priming and accumulation in the tumor microenvironment mediated the effect. Our data suggest that manipulating the microbiota may modulate cancer immunotherapy.', 'year': 2015, 'citationCount': 2492, 'authors': [{'authorId': '6010657', 'name': 'A. Sivan'}, {'authorId': '1899112', 'name': 'L. Corrales'}, {'authorId': '47063763', 'name': 'N. Hubert'}, {'authorId': '2111787256', 'name': 'Jason B. Williams'}, {'authorId': '1398500939', 'name': 'K. Aquino-Michaels'}, {'authorId': '3718121', 'name': 'Z. Earley'}, {'authorId': '1392752860', 'name': 'Franco W. Benyamin'}, {'authorId': '1768772854', 'name': 'Yuk Man Lei'}, {'authorId': '4216729', 'name': 'B. Jabri'}, {'authorId': '82924398', 'name': 'M. Alegre'}, {'authorId': '1702329', 'name': 'E. Chang'}, {'authorId': '3283308', 'name': 'T. Gajewski'}]}, {'paperId': '3c6e60e52c6a9f8b0998174030ebdb6a9fde5aa8', 'title': 'Rationally engineered Cas9 nucleases with improved specificity', 'abstract': 'Making the correct cut The CRISPR/Cas system is a prokaryotic immune system that targets and cuts out foreign DNA in bacteria. It has been adopted for gene editing because it can be designed to recognize and cut specific locations in the genome. A challenge in developing clinical applications is the potential for off-target effects that could result in DNA cleavage at the wrong locations. Slaymaker et al. used structure-guided engineering to improve the specificity of Streptococcus pyogenes Cas9 (SpCas9). They identified enhanced-specificity variants (eSpCas9) that display reduced off-target cleavage while maintaining robust on-target activity Science, this issue p. 84 Structure-guided engineering improves the genome editing specificity of the CRISPR-associated endonuclease Cas9. The RNA-guided endonuclease Cas9 is a versatile genome-editing tool with a broad range of applications from therapeutics to functional annotation of genes. Cas9 creates double-strand breaks (DSBs) at targeted genomic loci complementary to a short RNA guide. However, Cas9 can cleave off-target sites that are not fully complementary to the guide, which poses a major challenge for genome editing. Here, we use structure-guided protein engineering to improve the specificity of Streptococcus pyogenes Cas9 (SpCas9). Using targeted deep sequencing and unbiased whole-genome off-target analysis to assess Cas9-mediated DNA cleavage in human cells, we demonstrate that “enhanced specificity” SpCas9 (eSpCas9) variants reduce off-target effects and maintain robust on-target cleavage. Thus, eSpCas9 could be broadly useful for genome-editing applications requiring a high level of specificity.', 'year': 2015, 'citationCount': 1857, 'authors': [{'authorId': '4201929', 'name': 'I. Slaymaker'}, {'authorId': '3391994', 'name': 'L. Gao'}, {'authorId': '4627929', 'name': 'Bernd Zetsche'}, {'authorId': '153167506', 'name': 'David A. Scott'}, {'authorId': '5851153', 'name': 'Winston X. Yan'}, {'authorId': '145126988', 'name': 'Feng Zhang'}]}, {'paperId': '927a32a1b1c69a6041a586b71f52be1b957e5030', 'title': 'Promoting an open research culture', 'abstract': 'Author guidelines for journals could help to promote transparency, openness, and reproducibility Transparency, openness, and reproducibility are readily recognized as vital features of science (1, 2). When asked, most scientists embrace these features as disciplinary norms and values (3). Therefore, one might expect that these valued features would be routine in daily practice. Yet, a growing body of evidence suggests that this is not the case (4–6).', 'year': 2015, 'citationCount': 1653, 'authors': [{'authorId': '2862527', 'name': 'Brian A. Nosek'}, {'authorId': '2139865103', 'name': 'G. Alter'}, {'authorId': '8611013', 'name': 'G. Banks'}, {'authorId': '3237553', 'name': 'D. Borsboom'}, {'authorId': '72356049', 'name': 'S. Bowman'}, {'authorId': '6240718', 'name': 'S. Breckler'}, {'authorId': '2073872541', 'name': 'S. Buck'}, {'authorId': '46975669', 'name': 'C. Chambers'}, {'authorId': '2058346280', 'name': 'G. Chin'}, {'authorId': '2054652938', 'name': 'G. Christensen'}, {'authorId': '50230722', 'name': 'M. Contestabile'}, {'authorId': '3198576', 'name': 'A. Dafoe'}, {'authorId': '34840029', 'name': 'E. Eich'}, {'authorId': '2364509', 'name': 'J. Freese'}, {'authorId': '4249203', 'name': 'R. Glennerster'}, {'authorId': '13065618', 'name': 'D. Goroff'}, {'authorId': '48997800', 'name': 'D. P. Green'}, {'authorId': '119365490', 'name': 'B. Hesse'}, {'authorId': '32272151', 'name': 'M. Humphreys'}, {'authorId': '46636130', 'name': 'J. Ishiyama'}, {'authorId': '2124080032', 'name': 'D. Karlan'}, {'authorId': '47733218', 'name': 'A. Kraut'}, {'authorId': '47975256', 'name': 'A. Lupia'}, {'authorId': '3154113', 'name': 'P. Mabry'}, {'authorId': '4133783', 'name': 'T. Madon'}, {'authorId': '118685494', 'name': 'N. Malhotra'}, {'authorId': '1390039913', 'name': 'E. Mayo-Wilson'}, {'authorId': '114856072', 'name': 'M. McNutt'}, {'authorId': '144624624', 'name': 'E. Miguel'}, {'authorId': '4659343', 'name': 'E. Paluck'}, {'authorId': '2130902', 'name': 'U. Simonsohn'}, {'authorId': '4110370', 'name': 'C. Soderberg'}, {'authorId': '101916661', 'name': 'B. Spellman'}, {'authorId': '1398590331', 'name': 'J. Turitto'}, {'authorId': '80222152', 'name': 'G. VandenBos'}, {'authorId': '2809726', 'name': 'S. Vazire'}, {'authorId': '2505272', 'name': 'E. Wagenmakers'}, {'authorId': '153160363', 'name': 'R. Wilson'}, {'authorId': '2075675', 'name': 'T. Yarkoni'}]}, {'paperId': '0e166790fb686d43bb6983800d05c38766c79d39', 'title': 'The human transcriptome across tissues and individuals', 'abstract': 'Expression, genetic variation, and tissues Human genomes show extensive genetic variation across individuals, but we have only just started documenting the effects of this variation on the regulation of gene expression. Furthermore, only a few tissues have been examined per genetic variant. In order to examine how genetic expression varies among tissues within individuals, the Genotype-Tissue Expression (GTEx) Consortium collected 1641 postmortem samples covering 54 body sites from 175 individuals. They identified quantitative genetic traits that affect gene expression and determined which of these exhibit tissue-specific expression patterns. Melé et al. measured how transcription varies among tissues, and Rivas et al. looked at how truncated protein variants affect expression across tissues. Science, this issue p. 648, p. 660, p. 666; see also p. 640 RNA expression documents patterns of human transcriptome variation across individuals and tissues. [Also see Perspective by Gibson] Transcriptional regulation and posttranscriptional processing underlie many cellular and organismal phenotypes. We used RNA sequence data generated by Genotype-Tissue Expression (GTEx) project to investigate the patterns of transcriptome variation across individuals and tissues. Tissues exhibit characteristic transcriptional signatures that show stability in postmortem samples. These signatures are dominated by a relatively small number of genes—which is most clearly seen in blood—though few are exclusive to a particular tissue and vary more across tissues than individuals. Genes exhibiting high interindividual expression variation include disease candidates associated with sex, ethnicity, and age. Primary transcription is the major driver of cellular specificity, with splicing playing mostly a complementary role; except for the brain, which exhibits a more divergent splicing program. Variation in splicing, despite its stochasticity, may play in contrast a comparatively greater role in defining individual phenotypes.', 'year': 2015, 'citationCount': 1054, 'authors': [{'authorId': '2326247', 'name': 'Marta Melé'}, {'authorId': '145209580', 'name': 'Pedro G. Ferreira'}, {'authorId': '1704215264', 'name': 'F. Reverter'}, {'authorId': '46980374', 'name': 'D. DeLuca'}, {'authorId': '6596913', 'name': 'Jean Monlong'}, {'authorId': '1971387', 'name': 'M. Sammeth'}, {'authorId': '31774477', 'name': 'Taylor R. Young'}, {'authorId': '47824049', 'name': 'J. Goldmann'}, {'authorId': '3288223', 'name': 'D. Pervouchine'}, {'authorId': '2068615525', 'name': 'T. Sullivan'}, {'authorId': '1405949033', 'name': 'Rory Johnson'}, {'authorId': '4155963', 'name': 'A. Segrè'}, {'authorId': '3434507', 'name': 'S. Djebali'}, {'authorId': '4501384', 'name': 'Anastasia Niarchou'}, {'authorId': '2122677498', 'name': 'The GTEx Consortium'}, {'authorId': '2056823854', 'name': 'Fred A. Wright'}, {'authorId': '144946488', 'name': 'T. Lappalainen'}, {'authorId': '2060334738', 'name': 'Miquel Calvo'}, {'authorId': '2110594', 'name': 'G. Getz'}, {'authorId': '2939073', 'name': 'E. Dermitzakis'}, {'authorId': '4351471', 'name': 'K. Ardlie'}, {'authorId': '1853610', 'name': 'R. Guigó'}]}, {'paperId': 'd7b42a977e9c09654b5a4e65538fb56bc9252927', 'title': 'Spatially resolved, highly multiplexed RNA profiling in single cells', 'abstract': 'Multiplexed RNA imaging in single cells The basis of cellular function is where and when proteins are expressed and in what quantities. Single-molecule fluorescence in situ hybridization (smFISH) experiments quantify the copy number and location of mRNA molecules; however, the numbers of RNA species that can be simultaneously measured by smFISH has been limited. Using combinatorial labeling with error-robust encoding schemes, Chen et al. simultaneously imaged 100 to 1000 RNA species in a single cell. Such large-scale detection allows regulatory interactions to be analyzed at the transcriptome scale. Science, this issue p. 10.1126/science.aaa6090 A single-molecule imaging method allows simultaneous measurement of 1000 RNA species in single cells. INTRODUCTION The copy number and intracellular localization of RNA are important regulators of gene expression. Measurement of these properties at the transcriptome scale in single cells will give answers to many questions related to gene expression and regulation. Single-molecule RNA imaging approaches, such as single-molecule fluorescence in situ hybridization (smFISH), are powerful tools for counting and mapping RNA; however, the number of RNA species that can be simultaneously imaged in individual cells has been limited. This makes it challenging to perform transcriptomic analysis of single cells in a spatially resolved manner. Here, we report multiplexed error-robust FISH (MERFISH), a single-molecule imaging method that allows thousands of RNA species to be imaged in single cells by using combinatorial FISH labeling with encoding schemes capable of detecting and/or correcting errors. RATIONALE We labeled each cellular RNA with a set of encoding probes, which contain targeting sequences that bind the RNA and readout sequences that bind fluorescently labeled readout probes. Each RNA species is encoded with a particular combination of readout sequences. We used successive rounds of hybridization and imaging, each with a different readout probe, to identify the readout sequences bound to each RNA and to decode the RNA. In principle, combinatorial labeling allows the number of detectable RNA species to grow exponentially with the number of imaging rounds, but the detection errors also increase exponentially. To combat such accumulating errors, we exploited error-robust encoding schemes used in digital electronics, such as the extended Hamming code, in the design of our encoding probes but modified these schemes in order to account for the error properties in FISH measurements. We assigned each RNA a binary word in our modified Hamming code and encoded the RNA with a combination of readout sequences according to this binary word. RESULTS We first imaged 140 RNA species in human fibroblast cells using MERFISH with 16 rounds of hybridization and a modified Hamming code capable of both error detection and correction. We obtained ~80% detection efficiency and observed excellent correlation of RNA copy numbers determined with MERFISH with both bulk RNA sequencing data and conventional smFISH measurements of individual genes. Next, we used an alternative MERFISH encoding scheme, which is capable of detecting but not correcting errors, to image 1001 RNA species in individual cells using only 14 rounds of hybridization. The observed RNA copy numbers again correlate well with bulk sequencing data. However, the detection efficiency is only one-third that of the error-correcting encoding scheme. We performed correlation analysis of the 104 to 106 pairs of measured genes and identified many covarying gene groups that share common regulatory elements. Such grouping allowed us to hypothesize potential functions of ~100 unannotated or partially annotated genes of unknown functions. We further analyzed correlations in the spatial distributions of different RNA species and identified groups of RNAs with different distribution patterns in the cell. DISCUSSION This highly multiplexed imaging approach enables analyses based on the variation and correlation of copy numbers and spatial distributions of a large number of RNA species within single cells. Such analyses should facilitate the delineation of regulatory networks and in situ identification of cell types. We envision that this approach will allow spatially resolved transcriptomes to be determined for single cells. MERFISH for transcriptome imaging. Numerous RNA species can be identified, counted, and localized in a single cell by using MERFISH, a single-molecule imaging approach that uses combinatorial labeling and sequential imaging with encoding schemes capable of detection and/or correction of errors. This highly multiplexed measurement of individual RNAs can be used to compute the gene expression profile and noise, covariation in expression among different genes, and spatial distribution of RNAs within single cells. Knowledge of the expression profile and spatial landscape of the transcriptome in individual cells is essential for understanding the rich repertoire of cellular behaviors. Here, we report multiplexed error-robust fluorescence in situ hybridization (MERFISH), a single-molecule imaging approach that allows the copy numbers and spatial localizations of thousands of RNA species to be determined in single cells. Using error-robust encoding schemes to combat single-molecule labeling and detection errors, we demonstrated the imaging of 100 to 1000 distinct RNA species in hundreds of individual cells. Correlation analysis of the ~104 to 106 pairs of genes allowed us to constrain gene regulatory networks, predict novel functions for many unannotated genes, and identify distinct spatial distribution patterns of RNAs that correlate with properties of the encoded proteins.', 'year': 2015, 'citationCount': 1428, 'authors': [{'authorId': '50053851', 'name': 'Kok Hao Chen'}, {'authorId': '2291340', 'name': 'A. Boettiger'}, {'authorId': '2786991', 'name': 'J. Moffitt'}, {'authorId': '2116420386', 'name': 'Siyuan S Wang'}, {'authorId': '1837985', 'name': 'X. Zhuang'}]}, {'paperId': '2db355f222fc6b05df2e36033d1b1d24acb799ab', 'title': 'm6A mRNA methylation facilitates resolution of naïve pluripotency toward differentiation', 'abstract': 'mRNA modification regulates pluripotency When stem cells progress from an embryonic pluripotent state toward a particular lineage, molecular switches dismantle the transcription factor network that keeps the cell pluripotent. Geula et al. now show that N6-methyladenosine (m6A), a messenger RNA (mRNA) modification present on transcripts of pluripotency factors, drives this transition. Methylation destabilized mRNA transcripts and limited their translation efficiency, which promoted the timely decay of naïve pluripotency. This m6A methylation was also critical for mammalian development. Science, this issue p. 1002 A messenger RNA epigenetic modification regulates stem cell progression from the pluripotent to the differentiated state. Naïve and primed pluripotent states retain distinct molecular properties, yet limited knowledge exists on how their state transitions are regulated. Here, we identify Mettl3, an N6-methyladenosine (m6A) transferase, as a regulator for terminating murine naïve pluripotency. Mettl3 knockout preimplantation epiblasts and naïve embryonic stem cells are depleted for m6A in mRNAs, yet are viable. However, they fail to adequately terminate their naïve state and, subsequently, undergo aberrant and restricted lineage priming at the postimplantation stage, which leads to early embryonic lethality. m6A predominantly and directly reduces mRNA stability, including that of key naïve pluripotency-promoting transcripts. This study highlights a critical role for an mRNA epigenetic modification in vivo and identifies regulatory modules that functionally influence naïve and primed pluripotency in an opposing manner.', 'year': 2015, 'citationCount': 1173, 'authors': [{'authorId': '6819175', 'name': 'Shay Geula'}, {'authorId': '1398554907', 'name': 'S. Moshitch-Moshkovitz'}, {'authorId': '4119568', 'name': 'D. Dominissini'}, {'authorId': '31155224', 'name': 'A. Mansour'}, {'authorId': '2336656', 'name': 'N. Kol'}, {'authorId': '1398554898', 'name': 'M. Salmon-Divon'}, {'authorId': '6481645', 'name': 'Vera Hershkovitz'}, {'authorId': '49925624', 'name': 'Eyal Peer'}, {'authorId': '7998267', 'name': 'Nofar Mor'}, {'authorId': '47428977', 'name': 'Y. Manor'}, {'authorId': '1398555041', 'name': 'Moshe Shay Ben-Haim'}, {'authorId': '3184215', 'name': 'E. Eyal'}, {'authorId': '6895327', 'name': 'Sharon Yunger'}, {'authorId': '2735516', 'name': 'Y. Pinto'}, {'authorId': '5425973', 'name': 'Diego Jaitin'}, {'authorId': '4322985', 'name': 'Sergey V. Viukov'}, {'authorId': '6581218', 'name': 'Y. Rais'}, {'authorId': '4992332', 'name': 'V. Krupalnik'}, {'authorId': '5125935', 'name': 'Elad Chomsky'}, {'authorId': '48950453', 'name': 'Mirie Zerbib'}, {'authorId': '32317040', 'name': 'I. Maza'}, {'authorId': '8244084', 'name': 'Yoav Rechavi'}, {'authorId': '7526795', 'name': \"R'ada Massarwa\"}, {'authorId': '40379130', 'name': 'Suhair Hanna'}, {'authorId': '2960101', 'name': 'I. Amit'}, {'authorId': '2672226', 'name': 'E. Levanon'}, {'authorId': '3432577', 'name': 'N. Amariglio'}, {'authorId': '1397179745', 'name': 'Noam Stern-Ginossar'}, {'authorId': '2376964', 'name': 'Noa Novershtern'}, {'authorId': '1786579', 'name': 'G. Rechavi'}, {'authorId': '1820962', 'name': 'J. Hanna'}]}, {'paperId': 'b3634280958e617fbedad078961f816538da1944', 'title': 'Continuous liquid interface production of 3D objects', 'abstract': 'Fast, continuous, 3D printing Although three-dimensional (3D) printing is now possible using relatively small and low-cost machines, it is still a fairly slow process. This is because 3D printers require a series of steps to cure, replenish, and reposition themselves for each additive cycle. Tumbleston et al. devised a process to effectively grow solid structures out of a liquid bath. The key to the process is the creation of an oxygen-containing “dead zone” between the solid part and the liquid precursor where solidification cannot occur. The precursor liquid is then renewed by the upward movement of the growing solid part. This approach made structures tens of centimeters in size that could contain features with a resolution below 100 µm. Science, this issue p. 1349 Solid parts are elevated from a liquid resin pool at a speed of hundreds of millimeters per hour. Additive manufacturing processes such as 3D printing use time-consuming, stepwise layer-by-layer approaches to object fabrication. We demonstrate the continuous generation of monolithic polymeric parts up to tens of centimeters in size with feature resolution below 100 micrometers. Continuous liquid interface production is achieved with an oxygen-permeable window below the ultraviolet image projection plane, which creates a “dead zone” (persistent liquid interface) where photopolymerization is inhibited between the window and the polymerizing part. We delineate critical control parameters and show that complex solid parts can be drawn out of the resin at rates of hundreds of millimeters per hour. These print speeds allow parts to be produced in minutes instead of hours.', 'year': 2015, 'citationCount': 1456, 'authors': [{'authorId': '3856850', 'name': 'John R. Tumbleston'}, {'authorId': '2888642', 'name': 'D. Shirvanyants'}, {'authorId': '4737061', 'name': 'N. Ermoshkin'}, {'authorId': '5768526', 'name': 'R. Janusziewicz'}, {'authorId': '25711869', 'name': 'Ashley R. Johnson'}, {'authorId': '2070074460', 'name': 'David Kelly'}, {'authorId': '2118439764', 'name': 'Kai Chen'}, {'authorId': '69923437', 'name': 'R. Pinschmidt'}, {'authorId': '1850765', 'name': 'J. Rolland'}, {'authorId': '2661307', 'name': 'A. Ermoshkin'}, {'authorId': '49805613', 'name': 'E. Samulski'}, {'authorId': '6880755', 'name': 'J. Desimone'}]}, {'paperId': '667be50b805b2ac510150821c68e9f911cbe0d1b', 'title': 'Principles of connectivity among morphologically defined cell types in adult neocortex', 'abstract': \"A census of neocortical neurons Despite the importance of the brain's neocortex, we still do not completely understand the diversity and functional connections of its cell types. Jiang et al. recorded, labeled, and classified over 1200 interneurons and more than 400 pyramidal neurons in the mature mouse visual cortex. Fifteen major classes of interneurons fell into three types: some connect to all neurons, some connect to other interneurons, and some form synapses with pyramidal neurons. Science, this issue p. 10.1126/science.aac9462 The connections between more than 10,000 pairs of individually classified neurons in the visual cortex of adult mice are mapped. INTRODUCTION The intricate microcircuitry of the cerebral cortex is thought to be a critical substrate from which arise the impressive capabilities of the mammalian brain. Until now, our knowledge of the stereotypical connectivity in neocortical microcircuits has been pieced together from individual studies of the connectivity between small numbers of neuronal cell types. Here, we provide unbiased, large-scale profiling of neuronal cell types and connections to reveal the essential building blocks of the cortex and the principles governing their assembly into cortical circuits. Using advanced techniques for tissue slicing, multiple simultaneous whole-cell recording, and morphological reconstruction, we are able to provide a comprehensive view of the connectivity between diverse types of neurons, particularly among types of γ-aminobutyric acid–releasing (GABAergic) interneurons, in the adult animal. RATIONALE We took advantage of a method for preparing high-quality slices of adult tissue and combined this technique with octuple simultaneous, whole-cell recordings followed by an improved staining method that allowed detailed recovery of axonal and dendritic arbor morphology. These data allowed us to perform a census of morphologically and electrophysiologically defined neuronal types (primarily GABAergic interneurons) in neocortical layers 1, 2/3, and 5 (L1, L23, and L5, respectively) and to observe their connectivity patterns in adult animals. RESULTS Our large-scale, comprehensive profiling of neocortical neurons differentiated 15 major types of interneurons, in addition to two lamina-defined types of pyramidal neurons (L23 and L5). Cortical interneurons comprise two types in L1 (eNGC and SBC-like), seven in L23 (L23MC, L23NGC, BTC, BPC, DBC, L23BC, and ChC), and six in L5 (L5MC, L5NGC, L5BC, SC, HEC, and DC) (see the figure). Each type has stereotypical electrophysiological properties and morphological features and can be differentiated from all others by cell type–specific axonal geometry and axonal projection patterns. Importantly, each type of neuron has its own characteristic input-output connectivity profile, connecting with other constituent neuronal types with varying degrees of specificity in postsynaptic targets, laminar location, and synaptic characteristics. Despite specific connection patterns for each cell type, we found that a small number of simple connectivity motifs are repeated across layers and cell types defining a canonical cortical microcircuit. CONCLUSION Our comprehensive profiling of neuronal cell types and connections in adult neocortex provides the most complete wiring diagram of neocortical microcircuits to date. Compared with current genetic labels for cell class, which paint the cortex in broad strokes, our analysis of morphological and electrophysiological properties revealed new cell classes and allowed us to derive a small number of simple connectivity rules that were repeated across layers and cell types. This detailed blueprint of cortical wiring should aid efforts to identify specific circuit abnormalities in animal models of brain disease and may eventually provide a path toward the development of comprehensive circuit-based, cell type–specific interventions. Connectivity among morphologically defined cell types in adult neocortex. (A) Simultaneous octuple whole-cell recording to study connectivity followed by morphological reconstruction. (B) Synaptic connectivity between morphologically distinct types of neurons, including pyramidal (P) neurons. (C) Connectivity from NGCs to other cell types. This connectivity is believed to be nonsynaptic and mediated by volume transmission. Martinotti cell, MC; neurogliaform cell, NGC; basket cell, BC; single-bouquet cell-like cell, SBC-like; bitufted cell, BTC; bipolar cell, BPC, double-bouquet cell, DBC; chandelier cell, ChC; shrub cell, SC; horizontally elongated cell, HEC; deep-projecting cell, DC. Since the work of Ramón y Cajal in the late 19th and early 20th centuries, neuroscientists have speculated that a complete understanding of neuronal cell types and their connections is key to explaining complex brain functions. However, a complete census of the constituent cell types and their wiring diagram in mature neocortex remains elusive. By combining octuple whole-cell recordings with an optimized avidin-biotin-peroxidase staining technique, we carried out a morphological and electrophysiological census of neuronal types in layers 1, 2/3, and 5 of mature neocortex and mapped the connectivity between more than 11,000 pairs of identified neurons. We categorized 15 types of interneurons, and each exhibited a characteristic pattern of connectivity with other interneuron types and pyramidal cells. The essential connectivity structure of the neocortical microcircuit could be captured by only a few connectivity motifs.\", 'year': 2015, 'citationCount': 696, 'authors': [{'authorId': '46812938', 'name': 'Xiaolong Jiang'}, {'authorId': '2072819148', 'name': 'Shan Shen'}, {'authorId': '5686630', 'name': 'C. Cadwell'}, {'authorId': '1689077', 'name': 'Philipp Berens'}, {'authorId': '50095217', 'name': 'Fabian H Sinz'}, {'authorId': '1746183', 'name': 'Alexander S. Ecker'}, {'authorId': '4330752', 'name': 'Saumil S. Patel'}, {'authorId': '1739838', 'name': 'A. Tolias'}]}, {'paperId': '4414a1e13595c8ff7bdc861c5725836bc11d76e0', 'title': 'Exome sequencing in amyotrophic lateral sclerosis identifies risk genes and pathways', 'abstract': \"New players in Lou Gehrig's disease Amyotrophic lateral sclerosis (ALS), often referred to as “Lou Gehrig's disease,” is a progressive neurodegenerative disease that affects nerve cells in the brain and the spinal cord. Cirulli et al. sequenced the expressed genes of nearly 3000 ALS patients and compared them with those of more than 6000 controls (see the Perspective by Singleton and Traynor). They identified several proteins that were linked to disease in patients. One such protein, TBK1, is implicated in innate immunity and autophagy and may represent a therapeutic target. Science, this issue p. 1436; see also p. 1422 Analysis of the expressed genes of nearly 2900 patients with amyotrophic lateral sclerosis and about 6400 controls reveals a disease predisposition–associated gene. [Also see Perspective by Singleton and Traynor] Amyotrophic lateral sclerosis (ALS) is a devastating neurological disease with no effective treatment. We report the results of a moderate-scale sequencing study aimed at increasing the number of genes known to contribute to predisposition for ALS. We performed whole-exome sequencing of 2869 ALS patients and 6405 controls. Several known ALS genes were found to be associated, and TBK1 (the gene encoding TANK-binding kinase 1) was identified as an ALS gene. TBK1 is known to bind to and phosphorylate a number of proteins involved in innate immunity and autophagy, including optineurin (OPTN) and p62 (SQSTM1/sequestosome), both of which have also been implicated in ALS. These observations reveal a key role of the autophagic pathway in ALS and suggest specific targets for therapeutic intervention.\", 'year': 2015, 'citationCount': 827, 'authors': [{'authorId': '2853875', 'name': 'E. Cirulli'}, {'authorId': '6055820', 'name': 'Brittany N. Lasseigne'}, {'authorId': '2024504', 'name': 'S. Petrovski'}, {'authorId': '5755914', 'name': 'P. Sapp'}, {'authorId': '2546787', 'name': 'P. Dion'}, {'authorId': '3641674', 'name': 'C. Leblond'}, {'authorId': '6713179', 'name': 'Julien Couthouis'}, {'authorId': '2256947', 'name': 'Yi-Fan Lu'}, {'authorId': '2145348474', 'name': 'Quanli Wang'}, {'authorId': '34252586', 'name': 'Brian J. Krueger'}, {'authorId': '144653215', 'name': 'Z. Ren'}, {'authorId': '32045858', 'name': 'Jonathan E. M. Keebler'}, {'authorId': '48386356', 'name': 'Yujun Han'}, {'authorId': '33853109', 'name': 'S. Levy'}, {'authorId': '1795313', 'name': 'B. Boone'}, {'authorId': '5113017', 'name': 'Jack R. Wimbish'}, {'authorId': '31952392', 'name': 'L. Waite'}, {'authorId': '39049918', 'name': 'Angela L. Jones'}, {'authorId': '1695735', 'name': 'J. Carulli'}, {'authorId': '1380548039', 'name': 'A. Day-Williams'}, {'authorId': '5295959', 'name': 'J. Staropoli'}, {'authorId': '3573620', 'name': 'W. Xin'}, {'authorId': '4939557', 'name': 'A. Chesi'}, {'authorId': '144284776', 'name': 'Alya R. Raphael'}, {'authorId': '1412594994', 'name': 'D. McKenna-Yasek'}, {'authorId': '144166444', 'name': 'J. Cady'}, {'authorId': '8360115', 'name': 'J. M. B. Vianney de Jong'}, {'authorId': '4292338', 'name': 'K. Kenna'}, {'authorId': '5849128', 'name': 'Bradley N. Smith'}, {'authorId': '32677588', 'name': 'S. Topp'}, {'authorId': '2118099706', 'name': 'Jack W. Miller'}, {'authorId': '7790207', 'name': 'A. Gkazi'}, {'authorId': '87486204', 'name': 'A. Al-Chalabi'}, {'authorId': '4703755', 'name': 'L. H. van den Berg'}, {'authorId': '50075325', 'name': 'J. Veldink'}, {'authorId': '2989411', 'name': 'V. Silani'}, {'authorId': '5230585', 'name': 'N. Ticozzi'}, {'authorId': '144756102', 'name': 'C. Shaw'}, {'authorId': '6466995', 'name': 'R. Baloh'}, {'authorId': '1992098', 'name': 'S. Appel'}, {'authorId': '40023500', 'name': 'E. Simpson'}, {'authorId': '1398469732', 'name': 'C. Lagier-Tourenne'}, {'authorId': '34857044', 'name': 'S. Pulst'}, {'authorId': '34924662', 'name': 'S. Gibson'}, {'authorId': '2410201', 'name': 'J. Trojanowski'}, {'authorId': '143893347', 'name': 'L. Elman'}, {'authorId': '145315316', 'name': 'L. McCluskey'}, {'authorId': '144535321', 'name': 'M. Grossman'}, {'authorId': '3750137', 'name': 'N. Shneider'}, {'authorId': '145062232', 'name': 'W. Chung'}, {'authorId': '5606258', 'name': 'J. Ravits'}, {'authorId': '2069522', 'name': 'J. Glass'}, {'authorId': '4127881', 'name': 'K. Sims'}, {'authorId': '5159642', 'name': 'V. V. van Deerlin'}, {'authorId': '2551763', 'name': 'T. Maniatis'}, {'authorId': '5715284', 'name': 'Sebastian Hayes'}, {'authorId': '4096885', 'name': 'A. Ordureau'}, {'authorId': '21152395', 'name': 'S. Swarup'}, {'authorId': '4754915', 'name': 'J. Landers'}, {'authorId': '1995886', 'name': 'F. Baas'}, {'authorId': '144643055', 'name': 'A. Allen'}, {'authorId': '6241427', 'name': 'R. Bedlack'}, {'authorId': '143851342', 'name': 'J. Harper'}, {'authorId': '2102239', 'name': 'A. Gitler'}, {'authorId': '2153485', 'name': 'G. Rouleau'}, {'authorId': '2235964004', 'name': 'Robert H. Brown'}, {'authorId': '22104678', 'name': 'M. Harms'}, {'authorId': '31273465', 'name': 'G. Cooper'}, {'authorId': '2067989942', 'name': 'Tim Harris'}, {'authorId': '34819901', 'name': 'R. Myers'}, {'authorId': '50252657', 'name': 'D. Goldstein'}]}, {'paperId': '96511e6b1a6e0a231f08e933becd20687ca9c41d', 'title': 'High burden and pervasive positive selection of somatic mutations in normal human skin', 'abstract': \"Normal skin's curiously abnormal genome Within every tumor, a battle is being waged. As individual tumor cells acquire new mutations that promote their survival and growth, they clonally expand at the expense of tumor cells that are “less fit.” Martincorena et al. sequenced 234 biopsies of sun-exposed but physiologically normal skin from four individuals (see the Perspective by Brash). They found a surprisingly high burden of mutations, higher than that of many tumors. Many of the mutations known to drive the growth of cutaneous squamous cell carcinomas were already under strong positive selection. More than a quarter of normal skin cells carried a driver mutation, and every square centimeter of skin contained hundreds of competing mutant clones. Science, this issue p. 880; see also p. 867 Sun-exposed but physiologically normal human skin harbors an unexpectedly high number of cancer-causing mutations. [Also see Perspective by Brash] How somatic mutations accumulate in normal cells is central to understanding cancer development but is poorly understood. We performed ultradeep sequencing of 74 cancer genes in small (0.8 to 4.7 square millimeters) biopsies of normal skin. Across 234 biopsies of sun-exposed eyelid epidermis from four individuals, the burden of somatic mutations averaged two to six mutations per megabase per cell, similar to that seen in many cancers, and exhibited characteristic signatures of exposure to ultraviolet light. Remarkably, multiple cancer genes are under strong positive selection even in physiologically normal skin, including most of the key drivers of cutaneous squamous cell carcinomas. Positively selected mutations were found in 18 to 32% of normal skin cells at a density of ~140 driver mutations per square centimeter. We observed variability in the driver landscape among individuals and variability in the sizes of clonal expansions across genes. Thus, aged sun-exposed skin is a patchwork of thousands of evolving clones with over a quarter of cells carrying cancer-causing mutations while maintaining the physiological functions of epidermis.\", 'year': 2015, 'citationCount': 1159, 'authors': [{'authorId': '2748985', 'name': 'I. Martincorena'}, {'authorId': '145503694', 'name': 'A. Roshan'}, {'authorId': '2420343', 'name': 'M. Gerstung'}, {'authorId': '2059629181', 'name': 'P. Ellis'}, {'authorId': '2358657', 'name': 'P. Van Loo'}, {'authorId': '3846300', 'name': 'Stuart McLaren'}, {'authorId': '2824722', 'name': 'D. Wedge'}, {'authorId': '8037157', 'name': 'A. Fullam'}, {'authorId': '2757728', 'name': 'L. Alexandrov'}, {'authorId': '143863567', 'name': 'J. Tubío'}, {'authorId': '2933842', 'name': 'L. Stebbings'}, {'authorId': '143864634', 'name': 'A. Menzies'}, {'authorId': '152218260', 'name': 'S. Widaa'}, {'authorId': '144693778', 'name': 'M. Stratton'}, {'authorId': '145090665', 'name': 'P. Jones'}, {'authorId': '145761328', 'name': 'P. Campbell'}]}, {'paperId': 'abbf1e4d346fd561d60a46bafd0a1de6c2609a98', 'title': 'Slow adaptation in the face of rapid warming leads to collapse of the Gulf of Maine cod fishery', 'abstract': 'Several studies have documented fish populations changing in response to long-term warming. Over the past decade, sea surface temperatures in the Gulf of Maine increased faster than 99% of the global ocean. The warming, which was related to a northward shift in the Gulf Stream and to changes in the Atlantic Multidecadal Oscillation and Pacific Decadal Oscillation, led to reduced recruitment and increased mortality in the region’s Atlantic cod (Gadus morhua) stock. Failure to recognize the impact of warming on cod contributed to overfishing. Recovery of this fishery depends on sound management, but the size of the stock depends on future temperature conditions. The experience in the Gulf of Maine highlights the need to incorporate environmental factors into resource management. Warming waters prevented cod recovery in the North Atlantic. Double jeopardy In the best of worlds, exploited fish stocks are monitored so that harvest quotas protect the reproductive ability of the population. Climate change is likely to complicate this process substantially. Pershing et al. found that cod stocks declined continuously during intense warming in the North Atlantic. Fisheries quotas, even though they were responsibly set and followed by fishers, decreased the reproductive rate. Thus, managing fisheries in a warming world is going to be increasingly problematic. Science, this issue p. 809', 'year': 2015, 'citationCount': 645, 'authors': [{'authorId': '4666755', 'name': 'A. Pershing'}, {'authorId': '47099873', 'name': 'M. Alexander'}, {'authorId': '32328272', 'name': 'Christina M. Hernández'}, {'authorId': '46781149', 'name': 'L. Kerr'}, {'authorId': '48082771', 'name': 'A. Le Bris'}, {'authorId': '5046907', 'name': 'Katherine E. Mills'}, {'authorId': '31020656', 'name': 'J. Nye'}, {'authorId': '2558603', 'name': 'N. Record'}, {'authorId': '7688646', 'name': 'Hillary A. Scannell'}, {'authorId': '32999974', 'name': 'James D. Scott'}, {'authorId': '16789099', 'name': 'G. Sherwood'}, {'authorId': '145250501', 'name': 'A. Thomas'}]}, {'paperId': '81faacd7a521fd2e923227ab1dd01f199520e3df', 'title': 'Marine defaunation: Animal loss in the global ocean', 'abstract': 'Marine animals are disappearing, too The loss of animal species in terrestrial environments has been well documented and is continuing. Loss of species in marine environments has been slower than in terrestrial systems, but appears to be increasing rapidly. McCauley et al. review the recent patterns of species decline and loss in marine environments. Though they note many worrying declines, they also highlight approaches that might allow us to prevent the type of massive defaunation that has occurred on land. Science, this issue 10.1126/science.1255641 BACKGROUND Comparing patterns of terrestrial and marine defaunation helps to place human impacts on marine fauna in context and to navigate toward recovery. Defaunation began in earnest tens of thousands of years later in the oceans than it did on land. Although defaunation has been less severe in the oceans than on land, our effects on marine animals are increasing in pace and impact. Humans have caused few complete extinctions in the sea, but we are responsible for many ecological, commercial, and local extinctions. Despite our late start, humans have already powerfully changed virtually all major marine ecosystems. ADVANCES Humans have profoundly decreased the abundance of both large (e.g., whales) and small (e.g., anchovies) marine fauna. Such declines can generate waves of ecological change that travel both up and down marine food webs and can alter ocean ecosystem functioning. Human harvesters have also been a major force of evolutionary change in the oceans and have reshaped the genetic structure of marine animal populations. Climate change threatens to accelerate marine defaunation over the next century. The high mobility of many marine animals offers some increased, though limited, capacity for marine species to respond to climate stress, but it also exposes many species to increased risk from other stressors. Because humans are intensely reliant on ocean ecosystems for food and other ecosystem services, we are deeply affected by all of these forecasted changes. Three lessons emerge when comparing the marine and terrestrial defaunation experiences: (i) today’s low rates of marine extinction may be the prelude to a major extinction pulse, similar to that observed on land during the industrial revolution, as the footprint of human ocean use widens; (ii) effectively slowing ocean defaunation requires both protected areas and careful management of the intervening ocean matrix; and (iii) the terrestrial experience and current trends in ocean use suggest that habitat destruction is likely to become an increasingly dominant threat to ocean wildlife over the next 150 years. OUTLOOK Wildlife populations in the oceans have been badly damaged by human activity. Nevertheless, marine fauna generally are in better condition than terrestrial fauna: Fewer marine animal extinctions have occurred; many geographic ranges have shrunk less; and numerous ocean ecosystems remain more wild than terrestrial ecosystems. Consequently, meaningful rehabilitation of affected marine animal populations remains within the reach of managers. Human dependency on marine wildlife and the linked fate of marine and terrestrial fauna necessitate that we act quickly to slow the advance of marine defaunation. Timeline (log scale) of marine and terrestrial defaunation. The marine defaunation experience is much less advanced, even though humans have been harvesting ocean wildlife for thousands of years. The recent industrialization of this harvest, however, initiated an era of intense marine wildlife declines. If left unmanaged, we predict that marine habitat alteration, along with climate change (colored bar: IPCC warming), will exacerbate marine defaunation. Marine defaunation, or human-caused animal loss in the oceans, emerged forcefully only hundreds of years ago, whereas terrestrial defaunation has been occurring far longer. Though humans have caused few global marine extinctions, we have profoundly affected marine wildlife, altering the functioning and provisioning of services in every ocean. Current ocean trends, coupled with terrestrial defaunation lessons, suggest that marine defaunation rates will rapidly intensify as human use of the oceans industrializes. Though protected areas are a powerful tool to harness ocean productivity, especially when designed with future climate in mind, additional management strategies will be required. Overall, habitat degradation is likely to intensify as a major driver of marine wildlife loss. Proactive intervention can avert a marine defaunation disaster of the magnitude observed on land.', 'year': 2015, 'citationCount': 902, 'authors': [{'authorId': '35646245', 'name': 'D. McCauley'}, {'authorId': '27836011', 'name': 'M. Pinsky'}, {'authorId': '3857267', 'name': 'S. Palumbi'}, {'authorId': '34611309', 'name': 'J. Estes'}, {'authorId': '7591314', 'name': 'Francis H. Joyce'}, {'authorId': '32657386', 'name': 'R. R. Warner'}]}, {'paperId': '5fe8ed6e2c75a3b352be695f423cc2fac19446fa', 'title': 'Multiplex single-cell profiling of chromatin accessibility by combinatorial cellular indexing', 'abstract': 'Chromatin state and the single cell Identifying the chromatin state of any single cell, which may or may not have a different function or represent different stages relative to others collected within any single culture, experiment, or tissue, has been challenging. Cusanovitch et al. skirted previously identified technological limitations to identify regions of accessible chromatin at single-cell resolution. Combinatorial cellular indexing, a strategy for multiplex barcoding of thousands of single cells per experiment, was successfully used to investigate the genome-wide chromatin accessibility landscape in each of over 15,000 single cells. Science, this issue p. 910 Combinatorial indexing can identify chromatin states at single-cell resolution. Technical advances have enabled the collection of genome and transcriptome data sets with single-cell resolution. However, single-cell characterization of the epigenome has remained challenging. Furthermore, because cells must be physically separated before biochemical processing, conventional single-cell preparatory methods scale linearly. We applied combinatorial cellular indexing to measure chromatin accessibility in thousands of single cells per assay, circumventing the need for compartmentalization of individual cells. We report chromatin accessibility profiles from more than 15,000 single cells and use these data to cluster cells on the basis of chromatin accessibility landscapes. We identify modules of coordinately regulated chromatin accessibility at the level of single cells both between and within cell types, with a scalable method that may accelerate progress toward a human cell atlas.', 'year': 2015, 'citationCount': 892, 'authors': [{'authorId': '6639465', 'name': 'D. Cusanovich'}, {'authorId': '49491404', 'name': 'R. Daza'}, {'authorId': '5552754', 'name': 'Andrew C. Adey'}, {'authorId': '47777448', 'name': 'Hannah A. Pliner'}, {'authorId': '10803574', 'name': 'Lena Christiansen'}, {'authorId': '2147608', 'name': 'K. Gunderson'}, {'authorId': '6420805', 'name': 'F. Steemers'}, {'authorId': '2244164', 'name': 'C. Trapnell'}, {'authorId': '2431330', 'name': 'J. Shendure'}]}, {'paperId': '77d7aa4efa1f562a0749b104c77662562cc0f3c5', 'title': 'High-efficiency solution-processed perovskite solar cells with millimeter-scale grains', 'abstract': 'Large-crystal perovskite films The performance of organic-inorganic hybrid perovskite planar solar cells has steadily improved. One outstanding issue is that grain boundaries and defects in polycrystalline films degrade their output. Now, two studies report the growth of millimeter-scale single crystals. Nie et al. grew continuous, pinhole-free, thin iodochloride films with a hot-casting technique and report device efficiencies of 18%. Shi et al. used antisolvent vapor-assisted crystallization to grow millimeter-scale bromide and iodide cubic crystals with charge-carrier diffusion lengths exceeding 10 mm. Science, this issue p. 522, p. 519 Solution processing techniques enable the growth of high-quality, large-area perovskite crystals for solar cells. State-of-the-art photovoltaics use high-purity, large-area, wafer-scale single-crystalline semiconductors grown by sophisticated, high-temperature crystal growth processes. We demonstrate a solution-based hot-casting technique to grow continuous, pinhole-free thin films of organometallic perovskites with millimeter-scale crystalline grains. We fabricated planar solar cells with efficiencies approaching 18%, with little cell-to-cell variability. The devices show hysteresis-free photovoltaic response, which had been a fundamental bottleneck for the stable operation of perovskite devices. Characterization and modeling attribute the improved performance to reduced bulk defects and improved charge carrier mobility in large-grain devices. We anticipate that this technique will lead the field toward synthesis of wafer-scale crystalline perovskites, necessary for the fabrication of high-efficiency solar cells, and will be applicable to several other material systems plagued by polydispersity, defects, and grain boundary recombination in solution-processed thin films.', 'year': 2015, 'citationCount': 2824, 'authors': [{'authorId': '48638740', 'name': 'W. Nie'}, {'authorId': '3875114', 'name': 'H. Tsai'}, {'authorId': '2785212', 'name': 'R. Asadpour'}, {'authorId': '144223671', 'name': 'J. Blancon'}, {'authorId': '4541893', 'name': 'A. Neukirch'}, {'authorId': '144410378', 'name': 'Gautam Gupta'}, {'authorId': '88634292', 'name': 'J. Crochet'}, {'authorId': '5025124', 'name': 'M. Chhowalla'}, {'authorId': '6232252', 'name': 'S. Tretiak'}, {'authorId': '2110011191', 'name': 'M. Alam'}, {'authorId': '7643816', 'name': 'Hsing-lin Wang'}, {'authorId': '4788240', 'name': 'A. Mohite'}]}, {'paperId': '8595b62e8933f29f614d151a60d34fb228891b75', 'title': 'The dominant role of semi-arid ecosystems in the trend and variability of the land CO2 sink', 'abstract': 'The difference is found at the margins The terrestrial biosphere absorbs about a quarter of all anthropogenic carbon dioxide emissions, but the amount that they take up varies from year to year. Why? Combining models and observations, Ahlström et al. found that marginal ecosystems—semiarid savannas and low-latitude shrublands—are responsible for most of the variability. Biological productivity in these semiarid regions is water-limited and strongly associated with variations in precipitation, unlike wetter tropical areas. Understanding carbon uptake by these marginal lands may help to improve predictions of variations in the global carbon cycle. Science, this issue p. 895 Semi-arid regions cause most of the interannual variability of the terrestrial carbon dioxide sink. The growth rate of atmospheric carbon dioxide (CO2) concentrations since industrialization is characterized by large interannual variability, mostly resulting from variability in CO2 uptake by terrestrial ecosystems (typically termed carbon sink). However, the contributions of regional ecosystems to that variability are not well known. Using an ensemble of ecosystem and land-surface models and an empirical observation-based product of global gross primary production, we show that the mean sink, trend, and interannual variability in CO2 uptake by terrestrial ecosystems are dominated by distinct biogeographic regions. Whereas the mean sink is dominated by highly productive lands (mainly tropical forests), the trend and interannual variability of the sink are dominated by semi-arid ecosystems whose carbon balance is strongly associated with circulation-driven variations in both precipitation and temperature.', 'year': 2015, 'citationCount': 950, 'authors': [{'authorId': '3640646', 'name': 'A. Ahlström'}, {'authorId': '34992220', 'name': 'M. Raupach'}, {'authorId': '4156014', 'name': 'G. Schurgers'}, {'authorId': '1417792690', 'name': 'Benjamin Smith'}, {'authorId': '2792896', 'name': 'A. Arneth'}, {'authorId': '95799904', 'name': 'M. Jung'}, {'authorId': '2530948', 'name': 'M. Reichstein'}, {'authorId': '66251280', 'name': 'J. Canadell'}, {'authorId': '4213663', 'name': 'P. Friedlingstein'}, {'authorId': '145347290', 'name': 'Atul K. Jain'}, {'authorId': '46310681', 'name': 'Etsushi Kato'}, {'authorId': '144853616', 'name': 'B. Poulter'}, {'authorId': '3728155', 'name': 'S. Sitch'}, {'authorId': '35759515', 'name': 'B. Stocker'}, {'authorId': '2350727', 'name': 'N. Viovy'}, {'authorId': '49417469', 'name': 'Ying Ping Wang'}, {'authorId': '8407849', 'name': 'A. Wiltshire'}, {'authorId': '4640770', 'name': 'S. Zaehle'}, {'authorId': '2149303', 'name': 'N. Zeng'}]}, {'paperId': '1c8d59d11fcdf222bbe3773e00513f264d268be0', 'title': 'The DNA damage response induces inflammation and senescence by inhibiting autophagy of GATA4', 'abstract': 'Transcriptional control of cell senescence Senescent cells that have stopped proliferating secrete molecules that influence the cells around them. Prevention of this senescence-activated secretory phenotype seems to slow organismal aging. Kang et al. explored the regulatory process behind cell senescence and found that DNA damage led to stabilization of the transcription factor GATA4 (see the Perspective by Cassidy and Narita). Increased activity of GATA4 in senescent cells stimulated genes encoding secreted factors. GATA4 also accumulates in the brains of aging mice or humans. Science, this issue 10.1126/science.aaa5612; see also p. 1448 The transcription factor GATA4 promotes cell senescence. [Also see Perspective by Cassidy and Narita] INTRODUCTION Cellular senescence is a program of arrested proliferation and altered gene expression triggered by many stresses. Although it is a potent tumor-suppressive mechanism, senescence has been implicated in several pathological processes including aging, age-associated diseases, and (counterintuitively) tumorigenesis. One potential mechanism through which senescent cells exert such pleiotropic effects is the secretion of proinflammatory cytokines, chemokines, growth factors, and proteases, termed the senescence-associated secretory phenotype (SASP), which affects senescent cells and their microenvironment. The mechanism by which the SASP is initiated and maintained is not well characterized beyond the classical regulators of inflammation, including the transcription factors NF-κB and C/EBPβ. RATIONALE In senescence growth arrest, two core senescence-regulating pathways, p53 and p16INK4a/Rb, play a critical role. By contrast, the SASP does not depend on either p53 or p16INK4a, which suggests the existence of an independent senescence regulatory network that controls the SASP. Having observed high levels of induction of microRNA miR-146a during induced senescence in human fibroblasts, we developed a green fluorescent protein–tagged senescence reporter based on a miR-146a promoter fragment. This reporter responded to senescence-inducing stimuli, including replicative exhaustion, DNA damage, and oncogenic RAS activation—all of which activate the SASP. This system allowed us to identify additional regulators of senescence and the SASP. RESULTS Through miR-146a promoter analysis, we mapped the critical region for senescence-induced activity and identified the transcriptional regulator responsible for this regulation, GATA4, previously known as a regulator of embryonic development. Ectopic expression of GATA4 induced senescence, whereas disruption of GATA4 suppressed it, thus establishing GATA4 as a senescence regulator. GATA4 protein abundance, but not mRNA, increased during sene1scence, primarily as a result of increased protein stability. Under normal conditions, GATA4 binds the p62 autophagy adaptor and is degraded by selective autophagy. Upon senescence induction, however, this selective autophagy was suppressed through decreased interaction between GATA4 and p62, thereby stabilizing GATA4. GATA4 in turn induced TRAF3IP2 (tumor necrosis factor receptor–associated factor interacting protein 2) and IL1A (interleukin 1A), which activate NF-κB to initiate and maintain the SASP, thus facilitating senescence. GATA4 pathway activation depends on the key DNA damage response (DDR) kinases ATM (ataxia telangiectasia mutated) and ATR (ataxia telangiectasia and Rad3–related), as does senescence-associated activation of p53 and p16INK4a. However, the GATA4 pathway is independent of p53 and p16INK4a. Finally, GATA4 protein accumulated in multiple tissues in mice treated with senescence-inducing stimuli and during normal mouse and human aging, including many cell types in the brain; these findings raise the possibility that the GATA4 pathway drives age-dependent inflammation. CONCLUSION Our results indicate that GATA4 connects autophagy and the DDR to senescence and inflammation through TRAF3IP2 and IL1A activation of NF-κB. These findings establish GATA4 as a key switch activated by the DDR to regulate senescence, independently of p53 and p16INK4a. Our in vivo data indicate a potential role of GATA4 during aging and its associated inflammation. Because accumulation of senescent cells is thought to promote aging and aging-associated diseases through the resulting inflammatory response, inhibiting the GATA4 pathway may provide an avenue for therapeutic intervention. GATA4 functions as a key switch in the senescence regulatory network to activate the SASP. The nonsenescent state is maintained by inhibitory barriers that prevent cell cycle arrest and inflammation. Upon senescence-inducing signals, ATM and ATR relieve inhibition of the p53 and p16INK4a pathways to induce growth arrest and also block p62-dependent autophagic degradation of GATA4, resulting in NF-κB activation and SASP induction. Cellular senescence is a terminal stress-activated program controlled by the p53 and p16INK4a tumor suppressor proteins. A striking feature of senescence is the senescence-associated secretory phenotype (SASP), a pro-inflammatory response linked to tumor promotion and aging. We have identified the transcription factor GATA4 as a senescence and SASP regulator. GATA4 is stabilized in cells undergoing senescence and is required for the SASP. Normally, GATA4 is degraded by p62-mediated selective autophagy, but this regulation is suppressed during senescence, thereby stabilizing GATA4. GATA4 in turn activates the transcription factor NF-κB to initiate the SASP and facilitate senescence. GATA4 activation depends on the DNA damage response regulators ATM and ATR, but not on p53 or p16INK4a. GATA4 accumulates in multiple tissues, including the aging brain, and could contribute to aging and its associated inflammation.', 'year': 2015, 'citationCount': 631, 'authors': [{'authorId': '49206657', 'name': 'Chanhee Kang'}, {'authorId': '8285259', 'name': 'Qikai Xu'}, {'authorId': '144696763', 'name': 'Timothy D. Martin'}, {'authorId': '25857253', 'name': 'Mamie Z. Li'}, {'authorId': '9109024', 'name': 'M. Demaria'}, {'authorId': '30472762', 'name': 'Liviu Aron'}, {'authorId': '40629125', 'name': 'Tao Lu'}, {'authorId': '6749057', 'name': 'B. Yankner'}, {'authorId': '3050721', 'name': 'J. Campisi'}, {'authorId': '4339245', 'name': 'S. Elledge'}]}, {'paperId': 'd290f977586fac98c695346c9591e33763269987', 'title': 'Activation of proto-oncogenes by disruption of chromosome neighborhoods', 'abstract': 'The spread of bad neighborhoods Our genomes have complex three-dimensional (3D) arrangements that partition and regulate gene expression. Cancer cells frequently have their genomes grossly rearranged, disturbing this intricate 3D organization. Hnisz et al. show that the disruption of these 3D neighborhoods can bring oncogenes under the control of regulatory elements normally kept separate from them (see the Perspective by Wala and Beroukim). These novel juxtapositions can result in the inappropriate activation of oncogenes. Science, this issue p. 1454; see also p. 1398 Disrupting the boundaries between three-dimensional neighborhoods in the genome can activate cancer-promoting genes. [Also see Perspective by Wala and Beroukim] Oncogenes are activated through well-known chromosomal alterations such as gene fusion, translocation, and focal amplification. In light of recent evidence that the control of key genes depends on chromosome structures called insulated neighborhoods, we investigated whether proto-oncogenes occur within these structures and whether oncogene activation can occur via disruption of insulated neighborhood boundaries in cancer cells. We mapped insulated neighborhoods in T cell acute lymphoblastic leukemia (T-ALL) and found that tumor cell genomes contain recurrent microdeletions that eliminate the boundary sites of insulated neighborhoods containing prominent T-ALL proto-oncogenes. Perturbation of such boundaries in nonmalignant cells was sufficient to activate proto-oncogenes. Mutations affecting chromosome neighborhood boundaries were found in many types of cancer. Thus, oncogene activation can occur via genetic alterations that disrupt insulated neighborhoods in malignant cells.', 'year': 2015, 'citationCount': 793, 'authors': [{'authorId': '3787252', 'name': 'Denes Hnisz'}, {'authorId': '48537699', 'name': 'Abraham S. Weintraub'}, {'authorId': '5885383', 'name': 'Daniel S. Day'}, {'authorId': '7255958', 'name': 'Anne-Laure Valton'}, {'authorId': '3725971', 'name': 'R. Bak'}, {'authorId': '2109482219', 'name': 'Charles H. Li'}, {'authorId': '4188811', 'name': 'Johanna E. Goldmann'}, {'authorId': '2399615', 'name': 'B. Lajoie'}, {'authorId': '144073851', 'name': 'Z. Fan'}, {'authorId': '4066956', 'name': 'A. Sigova'}, {'authorId': '46291633', 'name': 'J. Reddy'}, {'authorId': '1398807023', 'name': 'Diego Borges-Rivera'}, {'authorId': '152240986', 'name': 'Tong Ihn Lee'}, {'authorId': '5203884', 'name': 'R. Jaenisch'}, {'authorId': '4046906', 'name': 'M. Porteus'}, {'authorId': '145696688', 'name': 'J. Dekker'}, {'authorId': '144787676', 'name': 'R. Young'}]}, {'paperId': 'e171cbf02cc7aa5a6f72d514799ca291523360e4', 'title': 'Low trap-state density and long carrier diffusion in organolead trihalide perovskite single crystals', 'abstract': 'Large-crystal perovskite films The performance of organic-inorganic hybrid perovskite planar solar cells has steadily improved. One outstanding issue is that grain boundaries and defects in polycrystalline films degrade their output. Now, two studies report the growth of millimeter-scale single crystals. Nie et al. grew continuous, pinhole-free, thin iodochloride films with a hot-casting technique and report device efficiencies of 18%. Shi et al. used antisolvent vapor-assisted crystallization to grow millimeter-scale bromide and iodide cubic crystals with charge-carrier diffusion lengths exceeding 10 mm. Science, this issue p. 522, p. 519 Solution processing techniques enable the growth of high-quality, large-area perovskite crystals for solar cells. The fundamental properties and ultimate performance limits of organolead trihalide MAPbX3 (MA = CH3NH3+; X = Br– or I–) perovskites remain obscured by extensive disorder in polycrystalline MAPbX3 films. We report an antisolvent vapor-assisted crystallization approach that enables us to create sizable crack-free MAPbX3 single crystals with volumes exceeding 100 cubic millimeters. These large single crystals enabled a detailed characterization of their optical and charge transport characteristics. We observed exceptionally low trap-state densities on the order of 109 to 1010 per cubic centimeter in MAPbX3 single crystals (comparable to the best photovoltaic-quality silicon) and charge carrier diffusion lengths exceeding 10 micrometers. These results were validated with density functional theory calculations.', 'year': 2015, 'citationCount': 3627, 'authors': [{'authorId': '144968738', 'name': 'D. Shi'}, {'authorId': '35620881', 'name': 'V. Adinolfi'}, {'authorId': '47486987', 'name': 'R. Comin'}, {'authorId': '1758918', 'name': 'M. Yuan'}, {'authorId': '6032563', 'name': 'E. Alarousu'}, {'authorId': '48368168', 'name': 'A. Buin'}, {'authorId': '2145266143', 'name': 'Yin Chen'}, {'authorId': '3891522', 'name': 'S. Hoogland'}, {'authorId': '6209438', 'name': 'A. Rothenberger'}, {'authorId': '7837348', 'name': 'K. Katsiev'}, {'authorId': '46368681', 'name': 'Y. Losovyj'}, {'authorId': '2149172030', 'name': 'Xin Zhang'}, {'authorId': '145977172', 'name': 'P. Dowben'}, {'authorId': '144348186', 'name': 'O. Mohammed'}, {'authorId': '1799992', 'name': 'E. Sargent'}, {'authorId': '3095691', 'name': 'O. Bakr'}]}, {'paperId': '26e7f0da2f0d586c60fc3a8bf8605d4b2a38ff78', 'title': 'RNA editing by ADAR1 prevents MDA5 sensing of endogenous dsRNA as nonself', 'abstract': 'RNA editing helps identify cellular RNAs Adenosine bases in messenger RNA (mRNAs) can be enzymatically modified and changed into inosine bases. This RNA “editing” is mediated by adenosine deaminase acting on RNA (ADAR) enzymes. Liddicoat et al. show that the in vivo targets of the principal editing enzyme, ADAR1, are long double-stranded RNA (dsRNA) structures in noncoding portions of cellular mRNAs. ADAR1-directed editing of these cellular targets is critical to avoid activation of an immune response to dsRNA in the cytoplasm, because dsRNA is also a marker of viral infection. Science, this issue p. 1115 The principal RNA-editing enzyme modifies cellular RNAs to prevent their erroneous identification as foreign RNA. Adenosine-to-inosine (A-to-I) editing is a highly prevalent posttranscriptional modification of RNA, mediated by ADAR (adenosine deaminase acting on RNA) enzymes. In addition to RNA editing, additional functions have been proposed for ADAR1. To determine the specific role of RNA editing by ADAR1, we generated mice with an editing-deficient knock-in mutation (Adar1E861A, where E861A denotes Glu861→Ala861). Adar1E861A/E861A embryos died at ~E13.5 (embryonic day 13.5), with activated interferon and double-stranded RNA (dsRNA)–sensing pathways. Genome-wide analysis of the in vivo substrates of ADAR1 identified clustered hyperediting within long dsRNA stem loops within 3′ untranslated regions of endogenous transcripts. Finally, embryonic death and phenotypes of Adar1E861A/E861A were rescued by concurrent deletion of the cytosolic sensor of dsRNA, MDA5. A-to-I editing of endogenous dsRNA is the essential function of ADAR1, preventing the activation of the cytosolic dsRNA response by endogenous transcripts.', 'year': 2015, 'citationCount': 578, 'authors': [{'authorId': '5785182', 'name': 'Brian J. Liddicoat'}, {'authorId': '5585182', 'name': 'Robert Piskol'}, {'authorId': '3027693', 'name': 'A. Chalk'}, {'authorId': '37579243', 'name': 'Gokul Ramaswami'}, {'authorId': '39661583', 'name': 'M. Higuchi'}, {'authorId': '5218250', 'name': 'Jochen C. Hartner'}, {'authorId': '1492117701', 'name': 'J. B. Li'}, {'authorId': '144870675', 'name': 'P. Seeburg'}, {'authorId': '4730920', 'name': 'Carl R Walkley'}]}, {'paperId': '08aa0f7272560b90d8cc4e49bd5defbdb20d1842', 'title': 'Boreal forest health and global change', 'abstract': 'The boreal forest, one of the largest biomes on Earth, provides ecosystem services that benefit society at levels ranging from local to global. Currently, about two-thirds of the area covered by this biome is under some form of management, mostly for wood production. Services such as climate regulation are also provided by both the unmanaged and managed boreal forests. Although most of the boreal forests have retained the resilience to cope with current disturbances, projected environmental changes of unprecedented speed and amplitude pose a substantial threat to their health. Management options to reduce these threats are available and could be implemented, but economic incentives and a greater focus on the boreal biome in international fora are needed to support further adaptation and mitigation actions.', 'year': 2015, 'citationCount': 683, 'authors': [{'authorId': '46813897', 'name': 'S. Gauthier'}, {'authorId': '33655263', 'name': 'P. Bernier'}, {'authorId': '5684568', 'name': 'T. Kuuluvainen'}, {'authorId': '2689353', 'name': 'A. Shvidenko'}, {'authorId': '1877912', 'name': 'D. Schepaschenko'}]}, {'paperId': '99a3c6c33c7c4470a550061f2aa25dcd406df9ce', 'title': 'Pervasive drought legacies in forest ecosystems and their implications for carbon cycle models', 'abstract': \"Drought effects on carbon cycling The response of forest ecosystems to drought is increasingly important in the context of a warming climate. Anderegg et al. studied a tree-ring database of 1338 forest sites from around the globe. They found that forests exhibit a drought “legacy effect” with 3 to 4 years' reduced growth following drought. During this postdrought delay, forests will be less able to act as a sink for carbon. Incorporating forest legacy effects into Earth system models will provide more accurate predictions of the effects of drought on the global carbon cycle. Science, this issue p. 528 Tree-ring data from 1338 forest sites show that after a drought tree growth is reduced for a further 3 to 4 years. The impacts of climate extremes on terrestrial ecosystems are poorly understood but important for predicting carbon cycle feedbacks to climate change. Coupled climate–carbon cycle models typically assume that vegetation recovery from extreme drought is immediate and complete, which conflicts with the understanding of basic plant physiology. We examined the recovery of stem growth in trees after severe drought at 1338 forest sites across the globe, comprising 49,339 site-years, and compared the results with simulated recovery in climate-vegetation models. We found pervasive and substantial “legacy effects” of reduced growth and incomplete recovery for 1 to 4 years after severe drought. Legacy effects were most prevalent in dry ecosystems, among Pinaceae, and among species with low hydraulic safety margins. In contrast, limited or no legacy effects after drought were simulated by current climate-vegetation models. Our results highlight hysteresis in ecosystem-level carbon cycling and delayed recovery from climate extremes.\", 'year': 2015, 'citationCount': 728, 'authors': [{'authorId': '3736387', 'name': 'W. Anderegg'}, {'authorId': '3736387', 'name': 'W. Anderegg'}, {'authorId': '2139023', 'name': 'C. Schwalm'}, {'authorId': '2939516', 'name': 'F. Biondi'}, {'authorId': '2359553', 'name': 'J. Camarero'}, {'authorId': '153749410', 'name': 'G. Koch'}, {'authorId': '3739443', 'name': 'M. Litvak'}, {'authorId': '2563825', 'name': 'K. Ogle'}, {'authorId': '144827840', 'name': 'J. Shaw'}, {'authorId': '3773288', 'name': 'E. Shevliakova'}, {'authorId': '1518746020', 'name': 'A. P. Williams'}, {'authorId': '2073843', 'name': 'A. Wolf'}, {'authorId': '6513992', 'name': 'E. Ziaco'}, {'authorId': '4955260', 'name': 'S. Pacala'}]}, {'paperId': 'e964eec826f48f6bec6e62319655e34764a13335', 'title': 'Gene essentiality and synthetic lethality in haploid human cells', 'abstract': \"Zeroing in on essential human genes More powerful genetic techniques are helping to define the list of genes required for the life of a human cell. Two papers used the CRISPR genome editing system and a gene trap method in haploid human cells to screen for essential genes (see the Perspective by Boone and Andrews). Wang et al.'s analysis of multiple cell lines indicates that it may be possible to find tumor-specific dependencies on particular genes. Blomen et al. investigate the phenomenon in which nonessential genes are required for fitness in the absence of another gene. Hence, complexity rather than robustness is the human strategy. Science, this issue p. 1096 and p. 1092; see also p. 1028 Systematic mutagenesis reveals essential genetic interactions required for human cells to keep growing. [Also see Perspective by Boone and Andrews] Although the genes essential for life have been identified in less complex model organisms, their elucidation in human cells has been hindered by technical barriers. We used extensive mutagenesis in haploid human cells to identify approximately 2000 genes required for optimal fitness under culture conditions. To study the principles of genetic interactions in human cells, we created a synthetic lethality network focused on the secretory pathway based exclusively on mutations. This revealed a genetic cross-talk governing Golgi homeostasis, an additional subunit of the human oligosaccharyltransferase complex, and a phosphatidylinositol 4-kinase β adaptor hijacked by viruses. The synthetic lethality map parallels observations made in yeast and projects a route forward to reveal genetic networks in diverse aspects of human cell biology.\", 'year': 2015, 'citationCount': 677, 'authors': [{'authorId': '4368790', 'name': 'V. Blomen'}, {'authorId': '34025797', 'name': 'P. Májek'}, {'authorId': '3994560', 'name': 'L. T. Jae'}, {'authorId': '4217436', 'name': 'J. Bigenzahn'}, {'authorId': '2676853', 'name': 'Joppe Nieuwenhuis'}, {'authorId': '8518116', 'name': 'J. Staring'}, {'authorId': '153530312', 'name': 'R. Sacco'}, {'authorId': '102248379', 'name': 'Ferdy R. van Diemen'}, {'authorId': '1820863502', 'name': 'N. Olk'}, {'authorId': '3697345', 'name': 'A. Stukalov'}, {'authorId': '4686719', 'name': 'C. Marceau'}, {'authorId': '144985777', 'name': 'H. Janssen'}, {'authorId': '16762589', 'name': 'J. Carette'}, {'authorId': '2502649', 'name': 'K. Bennett'}, {'authorId': '2248107', 'name': 'J. Colinge'}, {'authorId': '1387445843', 'name': 'G. Superti-Furga'}, {'authorId': '3658300', 'name': 'T. Brummelkamp'}]}, {'paperId': 'acfea18276e393b6e147bed8f04b0ce8bccc9aaf', 'title': 'Soil and human security in the 21st century', 'abstract': \"Global soil resources under stress The future of humanity is intertwined with the future of Earth's soil resources. Soil provides for agriculture, improves water quality, and buffers greenhouse gases in the atmosphere. Yet human activities, including agricultural soil erosion, are rapidly degrading soil faster than it is naturally replenished. At this rate, human security over the next century will be severely threatened by unsustainable soil management practices. Amundson et al. review recent advances in understanding global soil resources, including how carbon stored in soil responds to anthropogenic warming. Translating this knowledge into practice is the biggest challenge remaining. Science, this issue 10.1126/science.1261071 BACKGROUND Earth’s soil has formed by processes that have maintained a persistent and expansive global soil mantle, one that in turn provided the stage for the evolution of the vast diversity of life on land. The underlying stability of soil systems is controlled by their inherent balance between inputs and losses of nutrients and carbon. Human exploitation of these soil resources, beginning a few thousand years ago, allowed agriculture to become an enormous success. The vastness of the planet and its soil resources allowed agriculture to expand, with growing populations, or to move, when soil resources were depleted. However, the practice of farming greatly accelerated rates of erosion relative to soil production, and soil has been and continues to be lost at rates that are orders of magnitude greater than mechanisms that replenish soil. Additionally, agricultural practices greatly altered natural soil carbon balances and feedbacks. Cultivation thus began an ongoing slow ignition of Earth’s largest surficial reservoir of carbon—one that, when combined with the anthropogenic warming of many biomes, is capable of driving large positive feedbacks that will further increase the accumulation of atmospheric greenhouse gases and exacerbate associated climate change. ADVANCES The study of soil is now the domain of diverse schools of physical and biological science. Rapid advances in empirical and theoretical understanding of soil processes are occurring. These advances have brought an international, and global, perspective to the study of soil processes and focused the implications of soil stewardship for societal well-being. Major advances in the past decade include our first quantitative understanding of the natural rates of soil production, derived from isotopic methods developed by collaboration of geochemists and geomorphologists. Proliferation of research by soil and ecological scientists in the northern latitudes continues to illuminate and improve estimates of the magnitude of soil carbon storage in these regions and its sensitivity and response to warming. The role of soil processes in global carbon and climate models is entering a period of growing attention and increasing maturity. These activities in turn reveal the severity of soil-related issues at stake for the remainder of this century—the need to rapidly regain a balance to the physical and biological processes that drive and maintain soil properties, and the societal implications that will result if we do not. OUTLOOK Both great challenges and opportunities exist in regards to maintaining soil’s role in food, climate, and human security. Erosion continues to exceed natural rates of soil renewal even in highly developed countries. The recent focus by economists and natural scientists on potential future shortages of phosphorus fertilizer offers opportunities for novel partnerships to develop efficient methods of nutrient recycling and redistribution systems in urban settings. Possibly the most challenging issues will be to better understand the magnitude of global soil carbon feedbacks to climate change and to mitigating climate change in a timely fashion. The net results of human impacts on soil resources this century will be global in scale and will have direct impacts on human security for centuries to come. Large-scale erosion forming a gully system in the watershed of Lake Bogoria, Kenya. Accelerated soil erosion here is due to both overgrazing and improper agricultural management, which are partially due to political-social impacts of past colonization and inadequate resources and infrastructure. The erosion additionally affects the long-term future of Lake Bogoria because of rapid sedimentation. This example illustrates the disruption of the natural balance of soil production and erosion over geological time scales by human activity and the rapidity of the consequences of this imbalance. CREDIT: BRENT STIRTON/GETTY IMAGES Human security has and will continue to rely on Earth’s diverse soil resources. Yet we have now exploited the planet’s most productive soils. Soil erosion greatly exceeds rates of production in many agricultural regions. Nitrogen produced by fossil fuel and geological reservoirs of other fertilizers are headed toward possible scarcity, increased cost, and/or geopolitical conflict. Climate change is accelerating the microbial release of greenhouse gases from soil organic matter and will likely play a large role in our near-term climate future. In this Review, we highlight challenges facing Earth’s soil resources in the coming century. The direct and indirect response of soils to past and future human activities will play a major role in human prosperity and survival.\", 'year': 2015, 'citationCount': 832, 'authors': [{'authorId': '7249227', 'name': 'R. Amundson'}, {'authorId': '4885197', 'name': 'A. Berhe'}, {'authorId': '5631731', 'name': 'J. Hopmans'}, {'authorId': '82933112', 'name': 'C. Olson'}, {'authorId': '8311470', 'name': 'A. Sztein'}, {'authorId': '1870724', 'name': 'D. Sparks'}]}, {'paperId': '2846ecd7a2eb14e2262e7cd53296170ba49b76e2', 'title': 'Neutrophil extracellular traps license macrophages for cytokine production in atherosclerosis', 'abstract': 'Neutrophil NETs drive atherosclerosis The buildup of fats, cholesterol, and other substances in arteries causes atherosclerosis, which restricts blood flow and can lead to heart attacks and stroke. Inflammation contributes to the pathogenesis of atherosclerosis, but exactly how is not fully understood. Warnatsch et al. now show that immune cells called neutrophils release NETs (neutrophil extracellular traps) (see the Perspective by Nahrendorf and Swirski). These NETs are composed of DNA and antimicrobial proteins, and in the setting of atherosclerosis they activate innate immune signaling pathways in macrophages. This causes the macrophages to secrete proinflammatory cytokines, exacerbating the disease. Indirectly, NETS also attract a specialized subset of T cells that further amplify the proinflammatory response. Science, this issue p. 316; see also p. 237 Neutrophils release networks of extracellular fibers that contribute to the pathogenesis of atherosclerosis. [Also see Perspective by Nahrendorf and Swirski] Secretion of the cytokine interleukin-1β (IL-1β) by macrophages, a major driver of pathogenesis in atherosclerosis, requires two steps: Priming signals promote transcription of immature IL-1β, and then endogenous “danger” signals activate innate immune signaling complexes called inflammasomes to process IL-1β for secretion. Although cholesterol crystals are known to act as danger signals in atherosclerosis, what primes IL-1β transcription remains elusive. Using a murine model of atherosclerosis, we found that cholesterol crystals acted both as priming and danger signals for IL-1β production. Cholesterol crystals triggered neutrophils to release neutrophil extracellular traps (NETs). NETs primed macrophages for cytokine release, activating T helper 17 (TH17) cells that amplify immune cell recruitment in atherosclerotic plaques. Therefore, danger signals may drive sterile inflammation, such as that seen in atherosclerosis, through their interactions with neutrophils.', 'year': 2015, 'citationCount': 799, 'authors': [{'authorId': '3739915', 'name': 'A. Warnatsch'}, {'authorId': '145300176', 'name': 'M. Ioannou'}, {'authorId': '2153283114', 'name': 'Qian Wang'}, {'authorId': '49579944', 'name': 'V. Papayannopoulos'}]}, {'paperId': 'c68ca1258410b583cfdbe5a3733be5dbce0e03e3', 'title': 'Sea-level rise due to polar ice-sheet mass loss during past warm periods', 'abstract': 'Warming climate, melting ice, rising seas We know that the sea level will rise as climate warms. Nevertheless, accurate projections of how much sea-level rise will occur are difficult to make based solely on modern observations. Determining how ice sheets and sea level have varied in past warm periods can help us better understand how sensitive ice sheets are to higher temperatures. Dutton et al. review recent interdisciplinary progress in understanding this issue, based on data from four different warm intervals over the past 3 million years. Their synthesis provides a clear picture of the progress we have made and the hurdles that still exist. Science, this issue 10.1126/science.aaa4019 Reconstructing past magnitudes, rates, and sources of sea-level rise can help project what our warmer future may hold. BACKGROUND Although thermal expansion of seawater and melting of mountain glaciers have dominated global mean sea level (GMSL) rise over the last century, mass loss from the Greenland and Antarctic ice sheets is expected to exceed other contributions to GMSL rise under future warming. To better constrain polar ice-sheet response to warmer temperatures, we draw on evidence from interglacial periods in the geologic record that experienced warmer polar temperatures and higher GMSLs than present. Coastal records of sea level from these previous warm periods demonstrate geographic variability because of the influence of several geophysical processes that operate across a range of magnitudes and time scales. Inferring GMSL and ice-volume changes from these reconstructions is nontrivial and generally requires the use of geophysical models. ADVANCES Interdisciplinary studies of geologic archives have ushered in a new era of deciphering magnitudes, rates, and sources of sea-level rise. Advances in our understanding of polar ice-sheet response to warmer climates have been made through an increase in the number and geographic distribution of sea-level reconstructions, better ice-sheet constraints, and the recognition that several geophysical processes cause spatially complex patterns in sea level. In particular, accounting for glacial isostatic processes helps to decipher spatial variability in coastal sea-level records and has reconciled a number of site-specific sea-level reconstructions for warm periods that have occurred within the past several hundred thousand years. This enables us to infer that during recent interglacial periods, small increases in global mean temperature and just a few degrees of polar warming relative to the preindustrial period resulted in ≥6 m of GMSL rise. Mantle-driven dynamic topography introduces large uncertainties on longer time scales, affecting reconstructions for time periods such as the Pliocene (~3 million years ago), when atmospheric CO2 was ~400 parts per million (ppm), similar to that of the present. Both modeling and field evidence suggest that polar ice sheets were smaller during this time period, but because dynamic topography can cause tens of meters of vertical displacement at Earth’s surface on million-year time scales and uncertainty in model predictions of this signal are large, it is currently not possible to make a precise estimate of peak GMSL during the Pliocene. OUTLOOK Our present climate is warming to a level associated with significant polar ice-sheet loss in the past, but a number of challenges remain to further constrain ice-sheet sensitivity to climate change using paleo–sea level records. Improving our understanding of rates of GMSL rise due to polar ice-mass loss is perhaps the most societally relevant information the paleorecord can provide, yet robust estimates of rates of GMSL rise associated with polar ice-sheet retreat and/or collapse remain a weakness in existing sea-level reconstructions. Improving existing magnitudes, rates, and sources of GMSL rise will require a better (global) distribution of sea-level reconstructions with high temporal resolution and precise elevations and should include sites close to present and former ice sheets. Translating such sea-level data into a robust GMSL signal demands integration with geophysical models, which in turn can be tested through improved spatial and temporal sampling of coastal records. Further development is needed to refine estimates of past sea level from geochemical proxies. In particular, paired oxygen isotope and Mg/Ca data are currently unable to provide confident, quantitative estimates of peak sea level during these past warm periods. In some GMSL reconstructions, polar ice-sheet retreat is inferred from the total GMSL budget, but identifying the specific ice-sheet sources is currently hindered by limited field evidence at high latitudes. Given the paucity of such data, emerging geochemical and geophysical techniques show promise for identifying the sectors of the ice sheets that were most vulnerable to collapse in the past and perhaps will be again in the future. Peak global mean temperature, atmospheric CO2, maximum global mean sea level (GMSL), and source(s) of meltwater. Light blue shading indicates uncertainty of GMSL maximum. Red pie charts over Greenland and Antarctica denote fraction (not location) of ice retreat. Interdisciplinary studies of geologic archives have ushered in a new era of deciphering magnitudes, rates, and sources of sea-level rise from polar ice-sheet loss during past warm periods. Accounting for glacial isostatic processes helps to reconcile spatial variability in peak sea level during marine isotope stages 5e and 11, when the global mean reached 6 to 9 meters and 6 to 13 meters higher than present, respectively. Dynamic topography introduces large uncertainties on longer time scales, precluding robust sea-level estimates for intervals such as the Pliocene. Present climate is warming to a level associated with significant polar ice-sheet loss in the past. Here, we outline advances and challenges involved in constraining ice-sheet sensitivity to climate change with use of paleo–sea level records.', 'year': 2015, 'citationCount': 556, 'authors': [{'authorId': '47081678', 'name': 'A. Dutton'}, {'authorId': '104123258', 'name': 'A. Carlson'}, {'authorId': '40981288', 'name': 'A. Long'}, {'authorId': '29862195', 'name': 'G. Milne'}, {'authorId': '4367088', 'name': 'P. Clark'}, {'authorId': '104830825', 'name': 'R. DeConto'}, {'authorId': '3522204', 'name': 'B. Horton'}, {'authorId': '3522204', 'name': 'B. Horton'}, {'authorId': '1720524', 'name': 'S. Rahmstorf'}, {'authorId': '2226849', 'name': 'M. Raymo'}]}, {'paperId': '48b5bad16c3a98ccd22bea9965d821454be0fb65', 'title': 'Tissue residency of innate lymphoid cells in lymphoid and nonlymphoid organs', 'abstract': 'Establishing a longtime residency Innate lymphoid cells (ILCs) are a subset of immune cells that promote barrier immunity in tissues such as the gut and lungs and help to maintain immune homeostasis. Gasteiger et al. investigated how the body maintains its pools of ILCs in such peripheral tissues, as well as in immune tissues such as the lymph nodes and the spleen. In mice surgically joined to share their bloodstreams, unlike lymphocytes, most ILCs did not circulate through the blood. Instead, ILCs resided long term in tissues, even in the face of inflammation or infection. Science, this issue p. 981 Most innate lymphoid cells are locally maintained in tissues and do not circulate through the bloodstream. Innate lymphoid cells (ILCs) contribute to barrier immunity, tissue homeostasis, and immune regulation at various anatomical sites throughout the body. How ILCs maintain their presence in lymphoid and peripheral tissues thus far has been unclear. We found that in the lymphoid and nonlymphoid organs of adult mice, ILCs are tissue-resident cells that were maintained and expanded locally under physiologic conditions, upon systemic perturbation of immune homeostasis and during acute helminth infection. However, at later time points after infection, cells from hematogenous sources helped to partially replenish the pool of resident ILCs. Thus, ILCs are maintained by self-renewal in broadly different microenvironments and physiological settings. Such an extreme “sedentary” lifestyle is consistent with the proposed roles of ILCs as sentinels and local keepers of tissue function.', 'year': 2015, 'citationCount': 606, 'authors': [{'authorId': '5916140', 'name': 'G. Gasteiger'}, {'authorId': '46515943', 'name': 'Xiying Fan'}, {'authorId': '5348681', 'name': 'Stanislav Dikiy'}, {'authorId': '2107987539', 'name': 'Sue Y. Lee'}, {'authorId': '5146760', 'name': 'A. Rudensky'}]}, {'paperId': 'ee4bb392f3264abb684a43169240e20f05683e5a', 'title': 'RNA-Seq of single prostate CTCs implicates noncanonical Wnt signaling in antiandrogen resistance', 'abstract': 'Circulating signals of drug resistance Cancer drugs often lose their effectiveness because tumors acquire genetic changes that confer drug resistance. Ideally, patients would be switched to a different drug before tumor growth resumes, but this requires early knowledge of how resistance arose. Miyamoto et al. have developed a non-invasive method to spot resistance by sequencing RNA transcripts in single circulating tumor cells (CTCs) (see the Perspective by Nanus and Giannakakou). For example, in prostate cancer patients, drug resistance was triggered by activation of the Wnt signaling pathway. But CTCs are rare and fragile, and the technology needs further development before it is used in clinical practice. Science, this issue p. 1351; see also p. 1283 Analysis of circulating tumor cells from prostate cancer patients reveals a mechanism that contributes to treatment failure. [Also see Perspective by Nanus and Giannakakou] Prostate cancer is initially responsive to androgen deprivation, but the effectiveness of androgen receptor (AR) inhibitors in recurrent disease is variable. Biopsy of bone metastases is challenging; hence, sampling circulating tumor cells (CTCs) may reveal drug-resistance mechanisms. We established single-cell RNA-sequencing (RNA-Seq) profiles of 77 intact CTCs isolated from 13 patients (mean six CTCs per patient), by using microfluidic enrichment. Single CTCs from each individual display considerable heterogeneity, including expression of AR gene mutations and splicing variants. Retrospective analysis of CTCs from patients progressing under treatment with an AR inhibitor, compared with untreated cases, indicates activation of noncanonical Wnt signaling (P = 0.0064). Ectopic expression of Wnt5a in prostate cancer cells attenuates the antiproliferative effect of AR inhibition, whereas its suppression in drug-resistant cells restores partial sensitivity, a correlation also evident in an established mouse model. Thus, single-cell analysis of prostate CTCs reveals heterogeneity in signaling pathways that could contribute to treatment failure.', 'year': 2015, 'citationCount': 601, 'authors': [{'authorId': '3968473', 'name': 'D. Miyamoto'}, {'authorId': '145143275', 'name': 'Yu Zheng'}, {'authorId': '1726209', 'name': 'B. Wittner'}, {'authorId': '48723610', 'name': 'Richard J Lee'}, {'authorId': '50319431', 'name': 'Huili Zhu'}, {'authorId': '40267957', 'name': 'Katherine T. Broderick'}, {'authorId': '49277771', 'name': 'R. Desai'}, {'authorId': '50541900', 'name': 'Douglas B. Fox'}, {'authorId': '4577591', 'name': 'Brian W. Brannigan'}, {'authorId': '4257990', 'name': 'Julie Trautwein'}, {'authorId': '1985674', 'name': 'Kshitij S Arora'}, {'authorId': '6449022', 'name': 'Niyati Desai'}, {'authorId': '50331509', 'name': 'D. Dahl'}, {'authorId': '144346741', 'name': 'L. Sequist'}, {'authorId': '2209887632', 'name': 'Matthew R. Smith'}, {'authorId': '40250521', 'name': 'R. Kapur'}, {'authorId': '48251343', 'name': 'Chin-Lee Wu'}, {'authorId': '2281109', 'name': 'T. Shioda'}, {'authorId': '34631654', 'name': 'S. Ramaswamy'}, {'authorId': '2049483', 'name': 'D. Ting'}, {'authorId': '145527638', 'name': 'M. Toner'}, {'authorId': '144782680', 'name': 'S. Maheswaran'}, {'authorId': '2253481', 'name': 'D. Haber'}]}, {'paperId': '1ab87ca03b4fabcd691516a7a95d6ed325c809dd', 'title': 'Electrical switching of an antiferromagnet', 'abstract': \"Manipulating a stubborn magnet Spintronics is an alternative to conventional electronics, based on using the electron's spin rather than its charge. Spintronic devices, such as magnetic memory, have traditionally used ferromagnetic materials to encode the 1's and 0's of the binary code. A weakness of this approach—that strong magnetic fields can erase the encoded information—could be avoided by using antiferromagnets instead of ferromagnets. But manipulating the magnetic ordering of antiferromagnets is tricky. Now, Wadley et al. have found a way (see the Perspective by Marrows). Running currents along specific directions in the thin films of the antiferromagnetic compound CuMnAs reoriented the magnetic domains in the material. Science, this issue p. 587; see also p. 558 Transport and optical measurements are used to demonstrate the switching of domains in the antiferromagnetic compound CuMnAs. [Also see Perspective by Marrows] Antiferromagnets are hard to control by external magnetic fields because of the alternating directions of magnetic moments on individual atoms and the resulting zero net magnetization. However, relativistic quantum mechanics allows for generating current-induced internal fields whose sign alternates with the periodicity of the antiferromagnetic lattice. Using these fields, which couple strongly to the antiferromagnetic order, we demonstrate room-temperature electrical switching between stable configurations in antiferromagnetic CuMnAs thin-film devices by applied current with magnitudes of order 106 ampere per square centimeter. Electrical writing is combined in our solid-state memory with electrical readout and the stored magnetic state is insensitive to and produces no external magnetic field perturbations, which illustrates the unique merits of antiferromagnets for spintronics.\", 'year': 2015, 'citationCount': 971, 'authors': [{'authorId': '6529129', 'name': 'P. Wadley'}, {'authorId': '92662262', 'name': 'B. Howells'}, {'authorId': '7183375', 'name': 'J. Železný'}, {'authorId': '7183375', 'name': 'J. Železný'}, {'authorId': '122532770', 'name': 'C. Andrews'}, {'authorId': '116923455', 'name': 'V. Hills'}, {'authorId': '34810725', 'name': 'R. Campion'}, {'authorId': '153319812', 'name': 'V. Novák'}, {'authorId': '4373201', 'name': 'K. Olejník'}, {'authorId': '5090120', 'name': 'F. Maccherozzi'}, {'authorId': '4838511', 'name': 'S. Dhesi'}, {'authorId': '2107491459', 'name': 'S. Martin'}, {'authorId': '47426511', 'name': 'T. Wagner'}, {'authorId': '47426511', 'name': 'T. Wagner'}, {'authorId': '2678034', 'name': 'J. Wunderlich'}, {'authorId': '2678034', 'name': 'J. Wunderlich'}, {'authorId': '6559483', 'name': 'F. Freimuth'}, {'authorId': '6402565', 'name': 'Y. Mokrousov'}, {'authorId': '50241568', 'name': 'J. Kuneš'}, {'authorId': '1630347158', 'name': 'J. Chauhan'}, {'authorId': '50419899', 'name': 'M. Grzybowski'}, {'authorId': '50419899', 'name': 'M. Grzybowski'}, {'authorId': '144765908', 'name': 'A. Rushforth'}, {'authorId': '48918214', 'name': 'K. Edmonds'}, {'authorId': '46346797', 'name': 'B. Gallagher'}, {'authorId': '6620058', 'name': 'T. Jungwirth'}, {'authorId': '6620058', 'name': 'T. Jungwirth'}]}, {'paperId': '69de421683a59ea687cfbb8280b264793e41316d', 'title': 'Graphene, related two-dimensional crystals, and hybrid systems for energy conversion and storage', 'abstract': 'Background The integration of graphene in photovoltaic modules, fuel cells, batteries, supercapacitors, and devices for hydrogen generation offers opportunities to tackle challenges driven by the increasing global energy demand. Graphene’s two-dimensional (2D) nature leads to a theoretical surface-to-mass ratio of ~2600 m2/g, which combined with its high electrical conductivity and flexibility, gives it the potential to store electric charge, ions, or hydrogen. Other 2D crystals, such as transition metal chalcogenides (TMDs) and transition metal oxides, are also promising and are now gaining increasing attention for energy applications. The advantage of using such 2D crystals is linked to the possibility of creating and designing layered artificial structures with “on-demand” properties by means of spin-on processes, or layer-by-layer assembly. This approach exploits the availability of materials with metallic, semiconducting, and insulating properties. Advances The success of graphene and related materials (GRMs) for energy applications crucially depends on the development and optimization of production methods. High-volume liquid-phase exfoliation is being developed for a wide variety of layered materials. This technique is being optimized to control the flake size and to increase the edge-to-surface ratio, which is crucial for optimizing electrode performance in fuel cells and batteries. Micro- or nanocrystal or flake edge control can also be achieved through chemical synthesis. This is an ideal route for functionalization, in order to improve storage capacity. Large-area growth via chemical vapor deposition (CVD) has been demonstrated, producing material with high structural and electronic quality for the preparation of transparent conducting electrodes for displays and touch-screens, and is being evaluated for photovoltaic applications. CVD growth of other multicomponent layered materials is less mature and needs further development. Although many transfer techniques have been developed successfully, further improvement of high-volume manufacturing and transfer processes for multilayered heterostructures is needed. In this context, layer-by-layer assembly may enable the realization of devices with on-demand properties for targeted applications, such as photovoltaic devices in which photon absorption in TMDs is combined with charge transport in graphene. Outlook Substantial progress has been made on the preparation of GRMs at the laboratory level. However, cost-effective production of GRMs on an industrial scale is needed to create the future energy value chain. Applications that could benefit the most from GRMs include flexible electronics, batteries with efficient anodes and cathodes, supercapacitors with high energy density, and solar cells. The realization of GRMs with specific transport and insulating properties on demand is an important goal. Additional energy applications of GRMs comprise water splitting and hydrogen production. As an example, the edges of MoS2 single layers can oxidize fuels—such as hydrogen, methanol, and ethanol—in fuel cells, and GRM membranes can be used in fuel cells to improve proton exchange. Functionalized graphene can be exploited for water splitting and hydrogen production. Flexible and wearable devices and membranes incorporating GRMs can also generate electricity from motion, as well as from water and gas flows. Tailored GRMs for energy applications. The ability to produce GRMs with desired specific properties paves the way to their integration in a variety of energy devices. Solution processing and chemical vapor deposition are the ideal means to produce thin films that can be used as electrodes in energy devices (such as solar panels, batteries, fuel cells, or in hydrogen storage). Chemical synthesis is an attractive route to produce “active” elements in solar cell or thermoelectric devices. Graphene and related two-dimensional crystals and hybrid systems showcase several key properties that can address emerging energy needs, in particular for the ever growing market of portable and wearable energy conversion and storage devices. Graphene’s flexibility, large surface area, and chemical stability, combined with its excellent electrical and thermal conductivity, make it promising as a catalyst in fuel and dye-sensitized solar cells. Chemically functionalized graphene can also improve storage and diffusion of ionic species and electric charge in batteries and supercapacitors. Two-dimensional crystals provide optoelectronic and photocatalytic properties complementing those of graphene, enabling the realization of ultrathin-film photovoltaic devices or systems for hydrogen production. Here, we review the use of graphene and related materials for energy conversion and storage, outlining the roadmap for future applications. Layered materials power the cause Methods for storing and converting energy, including fuel cells, solar cells, and water splitting, often benefit from having materials with a large surface area. When combined with a high surface reactivity, high conductivity, or useful optical properties, two-dimensional layered materials become of notable interest for a range of applications. Bonaccorso et al. review the progress that has been made using graphene and other layered or two-dimensional materials at laboratory scales and the challenges in producing these materials in industrially relevant quantities. Science, this issue 10.1126/science.1246501', 'year': 2015, 'citationCount': 2686, 'authors': [{'authorId': '3040167', 'name': 'F. Bonaccorso'}, {'authorId': '50997465', 'name': 'L. Colombo'}, {'authorId': '5964001', 'name': 'Guihua Yu'}, {'authorId': '3992446', 'name': 'M. Stoller'}, {'authorId': '2359160', 'name': 'V. Tozzini'}, {'authorId': '153332598', 'name': 'A. Ferrari'}, {'authorId': '3197462', 'name': 'R. Ruoff'}, {'authorId': '2003758', 'name': 'V. Pellegrini'}]}, {'paperId': '894805e7984e34d93fbc5753c5e3c28facdb189d', 'title': 'K13-propeller mutations confer artemisinin resistance in Plasmodium falciparum clinical isolates', 'abstract': \"Mechanisms propelling drug resistance If it were to spread, resistance to the drug artemisinin would seriously derail the recent gains of global malaria control programs (see the Perspective by Sibley). Mutations in a region called the K13-propeller are predictive for artemisinin resistance in Southeast Asia. Mok et al. looked at the patterns of gene expression in parasites isolated from more than 1000 patients sampled in Africa, Bangladesh, and the Mekong region. A range of mutations that alter protein repair pathways and the timing of the parasite's developmental cycle were only found in parasites from the Mekong region. Straimer et al. genetically engineered the K13 region of parasites obtained from recent clinical isolates. Mutations in this region were indeed responsible for the resistance phenotypes. Science, this issue p. 431, p. 428; see also p. 373 Resistance to the primary antimalarial drug lies in mutations in protein repair and developmental pathways. [Also see Perspective by Sibley] The emergence of artemisinin resistance in Southeast Asia imperils efforts to reduce the global malaria burden. We genetically modified the Plasmodium falciparum K13 locus using zinc-finger nucleases and measured ring-stage survival rates after drug exposure in vitro; these rates correlate with parasite clearance half-lives in artemisinin-treated patients. With isolates from Cambodia, where resistance first emerged, survival rates decreased from 13 to 49% to 0.3 to 2.4% after the removal of K13 mutations. Conversely, survival rates in wild-type parasites increased from ≤0.6% to 2 to 29% after the insertion of K13 mutations. These mutations conferred elevated resistance to recent Cambodian isolates compared with that of reference lines, suggesting a contemporary contribution of additional genetic factors. Our data provide a conclusive rationale for worldwide K13-propeller sequencing to identify and eliminate artemisinin-resistant parasites.\", 'year': 2015, 'citationCount': 541, 'authors': [{'authorId': '4235059', 'name': 'J. Straimer'}, {'authorId': '3805330', 'name': 'Nina F. Gnädig'}, {'authorId': '3606394', 'name': 'B. Witkowski'}, {'authorId': '4086075', 'name': 'C. Amaratunga'}, {'authorId': '5151228', 'name': 'V. Duru'}, {'authorId': '3614601', 'name': 'A. P. Ramadani'}, {'authorId': '1694386255', 'name': 'Mélanie A. Dacheux'}, {'authorId': '4109824', 'name': 'N. Khim'}, {'authorId': '2152827305', 'name': 'Lei Zhang'}, {'authorId': '2056427851', 'name': 'Stephen C. Lam'}, {'authorId': '6635031', 'name': 'P. Gregory'}, {'authorId': '3571589', 'name': 'F. Urnov'}, {'authorId': '1398403785', 'name': 'O. Mercereau‐Puijalon'}, {'authorId': '1398426454', 'name': 'F. Benoit‐Vical'}, {'authorId': '87397838', 'name': 'R. Fairhurst'}, {'authorId': '144182939', 'name': 'D. Ménard'}, {'authorId': '2735233', 'name': 'D. Fidock'}]}, {'paperId': 'd73aca5a4e3a9b3dcde7a21e995efc67a820cd30', 'title': 'A dendritic cell vaccine increases the breadth and diversity of melanoma neoantigen-specific T cells', 'abstract': \"Giving antitumor T cells a boost Mutations allow tumors to divide, escape death, and resist treatment. But mutations can also cause tumors to express mutant proteins, which could potentially be exploited to drive antitumor T cell responses. Carreno et al. report the results of a small phase I trial seeking to do just this (see the Perspective by Delamarre et al.). They vaccinated three patients with advanced melanoma with personalized dendritic cell–based vaccines designed to activate T cells specific for mutations in the patients' cancer. T cells specific for mutant peptides did indeed expand. A next step will be to determine whether this promising strategy improves patient outcomes. Science, this issue p. 803; see also p. 760 Passenger mutations encode target melanoma antigens for human cancer immunotherapy. [Also see Perspective by Delamarre et al.] T cell immunity directed against tumor-encoded amino acid substitutions occurs in some melanoma patients. This implicates missense mutations as a source of patient-specific neoantigens. However, a systematic evaluation of these putative neoantigens as targets of antitumor immunity is lacking. Moreover, it remains unknown whether vaccination can augment such responses. We found that a dendritic cell vaccine led to an increase in naturally occurring neoantigen-specific immunity and revealed previously undetected human leukocyte antigen (HLA) class I–restricted neoantigens in patients with advanced melanoma. The presentation of neoantigens by HLA-A*02:01 in human melanoma was confirmed by mass spectrometry. Vaccination promoted a diverse neoantigen-specific T cell receptor (TCR) repertoire in terms of both TCR-β usage and clonal composition. Our results demonstrate that vaccination directed at tumor-encoded amino acid substitutions broadens the antigenic breadth and clonal diversity of antitumor immunity.\", 'year': 2015, 'citationCount': 904, 'authors': [{'authorId': '5194861', 'name': 'B. Carreno'}, {'authorId': '46540039', 'name': 'V. Magrini'}, {'authorId': '1398701413', 'name': 'M. Becker-Hapak'}, {'authorId': '5303952', 'name': 'S. Kaabinejadian'}, {'authorId': '3081783', 'name': 'J. Hundal'}, {'authorId': '3062663', 'name': 'A. Petti'}, {'authorId': '46762618', 'name': 'Amy Ly'}, {'authorId': '120626088', 'name': 'W. Lie'}, {'authorId': '3220399', 'name': 'W. Hildebrand'}, {'authorId': '3286001', 'name': 'E. Mardis'}, {'authorId': '4622371', 'name': 'G. Linette'}]}, {'paperId': 'adfc3c88d887818149b8aa672e96cd11313bbcd6', 'title': 'Patterns and ecological drivers of ocean viral communities', 'abstract': 'Viruses influence ecosystems by modulating microbial population size, diversity, metabolic outputs, and gene flow. Here, we use quantitative double-stranded DNA (dsDNA) viral-fraction metagenomes (viromes) and whole viral community morphological data sets from 43 Tara Oceans expedition samples to assess viral community patterns and structure in the upper ocean. Protein cluster cataloging defined pelagic upper-ocean viral community pan and core gene sets and suggested that this sequence space is well-sampled. Analyses of viral protein clusters, populations, and morphology revealed biogeographic patterns whereby viral communities were passively transported on oceanic currents and locally structured by environmental conditions that affect host community structure. Together, these investigations establish a global ocean dsDNA viromic data set with analyses supporting the seed-bank hypothesis to explain how oceanic viral communities maintain high local diversity.', 'year': 2015, 'citationCount': 554, 'authors': [{'authorId': '31932986', 'name': 'Jennifer R. Brum'}, {'authorId': '1398943556', 'name': 'J. C. Ignacio-Espinoza'}, {'authorId': '52161096', 'name': 'S. Roux'}, {'authorId': '4330984', 'name': 'Guilhem Doulcier'}, {'authorId': '6119340', 'name': 'S. Acinas'}, {'authorId': '6794377', 'name': 'A. Alberti'}, {'authorId': '2045252', 'name': 'Samuel Chaffron'}, {'authorId': '3529067', 'name': 'C. Cruaud'}, {'authorId': '2126901', 'name': 'C. de Vargas'}, {'authorId': '5262671', 'name': 'J. Gasol'}, {'authorId': '34168606', 'name': 'G. Gorsky'}, {'authorId': '49882697', 'name': 'A. Gregory'}, {'authorId': '10650860', 'name': 'L. Guidi'}, {'authorId': '2536107', 'name': 'P. Hingamp'}, {'authorId': '4206582', 'name': 'D. Iudicone'}, {'authorId': '2158864', 'name': 'F. Not'}, {'authorId': '2272089', 'name': 'H. Ogata'}, {'authorId': '5907934', 'name': 'S. Pesant'}, {'authorId': '4763837', 'name': 'B. Poulos'}, {'authorId': '6649875', 'name': 'S. Schwenck'}, {'authorId': '2154189', 'name': 'S. Speich'}, {'authorId': '6434378', 'name': 'Céline Dimier'}, {'authorId': '1396183973', 'name': 'S. Kandels-Lewis'}, {'authorId': '4157832', 'name': 'M. Picheral'}, {'authorId': '4309482', 'name': 'S. Searson'}, {'authorId': '3534315', 'name': 'P. Bork'}, {'authorId': '48244603', 'name': 'C. Bowler'}, {'authorId': '2067568', 'name': 'S. Sunagawa'}, {'authorId': '2839934', 'name': 'P. Wincker'}, {'authorId': '4192764', 'name': 'E. Karsenti'}, {'authorId': '5518928', 'name': 'M. Sullivan'}]}, {'paperId': '5f53968d110eb8413882b9c730fa7ac846b0e5e7', 'title': 'Lysosomal amino acid transporter SLC38A9 signals arginine sufficiency to mTORC1', 'abstract': 'Sensing amino acids at the lysosome The mTORC1 protein kinase is a complex of proteins that functions to regulate growth and metabolism. Activity of mTORC1 is sensitive to the abundance of amino acids, but how the sensing of amino acids is coupled to the control of mTORC1 has been unclear. Wang et al. searched for predicted membrane proteins that interacted with regulators of mTORC1. They identified a protein currently known only as SLC38A9. Interaction of SLC38A9 with mTORC1 regulators was sensitive to the presence of amino acids. SLC38A9 has sequence similarity to amino acid transporters. Effects of modulation of SLC38A9 in cultured human cells indicate that it may be the sensor that connects the abundance of arginine and leucine to mTORC1 activity. Science, this issue p. 188 A possible sensor for detecting and signaling amino acid abundance is identified. [Also see Perspective by Abraham] The mechanistic target of rapamycin complex 1 (mTORC1) protein kinase is a master growth regulator that responds to multiple environmental cues. Amino acids stimulate, in a Rag-, Ragulator-, and vacuolar adenosine triphosphatase–dependent fashion, the translocation of mTORC1 to the lysosomal surface, where it interacts with its activator Rheb. Here, we identify SLC38A9, an uncharacterized protein with sequence similarity to amino acid transporters, as a lysosomal transmembrane protein that interacts with the Rag guanosine triphosphatases (GTPases) and Ragulator in an amino acid–sensitive fashion. SLC38A9 transports arginine with a high Michaelis constant, and loss of SLC38A9 represses mTORC1 activation by amino acids, particularly arginine. Overexpression of SLC38A9 or just its Ragulator-binding domain makes mTORC1 signaling insensitive to amino acid starvation but not to Rag activity. Thus, SLC38A9 functions upstream of the Rag GTPases and is an excellent candidate for being an arginine sensor for the mTORC1 pathway.', 'year': 2015, 'citationCount': 529, 'authors': [{'authorId': '2155437961', 'name': 'Shuyu Wang'}, {'authorId': '4099866', 'name': 'Z. Tsun'}, {'authorId': '6862301', 'name': 'Rachel L. Wolfson'}, {'authorId': '2705753', 'name': 'K. Shen'}, {'authorId': '4587526', 'name': 'Gregory A. Wyant'}, {'authorId': '4606345', 'name': 'Molly E Plovanich'}, {'authorId': '143966902', 'name': 'Elizabeth D. Yuan'}, {'authorId': '10064557', 'name': 'Tony D. Jones'}, {'authorId': '3054115', 'name': 'Lynne Chantranupong'}, {'authorId': '4037235', 'name': 'W. Comb'}, {'authorId': '49981123', 'name': 'Tim Wang'}, {'authorId': '1397944980', 'name': 'L. Bar-Peled'}, {'authorId': '3034790', 'name': 'R. Zoncu'}, {'authorId': '48449774', 'name': 'Christoph Straub'}, {'authorId': '1787267307', 'name': 'Choah Kim'}, {'authorId': '2115930814', 'name': 'Jiwon Park'}, {'authorId': '2845457', 'name': 'B. Sabatini'}, {'authorId': '1843680', 'name': 'D. Sabatini'}]}, {'paperId': '009e98d679ea67ae8b7a71f0ccd62b9b0eaae3ef', 'title': 'Universal linear optics', 'abstract': \"Complex quantum optical circuitry Encoding and manipulating information in the states of single photons provides a potential platform for quantum computing and communication. Carolan et al. developed a reconfigurable integrated waveguide device fabricated in a glass chip (see the Perspective by Rohde and Dowling). The device allowed for universal linear optics transformations on six wave-guides using 15 integrated Mach-Zehnder interferometers, each of which was individually programmable. Functional performance in a number of applications in optics and quantum optics demonstrates the versatility of the device's reprogrammable architecture. Science, this issue p. 711; see also p. 696 A reconfigurable optical circuit provides a platform for a photonically-based quantum computer. [Also see Perspective by Rohde and Dowling] Linear optics underpins fundamental tests of quantum mechanics and quantum technologies. We demonstrate a single reprogrammable optical circuit that is sufficient to implement all possible linear optical protocols up to the size of that circuit. Our six-mode universal system consists of a cascade of 15 Mach-Zehnder interferometers with 30 thermo-optic phase shifters integrated into a single photonic chip that is electrically and optically interfaced for arbitrary setting of all phase shifters, input of up to six photons, and their measurement with a 12-single-photon detector system. We programmed this system to implement heralded quantum logic and entangling gates, boson sampling with verification tests, and six-dimensional complex Hadamards. We implemented 100 Haar random unitaries with an average fidelity of 0.999 ± 0.001. Our system can be rapidly reprogrammed to implement these and any other linear optical protocol, pointing the way to applications across fundamental science and quantum technologies.\", 'year': 2015, 'citationCount': 819, 'authors': [{'authorId': '48228088', 'name': 'J. Carolan'}, {'authorId': '144731834', 'name': 'C. Harrold'}, {'authorId': '40062159', 'name': 'Chris Sparrow'}, {'authorId': '1404394297', 'name': 'E. Martin-Lopez'}, {'authorId': '48070560', 'name': 'N. Russell'}, {'authorId': '14244862', 'name': 'J. Silverstone'}, {'authorId': '2990311', 'name': 'P. Shadbolt'}, {'authorId': '40184662', 'name': 'N. Matsuda'}, {'authorId': '3134178', 'name': 'M. Oguma'}, {'authorId': '2011983', 'name': 'M. Itoh'}, {'authorId': '48258157', 'name': 'G. Marshall'}, {'authorId': '1690351', 'name': 'M. Thompson'}, {'authorId': '33859373', 'name': 'J. Matthews'}, {'authorId': '32256564', 'name': 'T. Hashimoto'}, {'authorId': '8187633', 'name': \"J. O'Brien\"}, {'authorId': '39220451', 'name': 'A. Laing'}]}, {'paperId': 'e77709623a4c7824be6200933353883097ccd170', 'title': 'Function-led design of new porous materials', 'abstract': \"It's all about the holes From kitchen sieves and strainers to coffee filters, porous materials have a wide range of uses. On an industrial scale, they are used as sorbents, filters, membranes, and catalysts. Slater and Cooper review how each application will limit the materials that can be used, and also the size and connectivity of the pores required. They go on to compare and contrast a growing range of porous materials that are finding increasing use in academic and industrial applications. Science, this issue 10.1126/science.aaa8075 BACKGROUND Porous materials are important in established processes such as catalysis and molecular separations and in emerging technologies for energy and health. Porous zeolites have made the largest contribution to society so far, and that field is still developing rapidly. Other porous solids have also entered the scene in the past two decades, such as metal-organic frameworks (MOFs), covalent organic frameworks (COFs), and porous organic polymers. No single class of porous material is ideal for all purposes. For example, crystallinity and long-range order might enhance selectivity for a molecular separation while also reducing mechanical stability or processability with respect to less ordered structures. To have an impact on real applications, porous materials must be scalable and must satisfy multiple functional criteria such as long-term stability, selectivity, adsorption kinetics, and processability, all within a viable cost envelope. This presents a broad design challenge, and it requires us to be able to control structure and to understand multiple structure-property relationships at a detailed level. ADVANCES In addition to MOFs, COFs, and porous polymer networks, other classes of molecular porous solids have emerged in the past 10 years, such as polymers of intrinsic microporosity and porous organic cages. The range of possible functions for porous solids is thus much broader than before. For example, conjugated microporous polymers and some COFs have extended, conjugated structures that are not present in zeolites or MOFs and have led to porous organic photocatalysts and electronic materials. The crystal engineering approaches developed for zeolites, MOFs, and COFs cannot be applied directly to amorphous solids such as porous polymers, but analogous modular strategies have allowed functions such as porosity and electronic band gap to be controlled by choosing the appropriate molecular building blocks. Rapid advances in the computational prediction of structure and function offer a strategy for identifying the best porous materials for specific applications, for example, via large-scale screening of gas adsorption in hypothetical MOFs. OUTLOOK Advances in synthesis have produced new classes of functional porous solids as well as fundamental breakthroughs in areas such as selective carbon dioxide capture, molecular separations, and catalysis. As yet, these rapid developments in basic understanding are unmatched by large-scale commercial implementation, but enhanced functions (such as enzyme-like CO2 selectivity) and new processing options (such as soluble porous solids) present exciting opportunities. A general challenge will be to reengineer porous materials where scale-up is prohibited by cost, retaining the advanced function but using cheaper and more sustainable building blocks. It is therefore important to develop structure-property relationships to understand how promising materials work. Not all future opportunities for porous solids involve improving on existing materials or the development of more scalable preparation routes. For example, porous photocatalysts that can perform direct solar water splitting might provide a completely new platform for energy production. As we seek increasingly complex functions for porous materials, the use of in silico computational design to guide experiment will become more important. Porous materials can be defined by type or by function, but it is function that will determine the scope for practical applications. Our ability to design functions in porous solids has advanced markedly in the past two decades as a result of developments in modular synthesis, materials characterization, and (more recently) computational structure-property predictions. This figure is based on the pore channels, shown in yellow, for an organic cage molecule, a new type of solution-processable porous solid developed over the past 6 years. Porous solids are important as membranes, adsorbents, catalysts, and in other chemical applications. But for these materials to find greater use at an industrial scale, it is necessary to optimize multiple functions in addition to pore structure and surface area, such as stability, sorption kinetics, processability, mechanical properties, and thermal properties. Several different classes of porous solids exist, and there is no one-size-fits-all solution; it can therefore be challenging to choose the right type of porous material for a given job. Computational prediction of structure and properties has growing potential to complement experiment to identify the best porous materials for specific applications.\", 'year': 2015, 'citationCount': 1058, 'authors': [{'authorId': '47387234', 'name': 'A. Slater'}, {'authorId': '1754626', 'name': 'A. Cooper'}]}, {'paperId': 'f0f09a65a5924b61ab14c5d1197d444a983a656e', 'title': 'Near-unity photoluminescence quantum yield in MoS2', 'abstract': 'Brighter molybdenum layers The confined layers of molybdenum disulphide (MoS2) exhibit photoluminescence that is attractive for optolectronic applications. In practice, efficiencies are low, presumably because defects trap excitons before they can recombine and radiate light. Amani et al. show that treatment of monolayer MoS2 with a nonoxidizing organic superacid, bis(trifluoromethane) sulfonimide, increased luminescence efficiency in excess of 95%. The enhancement mechanism may be related to the shielding of defects, such as sulfur vacancies. Science, this issue p. 1065 Superacid treatment enhances the luminescence efficiency of monolayer molybdenum disulfide from 1% to >95%. Two-dimensional (2D) transition metal dichalcogenides have emerged as a promising material system for optoelectronic applications, but their primary figure of merit, the room-temperature photoluminescence quantum yield (QY), is extremely low. The prototypical 2D material molybdenum disulfide (MoS2) is reported to have a maximum QY of 0.6%, which indicates a considerable defect density. Here we report on an air-stable, solution-based chemical treatment by an organic superacid, which uniformly enhances the photoluminescence and minority carrier lifetime of MoS2 monolayers by more than two orders of magnitude. The treatment eliminates defect-mediated nonradiative recombination, thus resulting in a final QY of more than 95%, with a longest-observed lifetime of 10.8 ± 0.6 nanoseconds. Our ability to obtain optoelectronic monolayers with near-perfect properties opens the door for the development of highly efficient light-emitting diodes, lasers, and solar cells based on 2D materials.', 'year': 2015, 'citationCount': 704, 'authors': [{'authorId': '4795241', 'name': 'M. Amani'}, {'authorId': '2029675', 'name': 'D. Lien'}, {'authorId': '51220328', 'name': 'D. Kiriya'}, {'authorId': '2111222656', 'name': 'Jun Xiao'}, {'authorId': '3986759', 'name': 'A. Azcatl'}, {'authorId': '2072788421', 'name': 'Jiyoung Noh'}, {'authorId': '5910856', 'name': 'S. Madhvapathy'}, {'authorId': '6009973', 'name': 'R. Addou'}, {'authorId': '2222022011', 'name': 'Santosh Kc'}, {'authorId': '34126382', 'name': 'M. Dubey'}, {'authorId': '2198231', 'name': 'Kyeongjae Cho'}, {'authorId': '2974186', 'name': 'R. Wallace'}, {'authorId': '2144375736', 'name': 'Si-Chen Lee'}, {'authorId': '49264345', 'name': 'Jr-hau He'}, {'authorId': '40544683', 'name': 'J. Ager'}, {'authorId': '2190290509', 'name': 'Xiang Zhang'}, {'authorId': '1823951', 'name': 'E. Yablonovitch'}, {'authorId': '2402712', 'name': 'A. Javey'}]}, {'paperId': 'a8674e93d1e4b9cdc471c81a1edfea76ab23ad6a', 'title': 'Extended-resolution structured illumination imaging of endocytic and cytoskeletal dynamics', 'abstract': 'Adding to the super-resolution arsenal Structured illumination microscopy (SIM) uses light intensities that are orders of magnitude lower than other super-resolution methods. SIM is also far faster over cellular-sized fields of view. Li et al. used two approaches to improve the resolution of SIM to allow live cell imaging of dynamic cellular processes, including endocytosis and cytoskeleton remodeling. The contrast in performance between SIM and other techniques is due to a few key differences. Defining the practical resolution at the limited signal-to-noise ratios necessary for live cell imaging will require better imaging metrics. Science, this issue 10.1126/science.aab3500 Super-resolution imaging of fast dynamic processes in living cells is facilitated by improvements to structured illumination microscopy. INTRODUCTION Various methods of super-resolution (SR) fluorescence microscopy have the potential to follow the dynamic nanoscale interactions of specific macromolecular assemblies in living cells. However, this potential is often left unfulfilled, either owing to the method’s inability to follow these processes at the speeds dictated by nature or because they require intense light that can substantially perturb the very physiology one hopes to study. An exception is structured illumination microscopy (SIM), which can image live cells far faster and with orders of magnitude less light than required for other SR approaches. However, SIM’s resolution is usually limited to only a twofold gain beyond conventional optical microscopes, or ~100 nm with visible light. RATIONALE We endeavored to find ways to extend SIM to the sub-100-nm regime while retaining, to the greatest extent possible, the advantages that make it the preferred SR method for live-cell imaging. Our first solution used an ultrahigh numerical aperture (NA) lens and total internal reflection fluorescence (TIRF) to achieve 84-nm resolution at subsecond acquisition speeds over hundreds of time points in multiple colors near the basal plasma membrane. Our second exploited the spatially patterned activation of a recently developed, reversibly photoswitchable fluorescent protein to reach 45- to 62-nm resolution, also at subsecond acquisition, over ∼10 to 40 time points. RESULTS We used high-NA TIRF-SIM to image the dynamic associations of cortical filamentous actin with myosin IIA, paxillin, or clathrin, as well as paxillin with vinculin and clathrin with transferrin receptors. Thanks to the combination of high spatial and temporal resolution, we were able to measure the sizes of individual clathrin-coated pits through their initiation, growth, and internalization. We were also able to relate pit size to lifetime, identify and characterize localized hot spots of pit generation, and describe the interaction of actin with clathrin and its role in accelerating endocytosis. With nonlinear SIM by use of patterned activation (PA NL-SIM), we monitored the remodeling of the actin cytoskeleton and the dynamics of caveolae at the cell surface. By combining TIRF-SIM and PA NL-SIM for two-color imaging, we followed the dynamic association of actin with α-actinin in expanding filopodia and membrane ruffles and characterized shape changes in and the transport of early endosomes. Last, by combining PA NL-SIM with lattice light sheet microscopy, we observed, in three dimensions and across the entire volume of whole cells, the dynamics of the actin cytoskeleton, the fusion and fission of mitochondria, and the trafficking of vesicles to and from the Golgi apparatus, each at axial resolution fivefold better than that of conventional widefield microscopy. In addition, through direct experimental comparisons, we demonstrated that the resolution for our methods is comparable with or better than other SR approaches yet allowed us to image at far higher speeds, and for far longer durations. To understand why this is so, we developed a detailed theoretical model showing that our methods transmit the information encoded in spatial frequencies beyond the diffraction limit with much greater strength than do other alternatives and hence require far fewer photons emitted from the specimen, using far less intense light. CONCLUSION High-NA TIRF-SIM and PA NL-SIM fill an unmet need for minimally invasive tools to image live cells in the gap between the 100-nm resolution traditionally associated with SIM and the sub-60-nm regime of protein-specific structural imaging served by single-molecule localization microscopy. Two approaches for improved live-cell imaging at sub-100-nm resolution. (Left) Association of cortical actin (purple) with clathrin-coated pits (green), the latter seen as rings (inset) at 84-nm resolution via a combination of total internal reflection fluorescence and structured illumination microscopy at ultrahigh numerical aperture (high-NA TIRF-SIM). (Right) Progression of resolution improvement across the actin cytoskeleton of a COS-7 cell, from conventional, diffraction-limited TIRF (220-nm resolution), to TIRF-SIM (97-nm resolution), and nonlinear SIM based on the patterned activation of a reversibly photoswitchable fluorescent protein (PA NL-SIM, 62 nm resolution). (Left and right represent single frames from time-lapse movies over 91 and 30 frames, respectively. Scale bars, 2 μm (left); 3 μm (right). Super-resolution fluorescence microscopy is distinct among nanoscale imaging tools in its ability to image protein dynamics in living cells. Structured illumination microscopy (SIM) stands out in this regard because of its high speed and low illumination intensities, but typically offers only a twofold resolution gain. We extended the resolution of live-cell SIM through two approaches: ultrahigh numerical aperture SIM at 84-nanometer lateral resolution for more than 100 multicolor frames, and nonlinear SIM with patterned activation at 45- to 62-nanometer resolution for approximately 20 to 40 frames. We applied these approaches to image dynamics near the plasma membrane of spatially resolved assemblies of clathrin and caveolin, Rab5a in early endosomes, and α-actinin, often in relationship to cortical actin. In addition, we examined mitochondria, actin, and the Golgi apparatus dynamics in three dimensions.', 'year': 2015, 'citationCount': 564, 'authors': [{'authorId': '2108819888', 'name': 'Dong Li'}, {'authorId': '49971595', 'name': 'L. Shao'}, {'authorId': '50461904', 'name': 'Bi-Chang Chen'}, {'authorId': '2108286623', 'name': 'Xi Zhang'}, {'authorId': '4144655', 'name': 'Mingshu Zhang'}, {'authorId': '2069911672', 'name': 'Brian Moses'}, {'authorId': '5119148', 'name': 'D. Milkie'}, {'authorId': '39875282', 'name': 'Jordan R. Beach'}, {'authorId': '144477660', 'name': 'J. Hammer'}, {'authorId': '4611850', 'name': 'Mithun Pasham'}, {'authorId': '4218577', 'name': 'T. Kirchhausen'}, {'authorId': '48179239', 'name': 'M. A. Baird'}, {'authorId': '48949259', 'name': 'M. Davidson'}, {'authorId': '145731493', 'name': 'P. Xu'}, {'authorId': '1774970', 'name': 'E. Betzig'}]}, {'paperId': '5e218166d49017d40c651669c9e130e6071e1be4', 'title': 'A prudent path forward for genomic engineering and germline gene modification', 'abstract': 'A framework for open discourse on the use of CRISPR-Cas9 technology to manipulate the human genome is urgently needed Genome engineering technology offers unparalleled potential for modifying human and nonhuman genomes. In humans, it holds the promise of curing genetic disease, while in other organisms it provides methods to reshape the biosphere for the benefit of the environment and human societies. However, with such enormous opportunities come unknown risks to human health and well-being. In January, a group of interested stakeholders met in Napa, California (1), to discuss the scientific, medical, legal, and ethical implications of these new prospects for genome biology. The goal was to initiate an informed discussion of the uses of genome engineering technology, and to identify those areas where action is essential to prepare for future developments. The meeting identified immediate steps to take toward ensuring that the application of genome engineering technology is performed safely and ethically.', 'year': 2015, 'citationCount': 562, 'authors': [{'authorId': '123871122', 'name': 'D. Baltimore'}, {'authorId': '145479891', 'name': 'P. Berg'}, {'authorId': '5754463', 'name': 'M. Botchan'}, {'authorId': '145108881', 'name': 'D. Carroll'}, {'authorId': '6809758', 'name': 'R. Charo'}, {'authorId': '145892667', 'name': 'G. Church'}, {'authorId': '5129546', 'name': 'J. Corn'}, {'authorId': '2028224', 'name': 'G. Daley'}, {'authorId': '4448238', 'name': 'J. Doudna'}, {'authorId': '41191823', 'name': 'M. Fenner'}, {'authorId': '3277528', 'name': 'H. Greely'}, {'authorId': '3693015', 'name': 'M. Jinek'}, {'authorId': '2013804594', 'name': 'G. S. Martin'}, {'authorId': '8047750', 'name': 'E. Penhoet'}, {'authorId': '4689670', 'name': 'J. Puck'}, {'authorId': '2993789', 'name': 'S. H. Sternberg'}, {'authorId': '3168827', 'name': 'J. Weissman'}, {'authorId': '115645782', 'name': 'K. Yamamoto'}]}, {'paperId': '259d28767d1a4f297788ac09b7cc9b8d4aa9d7b5', 'title': 'Macroscale superlubricity enabled by graphene nanoscroll formation', 'abstract': 'Slip sliding away Many applications would benefit from ultralow friction conditions to minimize wear on the moving parts such as in hard drives or engines. On the very small scale, ultralow friction has been observed with graphite as a lubricant. Berman et al. achieved superlubricity using graphene in combination with crystalline diamond nanoparticles and diamondlike carbon (see the Perspective by Hone and Carpick). Simulations showed that sliding of the graphene patches around the tiny nanodiamond particles led to nanoscrolls with reduced contact area that slide easily against the amorphous diamondlike carbon surface. Science, this issue p. 1118; see also p. 1087 Nanodiamonds wrapped with graphene sheets lead to ultralow friction against a diamondlike carbon surface. [Also see Perspective by Hone and Carpick] Friction and wear remain as the primary modes of mechanical energy dissipation in moving mechanical assemblies; thus, it is desirable to minimize friction in a number of applications. We demonstrate that superlubricity can be realized at engineering scale when graphene is used in combination with nanodiamond particles and diamondlike carbon (DLC). Macroscopic superlubricity originates because graphene patches at a sliding interface wrap around nanodiamonds to form nanoscrolls with reduced contact area that slide against the DLC surface, achieving an incommensurate contact and substantially reduced coefficient of friction (~0.004). Atomistic simulations elucidate the overall mechanism and mesoscopic link bridging the nanoscale mechanics and macroscopic experimental observations.', 'year': 2015, 'citationCount': 605, 'authors': [{'authorId': '20991272', 'name': 'D. Berman'}, {'authorId': '3887827', 'name': 'Sanket A. Deshmukh'}, {'authorId': '2929500', 'name': 'S. Sankaranarayanan'}, {'authorId': '48715178', 'name': 'A. Erdemir'}, {'authorId': '2149699', 'name': 'A. Sumant'}]}, {'paperId': 'd6b3d4b1e5b081af3226d5fdbbc3661113f331ee', 'title': 'Readily processed protonic ceramic fuel cells with high performance at low temperatures', 'abstract': 'Cooler ceramic fuel cells Ceramic ion conductors can be used as electrolytes in fuel cells using natural gas. One drawback of such solid-oxide fuel cells that conduct oxygen ions is their high operating temperatures (at least 600°C). Duan et al. have made a proton-conducting ceramic fuel cell with a modified cathode material that exhibits high performance on methane fuel at 500°C (see the Perspective by Gorte). Science, this issue p. 1321; see also p. 1290 A proton-conduction cathode and simpler fabrication enable lower-temperature operation of methane-fueled ceramic fuel cells. [Also see Perspective by Gorte] Because of the generally lower activation energy associated with proton conduction in oxides compared to oxygen ion conduction, protonic ceramic fuel cells (PCFCs) should be able to operate at lower temperatures than solid oxide fuel cells (250° to 550°C versus ≥600°C) on hydrogen and hydrocarbon fuels if fabrication challenges and suitable cathodes can be developed. We fabricated the complete sandwich structure of PCFCs directly from raw precursor oxides with only one moderate-temperature processing step through the use of sintering agents such as copper oxide. We also developed a proton-, oxygen-ion–, and electron-hole–conducting PCFC-compatible cathode material, BaCo0.4Fe0.4Zr0.1Y0.1O3-δ (BCFZY0.1), that greatly improved oxygen reduction reaction kinetics at intermediate to low temperatures. We demonstrated high performance from five different types of PCFC button cells without degradation after 1400 hours. Power densities as high as 455 milliwatts per square centimeter at 500°C on H2 and 142 milliwatts per square centimeter on CH4 were achieved, and operation was possible even at 350°C.', 'year': 2015, 'citationCount': 811, 'authors': [{'authorId': '7906816', 'name': 'Chuancheng Duan'}, {'authorId': '48620856', 'name': 'J. Tong'}, {'authorId': '2144460956', 'name': 'Meng Shang'}, {'authorId': '4406457', 'name': 'S. Nikodemski'}, {'authorId': '2004546422', 'name': 'M. Sanders'}, {'authorId': '8176738', 'name': 'S. Ricote'}, {'authorId': '34965842', 'name': 'A. Almansoori'}, {'authorId': '1398376191', 'name': \"R. O'Hayre\"}]}, {'paperId': 'a52e39e2cd8f98c002dafd84183661db682152bf', 'title': 'De novo mutations in congenital heart disease with neurodevelopmental and other congenital anomalies', 'abstract': 'Putting both heart and brain at risk For reasons that are unclear, newborns with congenital heart disease (CHD) have a high risk of neurodevelopmental disabilities. Homsy et al. performed exome sequence analysis of 1200 CHD patients and their parents to identify spontaneously arising (de novo) mutations. Patients with both CHD and neurodevelopmental disorders had a much higher burden of damaging de novo mutations, particularly in genes with likely roles in both heart and brain development. Thus, clinical genotyping of patients with CHD may help to identify those at greatest risk of neurodevelopmental disabilities, allowing surveillance and early intervention. Science, this issue p. 1262 Genotyping of children with congenital heart disease may identify those at high risk of neurodevelopmental disorders. Congenital heart disease (CHD) patients have an increased prevalence of extracardiac congenital anomalies (CAs) and risk of neurodevelopmental disabilities (NDDs). Exome sequencing of 1213 CHD parent-offspring trios identified an excess of protein-damaging de novo mutations, especially in genes highly expressed in the developing heart and brain. These mutations accounted for 20% of patients with CHD, NDD, and CA but only 2% of patients with isolated CHD. Mutations altered genes involved in morphogenesis, chromatin modification, and transcriptional regulation, including multiple mutations in RBFOX2, a regulator of mRNA splicing. Genes mutated in other cohorts examined for NDD were enriched in CHD cases, particularly those with coexisting NDD. These findings reveal shared genetic contributions to CHD, NDD, and CA and provide opportunities for improved prognostic assessment and early therapeutic intervention in CHD patients.', 'year': 2015, 'citationCount': 578, 'authors': [{'authorId': '36079066', 'name': 'Jason Homsy'}, {'authorId': '47885166', 'name': 'S. Zaidi'}, {'authorId': '37082667', 'name': 'Yufeng Shen'}, {'authorId': '48123382', 'name': 'J. Ware'}, {'authorId': '6255853', 'name': 'K. Samocha'}, {'authorId': '1813912', 'name': 'K. Karczewski'}, {'authorId': '5085779', 'name': 'S. DePalma'}, {'authorId': '36608677', 'name': 'D. McKean'}, {'authorId': '6344438', 'name': 'H. Wakimoto'}, {'authorId': '5444722', 'name': 'Joshua M. Gorham'}, {'authorId': '8130226', 'name': 'S. Jin'}, {'authorId': '2069206170', 'name': 'J. Deanfield'}, {'authorId': '3562597', 'name': 'A. Giardini'}, {'authorId': '38146876', 'name': 'G. Porter'}, {'authorId': '51178340', 'name': 'R. Kim'}, {'authorId': '4223946', 'name': 'K. Bilguvar'}, {'authorId': '1393617500', 'name': 'F. López-Giráldez'}, {'authorId': '36337379', 'name': 'Irina R. Tikhonova'}, {'authorId': '2666293', 'name': 'S. Mane'}, {'authorId': '1399293379', 'name': 'Angela Romano-Adesman'}, {'authorId': '145671097', 'name': 'Hongjian Qi'}, {'authorId': '3549190', 'name': 'B. Vardarajan'}, {'authorId': '5616092', 'name': 'Lijiang Ma'}, {'authorId': '144524355', 'name': 'M. Daly'}, {'authorId': '4419743', 'name': 'A. Roberts'}, {'authorId': '12217730', 'name': 'M. Russell'}, {'authorId': '144766484', 'name': 'S. Mital'}, {'authorId': '75082917', 'name': 'J. Newburger'}, {'authorId': '144442425', 'name': 'J. Gaynor'}, {'authorId': '3672289', 'name': 'R. Breitbart'}, {'authorId': '2154137', 'name': 'I. Iossifov'}, {'authorId': '4232362', 'name': 'M. Ronemus'}, {'authorId': '4493069', 'name': 'Stephan J Sanders'}, {'authorId': '4790942', 'name': 'J. Kaltman'}, {'authorId': '1703476', 'name': 'J. Seidman'}, {'authorId': '2141645', 'name': 'M. Brueckner'}, {'authorId': '5215835', 'name': 'B. Gelb'}, {'authorId': '4520908', 'name': 'E. Goldmuntz'}, {'authorId': '3038250', 'name': 'R. Lifton'}, {'authorId': '5088534', 'name': 'C. Seidman'}, {'authorId': '145062232', 'name': 'W. Chung'}]}, {'paperId': 'b7b9ab1eaf18a67f4aa620619d8f21211c7c5964', 'title': 'Extensive introgression in a malaria vector species complex revealed by phylogenomics', 'abstract': 'Introduction The notion that species boundaries can be porous to introgression is increasingly accepted. Yet the broader role of introgression in evolution remains contentious and poorly documented, partly because of the challenges involved in accurately identifying introgression in the very groups where it is most likely to occur. Recently diverged species often have incomplete reproductive barriers and may hybridize where they overlap. However, because of retention and stochastic sorting of ancestral polymorphisms, inference of the correct species branching order is notoriously challenging for recent speciation events, especially those closely spaced in time. Without knowledge of species relationships, it is impossible to identify instances of introgression. Rationale Since the discovery that the single mosquito taxon described in 1902 as Anopheles gambiae was actually a complex of several closely related and morphologically indistinguishable sibling species, the correct species branching order has remained controversial and unresolved. This Afrotropical complex contains the world’s most important vectors of human malaria, owing to their close association with humans, as well as minor vectors and species that do not bite humans. On the basis of ecology and behavior, one might predict phylogenetic clustering of the three highly anthropophilic vector species. However, previous phylogenetic analyses of the complex based on a limited number of markers strongly disagree about relationships between the major vectors, potentially because of historical introgression between them. To investigate the history of the species complex, we used whole-genome reference assemblies, as well as dozens of resequenced individuals from the field. Results We observed a large amount of phylogenetic discordance between trees generated from the autosomes and X chromosome. The autosomes, which make up the majority of the genome, overwhelmingly supported the grouping of the three major vectors of malaria, An. gambiae, An. coluzzii, and An. arabiensis. In stark contrast, the X chromosome strongly supported the grouping of An. arabiensis with a species that plays no role in malaria transmission, An. quadriannulatus. Although the whole-genome consensus phylogeny unequivocally agrees with the autosomal topology, we found that the topology most often located on the X chromosome follows the historical species branching order, with pervasive introgression on the autosomes producing relationships that group the three highly anthropophilic species together. With knowledge of the correct species branching order, we are further able to uncover introgression between another species pair, as well as a complex history of balancing selection, introgression, and local adaptation of a large autosomal inversion that confers aridity tolerance. Conclusion We identify the correct species branching order of the An. gambiae species complex, resolving a contentious phylogeny. Notably, lineages leading to the principal vectors of human malaria were among the first in the complex to radiate and are not most closely related to each other. Pervasive autosomal introgression between these human malaria vectors, including nonsister vector species, suggests that traits enhancing vectorial capacity can be acquired not only through de novo mutation but also through a more rapid process of interspecific genetic exchange. Time-lapse photographs of an adult anopheline mosquito emerging from its pupal case. RELATED ITEMS IN ScienceD. E. Neafsey et al., Science 347, 1258522 (2015) Introgressive hybridization is now recognized as a widespread phenomenon, but its role in evolution remains contested. Here, we use newly available reference genome assemblies to investigate phylogenetic relationships and introgression in a medically important group of Afrotropical mosquito sibling species. We have identified the correct species branching order to resolve a contentious phylogeny and show that lineages leading to the principal vectors of human malaria were among the first to split. Pervasive autosomal introgression between these malaria vectors means that only a small fraction of the genome, mainly on the X chromosome, has not crossed species boundaries. Our results suggest that traits enhancing vectorial capacity may be gained through interspecific gene flow, including between nonsister species. Mosquito adaptability across genomes Virtually everyone has first-hand experience with mosquitoes. Few recognize the subtle biological distinctions among these bloodsucking flies that render some bites mere nuisances and others the initiation of a potentially life-threatening infection. By sequencing the genomes of several mosquitoes in depth, Neafsey et al. and Fontaine et al. reveal clues that explain the mystery of why only some species of one genus of mosquitoes are capable of transmitting human malaria (see the Perspective by Clark and Messer). Science, this issue 10.1126/science.1258524 and 10.1126/science.1258522; see also p. 27 Comparison of several genomes reveals the genetic history of mosquitoes’ ability to vector malaria among humans. [Also see Perspective by Clark and Messer]', 'year': 2015, 'citationCount': 501, 'authors': [{'authorId': '144101105', 'name': 'M. Fontaine'}, {'authorId': '2497743', 'name': 'J. Pease'}, {'authorId': '47174215', 'name': 'Aaron Steele'}, {'authorId': '3008998', 'name': 'R. Waterhouse'}, {'authorId': '4303568', 'name': 'D. Neafsey'}, {'authorId': '2721351', 'name': 'I. Sharakhov'}, {'authorId': '143700507', 'name': 'Xiaofang Jiang'}, {'authorId': '144957945', 'name': 'A. B. Hall'}, {'authorId': '3795051', 'name': 'F. Catteruccia'}, {'authorId': '4003555', 'name': 'E. Kakani'}, {'authorId': '153336006', 'name': 'Sara N. Mitchell'}, {'authorId': '38870301', 'name': 'Yi-Chieh Wu'}, {'authorId': '153144270', 'name': 'Hilary A. Smith'}, {'authorId': '145966718', 'name': 'R. R. Love'}, {'authorId': '1688049', 'name': 'M. Lawniczak'}, {'authorId': '6198906', 'name': 'M. Slotman'}, {'authorId': '1760521', 'name': 'S. Emrich'}, {'authorId': '2235214549', 'name': 'M. Hahn'}, {'authorId': '2524799', 'name': 'N. Besansky'}]}, {'paperId': '2a0e218758c8ca8a2eba1db681d5dbbda81a43e2', 'title': 'Immunogenicity of somatic mutations in human gastrointestinal cancers', 'abstract': 'Low mutation rate okay for T cells Cancers that tend to have high numbers of mutations, such as melanoma and smoking-induced lung cancer, respond well to immunotherapies, whereas those with fewer mutations, such as pancreatic cancer, do not. Tran et al. searched for tumor mutation–reactive T cells in 10 patients with metastatic gastrointestinal cancers, which have relatively low mutation burdens, and discovered that 9 out of 10 harbored such cells. T cells from one patient recognized a mutation common to many types of cancers. Engineering T cells to express this particular mutation-reactive T cell receptor may extend adoptive cell immunotherapy to a larger pool of patients than previously anticipated. Science, this issue p. 1387 Individuals with cancers that have low mutation frequencies often harbor mutation-reactive T cells. It is unknown whether the human immune system frequently mounts a T cell response against mutations expressed by common epithelial cancers. Using a next-generation sequencing approach combined with high-throughput immunologic screening, we demonstrated that tumor-infiltrating lymphocytes (TILs) from 9 out of 10 patients with metastatic gastrointestinal cancers contained CD4+ and/or CD8+ T cells that recognized one to three neo-epitopes derived from somatic mutations expressed by the patient’s own tumor. There were no immunogenic epitopes shared between these patients. However, we identified in one patient a human leukocyte antigen–C*08:02–restricted T cell receptor from CD8+ TILs that targeted the KRASG12D hotspot driver mutation found in many human cancers. Thus, a high frequency of patients with common gastrointestinal cancers harbor immunogenic mutations that can potentially be exploited for the development of highly personalized immunotherapies.', 'year': 2015, 'citationCount': 585, 'authors': [{'authorId': '2280776', 'name': 'E. Tran'}, {'authorId': '38535581', 'name': 'M. Ahmadzadeh'}, {'authorId': '48518323', 'name': 'Yong-Chen Lu'}, {'authorId': '5168824', 'name': 'A. Gros'}, {'authorId': '3622148', 'name': 'S. Turcotte'}, {'authorId': '2083707', 'name': 'P. Robbins'}, {'authorId': '145772328', 'name': 'J. Gartner'}, {'authorId': '37322255', 'name': 'Z. Zheng'}, {'authorId': '47001537', 'name': 'Yong F Li'}, {'authorId': '8485007', 'name': 'Satyajit Ray'}, {'authorId': '32109109', 'name': 'J. Wunderlich'}, {'authorId': '5105081', 'name': 'R. Somerville'}, {'authorId': '144243541', 'name': 'S. Rosenberg'}]}, {'paperId': 'a50c931f46adc7bdaf8d3ce2f0cf7c9b87b35f89', 'title': 'Platinum-based nanocages with subnanometer-thick walls and well-defined, controllable facets', 'abstract': 'Etching platinum nanocage catalysts Although platinum is an excellent catalyst for the oxygen reduction reaction that occurs in fuel cells, its scarcity continues to drive efforts to improve its utilization. Zhang et al. made nanocages of platinum by coating palladium nanocrystals with only a few layers of platinum and then etching away the palladium core (see the Perspective by Strasser). Platinum nanocages made using nanoscale octahedra and cubes of palladium displayed different catalytic activity for the oxygen reduction reaction. Science, this issue p. 412; see also p. 379 Nanocage electrocatalysts can increase the utilization of platinum and improve activity by controlling surface structure. [Also see Perspective by Strasser] A cost-effective catalyst should have a high dispersion of the active atoms, together with a controllable surface structure for the optimization of activity, selectivity, or both. We fabricated nanocages by depositing a few atomic layers of platinum (Pt) as conformal shells on palladium (Pd) nanocrystals with well-defined facets and then etching away the Pd templates. Density functional theory calculations suggest that the etching is initiated via a mechanism that involves the formation of vacancies through the removal of Pd atoms incorporated into the outermost layer during the deposition of Pt. With the use of Pd nanoscale cubes and octahedra as templates, we obtained Pt cubic and octahedral nanocages enclosed by {100} and {111} facets, respectively, which exhibited distinctive catalytic activities toward oxygen reduction.', 'year': 2015, 'citationCount': 732, 'authors': [{'authorId': '2152832439', 'name': 'Lei Zhang'}, {'authorId': '4612075', 'name': 'L. T. Roling'}, {'authorId': '2144706211', 'name': 'Xue Wang'}, {'authorId': '2422767', 'name': 'M. Vara'}, {'authorId': '2232765', 'name': 'M. Chi'}, {'authorId': '6207126', 'name': 'J. Liu'}, {'authorId': '91669921', 'name': 'Sang‐Il Choi'}, {'authorId': '2109225880', 'name': 'Jinho Park'}, {'authorId': '2279071', 'name': 'J. Herron'}, {'authorId': '36620090', 'name': 'Zhaoxiong Xie'}, {'authorId': '47249637', 'name': 'M. Mavrikakis'}, {'authorId': '46267791', 'name': 'Younan Xia'}]}, {'paperId': '20bbc7c4c761d90f0f81c1c5928922caaaeef4af', 'title': 'Undergraduate research experiences: Impacts and opportunities', 'abstract': 'Assessing the value of undergraduate research Undergraduate research experiences often engender enthusiasm in the students involved, but how useful are they in terms of enhancing student learning? Linn et al. review studies that focus on the effectiveness of undergraduate research programs. Undergraduate research experiences in a class were distinguished from those involving individualized participation in a research program. Mentoring emerges as both an important component of a successful experience and a target for improvement. Science, this issue 10.1126/science.1261757 For any undergraduate contemplating a career in scientific research, participating in authentic research seems like a good opportunity. But what are authentic research experiences? How do they benefit undergraduates? What forms of mentoring are successful? What needs improvement? And how can these experiences meet the needs of interested students while at the same time be cost-effective in large research universities? We review the research tackling these questions and find few answers. While most undergraduates give high ratings to research experiences, specific benefits have not been documented. Of the 60 empirical studies published in the last 5 years, only 4 directly measured gains in research capabilities or conceptual understanding. Most studies draw conclusions from self-report surveys or interviews, notoriously poor methods for documenting impacts. These studies leave us with few insights into what works and little idea about how to make the experiences more effective. BACKGROUND Most colleges and universities offer Undergraduate Research Experiences (UREs) and/or Course-based Undergraduate Research Experiences (CUREs) (Fig. 1). Two large surveys, the 2004 Freshman Survey and the 2008 College Senior Survey, administered at over 200 institutions, generated data about the impact of undergraduate research experiences on persistence in science and intention to pursue graduate school. These studies document that students appreciate undergraduate research experiences. The surveys are unable, however, to distinguish between UREs and CUREs. In addition, the value that undergraduate research adds cannot be disentangled from precollege preparation, especially for students from groups that are underrepresented in science. ADVANCES Designers of UREs expect students to benefit from participating in a scientific laboratory but have not determined optimal ways to orient and guide participants. Students often expect the URE to mimic their college laboratory experiences with procedural guidance and planned outcomes. During the first year of a URE, students often report spending most of their effort on setting up and conducting experiments and limited effort on understanding the investigation or interpreting the results. Students would benefit from an orientation that integrates their beliefs and expectations with the realities of the research experience. The few studies that measure changes in understanding of scientific practices or relevant science concepts report little or no gains after 1 year in a URE. Students who spend over a year in a URE often learn new methodological techniques, collect their own data, interpret findings, and formulate new research questions. The slow enculturation into lab activities may make sense, especially when students join labs investigating questions that do not arise in undergraduate education. The time and resources needed, however, limit the scalability of UREs. Students encounter new ideas during their research experiences but often need guidance to integrate these ideas with their expectations. We discuss ways that designers of UREs can speed up enculturation and strengthen guidance. Individual mentoring emerges as an effective way to guide students and improve learning from research experiences. Activities that could help students benefit from research experiences include discussion with mentors, participation in group meetings where current research is discussed, guided opportunities to explore relevant research literature, reflection on observations in weekly journals, and synthesis of their insights by creating research proposals, reports, or posters. We discuss ways to prepare mentors so that they can efficiently guide students. OUTLOOK Undergraduate research experiences absorb a lot of time, money, and effort. The costs and benefits of research experiences for building human capital, benefitting undergraduates, improving workforce diversity, and strengthening educational outcomes need better understanding. Making the best use of extramural funds and the (often voluntary) contributions of faculty to improve undergraduate research experiences requires a strong research base. More rigorous research is needed, and the field could benefit by building on insights from the learning sciences. We use the knowledge integration framework to interpret the available findings and to identify gaps in the research base. We discuss ways to develop validated, generalizable assessments such as methods for measuring ability to locate and interpret primary literature. We suggest techniques for developing criteria for evaluating mentoring interactions. We identify ways to strengthen mentoring and to ensure that research experiences meet the needs of diverse students. URE in action. Biofuel research engages a Berkeley undergraduate researcher during a summer internship with the Synthetic Biology Engineering Research Center (funded by NSF grant 1132670). Most undergraduates give high ratings to research experiences. Studies report that these experiences improve participation and persistence, often by strengthening students’ views of themselves as scientists. Yet, the evidence for these claims is weak. More than half the 60 studies reviewed rely on self-report surveys or interviews. Rather than introducing new images of science, research experiences may reinforce flawed images especially of research practices and conceptual understanding. The most convincing studies show benefits for mentoring and for communicating the nature of science, but the ideas that students learn are often isolated or fragmented rather than integrated and coherent. Rigorous research is needed to identify ways to design research experiences so that they promote integrated understanding. These studies need powerful and generalizable assessments that can document student progress, help distinguish effective and ineffective aspects of the experiences, and illustrate how students interpret the research experiences they encounter. To create research experiences that meet the needs of interested students and make effective use of scarce resources, we encourage systematic, iterative studies with multiple indicators of success.', 'year': 2015, 'citationCount': 508, 'authors': [{'authorId': '48454219', 'name': 'M. Linn'}, {'authorId': '2059332914', 'name': 'Erin Palmer'}, {'authorId': '5029957', 'name': 'A. Baranger'}, {'authorId': '2075139616', 'name': 'Elizabeth Gerard'}, {'authorId': '39650728', 'name': 'E. Stone'}]}, {'paperId': 'ac44c78e8b2a938a71010fbc604500a8900ce521', 'title': 'Site-selective arene C-H amination via photoredox catalysis', 'abstract': 'Lighting the way to aryl C-N bonding Medicinal chemists like to add N bonds to the C atoms of aromatic rings to make bioactive compounds. By harnessing the energy in visible light, Romero et al. made these links and transformed C-H into C-N bonds. They used a blue-absorbing acridinium ion to activate a ring C for an incoming N partner. A nitroxyl radical co-catalyst (TEMPO) then choreographed the transfer of the H atom to O. The reaction worked for a broad range of substrates, including ammonium as a N source. Science, this issue p. 1326 Two catalysts leverage visible light and oxygen to form carbon-nitrogen bond motifs of interest in pharmaceutical research. Over the past several decades, organometallic cross-coupling chemistry has developed into one of the most reliable approaches to assemble complex aromatic compounds from preoxidized starting materials. More recently, transition metal–catalyzed carbon-hydrogen activation has circumvented the need for preoxidized starting materials, but this approach is limited by a lack of practical amination protocols. Here, we present a blueprint for aromatic carbon-hydrogen functionalization via photoredox catalysis and describe the utility of this strategy for arene amination. An organic photoredox-based catalyst system, consisting of an acridinium photooxidant and a nitroxyl radical, promotes site-selective amination of a variety of simple and complex aromatics with heteroaromatic azoles of interest in pharmaceutical research. We also describe the atom-economical use of ammonia to form anilines, without the need for prefunctionalization of the aromatic component.', 'year': 2015, 'citationCount': 532, 'authors': [{'authorId': '2775062', 'name': 'N. Romero'}, {'authorId': '7223629', 'name': 'Kaila A. Margrey'}, {'authorId': '3294517', 'name': 'Nicholas E. S. Tay'}, {'authorId': '4072096', 'name': 'David A. Nicewicz'}]}, {'paperId': '7a1a1f38ed470d814e2e92f3b976e99096ada4ad', 'title': 'Patients with LRBA deficiency show CTLA4 loss and immune dysregulation responsive to abatacept therapy', 'abstract': 'Trafficking from bedside to bench Typically in translational research, a discovery in cell or molecular biology is later exploited to improve patient care. Occasionally, information flows in the opposite direction. Lo et al. found that patients with an autoimmune disorder caused by deficiency of a protein called LRBA responded dramatically to the drug abatacept (see the Perspective by Sansom). Abatacept contains a segment of a potent inhibitory immune receptor, CTLA4. Experiments prompted by this observation revealed the relationship between the two proteins: LRBA controls the intracellular trafficking and degradation of CTLA4. This information may further improve patient care, because other clinically approved drugs have the desired mechanism of action with potentially fewer side effects. Science, this issue p. 436; see also p. 377 A rare autoimmune disorder is caused by aberrant degradation of a potent inhibitory immune receptor. [Also see Perspective by Sansom] Mutations in the LRBA gene (encoding the lipopolysaccharide-responsive and beige-like anchor protein) cause a syndrome of autoimmunity, lymphoproliferation, and humoral immune deficiency. The biological role of LRBA in immunologic disease is unknown. We found that patients with LRBA deficiency manifested a dramatic and sustained improvement in response to abatacept, a CTLA4 (cytotoxic T lymphocyte antigen-4)–immunoglobulin fusion drug. Clinical responses and homology of LRBA to proteins controlling intracellular trafficking led us to hypothesize that it regulates CTLA4, a potent inhibitory immune receptor. We found that LRBA colocalized with CTLA4 in endosomal vesicles and that LRBA deficiency or knockdown increased CTLA4 turnover, which resulted in reduced levels of CTLA4 protein in FoxP3+ regulatory and activated conventional T cells. In LRBA-deficient cells, inhibition of lysosome degradation with chloroquine prevented CTLA4 loss. These findings elucidate a mechanism for CTLA4 trafficking and control of immune responses and suggest therapies for diseases involving the CTLA4 pathway.', 'year': 2015, 'citationCount': 502, 'authors': [{'authorId': '35263862', 'name': 'Bernice Lo'}, {'authorId': '5170321', 'name': 'Kejian Zhang'}, {'authorId': '2198230', 'name': 'Wei-Yu Lu'}, {'authorId': '123584816', 'name': 'Lixin Zheng'}, {'authorId': '145261216', 'name': 'Qian Zhang'}, {'authorId': '40295074', 'name': 'C. Kanellopoulou'}, {'authorId': None, 'name': 'Yu Zhang'}, {'authorId': '2144292726', 'name': 'Zhiduo Liu'}, {'authorId': '39402782', 'name': 'Jill M. Fritz'}, {'authorId': '34743560', 'name': 'R. Marsh'}, {'authorId': '5997093', 'name': 'Ammar Husami'}, {'authorId': '6443598', 'name': 'D. Kissell'}, {'authorId': '8284739', 'name': 'Shannon Nortman'}, {'authorId': '7985432', 'name': 'Vijaya Chaturvedi'}, {'authorId': '50321247', 'name': 'H. Haines'}, {'authorId': '144059319', 'name': 'L. Young'}, {'authorId': '1431187881', 'name': 'J. Mo'}, {'authorId': '6579125', 'name': 'A. Filipovich'}, {'authorId': '77416037', 'name': 'J. Bleesing'}, {'authorId': '10355163', 'name': 'P. Mustillo'}, {'authorId': '49649865', 'name': 'Michael C. Stephens'}, {'authorId': '40506732', 'name': 'C. Rueda'}, {'authorId': '5879639', 'name': 'C. Chougnet'}, {'authorId': '6682787', 'name': 'K. Hoebe'}, {'authorId': '143964722', 'name': 'J. McElwee'}, {'authorId': '145930892', 'name': 'J. Hughes'}, {'authorId': '1401313922', 'name': 'E. Karakoc‐Aydiner'}, {'authorId': '4563707', 'name': 'H. Matthews'}, {'authorId': '48194986', 'name': 'S. Price'}, {'authorId': '144213500', 'name': 'H. Su'}, {'authorId': '144632738', 'name': 'V. Rao'}, {'authorId': '145001039', 'name': 'M. Lenardo'}, {'authorId': '31907435', 'name': 'M. Jordan'}]}, {'paperId': '02c24ef5736d9c2809da31ca658d8ced1d81350b', 'title': 'CTCF establishes discrete functional chromatin domains at the Hox clusters during differentiation', 'abstract': 'Keeping repressed genes repressed Hox genes confer positional identity to cells and tissues. Maintaining precise spatial patterns of Hox gene expression is vital during metazoan development. The transcriptional repressor CTCF is involved in the regulation of chromatin architecture. Narendra et al. show that a CTCF protein binding site insulates regions of active and repressed Hox gene expression from each other. This protects heterochromatin containing repressed Hox genes from the encroaching spread of active chromatin. The CTCF protein appears to organize the active and repressed chromatin regions into distinct architectural domains. Science, this issue p. 1017 A DNA binding protein insulates and protects repressed topological domains in the genome from adjacent active regions. Polycomb and Trithorax group proteins encode the epigenetic memory of cellular positional identity by establishing inheritable domains of repressive and active chromatin within the Hox clusters. Here we demonstrate that the CCCTC-binding factor (CTCF) functions to insulate these adjacent yet antagonistic chromatin domains during embryonic stem cell differentiation into cervical motor neurons. Deletion of CTCF binding sites within the Hox clusters results in the expansion of active chromatin into the repressive domain. CTCF functions as an insulator by organizing Hox clusters into spatially disjoint domains. Ablation of CTCF binding disrupts topological boundaries such that caudal Hox genes leave the repressed domain and become subject to transcriptional activation. Hence, CTCF is required to insulate facultative heterochromatin from impinging euchromatin to produce discrete positional identities.', 'year': 2015, 'citationCount': 472, 'authors': [{'authorId': '2098133', 'name': 'V. Narendra'}, {'authorId': '144949609', 'name': 'Pedro P. Rocha'}, {'authorId': '6659023', 'name': 'Disi An'}, {'authorId': '3382835', 'name': 'R. Raviram'}, {'authorId': '3382289', 'name': 'J. Skok'}, {'authorId': '2565403', 'name': 'E. Mazzoni'}, {'authorId': '5850010', 'name': 'D. Reinberg'}]}, {'paperId': '29e96ad5f757fb36b8d3f4ec2e5ea11e7121e518', 'title': 'Transcribed enhancers lead waves of coordinated transcription in transitioning mammalian cells', 'abstract': 'Uncaging promoter and enhancer dynamics In order to understand cellular differentiation, it is important to understand the timing of the regulation of gene expression. Arner et al. used cap analysis of gene expression (CAGE) to analyze gene enhancer and promoter activities in a number of human and mouse cell types. The RNA of enhancers was transcribed first, followed by that of transcription factors, and finally by genes that are not transcription factors. Science, this issue p. 1010 The RNAs of enhancers are expressed before the genes they regulate during human and mouse cell differentiation. Although it is generally accepted that cellular differentiation requires changes to transcriptional networks, dynamic regulation of promoters and enhancers at specific sets of genes has not been previously studied en masse. Exploiting the fact that active promoters and enhancers are transcribed, we simultaneously measured their activity in 19 human and 14 mouse time courses covering a wide range of cell types and biological stimuli. Enhancer RNAs, then messenger RNAs encoding transcription factors, dominated the earliest responses. Binding sites for key lineage transcription factors were simultaneously overrepresented in enhancers and promoters active in each cellular system. Our data support a highly generalizable model in which enhancer transcription is the earliest event in successive waves of transcriptional change during cellular differentiation or activation.', 'year': 2015, 'citationCount': 511, 'authors': [{'authorId': '23521489', 'name': 'E. Arner'}, {'authorId': '143917592', 'name': 'C. Daub'}, {'authorId': '1397636244', 'name': 'Kristoffer Vitting-Seerup'}, {'authorId': '40479623', 'name': 'R. Andersson'}, {'authorId': '4166525', 'name': 'B. Lilje'}, {'authorId': '114341708', 'name': 'F. Drabløs'}, {'authorId': '2688127', 'name': 'A. Lennartsson'}, {'authorId': '4886769', 'name': 'Michelle Rönnerblad'}, {'authorId': '47320354', 'name': 'Olga Hrydziuszko'}, {'authorId': '2532012', 'name': 'Morana Vitezic'}, {'authorId': '143803510', 'name': 'T. Freeman'}, {'authorId': '2653439', 'name': 'Ahmad M. N. Alhendi'}, {'authorId': '4212074', 'name': 'P. Arner'}, {'authorId': '6948294', 'name': 'R. Axton'}, {'authorId': '145337559', 'name': 'J. Baillie'}, {'authorId': '8528842', 'name': 'A. Beckhouse'}, {'authorId': '3696831', 'name': 'B. Bodega'}, {'authorId': '144968403', 'name': 'J. Briggs'}, {'authorId': '6018346', 'name': 'F. Brombacher'}, {'authorId': '30335332', 'name': 'M. Davis'}, {'authorId': '4516324', 'name': 'M. Detmar'}, {'authorId': '4699789', 'name': 'A. Ehrlund'}, {'authorId': '5356829', 'name': 'Mitsuhiro Endoh'}, {'authorId': '46939083', 'name': 'A. Eslami'}, {'authorId': '10935716', 'name': 'M. Fagiolini'}, {'authorId': '34034984', 'name': 'L. Fairbairn'}, {'authorId': '3076070', 'name': 'G. Faulkner'}, {'authorId': '46750409', 'name': 'Carmelo Ferrai'}, {'authorId': '153395152', 'name': 'M. Fisher'}, {'authorId': '5226420', 'name': 'L. Forrester'}, {'authorId': '2230666', 'name': 'D. Goldowitz'}, {'authorId': '3261646', 'name': 'R. Guler'}, {'authorId': '9655717', 'name': 'Thomas J. Ha'}, {'authorId': '9034129', 'name': 'Mitsuko Hara'}, {'authorId': '47789855', 'name': 'M. Herlyn'}, {'authorId': '4258998', 'name': 'T. Ikawa'}, {'authorId': '145708643', 'name': 'C. Kai'}, {'authorId': '2135221', 'name': 'H. Kawamoto'}, {'authorId': '3254387', 'name': 'L. Khachigian'}, {'authorId': '144555597', 'name': 'S. Klinken'}, {'authorId': '33518402', 'name': 'S. Kojima'}, {'authorId': '2777281', 'name': 'H. Koseki'}, {'authorId': '49705905', 'name': 'Sarah Klein'}, {'authorId': '6826212', 'name': 'N. Mejhert'}, {'authorId': '4971860', 'name': 'K. Miyaguchi'}, {'authorId': '50148706', 'name': 'Y. Mizuno'}, {'authorId': '34788980', 'name': 'M. Morimoto'}, {'authorId': '50525373', 'name': 'Kelly J. Morris'}, {'authorId': '3440534', 'name': 'C. Mummery'}, {'authorId': '2351194', 'name': 'Y. Nakachi'}, {'authorId': '1786708', 'name': 'S. Ogishima'}, {'authorId': '1397636249', 'name': 'M. Okada‐Hatakeyama'}, {'authorId': '50117403', 'name': 'Y. Okazaki'}, {'authorId': '37788422', 'name': 'V. Orlando'}, {'authorId': '3745403', 'name': 'D. Ovchinnikov'}, {'authorId': '3517695', 'name': 'R. Passier'}, {'authorId': '8576273', 'name': 'M. Patrikakis'}, {'authorId': '144656656', 'name': 'A. Pombo'}, {'authorId': '5128673', 'name': 'Xian-Yang Qin'}, {'authorId': '47115698', 'name': 'Sugata Roy'}, {'authorId': '145506513', 'name': 'Hiroki Sato'}, {'authorId': '5481867', 'name': 'Suzana Savvi'}, {'authorId': '49984299', 'name': 'A. Saxena'}, {'authorId': '2949707', 'name': 'Anita Schwegmann'}, {'authorId': '145080244', 'name': 'D. Sugiyama'}, {'authorId': '145354896', 'name': 'R. Swoboda'}, {'authorId': '143848108', 'name': 'Hiroshi Tanaka'}, {'authorId': '5779951', 'name': 'A. Tomoiu'}, {'authorId': '4853058', 'name': 'Louise N. Winteringham'}, {'authorId': '4431070', 'name': 'E. Wolvetang'}, {'authorId': '1400621142', 'name': 'Chiyo Yanagi-Mizuochi'}, {'authorId': '47848477', 'name': 'M. Yoneda'}, {'authorId': '4654219', 'name': 'S. Zabierowski'}, {'authorId': '1396200156', 'name': 'Peter G. Zhang'}, {'authorId': '145237763', 'name': 'I. Abugessaisa'}, {'authorId': '5942363', 'name': 'N. Bertin'}, {'authorId': '2389396', 'name': 'A. Diehl'}, {'authorId': '3245060', 'name': 'S. Fukuda'}, {'authorId': '47876805', 'name': 'M. Furuno'}, {'authorId': '3434398', 'name': 'J. Harshbarger'}, {'authorId': '145629751', 'name': 'A. Hasegawa'}, {'authorId': '27021163', 'name': 'Fumi Hori'}, {'authorId': '1400621118', 'name': 'Sachi Ishikawa-Kato'}, {'authorId': '11917019', 'name': 'Y. Ishizu'}, {'authorId': '144562623', 'name': 'M. Itoh'}, {'authorId': '6461494', 'name': 'T. Kawashima'}, {'authorId': '34833114', 'name': 'M. Kojima'}, {'authorId': '35970974', 'name': 'N. Kondo'}, {'authorId': '2124452', 'name': 'M. Lizio'}, {'authorId': '35466559', 'name': 'T. Meehan'}, {'authorId': '52038267', 'name': 'C. Mungall'}, {'authorId': '11168299', 'name': 'Mitsuyoshi Murata'}, {'authorId': '1400621107', 'name': 'Hiromi Nishiyori-Sueki'}, {'authorId': '48534111', 'name': 'S. Şahin'}, {'authorId': '1400621116', 'name': 'Sayaka Nagao-Sato'}, {'authorId': '3177339', 'name': 'J. Severin'}, {'authorId': '3207348', 'name': 'M. Hoon'}, {'authorId': '1695117', 'name': 'J. Kawai'}, {'authorId': '2572145', 'name': 'T. Kasukawa'}, {'authorId': '2775888', 'name': 'T. Lassmann'}, {'authorId': '2111430835', 'name': 'Harukazu Suzuki'}, {'authorId': '2657593', 'name': 'H. Kawaji'}, {'authorId': '1948404', 'name': 'K. Summers'}, {'authorId': '49078862', 'name': 'C. Wells'}, {'authorId': '2330321', 'name': 'D. Hume'}, {'authorId': '144627172', 'name': 'A. Forrest'}, {'authorId': '3275066', 'name': 'A. Sandelin'}, {'authorId': '2051569', 'name': 'Piero Carninci'}, {'authorId': '3345237', 'name': 'Y. Hayashizaki'}]}, {'paperId': '42df2ef9b0f3fa9a1d52fe29a2f09fea6d0a937c', 'title': 'Deposition, exhumation, and paleoclimate of an ancient lake deposit, Gale crater, Mars', 'abstract': 'Ancient lake system at Gale crater Since 2012, the Curiosity rover has been diligently studying rocky outcrops on Mars, looking for clues about past water, climate, and habitability. Grotzinger et al. describe the analysis of a huge section of sedimentary rocks near Gale crater, where Mount Sharp now stands (see the Perspective by Chan). The features within these sediments are reminiscent of delta, stream, and lake deposits on Earth. Although individual lakes were probably transient, it is likely that there was enough water to fill in low-lying depressions such as impact craters for up to 10,000 years. Wind-driven erosion removed many of these deposits, creating Mount Sharp. Science, this issue p.10.1126/science.aac7575, see also p. 167 Mount Sharp now stands where there was once a large intercrater lake system. [Also see Perspective by Chan] INTRODUCTION Remote observational data suggest that large bodies of standing water existed on the surface of Mars in its early history. This would have required a much wetter climate than that of the present, implying greater availability of water on a global basis and enhanced potential for global habitability. However, based on assumptions of a vast water inventory and models of atmospheric erosion, theoretical studies suggest a climate that was wetter but not by enough to sustain large lakes, even in depressions such as impact craters. RATIONALE The Mars Science Laboratory mission’s rover, Curiosity, provides the capability to test hypotheses about Mars’s past climate. The focus of the mission is the exploration of a ~5-km-high mountain, Aeolis Mons (informally known as Mount Sharp), located near the center of the ~140-km-wide Gale impact crater. Mount Sharp is underlain by hundreds of meters of sedimentary rock strata deposited ~3.6 billion to 3.2 billion years ago. These sediments accumulated in aqueous environments, recording the history of Mars’s ancient climate. Because of Curiosity’s ability to study these strata where they are exposed near the base of Mount Sharp, we can directly test the hypothesis that large impact craters were capable of accumulating and storing water as lakes for substantial periods of time. RESULTS Over the course of 2 years, Curiosity studied dozens of outcrops distributed along a ~9-km transect that also rose ~75 m in elevation. Image data were used to measure the geometry and grain sizes of strata and to survey the textures associated with sediment deposition and diagenesis. Erosion of Gale’s northern crater wall and rim generated gravel and sand that were transported southward in shallow streams. Over time, these stream deposits advanced toward the crater interior, transitioning downstream into finer-grained (sand-sized), southward-advancing delta deposits. These deltas marked the boundary of an ancient lake where the finest (mud-sized) sediments accumulated, infilling both the crater and its internal lake basin. After infilling of the crater, the sedimentary deposits in Gale crater were exhumed, probably by wind-driven erosion, creating Mount Sharp. The ancient stream and lake deposits are erosional remnants of superimposed depositional sequences that once extended at least 75 m, and perhaps several hundreds of meters, above the current elevation of the crater floor. Although the modern landscape dips northward away from Mount Sharp, the ancient sedimentary deposits were laid down along a profile that projected southward beneath Mount Sharp and indicate that a basin once existed where today there is a mountain. CONCLUSION Our observations suggest that individual lakes were stable on the ancient surface of Mars for 100 to 10,000 years, a minimum duration when each lake was stable both thermally (as liquid water) and in terms of mass balance (with inputs effectively matching evaporation and loss of water to colder regions). We estimate that the stratigraphy traversed thus far by Curiosity would have required 10,000 to 10,000,000 years to accumulate, and even longer if overlying strata are included. Though individual lakes may have come and gone, they were probably linked in time through a common groundwater table. Over the long term, this water table must have risen at least tens of meters to enable accumulation of the delta and lake deposits observed by Curiosity in Gale crater. Inclined strata in the foreground dip southward toward Mount Sharp and represent ancient delta deposits. These deposits transition into strata in the mid-field that were deposited in ancient lakes. The buttes and mesas in the background contain younger deposits that overlie and postdate the lake deposits beneath Mount Sharp. The outcrop in the foreground is about 6 m wide, and the buttes and mesas in the background are hundreds of meters wide and tens of meters high. The image has been white-balanced. [Credit: NASA/Caltech/JPL/MSSS] The landforms of northern Gale crater on Mars expose thick sequences of sedimentary rocks. Based on images obtained by the Curiosity rover, we interpret these outcrops as evidence for past fluvial, deltaic, and lacustrine environments. Degradation of the crater wall and rim probably supplied these sediments, which advanced inward from the wall, infilling both the crater and an internal lake basin to a thickness of at least 75 meters. This intracrater lake system probably existed intermittently for thousands to millions of years, implying a relatively wet climate that supplied moisture to the crater rim and transported sediment via streams into the lake basin. The deposits in Gale crater were then exhumed, probably by wind-driven erosion, creating Aeolis Mons (Mount Sharp).', 'year': 2015, 'citationCount': 507, 'authors': [{'authorId': '3844614', 'name': 'J. Grotzinger'}, {'authorId': '1747899867', 'name': 'Sanjeev Gupta'}, {'authorId': '2185806', 'name': 'M. Malin'}, {'authorId': '9216016', 'name': 'D. Rubin'}, {'authorId': '2167952988', 'name': 'J. Schieber'}, {'authorId': '3674499', 'name': 'K. Siebach'}, {'authorId': '2086032', 'name': 'D. Sumner'}, {'authorId': '3558841', 'name': 'K. Stack'}, {'authorId': '35070652', 'name': 'A. Vasavada'}, {'authorId': '2144646', 'name': 'R. Arvidson'}, {'authorId': '4373022', 'name': 'F. Calef'}, {'authorId': '46621475', 'name': 'L. Edgar'}, {'authorId': '2057396505', 'name': 'W. Fischer'}, {'authorId': '119605866', 'name': 'J. Grant'}, {'authorId': '3957803', 'name': 'J. Griffes'}, {'authorId': '4441973', 'name': 'L. Kah'}, {'authorId': '143785554', 'name': 'M. Lamb'}, {'authorId': '143671882', 'name': 'K. Lewis'}, {'authorId': '145544465', 'name': 'N. Mangold'}, {'authorId': '38372442', 'name': 'M. Minitti'}, {'authorId': '7001762', 'name': 'M. Palucis'}, {'authorId': '49565924', 'name': 'M. Rice'}, {'authorId': '2158137751', 'name': 'Rebecca M. E. Williams'}, {'authorId': '5962704', 'name': 'R. Yingst'}, {'authorId': '2068472632', 'name': 'D. Blake'}, {'authorId': '145681115', 'name': 'D. Blaney'}, {'authorId': '40098549', 'name': 'P. Conrad'}, {'authorId': '4047178', 'name': 'J. Crisp'}, {'authorId': '1406001779', 'name': 'W. Dietrich'}, {'authorId': '4718864', 'name': 'G. Dromart'}, {'authorId': '4759990', 'name': 'K. Edgett'}, {'authorId': '145657190', 'name': 'R. Ewing'}, {'authorId': '2714162', 'name': 'R. Gellert'}, {'authorId': '6882251', 'name': 'J. Hurowitz'}, {'authorId': '98112552', 'name': 'G. Kocurek'}, {'authorId': '4579249', 'name': 'P. Mahaffy'}, {'authorId': '41210735', 'name': 'M. McBride'}, {'authorId': '144183259', 'name': 'S. McLennan'}, {'authorId': '6507728', 'name': 'M. Mischna'}, {'authorId': '153607306', 'name': 'Doug Ming'}, {'authorId': '32008334', 'name': 'R. Milliken'}, {'authorId': '4763968', 'name': 'H. Newsom'}, {'authorId': '40335319', 'name': 'D. Oehler'}, {'authorId': '48123456', 'name': 'T. Parker'}, {'authorId': '4803566', 'name': 'D. Vaniman'}, {'authorId': '145952447', 'name': 'R. Wiens'}, {'authorId': '115224202', 'name': 'S. Wilson'}]}, {'paperId': '8e9776716ff99195a5b9b87474f2e4a48beba883', 'title': 'Farm dust and endotoxin protect against allergy through A20 induction in lung epithelial cells', 'abstract': 'How farming protects against allergies People who grow up on dairy farms only rarely develop asthma or allergies. This is probably because as children, they breathe air containing bacterial components, which reduce the overall reactivity of the immune system. Schuijs et al. chronically exposed mice to bacterial endotoxin before they received an allergic stimulus. The protocol indeed protected them from developing an allergic response. Protection relied on a particular enzyme: A20. In humans, a variant of A20 correlates with increased susceptibility to asthma and allergy in children growing up on farms. Science, this issue p. 1106 Chronic exposure to bacterial endotoxin protects against allergy through the enzyme A20. Growing up on a dairy farm protects children from allergy, hay fever, and asthma. A mechanism linking exposure to this endotoxin (bacterial lipopolysaccharide)–rich environment with protection has remained elusive. Here we show that chronic exposure to low-dose endotoxin or farm dust protects mice from developing house dust mite (HDM)–induced asthma. Endotoxin reduced epithelial cell cytokines that activate dendritic cells (DCs), thus suppressing type 2 immunity to HDMs. Loss of the ubiquitin-modifying enzyme A20 in lung epithelium abolished the protective effect. A single-nucleotide polymorphism in the gene encoding A20 was associated with allergy and asthma risk in children growing up on farms. Thus, the farming environment protects from allergy by modifying the communication between barrier epithelial cells and DCs through A20 induction.', 'year': 2015, 'citationCount': 477, 'authors': [{'authorId': '4314078', 'name': 'M. Schuijs'}, {'authorId': '4085503', 'name': 'M. Willart'}, {'authorId': '50071230', 'name': 'Karl Vergote'}, {'authorId': '2198038', 'name': 'D. Gras'}, {'authorId': '5935685', 'name': 'K. Deswarte'}, {'authorId': '6207885', 'name': 'M. Ege'}, {'authorId': '39913579', 'name': 'Filipe Branco Madeira'}, {'authorId': '79930198', 'name': 'R. Beyaert'}, {'authorId': '3941663', 'name': 'G. van Loo'}, {'authorId': '5868696', 'name': 'F. Bracher'}, {'authorId': '116242153', 'name': 'E. von Mutius'}, {'authorId': '145813672', 'name': 'P. Chanez'}, {'authorId': '50382649', 'name': 'B. Lambrecht'}, {'authorId': '143816987', 'name': 'H. Hammad'}]}, {'paperId': '5c7ba6c1f0902007eb5a3aedf889c6a67654361d', 'title': 'Titin mutations in iPS cells define sarcomere insufficiency as a cause of dilated cardiomyopathy', 'abstract': 'A giant disruption of the heart Certain forms of heart failure originate from genetic mutations. Understanding how the culprit mutant proteins alter normal heart function could lead to more effective treatments. One candidate is the giant protein tintin, which is mutated in a subset of patients with dilated cardiomyopathy. Through a combination of patient-derived stem cells, tissue engineering, and gene editing, Hinson et al. found that disease-associated titin mutations disrupt the function of the contractile unit in heart muscle. As a result, the heart does not respond properly to mechanical and other forms of stress. Science, this issue p. 982 Mutations in titin cause heart disease by disrupting the sarcomere, which normally helps the heart adapt to stress. Human mutations that truncate the massive sarcomere protein titin [TTN-truncating variants (TTNtvs)] are the most common genetic cause for dilated cardiomyopathy (DCM), a major cause of heart failure and premature death. Here we show that cardiac microtissues engineered from human induced pluripotent stem (iPS) cells are a powerful system for evaluating the pathogenicity of titin gene variants. We found that certain missense mutations, like TTNtvs, diminish contractile performance and are pathogenic. By combining functional analyses with RNA sequencing, we explain why truncations in the A-band domain of TTN cause DCM, whereas truncations in the I band are better tolerated. Finally, we demonstrate that mutant titin protein in iPS cell–derived cardiomyocytes results in sarcomere insufficiency, impaired responses to mechanical and β-adrenergic stress, and attenuated growth factor and cell signaling activation. Our findings indicate that titin mutations cause DCM by disrupting critical linkages between sarcomerogenesis and adaptive remodeling.', 'year': 2015, 'citationCount': 479, 'authors': [{'authorId': '40393910', 'name': 'J. T. Hinson'}, {'authorId': '4356579', 'name': 'Anant Chopra'}, {'authorId': '2234608653', 'name': 'N. Nafissi'}, {'authorId': '5719705', 'name': 'W. Polacheck'}, {'authorId': '3197049', 'name': 'Craig C. Benson'}, {'authorId': '6786388', 'name': 'Sandra Swist'}, {'authorId': '5444722', 'name': 'Joshua M. Gorham'}, {'authorId': '2967726', 'name': 'Luhan Yang'}, {'authorId': '48327688', 'name': 'S. Schafer'}, {'authorId': '50741905', 'name': 'C. Sheng'}, {'authorId': '2113192', 'name': 'A. Haghighi'}, {'authorId': '36079066', 'name': 'Jason Homsy'}, {'authorId': '5182815', 'name': 'N. Hubner'}, {'authorId': '145892667', 'name': 'G. Church'}, {'authorId': '145502615', 'name': 'S. Cook'}, {'authorId': '2125678', 'name': 'W. Linke'}, {'authorId': '2110195832', 'name': 'Christopher S. Chen'}, {'authorId': '1703476', 'name': 'J. Seidman'}, {'authorId': '5088534', 'name': 'C. Seidman'}, {'authorId': '5088534', 'name': 'C. Seidman'}, {'authorId': '5088534', 'name': 'C. Seidman'}]}, {'paperId': '33e32c7b267441671aab77ac9a0a13cc6e14f3ab', 'title': 'Somatic mutation in single human neurons tracks developmental and transcriptional history', 'abstract': 'Individualized neuronal mutations in the human brain The neurons of the human brain can last for decades, carrying out computational and signaling functions. Lodato et al. analyzed the DNA of individual neurons sampled from postmortem human brains and found that individual neurons acquired somatic mutations (see the Perspective by Linnarsson). The mechanism of mutation involved gene transcription rather than DNA replication. Thus, postmitotic neurons would seem to be their own worst enemy: Genes used for neuronal function are the very genes put most at risk of somatic mutation. Science, this issue p. 94; see also p. 37 Human brains are built from intermingled clones of cells that carry mutations linked to their use of particular neuronal genes. [Also see Perspective by Linnarsson] Neurons live for decades in a postmitotic state, their genomes susceptible to DNA damage. Here we survey the landscape of somatic single-nucleotide variants (SNVs) in the human brain. We identified thousands of somatic SNVs by single-cell sequencing of 36 neurons from the cerebral cortex of three normal individuals. Unlike germline and cancer SNVs, which are often caused by errors in DNA replication, neuronal mutations appear to reflect damage during active transcription. Somatic mutations create nested lineage trees, allowing them to be dated relative to developmental landmarks and revealing a polyclonal architecture of the human cerebral cortex. Thus, somatic mutations in the brain represent a durable and ongoing record of neuronal life history, from development through postmitotic function.', 'year': 2015, 'citationCount': 463, 'authors': [{'authorId': '4760282', 'name': 'Michael A. Lodato'}, {'authorId': '4878598', 'name': 'Mollie B. Woodworth'}, {'authorId': '153310467', 'name': 'Semin Lee'}, {'authorId': '6021345', 'name': 'G. Evrony'}, {'authorId': '48781169', 'name': 'Bhaven K. Mehta'}, {'authorId': '39868617', 'name': 'Amir Karger'}, {'authorId': '2117180651', 'name': 'Soohyun Lee'}, {'authorId': '3075784', 'name': 'T. Chittenden'}, {'authorId': '1398498130', 'name': 'A. D’Gama'}, {'authorId': '34370609', 'name': 'Xuyu Cai'}, {'authorId': '2403971', 'name': 'Lovelace J. Luquette'}, {'authorId': '143790536', 'name': 'E. Lee'}, {'authorId': '2068953915', 'name': 'Peter J. Park'}, {'authorId': '2067645419', 'name': 'C. Walsh'}]}, {'paperId': '4eb6a2628db47351f2d71f21fb6b1520a0082fa3', 'title': 'U-Pb geochronology of the Deccan Traps and relation to the end-Cretaceous mass extinction', 'abstract': 'Dating the influence of Deccan Traps eruptions The Deccan Traps flood basalts in India represent over a million cubic kilometers of erupted lava. These massive eruptions occurred around the same time as the end-Cretaceous mass extinction some 65 million years ago, which famously wiped out all nonavian dinosaurs. Schoene et al. determined the precise timing and duration of the main phase of the eruptions, which lasted over 750,000 years and occurred just 250,000 years before the Cretaceous-Paleogene boundary. The relative contribution of these eruptions and of the Chicxulub impact in Mexico to the mass extinction remains unclear, but both provide potential kill mechanisms. Science, this issue p. 182 The main phase of the Deccan Traps eruption began 250,000 years before the end-Cretaceous extinction and lasted 750,000 years. The Chicxulub asteroid impact (Mexico) and the eruption of the massive Deccan volcanic province (India) are two proposed causes of the end-Cretaceous mass extinction, which includes the demise of nonavian dinosaurs. Despite widespread acceptance of the impact hypothesis, the lack of a high-resolution eruption timeline for the Deccan basalts has prevented full assessment of their relationship to the mass extinction. Here we apply uranium-lead (U-Pb) zircon geochronology to Deccan rocks and show that the main phase of eruptions initiated ~250,000 years before the Cretaceous-Paleogene boundary and that >1.1 million cubic kilometers of basalt erupted in ~750,000 years. Our results are consistent with the hypothesis that the Deccan Traps contributed to the latest Cretaceous environmental change and biologic turnover that culminated in the marine and terrestrial mass extinctions.', 'year': 2015, 'citationCount': 391, 'authors': [{'authorId': '4761805', 'name': 'B. Schoene'}, {'authorId': '7981222', 'name': 'K. Samperton'}, {'authorId': '6210862', 'name': 'M. Eddy'}, {'authorId': '35423404', 'name': 'G. Keller'}, {'authorId': '7769845', 'name': 'T. Adatte'}, {'authorId': '4518129', 'name': 'S. Bowring'}, {'authorId': '15059237', 'name': 'S. Khadri'}, {'authorId': '7453609', 'name': 'B. Gertsch'}]}, {'paperId': '385514ac7a1908821811b6b662cca262bd047c78', 'title': 'Correlated gene expression supports synchronous activity in brain networks', 'abstract': 'Cooperating brain regions express similar genes When the brain is at rest, a number of distinct areas are functionally connected. They tend to be organized in networks. Richiardi et al. compared brain imaging and gene expression data to build computational models of these networks. These functional networks are underpinned by the correlated expression of a core set of 161 genes. In this set, genes coding for ion channels and other synaptic functions such as neurotransmitter release dominate. Science, this issue p. 1241 Gene expression is more similar than expected by chance in brain regions that are functionally connected. During rest, brain activity is synchronized between different regions widely distributed throughout the brain, forming functional networks. However, the molecular mechanisms supporting functional connectivity remain undefined. We show that functional brain networks defined with resting-state functional magnetic resonance imaging can be recapitulated by using measures of correlated gene expression in a post mortem brain tissue data set. The set of 136 genes we identify is significantly enriched for ion channels. Polymorphisms in this set of genes significantly affect resting-state functional connectivity in a large sample of healthy adolescents. Expression levels of these genes are also significantly associated with axonal connectivity in the mouse. The results provide convergent, multimodal evidence that resting-state functional networks correlate with the orchestrated activity of dozens of genes linked to ion channel activity and synaptic function.', 'year': 2015, 'citationCount': 488, 'authors': [{'authorId': '1994765', 'name': 'J. Richiardi'}, {'authorId': '1712994', 'name': 'A. Altmann'}, {'authorId': '50670546', 'name': 'A. Milazzo'}, {'authorId': '2157872630', 'name': 'Catie Chang'}, {'authorId': '2130783982', 'name': 'M. Chakravarty'}, {'authorId': '2131952', 'name': 'T. Banaschewski'}, {'authorId': '2117363497', 'name': 'G. Barker'}, {'authorId': '1932550', 'name': 'A. Bokde'}, {'authorId': '2531464', 'name': 'U. Bromberg'}, {'authorId': '2076663752', 'name': 'C. Büchel'}, {'authorId': '2683022', 'name': 'P. Conrod'}, {'authorId': '1398728945', 'name': 'M. Fauth-Bühler'}, {'authorId': '2072211482', 'name': 'Herta Flor'}, {'authorId': '2470189', 'name': 'V. Frouin'}, {'authorId': '2234167987', 'name': 'Jurgen Gallinat'}, {'authorId': '2943065', 'name': 'H. Garavan'}, {'authorId': '46384297', 'name': 'P. Gowland'}, {'authorId': '2064205252', 'name': 'A. Heinz'}, {'authorId': '2181399', 'name': 'H. Lemaître'}, {'authorId': '2227123074', 'name': 'Karl Mann'}, {'authorId': '2451948', 'name': 'J. Martinot'}, {'authorId': '2700120', 'name': 'F. Nees'}, {'authorId': '2328876', 'name': 'T. Paus'}, {'authorId': '3211950', 'name': 'Z. Pausova'}, {'authorId': '3100404', 'name': 'M. Rietschel'}, {'authorId': '2114808096', 'name': 'T. Robbins'}, {'authorId': '2272108', 'name': 'M. Smolka'}, {'authorId': '3226472', 'name': 'R. Spanagel'}, {'authorId': '2416099', 'name': 'A. Ströhle'}, {'authorId': '1904764', 'name': 'G. Schumann'}, {'authorId': '46760867', 'name': 'M. Hawrylycz'}, {'authorId': '144063147', 'name': 'J. Poline'}, {'authorId': '3266708', 'name': 'M. Greicius'}]}, {'paperId': '4071aac026357483b3a371f8416c0bc5c7e69720', 'title': 'HIV-1 neutralizing antibodies induced by native-like envelope trimers', 'abstract': \"Steps in the right direction HIV-1 mutates rapidly, making it difficult to design a vaccine that will protect people against all of the virus' iterations. A potential successful vaccine design might protect by eliciting broadly neutralizing antibodies (bNAbs), which target specific regions on HIV-1's trimeric envelope glycoprotein (Env) (see the Perspective by Mascola). Jardine et al. used mice engineered to express germline-reverted heavy chains of a particular bNAb and immunized them with an Env-based immunogen designed to bind to precursors of that bNAb. Sanders et al. compared rabbits and monkeys immunized with Env trimers that adopt a nativelike conformation. In both cases, immunized animals produced antibodies that shared similarities with bNAbs. Boosting these animals with other immunogens may drive these antibodies to further mutate into the longsought bNAbs. Chen et al. report that retaining the cytoplasmic domain of Env proteins may be important to attract bNAbs. Removing the cytoplasmic domain may distract the immune response and instead generate antibodies that target epitopes on Env that would not lead to protection. Science, this issue p. 139, 10.1126/science.aac4223, p. 156; see also p. 191 Recombinant, native-like, HIV-1 envelope trimers induce neutralizing antibody responses in animal models. [Also see Perspective by Mascola] INTRODUCTION A major goal of HIV-1 vaccine development is to identify immunogens capable of inducing protective titers of broadly neutralizing antibodies (bNAbs) against circulating, neutralization-resistant (tier 2) viruses. The envelope glycoprotein (Env) trimer on the virus surface is the only bNAb target and accordingly serves as the basis for recombinant protein immunogens intended to induce bNAbs. We have engineered soluble, recombinant trimers based on the BG505 clade A and B41 clade B tier 2 viruses. These proteolytically cleaved and stabilized trimers, designated BG505 and B41 SOSIP.664 gp140, display multiple bNAb epitopes and have antigenic properties that mimic viral Env. High-resolution structures of the BG505 trimers reinforce this perspective. In contrast, other trimer designs (uncleaved gp140) that lack the SOSIP modifications and are based on eliminating the intersubunit proteolytic cleavage site adopt non-native configurations. RATIONALE To determine the NAb response elicited by native-like trimers, we immunized rabbits and macaques with BG505 SOSIP.664 gp140 proteins, and rabbits with their B41 counterparts. For comparison with the BG505 trimers, we also tested gp120 monomers and uncleaved gp140 proteins of the same genotype. We characterized the resulting antibody responses by quantifying their ability to neutralize the autologous tier 2 viruses and multiple heterologous tier 1 (neutralization-sensitive) and tier 2 viruses. We mapped the BG505 NAb responses using various techniques, including a large panel of Env-pseudotyped virus mutants. RESULTS BG505 SOSIP.664 trimers consistently induced potent NAbs against the autologous tier 2 virus in rabbits and similar but weaker responses in macaques. Cross-reactive NAbs against the more sensitive tier 1 viruses were also induced in all the immunized animals, but heterologous neutralization of tier 2 viruses was seen only weakly and sporadically. The rabbit responses to B41 SOSIP.664 trimers were qualitatively similar to those seen in the BG505 trimer recipients. The autologous tier 2 NAb titers to both SOSIP.664 trimers were greater than any such responses previously observed using various uncleaved gp140 proteins, including the BG505 version used in this study. The rabbit tier 2 and tier 1 NAb responses to the BG505 SOSIP.664 trimers were uncorrelated. Mapping studies showed the tier 2 NAbs recognized conformational epitopes that differed between animals. In some cases, these NAbs targeted components of the glycan shield previously associated with bNAb epitopes. In contrast, the tier 1 NAbs targeted linear epitopes in the gp120 V3 region. CONCLUSIONS Although an autologous tier 2 NAb response is not sufficient for vaccine protection against HIV-1, it may be a necessary step in various strategies intended to induce bNAbs. Thus, our findings indicate that native-like trimers represent a promising starting point for the further development of recombinant Env immunogens intended to broaden the NAb response. The high-resolution structures of SOSIP.664 trimers allow improvements to be rationally designed. Relevant strategies include the further stabilization of trimer-associated bNAb epitopes while reducing the antigenicity of V3 and other non-neutralizing epitopes that may be distractive; immunizing with longitudinal series of SOSIP.664 trimers from infected individuals who generate bNAbs; using trimer cocktails (e.g., from clades A, B, and C); and priming with trimer variants that trigger desirable germline responses before boosting with wild-type trimers. Autologous tier 2 NAb responses in animals immunized with native-like (SOSIP.664) trimers or non-native uncleaved gp140s. (Left) Representative reference-free 2D class averages of negative-stain electron microscopy (EM) images of the uncleaved BG505 gp140 and the native-like BG505 SOSIP.664 trimers. The illustrations (orange) of the two categories of Env protein are based on negative-stain EM. (Right) Autologous tier 2 serum NAb titers (IC50) in 50 animals immunized with various uncleaved gp140 proteins, compared to titers in the 30 rabbits given the BG505 or B41 SOSIP.664 trimers described here (see legend of Fig. 2 in the full article for details; P < 0.0001, two-tailed Mann-Whitney test). A challenge for HIV-1 immunogen design is the difficulty of inducing neutralizing antibodies (NAbs) against neutralization-resistant (tier 2) viruses that dominate human transmissions. We show that a soluble recombinant HIV-1 envelope glycoprotein trimer that adopts a native conformation, BG505 SOSIP.664, induced NAbs potently against the sequence-matched tier 2 virus in rabbits and similar but weaker responses in macaques. The trimer also consistently induced cross-reactive NAbs against more sensitive (tier 1) viruses. Tier 2 NAbs recognized conformational epitopes that differed between animals and in some cases overlapped with those recognized by broadly neutralizing antibodies (bNAbs), whereas tier 1 responses targeted linear V3 epitopes. A second trimer, B41 SOSIP.664, also induced a strong autologous tier 2 NAb response in rabbits. Thus, native-like trimers represent a promising starting point for the development of HIV-1 vaccines aimed at inducing bNAbs.\", 'year': 2015, 'citationCount': 473, 'authors': [{'authorId': '11253121', 'name': 'R. Sanders'}, {'authorId': '1753122', 'name': 'M. V. van Gils'}, {'authorId': '6913448', 'name': 'R. Derking'}, {'authorId': '4498862', 'name': 'D. Sok'}, {'authorId': '3532587', 'name': 'T. Ketas'}, {'authorId': '4224266', 'name': 'J. Burger'}, {'authorId': '5403524', 'name': 'G. Ozorowski'}, {'authorId': '3720967', 'name': 'A. Cupo'}, {'authorId': '5582265', 'name': 'Cassandra Simonich'}, {'authorId': '49823402', 'name': 'Leslie Goo'}, {'authorId': '40190515', 'name': 'H. Arendt'}, {'authorId': '2109106668', 'name': 'Helen J. Kim'}, {'authorId': '2155633982', 'name': 'Jeong Hyun Lee'}, {'authorId': '5996931', 'name': 'P. Pugach'}, {'authorId': '47638142', 'name': 'Melissa Williams'}, {'authorId': '8115982', 'name': 'Gargi Debnath'}, {'authorId': '4251219', 'name': 'B. Moldt'}, {'authorId': '114511784', 'name': 'M. V. van Breemen'}, {'authorId': '3550089', 'name': 'G. Işık'}, {'authorId': '1398898273', 'name': 'M. Medina-Ramírez'}, {'authorId': '6295820', 'name': 'J. Back'}, {'authorId': '4282690', 'name': 'W. Koff'}, {'authorId': '143931879', 'name': 'J. Julien'}, {'authorId': '4012471', 'name': 'E. Rakasz'}, {'authorId': '2785079', 'name': 'M. Seaman'}, {'authorId': '34071496', 'name': 'M. Guttman'}, {'authorId': '1574068297', 'name': 'Kelly K. Lee'}, {'authorId': '4064627', 'name': 'P. Klasse'}, {'authorId': '4625245', 'name': 'C. Labranche'}, {'authorId': '2417923', 'name': 'W. Schief'}, {'authorId': '2069848127', 'name': 'I. Wilson'}, {'authorId': '5093157', 'name': 'J. Overbaugh'}, {'authorId': '144482154', 'name': 'D. Burton'}, {'authorId': '144621454', 'name': 'A. Ward'}, {'authorId': '144524278', 'name': 'D. Montefiori'}, {'authorId': '34969108', 'name': 'H. Dean'}, {'authorId': '1405933166', 'name': 'John P. Moore'}]}, {'paperId': '4b66f21a5d0bb8618f8cbb0eae90110d1cc05c0e', 'title': 'Negative local resistance caused by viscous electron backflow in graphene', 'abstract': 'Electrons that flow like a fluid Electrons inside a conductor are often described as flowing in response to an electric field. This flow rarely resembles anything like the familiar flow of water through a pipe, but three groups describe counterexamples (see the Perspective by Zaanen). Moll et al. found that the viscosity of the electron fluid in thin wires of PdCoO2 had a major effect on the flow, much like what happens in regular fluids. Bandurin et al. found evidence in graphene of electron whirlpools similar to those formed by viscous fluid flowing through a small opening. Finally, Crossno et al. observed a huge increase of thermal transport in graphene, a signature of so-called Dirac fluids. Science, this issue p. 1061, 1055, 1058; see also p. 1026 Proximity transport measurements find evidence of whirlpools of graphene’s viscous electronic fluid. [Also see Perspective by Zaanen] Graphene hosts a unique electron system in which electron-phonon scattering is extremely weak but electron-electron collisions are sufficiently frequent to provide local equilibrium above the temperature of liquid nitrogen. Under these conditions, electrons can behave as a viscous liquid and exhibit hydrodynamic phenomena similar to classical liquids. Here we report strong evidence for this transport regime. We found that doped graphene exhibits an anomalous (negative) voltage drop near current-injection contacts, which is attributed to the formation of submicrometer-size whirlpools in the electron flow. The viscosity of graphene’s electron liquid is found to be ~0.1 square meters per second, an order of magnitude higher than that of honey, in agreement with many-body theory. Our work demonstrates the possibility of studying electron hydrodynamics using high-quality graphene.', 'year': 2015, 'citationCount': 496, 'authors': [{'authorId': '8368425', 'name': 'D. Bandurin'}, {'authorId': '41205715', 'name': 'I. Torre'}, {'authorId': '31912825', 'name': 'R. K. Kumar'}, {'authorId': '102321688', 'name': 'M. Shalom'}, {'authorId': '4434710', 'name': 'A. Tomadin'}, {'authorId': '3992470', 'name': 'A. Principi'}, {'authorId': '6194428', 'name': 'G. Auton'}, {'authorId': '4352409', 'name': 'E. Khestanova'}, {'authorId': '49912197', 'name': 'K. Novoselov'}, {'authorId': '5224749', 'name': 'I. Grigorieva'}, {'authorId': '1994032', 'name': 'L. Ponomarenko'}, {'authorId': '5288936', 'name': 'SUPARNA DUTTASINHA'}, {'authorId': '4943878', 'name': 'M. Polini'}]}, {'paperId': 'ff04a23ed941f8829db26e23db81efb6e99423f1', 'title': 'SARM1 activation triggers axon degeneration locally via NAD+ destruction', 'abstract': 'SARM1-driven axon degeneration Axons, the long protrusions of nerve cells, are programmed to self-destruct under certain conditions that occur during development, stress, or disease states. Gerdts et al. outline a biochemical mechanism that controls such axon degeneration. The authors designed versions of SARM1 (sterile alpha and TIR motif—constraining 1) that could be activated or inhibited in cells. Their experiments showed that the activation of SARM1 was necessary and sufficient to cause axon destruction in cultured mouse neurons. SARM1-mediated destruction was associated with depletion of the metabolic cofactor NAD+ from cells. Science, this issue p. 453 A critical role is elucidated for destruction of a metabolic cofactor in neuronal degeneration. Axon degeneration is an intrinsic self-destruction program that underlies axon loss during injury and disease. Sterile alpha and TIR motif–containing 1 (SARM1) protein is an essential mediator of axon degeneration. We report that SARM1 initiates a local destruction program involving rapid breakdown of nicotinamide adenine dinucleotide (NAD+) after injury. We used an engineered protease-sensitized SARM1 to demonstrate that SARM1 activity is required after axon injury to induce axon degeneration. Dimerization of the Toll–interleukin receptor (TIR) domain of SARM1 alone was sufficient to induce locally mediated axon degeneration. Formation of the SARM1 TIR dimer triggered rapid breakdown of NAD+, whereas SARM1-induced axon destruction could be counteracted by increased NAD+ synthesis. SARM1-induced depletion of NAD+ may explain the potent axon protection in Wallerian degeneration slow (Wlds) mutant mice.', 'year': 2015, 'citationCount': 410, 'authors': [{'authorId': '40641371', 'name': 'Josiah Gerdts'}, {'authorId': '3987255', 'name': 'E. Brace'}, {'authorId': '48346296', 'name': 'Y. Sasaki'}, {'authorId': '4204986', 'name': 'A. Diantonio'}, {'authorId': '2178644', 'name': 'J. Milbrandt'}]}, {'paperId': '2649ec70d571c681cb2677f242d8445dfda69f32', 'title': 'Early Homo at 2.8 Ma from Ledi-Geraru, Afar, Ethiopia', 'abstract': 'Finding Homo nearly 3 million years ago The fossil record of humans is notoriously patchy and incomplete. Even so, skeletal remains and artifacts unearthed in Africa in recent decades have done much to illuminate human evolution. But what is the origin of the genus Homo? Villmoare et al. found a fossil mandible and teeth from the Afar region in Ethiopia. The find extends the record of recognizable Homo by at least half a million years, to almost 2.8 million years ago. The morphological traits of the fossil align more closely with Homo than with any other hominid genus. DiMaggio et al. confirm the ancient date of the site and suggest that these early humans lived in a setting that was more open and arid than previously thought. Science, this issue p. 1352, p. 1355 A fossil mandible and teeth from Ethiopia extend the record of the genus Homo by almost 0.5 million years. Our understanding of the origin of the genus Homo has been hampered by a limited fossil record in eastern Africa between 2.0 and 3.0 million years ago (Ma). Here we report the discovery of a partial hominin mandible with teeth from the Ledi-Geraru research area, Afar Regional State, Ethiopia, that establishes the presence of Homo at 2.80 to 2.75 Ma. This specimen combines primitive traits seen in early Australopithecus with derived morphology observed in later Homo, confirming that dentognathic departures from the australopith pattern occurred early in the Homo lineage. The Ledi-Geraru discovery has implications for hypotheses about the timing and place of origin of the genus Homo.', 'year': 2015, 'citationCount': 304, 'authors': [{'authorId': '3856850', 'name': 'John R. Tumbleston'}, {'authorId': '2888642', 'name': 'D. Shirvanyants'}, {'authorId': '4737061', 'name': 'N. Ermoshkin'}, {'authorId': '5768526', 'name': 'R. Janusziewicz'}, {'authorId': '25711869', 'name': 'Ashley R. Johnson'}, {'authorId': '2070074460', 'name': 'David Kelly'}, {'authorId': '2205902206', 'name': 'Kai-Feng Chen'}, {'authorId': '69923437', 'name': 'R. Pinschmidt'}, {'authorId': '1850765', 'name': 'J. Rolland'}, {'authorId': '2661307', 'name': 'A. Ermoshkin'}, {'authorId': '49805613', 'name': 'E. Samulski'}, {'authorId': '6880755', 'name': 'J. Desimone'}]}, {'paperId': 'da7c38b5db28486f6027ea7a4f142eb742a3df70', 'title': 'Genomic evidence for the Pleistocene and recent population history of Native Americans', 'abstract': 'Genetic history of Native Americans Several theories have been put forth as to the origin and timing of when Native American ancestors entered the Americas. To clarify this controversy, Raghavan et al. examined the genomic variation among ancient and modern individuals from Asia and the Americas. There is no evidence for multiple waves of entry or recurrent gene flow with Asians in northern populations. The earliest migrations occurred no earlier than 23,000 years ago from Siberian ancestors. Amerindians and Athabascans originated from a single population, splitting approximately 13,000 years ago. Science, this issue 10.1126/science.aab3884 Genetic variation within ancient and extant Native American populations informs on their migration into the Americas. INTRODUCTION The consensus view on the peopling of the Americas is that ancestors of modern Native Americans entered the Americas from Siberia via the Bering Land Bridge and that this occurred at least ~14.6 thousand years ago (ka). However, the number and timing of migrations into the Americas remain controversial, with conflicting interpretations based on anatomical and genetic evidence. RATIONALE In this study, we address four major unresolved issues regarding the Pleistocene and recent population history of Native Americans: (i) the timing of their divergence from their ancestral group, (ii) the number of migrations into the Americas, (iii) whether there was ~15,000 years of isolation of ancestral Native Americans in Beringia (Beringian Incubation Model), and (iv) whether there was post-Pleistocene survival of relict populations in the Americas related to Australo-Melanesians, as suggested by apparent differences in cranial morphologies between some early (“Paleoamerican”) remains and those of more recent Native Americans. We generated 31 high-coverage modern genomes from the Americas, Siberia, and Oceania; 23 ancient genomic sequences from the Americas dating between ~0.2 and 6 ka; and SNP chip genotype data from 79 present-day individuals belonging to 28 populations from the Americas and Siberia. The above data sets were analyzed together with published modern and ancient genomic data from worldwide populations, after masking some present-day Native Americans for recent European admixture. RESULTS Using three different methods, we determined the divergence time for all Native Americans (Athabascans and Amerindians) from their Siberian ancestors to be ~20 ka, and no earlier than ~23 ka. Furthermore, we dated the divergence between Athabascans (northern Native American branch, together with northern North American Amerindians) and southern North Americans and South and Central Americans (southern Native American branch) to be ~13 ka. Similar divergence times from East Asian populations and a divergence time between the two branches that is close in age to the earliest well-established archaeological sites in the Americas suggest that the split between the branches occurred within the Americas. We additionally found that several sequenced Holocene individuals from the Americas are related to present-day populations from the same geographical regions, implying genetic continuity of ancient and modern populations in some parts of the Americas over at least the past 8500 years. Moreover, our results suggest that there has been gene flow between some Native Americans from both North and South America and groups related to East Asians and Australo-Melanesians, the latter possibly through an East Asian route that might have included ancestors of modern Aleutian Islanders. Last, using both genomic and morphometric analyses, we found that historical Native American groups such as the Pericúes and Fuego-Patagonians were not “relicts” of Paleoamericans, and hence, our results do not support an early migration of populations directly related to Australo-Melanesians into the Americas. CONCLUSION Our results provide an upper bound of ~23 ka on the initial divergence of ancestral Native Americans from their East Asian ancestors, followed by a short isolation period of no more than ~8000 years, and subsequent entrance and spread across the Americas. The data presented are consistent with a single-migration model for all Native Americans, with later gene flow from sources related to East Asians and, indirectly, Australo-Melanesians. The single wave diversified ~13 ka, likely within the Americas, giving rise to the northern and southern branches of present-day Native Americans. Population history of present-day Native Americans. The ancestors of all Native Americans entered the Americas as a single migration wave from Siberia (purple) no earlier than ~23 ka, separate from the Inuit (green), and diversified into “northern” and “southern” Native American branches ~13 ka. There is evidence of post-divergence gene flow between some Native Americans and groups related to East Asians/Inuit and Australo-Melanesians (yellow). How and when the Americas were populated remains contentious. Using ancient and modern genome-wide data, we found that the ancestors of all present-day Native Americans, including Athabascans and Amerindians, entered the Americas as a single migration wave from Siberia no earlier than 23 thousand years ago (ka) and after no more than an 8000-year isolation period in Beringia. After their arrival to the Americas, ancestral Native Americans diversified into two basal genetic branches around 13 ka, one that is now dispersed across North and South America and the other restricted to North America. Subsequent gene flow resulted in some Native Americans sharing ancestry with present-day East Asians (including Siberians) and, more distantly, Australo-Melanesians. Putative “Paleoamerican” relict populations, including the historical Mexican Pericúes and South American Fuego-Patagonians, are not directly related to modern Australo-Melanesians as suggested by the Paleoamerican Model.', 'year': 2015, 'citationCount': 443, 'authors': [{'authorId': '40300212', 'name': 'M. Raghavan'}, {'authorId': '3376543', 'name': 'Matthias Steinrücken'}, {'authorId': '31712306', 'name': 'Kelley Harris'}, {'authorId': '6020526', 'name': 'S. Schiffels'}, {'authorId': '49743174', 'name': 'S. Rasmussen'}, {'authorId': '2087800', 'name': 'Michael Degiorgio'}, {'authorId': '2628454', 'name': 'A. Albrechtsen'}, {'authorId': '2249014', 'name': 'Cristina E. Valdiosera'}, {'authorId': '1404276333', 'name': 'María C. Ávila-Arcos'}, {'authorId': '2491508', 'name': 'Anna-Sapfo Malaspinas'}, {'authorId': '144213877', 'name': 'A. Eriksson'}, {'authorId': '2318765', 'name': 'Ida Moltke'}, {'authorId': '5369838', 'name': 'M. Metspalu'}, {'authorId': '5091885', 'name': 'J. Homburger'}, {'authorId': '152921524', 'name': 'J. Wall'}, {'authorId': '2588473', 'name': 'Omar E. Cornejo'}, {'authorId': '1398610269', 'name': 'J. V. Moreno-Mayar'}, {'authorId': '2806324', 'name': 'T. Korneliussen'}, {'authorId': '4268632', 'name': 'T. Pierre'}, {'authorId': '153154198', 'name': 'M. Rasmussen'}, {'authorId': '34783432', 'name': 'P. Campos'}, {'authorId': '21107284', 'name': 'P. Damgaard'}, {'authorId': '3878367', 'name': 'M. Allentoft'}, {'authorId': '4990987', 'name': 'J. Lindo'}, {'authorId': '6160843', 'name': 'E. Metspalu'}, {'authorId': '1398719145', 'name': 'Ricardo Rodríguez-Varela'}, {'authorId': '145539929', 'name': 'J. Mansilla'}, {'authorId': '17599631', 'name': 'Celeste N. Henrickson'}, {'authorId': '1398045956', 'name': 'Andaine Seguin-Orlando'}, {'authorId': '5283621', 'name': 'H. Malmström'}, {'authorId': '32687097', 'name': 'T. Stafford'}, {'authorId': '1472926470', 'name': 'S. Shringarpure'}, {'authorId': '1398672485', 'name': 'A. Moreno-Estrada'}, {'authorId': '2038457', 'name': 'Monika Karmin'}, {'authorId': '5549881', 'name': 'K. Tambets'}, {'authorId': '47269529', 'name': 'Anders Bergström'}, {'authorId': '33496542', 'name': 'Yali Xue'}, {'authorId': '3584737', 'name': 'Vera M. Warmuth'}, {'authorId': '104308183', 'name': 'A. Friend'}, {'authorId': '4443483', 'name': 'J. Singarayer'}, {'authorId': '2948218', 'name': 'P. Valdes'}, {'authorId': '2056491', 'name': 'F. Balloux'}, {'authorId': '6924143', 'name': 'I. Leboreiro'}, {'authorId': '30012904', 'name': 'J. Vera'}, {'authorId': '1398119682', 'name': 'H. Rangel-Villalobos'}, {'authorId': '4674234', 'name': 'D. Pettener'}, {'authorId': '6904055', 'name': 'D. Luiselli'}, {'authorId': '104380079', 'name': 'L. Davis'}, {'authorId': '145027324', 'name': 'E. Heyer'}, {'authorId': '35251830', 'name': 'C. Zollikofer'}, {'authorId': '4389705', 'name': 'Marcia S. Ponce de León'}, {'authorId': '2135808573', 'name': 'Colin Smith'}, {'authorId': '4384505', 'name': 'V. Grimes'}, {'authorId': '46634137', 'name': 'Kelly A. Pike'}, {'authorId': '81782215', 'name': 'M. Deal'}, {'authorId': '47655297', 'name': 'B. Fuller'}, {'authorId': '50603710', 'name': 'B. Arriaza'}, {'authorId': '4607947', 'name': 'V. Standen'}, {'authorId': '120016142', 'name': 'Maria F. Luz'}, {'authorId': '47193911', 'name': 'F. Ricaut'}, {'authorId': '8307737', 'name': 'N. Guidon'}, {'authorId': '3529431', 'name': 'L. Osipova'}, {'authorId': '6297765', 'name': 'M. Voevoda'}, {'authorId': '5537046', 'name': 'O. Posukh'}, {'authorId': '144071337', 'name': 'O. Balanovsky'}, {'authorId': '3755286', 'name': 'M. Lavryashina'}, {'authorId': '4400741', 'name': 'Y. Bogunov'}, {'authorId': '145568845', 'name': 'E. Khusnutdinova'}, {'authorId': '144314334', 'name': 'M. Gubina'}, {'authorId': '3869128', 'name': 'E. Balanovska'}, {'authorId': '144749924', 'name': 'S. Fedorova'}, {'authorId': '144633425', 'name': 'S. Litvinov'}, {'authorId': '2640106', 'name': 'B. Malyarchuk'}, {'authorId': '6149225', 'name': 'M. Derenko'}, {'authorId': '34533721', 'name': 'M. Mosher'}, {'authorId': '2056167967', 'name': 'David R. Archer'}, {'authorId': '5639635', 'name': 'J. Cybulski'}, {'authorId': '4819572', 'name': 'Barbara Petzelt'}, {'authorId': '40215642', 'name': 'Joycelynn Mitchell'}, {'authorId': '6246512', 'name': 'Rosita Worl'}, {'authorId': '2754682', 'name': 'P. Norman'}, {'authorId': '31740540', 'name': 'P. Parham'}, {'authorId': '5324079', 'name': 'B. Kemp'}, {'authorId': '8019318', 'name': 'T. Kivisild'}, {'authorId': '1396409355', 'name': 'C. Tyler-Smith'}, {'authorId': '2664570', 'name': 'M. Sandhu'}, {'authorId': '2066467070', 'name': 'Michael Crawford'}, {'authorId': '6637782', 'name': 'R. Villems'}, {'authorId': '38129056', 'name': 'D. G. Smith'}, {'authorId': '47743436', 'name': 'M. Waters'}, {'authorId': '46860177', 'name': 'T. Goebel'}, {'authorId': '145368911', 'name': 'John R. Johnson'}, {'authorId': '4155430', 'name': 'R. Malhi'}, {'authorId': '2827710', 'name': 'M. Jakobsson'}, {'authorId': '47324013', 'name': 'D. Meltzer'}, {'authorId': '3277575', 'name': 'A. Manica'}, {'authorId': '144187514', 'name': 'R. Durbin'}, {'authorId': '33858917', 'name': 'C. Bustamante'}, {'authorId': '145229109', 'name': 'Yun S. Song'}, {'authorId': '144471919', 'name': 'R. Nielsen'}, {'authorId': '1812266', 'name': 'E. Willerslev'}]}, {'paperId': '4a0a3f003a75b908f1ae7933a0a0d8fce6c60969', 'title': '2.2 Å resolution cryo-EM structure of β-galactosidase in complex with a cell-permeant inhibitor', 'abstract': 'Pushing the limits of electron microscopy Recent advances in cryo–electron microscopy (cryo-EM) allow structures of large macromolecules to be determined at near-atomic resolution. So far, though, resolutions approaching 2 Å, where features key to drug design are revealed, remain the province of x-ray crystallography. Bartesaghi et al. achieved a resolution of 2.2 Å for a 465-kD ligand-bound protein complex using cryo-EM. The density map is detailed enough to show close to 800 water molecules, magnesium and sodium ions, and precise side-chain conformations. These results bring routine use of cryo-EM in rational drug design a step closer. Science, this issue p. 1147 Advances in electron microscopy allow protein structure determination at resolutions useful in drug discovery. Cryo–electron microscopy (cryo-EM) is rapidly emerging as a powerful tool for protein structure determination at high resolution. Here we report the structure of a complex between Escherichia coli β-galactosidase and the cell-permeant inhibitor phenylethyl β-d-thiogalactopyranoside (PETG), determined by cryo-EM at an average resolution of ~2.2 angstroms (Å). Besides the PETG ligand, we identified densities in the map for ~800 water molecules and for magnesium and sodium ions. Although it is likely that continued advances in detector technology may further enhance resolution, our findings demonstrate that preparation of specimens of adequate quality and intrinsic protein flexibility, rather than imaging or image-processing technologies, now represent the major bottlenecks to routinely achieving resolutions close to 2 Å using single-particle cryo-EM.', 'year': 2015, 'citationCount': 439, 'authors': [{'authorId': '1920144', 'name': 'A. Bartesaghi'}, {'authorId': '6278602', 'name': 'A. Merk'}, {'authorId': '153564608', 'name': 'Soojay Banerjee'}, {'authorId': '31544014', 'name': 'D. Matthies'}, {'authorId': '2944941', 'name': 'Xiongwu Wu'}, {'authorId': '48705866', 'name': 'J. Milne'}, {'authorId': '145690187', 'name': 'S. Subramaniam'}]}, {'paperId': 'bc4ff07b48d8033a1a00e6c33753dcf90cde9ea2', 'title': 'Gravitational waves from binary supermassive black holes missing in pulsar observations', 'abstract': 'Placing bounds on gravitational wave detection Gravitational waves are expected to be generated by the interaction of the massive bodies in black-hole binary systems. As gravitational waves distort spacetime, it should be possible to verify their existence as they interfere with the pulses emitted by millisecond pulsars. However, after monitoring 24 pulsars with the Parkes radio telescope for 12 years, Shannon et al. found no detectable variation in pulsar records. This nondetection result indicates that a new detection strategy for gravitational waves is needed. Science, this issue p. 1522 A lack of observed variations in the timing of pulsars places constraints on the detection of gravitational waves. Gravitational waves are expected to be radiated by supermassive black hole binaries formed during galaxy mergers. A stochastic superposition of gravitational waves from all such binary systems would modulate the arrival times of pulses from radio pulsars. Using observations of millisecond pulsars obtained with the Parkes radio telescope, we constrained the characteristic amplitude of this background, Ac,yr, to be <1.0 × 10−15 with 95% confidence. This limit excludes predicted ranges for Ac,yr from current models with 91 to 99.7% probability. We conclude that binary evolution is either stalled or dramatically accelerated by galactic-center environments and that higher-cadence and shorter-wavelength observations would be more sensitive to gravitational waves.', 'year': 2015, 'citationCount': 358, 'authors': [{'authorId': '49037918', 'name': 'R. Shannon'}, {'authorId': '2089367767', 'name': 'Ravi'}, {'authorId': '5872642', 'name': 'L. Lentati'}, {'authorId': '1381303595', 'name': 'P. Lasky'}, {'authorId': '145913658', 'name': 'G. Hobbs'}, {'authorId': '39343524', 'name': 'M. Kerr'}, {'authorId': '4701856', 'name': 'R. Manchester'}, {'authorId': '4919284', 'name': 'W. Coles'}, {'authorId': '123786554', 'name': 'Y. Levin'}, {'authorId': '5903194', 'name': 'M. Bailes'}, {'authorId': '2229714046', 'name': 'Ndr Bhat'}, {'authorId': '1388067048', 'name': 'S. Burke-Spolaor'}, {'authorId': '2087720656', 'name': 'Shi Dai'}, {'authorId': '49918437', 'name': 'M. Keith'}, {'authorId': '97646607', 'name': 'S. Osłowski'}, {'authorId': '47314221', 'name': 'D. Reardon'}, {'authorId': '2229712953', 'name': 'W. V. Strateni'}, {'authorId': '39700583', 'name': 'L. Toomey'}, {'authorId': '97773615', 'name': 'J. Wang'}, {'authorId': '144731348', 'name': 'L. Wen'}, {'authorId': '93456160', 'name': 'J. Wyithe'}, {'authorId': '6248124', 'name': 'Xiaoshuai Zhu'}]}, {'paperId': 'a2eb1a77dd5233286045e8bafeffbc6d158f15cb', 'title': 'Two-pore channels control Ebola virus host cell entry and are drug targets for disease treatment', 'abstract': \"Channeling Ebola virus entry into the cell The current outbreak of Ebola virus in West Africa highlights the need for antiviral therapies. One strategy would be to block the Ebola virus's ability to enter host cells. Cells engulf Ebola virus particles, which then traffic into the cell in structures called endosomes. Sakurai et al. now report that the Ebola virus requires calcium channels called two-pore channels (TPCs) in endosomal membranes for successful entry (see the Perspective by Falzarano and Feldmann). The Ebola virus could not enter cells lacking TPCs or cells treated with a TPC inhibitor. Blocking TPCs therapeutically allowed 50% of mice to survive an ordinarily lethal Ebola virus infection. Science, this issue p. 995; see also p. 947 Ebola virus requires endosomal two-pore calcium channels to enter host cells. [Also see Perspective by Falzarano and Feldmann] Ebola virus causes sporadic outbreaks of lethal hemorrhagic fever in humans, but there is no currently approved therapy. Cells take up Ebola virus by macropinocytosis, followed by trafficking through endosomal vesicles. However, few factors controlling endosomal virus movement are known. Here we find that Ebola virus entry into host cells requires the endosomal calcium channels called two-pore channels (TPCs). Disrupting TPC function by gene knockout, small interfering RNAs, or small-molecule inhibitors halted virus trafficking and prevented infection. Tetrandrine, the most potent small molecule that we tested, inhibited infection of human macrophages, the primary target of Ebola virus in vivo, and also showed therapeutic efficacy in mice. Therefore, TPC proteins play a key role in Ebola virus infection and may be effective targets for antiviral therapy.\", 'year': 2015, 'citationCount': 409, 'authors': [{'authorId': '6471111', 'name': 'Y. Sakurai'}, {'authorId': '4367895', 'name': 'A. Kolokoltsov'}, {'authorId': '48239565', 'name': 'Cheng-Chang Chen'}, {'authorId': '46319361', 'name': 'Michael W Tidwell'}, {'authorId': '8610145', 'name': 'William Bauta'}, {'authorId': '4158787', 'name': 'N. Klugbauer'}, {'authorId': '2075378825', 'name': 'C. Grimm'}, {'authorId': '1398097274', 'name': 'C. Wahl-Schott'}, {'authorId': '144893002', 'name': 'M. Biel'}, {'authorId': '144524415', 'name': 'R. Davey'}]}, {'paperId': '16db92f38e2c33f5902d30a233d57e12bce69bda', 'title': 'The structure of the dynactin complex and its interaction with dynein', 'abstract': 'Making a molecular motor fit for purpose Dynactin is an essential cofactor of the microtubule motor, cytoplasmic dynein. Dynactin contains 23 subunits built around a short filament of an actin-related protein (Arp1). How dynactin is assembled, how it functions with dynein, and why it is built around an actin-like filament is unclear. Urnavicius et al. combined cryo–electron microscopy structural studies and a crystal structure to determine the three-dimensional architecture of dynactin and how it interacts with dynein. Science, this issue p. 1441 Cryo–electron microscopy structures reveal how the 23-subunit dynactin complex is assembled and binds to the motor protein cytoplasmic dynein-1. Dynactin is an essential cofactor for the microtubule motor cytoplasmic dynein-1. We report the structure of the 23-subunit dynactin complex by cryo-electron microscopy to 4.0 angstroms. Our reconstruction reveals how dynactin is built around a filament containing eight copies of the actin-related protein Arp1 and one of β-actin. The filament is capped at each end by distinct protein complexes, and its length is defined by elongated peptides that emerge from the α-helical shoulder domain. A further 8.2 angstrom structure of the complex between dynein, dynactin, and the motility-inducing cargo adaptor Bicaudal-D2 shows how the translational symmetry of the dynein tail matches that of the dynactin filament. The Bicaudal-D2 coiled coil runs between dynein and dynactin to stabilize the mutually dependent interactions between all three components.', 'year': 2015, 'citationCount': 344, 'authors': [{'authorId': '5205544', 'name': 'L. Urnavicius'}, {'authorId': '2158519970', 'name': 'Kai Zhang'}, {'authorId': '3639450', 'name': 'A. Diamant'}, {'authorId': '46504814', 'name': 'C. Motz'}, {'authorId': '6344076', 'name': 'Max A. Schlager'}, {'authorId': '47730624', 'name': 'Minmin Yu'}, {'authorId': '91998801', 'name': 'Nisha A. Patel'}, {'authorId': '144199447', 'name': 'C. Robinson'}, {'authorId': '31454155', 'name': 'A. Carter'}]}, {'paperId': '05fdf5bd17f6bebf0e478311d2874ef8012c47d0', 'title': 'On the nucleus structure and activity of comet 67P/Churyumov-Gerasimenko', 'abstract': 'Images from the OSIRIS scientific imaging system onboard Rosetta show that the nucleus of 67P/Churyumov-Gerasimenko consists of two lobes connected by a short neck. The nucleus has a bulk density less than half that of water. Activity at a distance from the Sun of >3 astronomical units is predominantly from the neck, where jets have been seen consistently. The nucleus rotates about the principal axis of momentum. The surface morphology suggests that the removal of larger volumes of material, possibly via explosive release of subsurface pressure or via creation of overhangs by sublimation, may be a major mass loss process. The shape raises the question of whether the two lobes represent a contact binary formed 4.5 billion years ago, or a single body where a gap has evolved via mass loss.', 'year': 2015, 'citationCount': 386, 'authors': [{'authorId': '50246256', 'name': 'H. Sierks'}, {'authorId': '39419736', 'name': 'C. Barbieri'}, {'authorId': '50403997', 'name': 'P. Lamy'}, {'authorId': '91031514', 'name': 'R. Rodrigo'}, {'authorId': '69029744', 'name': 'D. Koschny'}, {'authorId': '6973900', 'name': 'H. Rickman'}, {'authorId': '144446632', 'name': 'H. Keller'}, {'authorId': '145969939', 'name': 'J. Agarwal'}, {'authorId': '1398024389', 'name': 'M. A’Hearn'}, {'authorId': '48286249', 'name': 'F. Angrilli'}, {'authorId': '102857838', 'name': 'A. Auger'}, {'authorId': '2070612652', 'name': 'M. Barucci'}, {'authorId': '102414410', 'name': 'J. Bertaux'}, {'authorId': '47679133', 'name': 'I. Bertini'}, {'authorId': '47071682', 'name': 'S. Besse'}, {'authorId': '6691955', 'name': 'D. Bodewits'}, {'authorId': '2721300', 'name': 'C. Capanna'}, {'authorId': '3939528', 'name': 'G. Cremonese'}, {'authorId': '2896324', 'name': 'V. Da Deppo'}, {'authorId': '3984918', 'name': 'B. Davidsson'}, {'authorId': '2317823', 'name': 'S. Debei'}, {'authorId': '50129229', 'name': 'M. De Cecco'}, {'authorId': '46341917', 'name': 'F. Ferri'}, {'authorId': '2564082', 'name': 'S. Fornasier'}, {'authorId': '14100462', 'name': 'M. Fulle'}, {'authorId': '4755964', 'name': 'R. Gaskell'}, {'authorId': '2215595', 'name': 'L. Giacomini'}, {'authorId': '3155836', 'name': 'O. Groussin'}, {'authorId': '1403005524', 'name': 'P. Gutierrez-Marques'}, {'authorId': '49672000', 'name': 'P. Gutiérrez'}, {'authorId': '3354765', 'name': 'C. Güttler'}, {'authorId': '4257859', 'name': 'N. Hoekzema'}, {'authorId': '5303188', 'name': 'S. Hviid'}, {'authorId': '145155207', 'name': 'W. Ip'}, {'authorId': '3240472', 'name': 'L. Jorda'}, {'authorId': '102231574', 'name': 'J. Knollenberg'}, {'authorId': '134014421', 'name': 'G. Kovacs'}, {'authorId': '143828408', 'name': 'J. Kramm'}, {'authorId': '2250203', 'name': 'E. Kührt'}, {'authorId': '48974673', 'name': 'M. Küppers'}, {'authorId': '133961675', 'name': 'F. La Forgia'}, {'authorId': '145621605', 'name': 'L. Lara'}, {'authorId': '144479967', 'name': 'M. Lazzarin'}, {'authorId': '5077738', 'name': 'C. Leyrat'}, {'authorId': '114954861', 'name': 'J. Lopez Moreno'}, {'authorId': '6016807', 'name': 'S. Magrin'}, {'authorId': '50786404', 'name': 'S. Marchi'}, {'authorId': '144372011', 'name': 'F. Marzari'}, {'authorId': '2761573', 'name': 'M. Massironi'}, {'authorId': '144921681', 'name': 'H. Michalik'}, {'authorId': '5261364', 'name': 'R. Moissl'}, {'authorId': '144381122', 'name': 'S. Mottola'}, {'authorId': '2704601', 'name': 'G. Naletto'}, {'authorId': '5300286', 'name': 'N. Oklay'}, {'authorId': '1398024053', 'name': 'M. Pajola'}, {'authorId': '35197843', 'name': 'M. Pertile'}, {'authorId': '4614024', 'name': 'F. Preusker'}, {'authorId': '143930936', 'name': 'L. Sabau'}, {'authorId': '143715006', 'name': 'F. Scholten'}, {'authorId': '2155760', 'name': 'C. Snodgrass'}, {'authorId': '144570575', 'name': 'N. Thomas'}, {'authorId': '5653933', 'name': 'C. Tubiana'}, {'authorId': '12647806', 'name': 'J. Vincent'}, {'authorId': '114386622', 'name': 'K. Wenzel'}, {'authorId': '2592622', 'name': 'M. Zaccariotto'}, {'authorId': '2632184', 'name': 'M. Pätzold'}]}, {'paperId': 'a62607e79840a2ca96508794944b864beb9a166d', 'title': 'Adaptive synergy between catechol and lysine promotes wet adhesion by surface salt displacement', 'abstract': 'Keeping it sticky when wet Some biological molecules are remarkably sticky, even to surfaces submerged in water. Mussel adhesion, for example, is based on the overproduction of dihydroxyphenylalanine (DOPA) and proteins with a high abundance of cationic amine residues such as lysine. Using bacterial iron chelators consisting of paired DOPA and lysine groups as analogs for the mussel proteins, Maier et al. show that these two functional groups synergistically enhance interfacial adhesion (see the Perspective by Wilker). The lysine appears to displace hydrated cations from the surface, thus giving a dry patch for better adhesion. Science, this issue p. 628; see also p. 582 Biologically inspired adhesives reveal a synergistic interplay that enables wet adhesion to mineral surfaces. [Also see Perspective by Wilker] In physiological fluids and seawater, adhesion of synthetic polymers to solid surfaces is severely limited by high salt, pH, and hydration, yet these conditions have not deterred the evolution of effective adhesion by mussels. Mussel foot proteins provide insights about adhesive adaptations: Notably, the abundance and proximity of catecholic Dopa (3,4-dihydroxyphenylalanine) and lysine residues hint at a synergistic interplay in adhesion. Certain siderophores—bacterial iron chelators—consist of paired catechol and lysine functionalities, thereby providing a convenient experimental platform to explore molecular synergies in bioadhesion. These siderophores and synthetic analogs exhibit robust adhesion energies (Ead ≥−15 millijoules per square meter) to mica in saline pH 3.5 to 7.5 and resist oxidation. The adjacent catechol-lysine placement provides a “one-two punch,” whereby lysine evicts hydrated cations from the mineral surface, allowing catechol binding to underlying oxides.', 'year': 2015, 'citationCount': 485, 'authors': [{'authorId': '7560283', 'name': 'G. Maier'}, {'authorId': '38478808', 'name': 'Michael V. Rapp'}, {'authorId': '143955060', 'name': 'J. Waite'}, {'authorId': '6637150', 'name': 'J. Israelachvili'}, {'authorId': '144237978', 'name': 'A. Butler'}]}, {'paperId': '5f1febdd44de7d6819f180fe1e5c02a5d58dcb2e', 'title': 'Quantum squeezing of motion in a mechanical resonator', 'abstract': 'Manipulation of a quantum squeeze The uncertainty principle of quantum mechanics dictates that even when a system is cooled to its ground state, there are still fluctuations. This zero-point motion is unavoidable but can be manipulated. Wollman et al. demonstrate such manipulation with the motion of a micrometer-sized mechanical system. By driving up the fluctuations in one of the variables of the system, they are able to squeeze the other related variable below the expected zero-point limit. Quantum squeezing will be important for realizing ultrasensitive sensors and detectors. Science, this issue p. 952 The fluctuating motion of a mechanical system can be squeezed below the zero-point limit. According to quantum mechanics, a harmonic oscillator can never be completely at rest. Even in the ground state, its position will always have fluctuations, called the zero-point motion. Although the zero-point fluctuations are unavoidable, they can be manipulated. Using microwave frequency radiation pressure, we have manipulated the thermal fluctuations of a micrometer-scale mechanical resonator to produce a stationary quadrature-squeezed state with a minimum variance of 0.80 times that of the ground state. We also performed phase-sensitive, back-action evading measurements of a thermal state squeezed to 1.09 times the zero-point level. Our results are relevant to the quantum engineering of states of matter at large length scales, the study of decoherence of large quantum systems, and for the realization of ultrasensitive sensing of force and motion.', 'year': 2015, 'citationCount': 494, 'authors': [{'authorId': '40638497', 'name': 'E. Wollman'}, {'authorId': '39769515', 'name': 'C. Lei'}, {'authorId': '11079370', 'name': 'A. Weinstein'}, {'authorId': '2091526', 'name': 'J. Suh'}, {'authorId': '4040239', 'name': 'Andreas Kronwald'}, {'authorId': '48057862', 'name': 'F. Marquardt'}, {'authorId': '38315299', 'name': 'A. Clerk'}, {'authorId': '145384249', 'name': 'K. Schwab'}]}, {'paperId': '04fa534903823fbe7c63c3c4e07402ac2f03d767', 'title': 'Discovery and spectroscopy of the young jovian planet 51 Eri b with the Gemini Planet Imager', 'abstract': 'An exoplanet extracted from the bright Direct imaging of Jupiter-like exoplanets around young stars provides a glimpse into how our solar system formed. The brightness of young stars requires the use of next-generation devices such as the Gemini Planet Imager (GPI). Using the GPI, Macintosh et al. discovered a Jupiter-like planet orbiting a young star, 51 Eridani (see the Perspective by Mawet). The planet, 51 Eri b, has a methane signature and is probably the smallest exoplanet that has been directly imaged. These findings open the door to understanding solar system origins and herald the dawn of a new era in next-generation planetary imaging. Science, this issue p. 64; see also p. 39 The Gemini Planet Imager detects a Jupiter-like exoplanet orbiting the young star 51 Eridani. [Also see Perspective by Mawet] Directly detecting thermal emission from young extrasolar planets allows measurement of their atmospheric compositions and luminosities, which are influenced by their formation mechanisms. Using the Gemini Planet Imager, we discovered a planet orbiting the ~20-million-year-old star 51 Eridani at a projected separation of 13 astronomical units. Near-infrared observations show a spectrum with strong methane and water-vapor absorption. Modeling of the spectra and photometry yields a luminosity (normalized by the luminosity of the Sun) of 1.6 to 4.0 × 10−6 and an effective temperature of 600 to 750 kelvin. For this age and luminosity, “hot-start” formation models indicate a mass twice that of Jupiter. This planet also has a sufficiently low luminosity to be consistent with the “cold-start” core-accretion process that may have formed Jupiter.', 'year': 2015, 'citationCount': 419, 'authors': [{'authorId': '34346058', 'name': 'B. Macintosh'}, {'authorId': '34346058', 'name': 'B. Macintosh'}, {'authorId': '47581492', 'name': 'J. Graham'}, {'authorId': '32298861', 'name': 'T. Barman'}, {'authorId': '51056549', 'name': 'R. D. Rosa'}, {'authorId': '3510736', 'name': 'Q. Konopacky'}, {'authorId': '143790271', 'name': 'M. Marley'}, {'authorId': '5818645', 'name': 'C. Marois'}, {'authorId': '5818645', 'name': 'C. Marois'}, {'authorId': '144710553', 'name': 'E. Nielsen'}, {'authorId': '48643723', 'name': 'L. Pueyo'}, {'authorId': '145203539', 'name': 'A. Rajan'}, {'authorId': '21741189', 'name': 'J. Rameau'}, {'authorId': '5297870', 'name': 'D. Saumon'}, {'authorId': '49605872', 'name': 'Jason J. Wang'}, {'authorId': '49036366', 'name': 'J. Patience'}, {'authorId': '100858086', 'name': 'M. Ammons'}, {'authorId': '49782357', 'name': 'P. Arriaga'}, {'authorId': '5709961', 'name': 'É. Artigau'}, {'authorId': '144615444', 'name': 'S. Beckwith'}, {'authorId': '1387963762', 'name': 'J. Brewster'}, {'authorId': '46189399', 'name': 'S. Bruzzone'}, {'authorId': '41130114', 'name': 'J. Bulger'}, {'authorId': '41130114', 'name': 'J. Bulger'}, {'authorId': '145248197', 'name': 'B. Burningham'}, {'authorId': '145248197', 'name': 'B. Burningham'}, {'authorId': '144174723', 'name': 'A. Burrows'}, {'authorId': '48240220', 'name': 'Christine H. Chen'}, {'authorId': '3423834', 'name': 'E. Chiang'}, {'authorId': '5530835', 'name': 'J. Chilcote'}, {'authorId': '144597346', 'name': 'R. Dawson'}, {'authorId': '1710497627', 'name': 'Ruobing Dong'}, {'authorId': '48813871', 'name': 'R. Doyon'}, {'authorId': '46229682', 'name': 'Z. Draper'}, {'authorId': '46229682', 'name': 'Z. Draper'}, {'authorId': '49002627', 'name': 'G. Duchêne'}, {'authorId': '49002627', 'name': 'G. Duchêne'}, {'authorId': '49584561', 'name': 'T. Esposito'}, {'authorId': '6032941', 'name': 'D. Fabrycky'}, {'authorId': '2971243', 'name': 'M. Fitzgerald'}, {'authorId': '77166014', 'name': 'K. Follette'}, {'authorId': '2568738', 'name': 'J. Fortney'}, {'authorId': '50520275', 'name': 'B. Gerard'}, {'authorId': '50520275', 'name': 'B. Gerard'}, {'authorId': '3702656', 'name': 'S. Goodsell'}, {'authorId': '38989004', 'name': 'A. Greenbaum'}, {'authorId': '38989004', 'name': 'A. Greenbaum'}, {'authorId': '6482776', 'name': 'P. Hibon'}, {'authorId': '3297911', 'name': 'S. Hinkley'}, {'authorId': '4375416', 'name': 'T. Cotten'}, {'authorId': '145466741', 'name': 'L. Hung'}, {'authorId': '7195935', 'name': 'P. Ingraham'}, {'authorId': '1403781545', 'name': 'Mara Johnson-Groh'}, {'authorId': '1403781545', 'name': 'Mara Johnson-Groh'}, {'authorId': '5189125', 'name': 'P. Kalas'}, {'authorId': '32630686', 'name': 'D. Lafreniére'}, {'authorId': '38566478', 'name': 'J. Larkin'}, {'authorId': '2108937331', 'name': 'Jung-eun Lee'}, {'authorId': '3067786', 'name': 'M. Line'}, {'authorId': '96525405', 'name': 'D. Long'}, {'authorId': '51073120', 'name': 'J. Maire'}, {'authorId': '3771660', 'name': 'F. Marchis'}, {'authorId': '48140967', 'name': 'B. Matthews'}, {'authorId': '48140967', 'name': 'B. Matthews'}, {'authorId': '6825250', 'name': 'C. Max'}, {'authorId': '144630381', 'name': 'S. Metchev'}, {'authorId': '144630381', 'name': 'S. Metchev'}, {'authorId': '1380251886', 'name': 'M. Millar-Blanchaer'}, {'authorId': '46818187', 'name': 'T. Mittal'}, {'authorId': '46205305', 'name': 'C. Morley'}, {'authorId': '5432844', 'name': 'K. Morzinski'}, {'authorId': '1390047812', 'name': 'R. Murray-Clay'}, {'authorId': '2337669', 'name': 'R. Oppenheimer'}, {'authorId': '39555776', 'name': 'D. Palmer'}, {'authorId': '49833435', 'name': 'R. Patel'}, {'authorId': '5095283', 'name': 'M. Perrin'}, {'authorId': '5335319', 'name': 'L. Poyneer'}, {'authorId': '46187849', 'name': 'R. Rafikov'}, {'authorId': '3822852', 'name': 'F. Rantakyrö'}, {'authorId': '3483952', 'name': 'E. Rice'}, {'authorId': '3483952', 'name': 'E. Rice'}, {'authorId': '40865906', 'name': 'P. Rojo'}, {'authorId': '41222111', 'name': 'A. Rudy'}, {'authorId': '97341200', 'name': 'J. Ruffio'}, {'authorId': '145937717', 'name': 'M. Ruiz'}, {'authorId': '3503160', 'name': 'N. Sadakuni'}, {'authorId': '6058233', 'name': 'L. Saddlemyer'}, {'authorId': '51373537', 'name': 'M. Salama'}, {'authorId': '46834135', 'name': 'D. Savransky'}, {'authorId': '14819041', 'name': 'A. Schneider'}, {'authorId': '3004825', 'name': 'A. Sivaramakrishnan'}, {'authorId': '4965490', 'name': 'I. Song'}, {'authorId': '3392617', 'name': 'R. Soummer'}, {'authorId': '39334803', 'name': 'S. Thomas'}, {'authorId': '4255387', 'name': 'G. Vasisht'}, {'authorId': '145535282', 'name': 'J. Wallace'}, {'authorId': '1380251779', 'name': 'K. Ward-Duong'}, {'authorId': '5131976', 'name': 'S. Wiktorowicz'}, {'authorId': '32886400', 'name': 'S. Wolff'}, {'authorId': '32886400', 'name': 'S. Wolff'}, {'authorId': '47522098', 'name': 'B. Zuckerman'}]}, {'paperId': '056c4b46e774c75351b9c53880ec31510644d67e', 'title': 'A stable trimeric influenza hemagglutinin stem as a broadly protective immunogen', 'abstract': 'Flu vaccine candidate STEMs the tide Every year we need a new flu vaccine, because influenza virus constantly mutates the major target of antibodies to flu: the “head” region of the viral hemagglutinin (HA) protein. Avoiding the problem of mutation requires a vaccine that elicits antibodies against the more conserved “stem” region of HA. During infection, antibodies are occasionally produced that recognize the stem and that neutralize a broad range of influenza virus strains. Impagliazzo et al. engineered an HA stem–only vaccine candidate that elicited broadly neutralizing antibodies in mice and nonhuman primates and that protected mice against multiple influenza strains. Science, this issue p. 1301 An engineered stem-only hemagglutinin vaccine candidate provides broad protection in an animal model of influenza infection. The identification of human broadly neutralizing antibodies (bnAbs) targeting the hemagglutinin (HA) stem revitalized hopes of developing a universal influenza vaccine. Using a rational design and library approach, we engineered stable HA stem antigens (“mini-HAs”) based on an H1 subtype sequence. Our most advanced candidate exhibits structural and bnAb binding properties comparable to those of full-length HA, completely protects mice in lethal heterologous and heterosubtypic challenge models, and reduces fever after sublethal challenge in cynomolgus monkeys. Antibodies elicited by this mini-HA in mice and nonhuman primates bound a wide range of HAs, competed with human bnAbs for HA stem binding, neutralized H5N1 viruses, and mediated antibody-dependent effector activity. These results represent a proof of concept for the design of HA stem mimics that elicit bnAbs against influenza A group 1 viruses.', 'year': 2015, 'citationCount': 448, 'authors': [{'authorId': '6029922', 'name': 'A. Impagliazzo'}, {'authorId': '5223281', 'name': 'F. Milder'}, {'authorId': '6919469', 'name': 'H. Kuipers'}, {'authorId': '20651034', 'name': 'M. Wagner'}, {'authorId': '2303500', 'name': 'Xueyong Zhu'}, {'authorId': '33864211', 'name': 'Ryan M. B. Hoffman'}, {'authorId': '1768780382', 'name': 'Ruud van Meersbergen'}, {'authorId': '7656722', 'name': 'Jeroen Huizingh'}, {'authorId': '47810501', 'name': 'P. Wanningen'}, {'authorId': '8383421', 'name': 'Johan W. A. Verspuij'}, {'authorId': '2059170515', 'name': 'M. de Man'}, {'authorId': '3650958', 'name': 'Zhaoqing Ding'}, {'authorId': '47618166', 'name': 'Adrian Apetri'}, {'authorId': '5864616', 'name': 'Başak Kükrer'}, {'authorId': '1398950001', 'name': 'Eveline Sneekes-Vriese'}, {'authorId': '50309292', 'name': 'Danuta Tomkiewicz'}, {'authorId': '8558947', 'name': 'N. S. Laursen'}, {'authorId': '1406042252', 'name': 'Peter S. Lee'}, {'authorId': '46278598', 'name': 'A. Zakrzewska'}, {'authorId': '6556142', 'name': 'Liesbeth Dekking'}, {'authorId': '3193831', 'name': 'J. Tolboom'}, {'authorId': '7424762', 'name': 'Lisanne Tettero'}, {'authorId': '1768782361', 'name': 'Sander van Meerten'}, {'authorId': '7985846', 'name': 'Wenli Yu'}, {'authorId': '4754966', 'name': 'W. Koudstaal'}, {'authorId': '145531112', 'name': 'J. Goudsmit'}, {'authorId': '144621454', 'name': 'A. Ward'}, {'authorId': '6759290', 'name': 'W. Meijberg'}, {'authorId': '143791278', 'name': 'I. Wilson'}, {'authorId': '33475599', 'name': 'K. Radošević'}]}, {'paperId': 'c0abb175c6c78964ac056acd699722206183c251', 'title': 'Alternative lengthening of telomeres renders cancer cells hypersensitive to ATR inhibitors.', 'abstract': 'Cancer cells rely on telomerase or the alternative lengthening of telomeres (ALT) pathway to overcome replicative mortality. ALT is mediated by recombination and is prevalent in a subset of human cancers, yet whether it can be exploited therapeutically remains unknown. Loss of the chromatin-remodeling protein ATRX associates with ALT in cancers. Here, we show that ATRX loss compromises cell-cycle regulation of the telomeric noncoding RNA TERRA and leads to persistent association of replication protein A (RPA) with telomeres after DNA replication, creating a recombinogenic nucleoprotein structure. Inhibition of the protein kinase ATR, a critical regulator of recombination recruited by RPA, disrupts ALT and triggers chromosome fragmentation and apoptosis in ALT cells. The cell death induced by ATR inhibitors is highly selective for cancer cells that rely on ALT, suggesting that such inhibitors may be useful for treatment of ALT-positive cancers.', 'year': 2015, 'citationCount': 383, 'authors': [{'authorId': '144325445', 'name': 'R. L. Flynn'}, {'authorId': '6625525', 'name': 'Kelli E. Cox'}, {'authorId': '6612562', 'name': 'M. Jeitany'}, {'authorId': '34972222', 'name': 'H. Wakimoto'}, {'authorId': '7681457', 'name': 'Alysia R. Bryll'}, {'authorId': '6933996', 'name': 'Neil J. Ganem'}, {'authorId': '38161554', 'name': 'F. Bersani'}, {'authorId': '13583738', 'name': 'J. R. Pineda'}, {'authorId': '3728246', 'name': 'M. Suvà'}, {'authorId': '1714039', 'name': 'C. Benes'}, {'authorId': '2253481', 'name': 'D. Haber'}, {'authorId': '3726742', 'name': 'F. Boussin'}, {'authorId': '2234244492', 'name': 'L. Zou'}]}, {'paperId': 'cc510777bf01063e970efc8c9b24bff1c434e2d0', 'title': '67P/Churyumov-Gerasimenko, a Jupiter family comet with a high D/H ratio', 'abstract': 'The provenance of water and organic compounds on Earth and other terrestrial planets has been discussed for a long time without reaching a consensus. One of the best means to distinguish between different scenarios is by determining the deuterium-to-hydrogen (D/H) ratios in the reservoirs for comets and Earth’s oceans. Here, we report the direct in situ measurement of the D/H ratio in the Jupiter family comet 67P/Churyumov-Gerasimenko by the ROSINA mass spectrometer aboard the European Space Agency’s Rosetta spacecraft, which is found to be (5.3 ± 0.7) × 10−4—that is, approximately three times the terrestrial value. Previous cometary measurements and our new finding suggest a wide range of D/H ratios in the water within Jupiter family objects and preclude the idea that this reservoir is solely composed of Earth ocean–like water.', 'year': 2015, 'citationCount': 384, 'authors': [{'authorId': '4512323', 'name': 'K. Altwegg'}, {'authorId': '3783380', 'name': 'H. Balsiger'}, {'authorId': '1403746229', 'name': 'A. Bar-Nun'}, {'authorId': '2282158', 'name': 'J. Berthelier'}, {'authorId': '6862522', 'name': 'A. Bieler'}, {'authorId': '6862522', 'name': 'A. Bieler'}, {'authorId': '4019296', 'name': 'P. Bochsler'}, {'authorId': '100535580', 'name': 'C. Briois'}, {'authorId': '4612766', 'name': 'U. Calmonte'}, {'authorId': '2894238', 'name': 'M. Combi'}, {'authorId': '144087995', 'name': 'J. Keyser'}, {'authorId': '4793465', 'name': 'P. Eberhardt'}, {'authorId': '1694493', 'name': 'B. Fiethe'}, {'authorId': '47853914', 'name': 'S. Fuselier'}, {'authorId': '69455147', 'name': 'S. Gasc'}, {'authorId': '2358371', 'name': 'T. Gombosi'}, {'authorId': '34594942', 'name': 'K. Hansen'}, {'authorId': '115165858', 'name': 'M. Hässig'}, {'authorId': '115165858', 'name': 'M. Hässig'}, {'authorId': '145555577', 'name': 'A. Jäckel'}, {'authorId': '144047212', 'name': 'E. Kopp'}, {'authorId': '2881282', 'name': 'A. Korth'}, {'authorId': '39947404', 'name': 'L. Leroy'}, {'authorId': '1931940', 'name': 'U. Mall'}, {'authorId': '2490322', 'name': 'B. Marty'}, {'authorId': '5055989', 'name': 'O. Mousis'}, {'authorId': '2376017', 'name': 'E. Neefs'}, {'authorId': '6309276', 'name': 'T. Owen'}, {'authorId': '3172579', 'name': 'H. Rème'}, {'authorId': '3172579', 'name': 'H. Rème'}, {'authorId': '145337044', 'name': 'M. Rubin'}, {'authorId': '2094410987', 'name': 'T. Sémon'}, {'authorId': '143664095', 'name': 'C. Tzou'}, {'authorId': '28411223', 'name': 'H. Waite'}, {'authorId': '3101090', 'name': 'P. Wurz'}]}, {'paperId': 'ae861867a6693094db5af7a9ae45b204407426e2', 'title': 'Transcription factor trapping by RNA in gene regulatory elements', 'abstract': 'Noncoding RNA helps protein binding Besides reading the coding regions of genes, RNA polymerase generates RNA at promoter-proximal and -distal DNA elements, but the function of these molecules is largely unknown. Sigova et al. show that these RNAs facilitate interactions between gene regulators and the regulatory elements they occupy. Nascent RNA associates with the transcription factor YY1 and increases its ability to bind DNA. Thus, transcription at active regulatory elements may provide a positive feedback loop that reinforces regulatory elements contributing to the stability of gene expression programs. Science, this issue p. 978 Nascent RNAs facilitate interactions between gene regulators and regulatory elements. Transcription factors (TFs) bind specific sequences in promoter-proximal and -distal DNA elements to regulate gene transcription. RNA is transcribed from both of these DNA elements, and some DNA binding TFs bind RNA. Hence, RNA transcribed from regulatory elements may contribute to stable TF occupancy at these sites. We show that the ubiquitously expressed TF Yin-Yang 1 (YY1) binds to both gene regulatory elements and their associated RNA species across the entire genome. Reduced transcription of regulatory elements diminishes YY1 occupancy, whereas artificial tethering of RNA enhances YY1 occupancy at these elements. We propose that RNA makes a modest but important contribution to the maintenance of certain TFs at gene regulatory elements and suggest that transcription of regulatory elements produces a positive-feedback loop that contributes to the stability of gene expression programs.', 'year': 2015, 'citationCount': 367, 'authors': [{'authorId': '4066956', 'name': 'A. Sigova'}, {'authorId': '41330093', 'name': 'B. Abraham'}, {'authorId': '48552586', 'name': 'X. Ji'}, {'authorId': '3622415', 'name': 'B. Molinie'}, {'authorId': '3607792', 'name': 'N. Hannett'}, {'authorId': '49813841', 'name': 'Y. Guo'}, {'authorId': '4029700', 'name': 'Mohini Jangi'}, {'authorId': '3649962', 'name': 'Cosmas C. Giallourakis'}, {'authorId': '2065901', 'name': 'P. Sharp'}, {'authorId': '144787676', 'name': 'R. Young'}]}, {'paperId': '5036e687656efa470275eb6813d09137670511b6', 'title': 'The structure of the human mitochondrial ribosome', 'abstract': 'The whole mitoribosome at high resolution Mitochondria are thought to be the descendents of a prokaryotic cell that took up residence in a protoeukaryotic cell. Mitochondria retain a few genes involved in oxidative phosphorylation. To translate these genes, mitochondria contain highly divergent mitochondrial ribosomes, or mitoribosomes. Amunts et al. determined the high-resolution structures of complete mammalian mitoribosomes using cryoelectron microscopy. Mitoribosomes include an unusual mRNA binding channel. The findings elucidate how aminoglycoside antibiotics can inadvertently inhibit mitoribosomes and how mutations in mitoribosomes can lead to disease. Science, this issue p. 95 Structures of mammalian mitoribosomes reveal details underlying the function of this divergent class of ribosome. The highly divergent ribosomes of human mitochondria (mitoribosomes) synthesize 13 essential proteins of oxidative phosphorylation complexes. We have determined the structure of the intact mitoribosome to 3.5 angstrom resolution by means of single-particle electron cryogenic microscopy. It reveals 80 extensively interconnected proteins, 36 of which are specific to mitochondria, and three ribosomal RNA molecules. The head domain of the small subunit, particularly the messenger (mRNA) channel, is highly remodeled. Many intersubunit bridges are specific to the mitoribosome, which adopts conformations involving ratcheting or rolling of the small subunit that are distinct from those seen in bacteria or eukaryotes. An intrinsic guanosine triphosphatase mediates a contact between the head and central protuberance. The structure provides a reference for analysis of mutations that cause severe pathologies and for future drug design.', 'year': 2015, 'citationCount': 401, 'authors': [{'authorId': '3681289', 'name': 'A. Amunts'}, {'authorId': '144686935', 'name': 'Alan Brown'}, {'authorId': '95552201', 'name': 'J. Toots'}, {'authorId': '46489599', 'name': 'S. Scheres'}, {'authorId': '145575131', 'name': 'V. Ramakrishnan'}]}, {'paperId': '074fecabdd0f93f3ed501a37cf9f69e6bf151504', 'title': 'Dust measurements in the coma of comet 67P/Churyumov-Gerasimenko inbound to the Sun', 'abstract': 'Critical measurements for understanding accretion and the dust/gas ratio in the solar nebula, where planets were forming 4.5 billion years ago, are being obtained by the GIADA (Grain Impact Analyser and Dust Accumulator) experiment on the European Space Agency’s Rosetta spacecraft orbiting comet 67P/Churyumov-Gerasimenko. Between 3.6 and 3.4 astronomical units inbound, GIADA and OSIRIS (Optical, Spectroscopic, and Infrared Remote Imaging System) detected 35 outflowing grains of mass 10−10 to 10−7 kilograms, and 48 grains of mass 10−5 to 10−2 kilograms, respectively. Combined with gas data from the MIRO (Microwave Instrument for the Rosetta Orbiter) and ROSINA (Rosetta Orbiter Spectrometer for Ion and Neutral Analysis) instruments, we find a dust/gas mass ratio of 4 ± 2 averaged over the sunlit nucleus surface. A cloud of larger grains also encircles the nucleus in bound orbits from the previous perihelion. The largest orbiting clumps are meter-sized, confirming the dust/gas ratio of 3 inferred at perihelion from models of dust comae and trails.', 'year': 2015, 'citationCount': 334, 'authors': [{'authorId': '5193844', 'name': 'A. Rotundi'}, {'authorId': '50246256', 'name': 'H. Sierks'}, {'authorId': '113142748', 'name': 'V. Della Corte'}, {'authorId': '14100462', 'name': 'M. Fulle'}, {'authorId': '49672000', 'name': 'P. Gutiérrez'}, {'authorId': '145621605', 'name': 'L. Lara'}, {'authorId': '39419736', 'name': 'C. Barbieri'}, {'authorId': '50403997', 'name': 'P. Lamy'}, {'authorId': '91031514', 'name': 'R. Rodrigo'}, {'authorId': '69029744', 'name': 'D. Koschny'}, {'authorId': '6973900', 'name': 'H. Rickman'}, {'authorId': '144446632', 'name': 'H. Keller'}, {'authorId': '1393618617', 'name': 'J. Lopez‐Moreno'}, {'authorId': '48649617', 'name': 'M. Accolla'}, {'authorId': '145969939', 'name': 'J. Agarwal'}, {'authorId': '1398024389', 'name': 'M. A’Hearn'}, {'authorId': '1888631', 'name': 'N. Altobelli'}, {'authorId': '48286249', 'name': 'F. Angrilli'}, {'authorId': '145885511', 'name': 'M. Barucci'}, {'authorId': '102414410', 'name': 'J. Bertaux'}, {'authorId': '47679133', 'name': 'I. Bertini'}, {'authorId': '6691955', 'name': 'D. Bodewits'}, {'authorId': '48366207', 'name': 'E. Bussoletti'}, {'authorId': '5058495', 'name': 'L. Colangeli'}, {'authorId': '2762299', 'name': 'M. Cosi'}, {'authorId': '3939528', 'name': 'G. Cremonese'}, {'authorId': '31844775', 'name': 'J. Crifo'}, {'authorId': '2896324', 'name': 'V. Da Deppo'}, {'authorId': '3984918', 'name': 'B. Davidsson'}, {'authorId': '2317823', 'name': 'S. Debei'}, {'authorId': '50129229', 'name': 'M. De Cecco'}, {'authorId': '40416093', 'name': 'F. Esposito'}, {'authorId': '145605911', 'name': 'M. Ferrari'}, {'authorId': '2564082', 'name': 'S. Fornasier'}, {'authorId': '8294277', 'name': 'F. Giovane'}, {'authorId': '5047888', 'name': 'B. Gustafson'}, {'authorId': '4616264', 'name': 'S. Green'}, {'authorId': '3155836', 'name': 'O. Groussin'}, {'authorId': '144003691', 'name': 'E. Grün'}, {'authorId': '3354765', 'name': 'C. Güttler'}, {'authorId': '143947538', 'name': 'M. Herranz'}, {'authorId': '5303188', 'name': 'S. Hviid'}, {'authorId': '2095291385', 'name': 'W. Ip'}, {'authorId': '27392662', 'name': 'S. Ivanovski'}, {'authorId': '31930502', 'name': 'J. M. Jerónimo'}, {'authorId': '3240472', 'name': 'L. Jorda'}, {'authorId': '152615704', 'name': 'J. Knollenberg'}, {'authorId': '8510904', 'name': 'R. Kramm'}, {'authorId': '2250203', 'name': 'E. Kührt'}, {'authorId': '48974673', 'name': 'M. Küppers'}, {'authorId': '144479967', 'name': 'M. Lazzarin'}, {'authorId': '34670298', 'name': 'M. Leese'}, {'authorId': '1398306080', 'name': 'A. López-Jiménez'}, {'authorId': '2083713398', 'name': 'F. Lucarelli'}, {'authorId': '2891649', 'name': 'S. Lowry'}, {'authorId': '144372011', 'name': 'F. Marzari'}, {'authorId': '7642818', 'name': 'E. Epifani'}, {'authorId': '144186953', 'name': 'J. McDonnell'}, {'authorId': '4105305', 'name': 'V. Mennella'}, {'authorId': '144921681', 'name': 'H. Michalik'}, {'authorId': '143666794', 'name': 'A. Molina'}, {'authorId': '153285004', 'name': 'R. Morales'}, {'authorId': '1572187912', 'name': 'F. Moreno'}, {'authorId': '144381122', 'name': 'S. Mottola'}, {'authorId': '2704601', 'name': 'G. Naletto'}, {'authorId': '5300286', 'name': 'N. Oklay'}, {'authorId': '145786944', 'name': 'J. Ortiz'}, {'authorId': '2885404', 'name': 'E. Palomba'}, {'authorId': '144685886', 'name': 'P. Palumbo'}, {'authorId': '122587024', 'name': 'J. Perrin'}, {'authorId': '2116901131', 'name': 'Julio Rodríguez'}, {'authorId': '143930936', 'name': 'L. Sabau'}, {'authorId': '2155760', 'name': 'C. Snodgrass'}, {'authorId': '8154401', 'name': 'R. Sordini'}, {'authorId': '144570575', 'name': 'N. Thomas'}, {'authorId': '5653933', 'name': 'C. Tubiana'}, {'authorId': '12647806', 'name': 'J. Vincent'}, {'authorId': '4905966', 'name': 'P. Weissman'}, {'authorId': '114386622', 'name': 'K. Wenzel'}, {'authorId': '152222796', 'name': 'V. Zakharov'}, {'authorId': '5101497', 'name': 'J. Zarnecki'}]}, {'paperId': 'e08354cc53e6b3b2b10233777989f2c28d1d1911', 'title': 'The transcription factor GABP selectively binds and activates the mutant TERT promoter in cancer', 'abstract': \"A mutant promoter's partner in crime Telomerase is an enzyme that maintains the ends of chromosomes. TERT, the gene coding for the enzyme's catalytic subunit, is not expressed in healthy somatic cells, but its expression is reactivated in the majority of human cancers. The resultant high levels of telomerase help cancer cells survive and multiply. Recurrent mutations in the promoter region of TERT are associated with high telomerase levels in multiple cancer types. Bell et al. show that a specific transcription factor called GABP is selectively recruited to the mutant form of the TERT promoter, which activates TERT gene expression Science, this issue p. 1036 Cancer-associated mutations in the promoter of the telomerase gene allow increased activation by transcription factor binding. Reactivation of telomerase reverse transcriptase (TERT) expression enables cells to overcome replicative senescence and escape apoptosis, which are fundamental steps in the initiation of human cancer. Multiple cancer types, including up to 83% of glioblastomas (GBMs), harbor highly recurrent TERT promoter mutations of unknown function but specific to two nucleotide positions. We identified the functional consequence of these mutations in GBMs to be recruitment of the multimeric GA-binding protein (GABP) transcription factor specifically to the mutant promoter. Allelic recruitment of GABP is consistently observed across four cancer types, highlighting a shared mechanism underlying TERT reactivation. Tandem flanking native E26 transformation-specific motifs critically cooperate with these mutations to activate TERT, probably by facilitating GABP heterotetramer binding. GABP thus directly links TERT promoter mutations to aberrant expression in multiple cancers.\", 'year': 2015, 'citationCount': 421, 'authors': [{'authorId': '144450987', 'name': 'Robert J. A. Bell'}, {'authorId': '4281133', 'name': 'H. T. Rube'}, {'authorId': '4338966', 'name': 'Alex Kreig'}, {'authorId': '38557296', 'name': 'A. Mancini'}, {'authorId': '30033687', 'name': 'S. Fouse'}, {'authorId': '31430164', 'name': 'R. Nagarajan'}, {'authorId': '4630388', 'name': 'Serah Choi'}, {'authorId': '4824870', 'name': 'C. Hong'}, {'authorId': '3957980', 'name': 'Daniel He'}, {'authorId': '4798798', 'name': 'M. Pekmezci'}, {'authorId': '2704556', 'name': 'J. Wiencke'}, {'authorId': '2074070', 'name': 'M. Wrensch'}, {'authorId': '18014955', 'name': 'Susan M. Chang'}, {'authorId': '2196573', 'name': 'K. Walsh'}, {'authorId': '5862538', 'name': 'S. Myong'}, {'authorId': '143729167', 'name': 'Jun S. Song'}, {'authorId': '2552233', 'name': 'J. Costello'}]}, {'paperId': 'c2f0f097f803926fedb9d4f7c71002f7ca32ba95', 'title': 'Mars methane detection and variability at Gale crater', 'abstract': 'Of water and methane on Mars The Curiosity rover has been collecting data for the past 2 years, since its delivery to Mars (see the Perspective by Zahnle). Many studies now suggest that many millions of years ago, Mars was warmer and wetter than it is today. But those conditions required an atmosphere that seems to have vanished. Using the Curiosity rover, Mahaffy et al. measured the ratio of deuterium to hydrogen in clays that were formed 3.0 to 3.7 billion years ago. Hydrogen escapes more readily than deuterium, so this ratio offers a snapshot measure of the ancient atmosphere that can help constrain when and how it disappeared. Most methane on Earth has a biological origin, so planetary scientists have keenly pursued its detection in the martian atmosphere as well. Now, Webster et al. have precisely confirmed the presence of methane in the martian atmosphere with the instruments aboard the Curiosity rover at Gale crater. Science, this issue p. 412, p. 415; see also p. 370 Curiosity confirms the presence and variability of atmospheric methane, implying episodic production from an unknown source. [Also see Perspective by Zahnle] Reports of plumes or patches of methane in the martian atmosphere that vary over monthly time scales have defied explanation to date. From in situ measurements made over a 20-month period by the tunable laser spectrometer of the Sample Analysis at Mars instrument suite on Curiosity at Gale crater, we report detection of background levels of atmospheric methane of mean value 0.69 ± 0.25 parts per billion by volume (ppbv) at the 95% confidence interval (CI). This abundance is lower than model estimates of ultraviolet degradation of accreted interplanetary dust particles or carbonaceous chondrite material. Additionally, in four sequential measurements spanning a 60-sol period (where 1 sol is a martian day), we observed elevated levels of methane of 7.2 ± 2.1 ppbv (95% CI), implying that Mars is episodically producing methane from an additional unknown source.', 'year': 2015, 'citationCount': 408, 'authors': [{'authorId': '1997113', 'name': 'C. Webster'}, {'authorId': '4579249', 'name': 'P. Mahaffy'}, {'authorId': '1934029', 'name': 'S. Atreya'}, {'authorId': '39051505', 'name': 'G. Flesch'}, {'authorId': '6507728', 'name': 'M. Mischna'}, {'authorId': '7166207', 'name': 'P. Meslin'}, {'authorId': '37318312', 'name': 'K. Farley'}, {'authorId': '40098549', 'name': 'P. Conrad'}, {'authorId': '36072221', 'name': 'L. Christensen'}, {'authorId': '47907676', 'name': 'A. Pavlov'}, {'authorId': '1400376253', 'name': 'J. Martín‐Torres'}, {'authorId': '70175219', 'name': 'M. Zorzano'}, {'authorId': '7924487', 'name': 'T. McConnochie'}, {'authorId': '6309276', 'name': 'T. Owen'}, {'authorId': '5905992', 'name': 'J. Eigenbrode'}, {'authorId': '6766468', 'name': 'D. Glavin'}, {'authorId': '145255661', 'name': 'A. Steele'}, {'authorId': '4289200', 'name': 'C. Malespin'}, {'authorId': '1927274', 'name': 'P. Archer'}, {'authorId': '2193015', 'name': 'B. Sutter'}, {'authorId': '46708768', 'name': 'P. Coll'}, {'authorId': '34580847', 'name': 'C. Freissinet'}, {'authorId': '87435846', 'name': 'C. Mckay'}, {'authorId': '35101299', 'name': 'J. Moores'}, {'authorId': '6827709', 'name': 'S. Schwenzer'}, {'authorId': '4386694', 'name': 'J. Bridges'}, {'authorId': '1388991208', 'name': 'R. Navarro‐González'}, {'authorId': '2714162', 'name': 'R. Gellert'}, {'authorId': '32600129', 'name': 'M. Lemmon'}]}, {'paperId': '6ea6bdab5a986a06160615e48cf385f4784f973e', 'title': 'The Atlantic Multidecadal Oscillation without a role for ocean circulation', 'abstract': 'Ocean circulation changes not needed What causes the pattern of sea surface temperature change that is seen in the North Atlantic Ocean? This naturally occurring quasi-cyclical variation, known as the Atlantic Multidecadal Oscillation (AMO), affects weather and climate. Some have suggested that the AMO is a consequence of variable large-scale ocean circulation. Clement et al. suggest otherwise. They find that the pattern of AMO variability can be produced in a model that does not include ocean circulation changes, but only the effects of changes in air temperatures and winds. Science, this issue p. 320 The Atlantic Multidecadal Oscillation does not depend on variable whole-ocean circulation. The Atlantic Multidecadal Oscillation (AMO) is a major mode of climate variability with important societal impacts. Most previous explanations identify the driver of the AMO as the ocean circulation, specifically the Atlantic Meridional Overturning Circulation (AMOC). Here we show that the main features of the observed AMO are reproduced in models where the ocean heat transport is prescribed and thus cannot be the driver. Allowing the ocean circulation to interact with the atmosphere does not significantly alter the characteristics of the AMO in the current generation of climate models. These results suggest that the AMO is the response to stochastic forcing from the mid-latitude atmospheric circulation, with thermal coupling playing a role in the tropics. In this view, the AMOC and other ocean circulation changes would be largely a response to, not a cause of, the AMO.', 'year': 2015, 'citationCount': 303, 'authors': [{'authorId': '38500930', 'name': 'A. Clement'}, {'authorId': '48858437', 'name': 'K. Bellomo'}, {'authorId': '48395861', 'name': 'L. Murphy'}, {'authorId': '2409880', 'name': 'M. Cane'}, {'authorId': '4969561', 'name': 'T. Mauritsen'}, {'authorId': '152424253', 'name': 'G. Rädel'}, {'authorId': '121696502', 'name': 'B. Stevens'}]}, {'paperId': 'be652a4dc3d9217f054013a31c3ab081205a1c7a', 'title': 'Preventing proteostasis diseases by selective inhibition of a phosphatase regulatory subunit', 'abstract': 'Giving protein folding a helping hand The reversible phosphorylation of proteins controls virtually all aspects of cell and organismal function. Targeting phosphorylation offers a broad range of therapeutic opportunities, and thus kinases have become important therapeutic targets. As targets, phosphatases should be as attractive, but in fact they are more challenging to manipulate. Das et al. have found a safe and specific inhibitor, called Sephin1, that targets a regulatory subunit of protein phosphatase 1 in vivo. Sephin1 binds and inhibits PPP1R15A, but not the related regulatory phosphatase PPP1R15B. In mice, Sephin1 prolonged a stress-induced phospho-signaling pathway to prevent the pathological defects of the unrelated protein-misfolding diseases Charcot-Marie-Tooth 1B and amyotrophic lateral sclerosis. Science, this issue p. 239 Sephin1 selectively inhibits a protein phosphatase to prevent two protein misfolding diseases in mice. Protein phosphorylation regulates virtually all biological processes. Although protein kinases are popular drug targets, targeting protein phosphatases remains a challenge. Here, we describe Sephin1 (selective inhibitor of a holophosphatase), a small molecule that safely and selectively inhibited a regulatory subunit of protein phosphatase 1 in vivo. Sephin1 selectively bound and inhibited the stress-induced PPP1R15A, but not the related and constitutive PPP1R15B, to prolong the benefit of an adaptive phospho-signaling pathway, protecting cells from otherwise lethal protein misfolding stress. In vivo, Sephin1 safely prevented the motor, morphological, and molecular defects of two otherwise unrelated protein-misfolding diseases in mice, Charcot-Marie-Tooth 1B, and amyotrophic lateral sclerosis. Thus, regulatory subunits of phosphatases are drug targets, a property exploited here to safely prevent two protein misfolding diseases.', 'year': 2015, 'citationCount': 331, 'authors': [{'authorId': '153163469', 'name': 'I. Das'}, {'authorId': '5619187', 'name': 'Agnieszka Krzyżosiak'}, {'authorId': '8020826', 'name': 'Kim Schneider'}, {'authorId': '6452817', 'name': 'L. Wrabetz'}, {'authorId': '1401066945', 'name': 'M. D’Antonio'}, {'authorId': '40226529', 'name': 'N. Barry'}, {'authorId': '12051959', 'name': 'A. Sigurdardottir'}, {'authorId': '35866732', 'name': 'A. Bertolotti'}]}, {'paperId': 'd66b125cf23f2b06d7ba717fdbb00e73d620cf08', 'title': 'The Pluto system: Initial results from its exploration by New Horizons', 'abstract': \"New Horizons' views of Pluto The flyby of Pluto and its moon Charon by the New Horizons spacecraft generated news coverage around the world. Now Stern et al. report the first scientific results from the high-speed encounter. The surface of Pluto is surprisingly diverse, with large regions of differing brightness and composition. There is ample evidence for ongoing rich geological processes that act to sculpt its surface. Charon's surface is similarly complex, with numerous relief structures and varied coloration. Pluto's atmosphere is extensive but less dense than expected, whereas Charon has no detectable atmosphere. Science, this issue p. 10.1126/science.aad1815 The first scientific results from New Horizon’s flyby of Pluto reveal a rich and diverse geology. INTRODUCTION Pluto was discovered in 1930 and was long thought to be a misfit or anomaly in the solar system. However, the 1992 discovery of the Kuiper Belt—a torus-shaped region beyond Neptune’s orbit, and the largest structure in our three-zoned planetary system—provided new context, showing Pluto to be the largest of a new class of small planets formed in the outer solar system during the ancient era of planetary accretion ~4.5 billion years ago. NASA’s New Horizons spacecraft made the first exploration of Pluto, culminating on 14 July 2015; it collected numerous remote sensing and in situ measurements of Pluto and its system of five moons. We report the first scientific results and interpretations of that flyby. RATIONALE The New Horizons spacecraft completed a close approach to the Pluto system at a distance of 13,691 km from Pluto’s center. The spacecraft carries a sophisticated suite of scientific instruments, including the Ralph multicolor/panchromatic mapper and mapping infrared composition spectrometer; the LORRI long-focal-length panchromatic visible imager; the Alice extreme/far ultraviolet mapping spectrograph; twin REX radio science experiments; the SWAP solar wind detector; the PEPSSI high-energy charged particle spectrometer; and VBSDC, a dust impact detector. Together these instruments collected more than 50 gigabits of data on the Pluto system near the time of the spacecraft’s closest approach. RESULTS We found that Pluto’s surface displays a wide variety of landforms and terrain ages, as well as substantial albedo, color, and compositional variation. Evidence was also found for a water ice–rich crust, geologically young surface units, tectonic extension, surface volatile ice convection, possible wind streaks, volatile transport, and glacial flow. Pluto’s atmosphere is highly extended, with trace hydrocarbons, a global haze layer, and a surface pressure near 10 microbars. The bulk densities of Pluto and Charon were found to differ by less than 10%, which is consistent with bulk rock contents for the two bodies that are likewise similar. This could imply that both precursor bodies were undifferentiated (or only modestly differentiated) prior to their collision—which would have profound implications for the timing, the duration, and even the mechanism of accretion in the ancestral Kuiper Belt. Pluto’s large moon Charon displays extensional tectonics and extensive resurfacing, as well as possible evidence for a heterogeneous crustal composition; its north pole displays puzzling dark terrain. The sizes of Pluto’s small satellites Nix and Hydra were measured for the first time, as were their surface reflectivities, which are puzzlingly higher than Charon’s. No new satellites were detected. CONCLUSION The New Horizons encounter revealed that Pluto displays a surprisingly wide variety of geological landforms, including those resulting from glaciological and surface-atmosphere interactions as well as impact, tectonic, possible cryovolcanic, and mass-wasting processes. This suggests that other small planets of the Kuiper Belt, such as Eris, Makemake, and Haumea, could express similarly complex histories that rival those of terrestrial planets. Pluto’s diverse surface geology and long-term activity also raise fundamental questions about how it has remained active many billions of years after its formation. Pluto mosaic made from New Horizons LORRI images taken 14 July 2015 from a distance of 80,000 km. This view is projected from a point 1800 km above Pluto’s equator, looking northeast over the dark, cratered, informally named Cthulhu Regio toward the bright, smooth expanse of icy plains informally called Sputnik Planum. Pluto’s north pole is off the image to the left. This image mosaic was produced with panchromatic images from the New Horizons LORRI camera, with color overlaid from the Ralph color mapper onboard New Horizons. The Pluto system was recently explored by NASA’s New Horizons spacecraft, making closest approach on 14 July 2015. Pluto’s surface displays diverse landforms, terrain ages, albedos, colors, and composition gradients. Evidence is found for a water-ice crust, geologically young surface units, surface ice convection, wind streaks, volatile transport, and glacial flow. Pluto’s atmosphere is highly extended, with trace hydrocarbons, a global haze layer, and a surface pressure near 10 microbars. Pluto’s diverse surface geology and long-term activity raise fundamental questions about how small planets remain active many billions of years after formation. Pluto’s large moon Charon displays tectonics and evidence for a heterogeneous crustal composition; its north pole displays puzzling dark terrain. Small satellites Hydra and Nix have higher albedos than expected.\", 'year': 2015, 'citationCount': 366, 'authors': [{'authorId': '145615615', 'name': 'S. Stern'}, {'authorId': '2596024', 'name': 'F. Bagenal'}, {'authorId': '144810318', 'name': 'K. Ennico'}, {'authorId': '144398834', 'name': 'G. Gladstone'}, {'authorId': '144079258', 'name': 'W. Grundy'}, {'authorId': '145741171', 'name': 'W. McKinnon'}, {'authorId': '38254933', 'name': 'J. Moore'}, {'authorId': '5752279', 'name': 'C. Olkin'}, {'authorId': '145837796', 'name': 'J. Spencer'}, {'authorId': '4612630', 'name': 'H. Weaver'}, {'authorId': '144667642', 'name': 'L. Young'}, {'authorId': '35561159', 'name': 'T. Andert'}, {'authorId': '48457533', 'name': 'J. Andrews'}, {'authorId': '36631430', 'name': 'M. Banks'}, {'authorId': '122579257', 'name': 'B. Bauer'}, {'authorId': '101412277', 'name': 'J. Bauman'}, {'authorId': '6895822', 'name': 'O. Barnouin'}, {'authorId': '5484323', 'name': 'P. Bedini'}, {'authorId': '3733800', 'name': 'K. Beisser'}, {'authorId': '49503161', 'name': 'R. Beyer'}, {'authorId': '1813837', 'name': 'S. Bhaskaran'}, {'authorId': '6270608', 'name': 'R. Binzel'}, {'authorId': '4234897', 'name': 'E. Birath'}, {'authorId': '34057526', 'name': 'M. Bird'}, {'authorId': '90120170', 'name': 'D. Bogan'}, {'authorId': '48789119', 'name': 'A. Bowman'}, {'authorId': '47621602', 'name': 'V. Bray'}, {'authorId': '49509606', 'name': 'M. Brozović'}, {'authorId': '146338036', 'name': 'C. Bryan'}, {'authorId': '2106296101', 'name': 'M. R. Buckley'}, {'authorId': '2823138', 'name': 'M. Buie'}, {'authorId': '6498124', 'name': 'B. Buratti'}, {'authorId': '93963585', 'name': 'S. Bushman'}, {'authorId': '70707164', 'name': 'A. Calloway'}, {'authorId': '47589465', 'name': 'B. Carcich'}, {'authorId': '32584940', 'name': 'A. Cheng'}, {'authorId': '8189338', 'name': 'S. Conard'}, {'authorId': '91092942', 'name': 'C. A. Conrad'}, {'authorId': '89089970', 'name': 'J. Cook'}, {'authorId': '144051326', 'name': 'D. Cruikshank'}, {'authorId': '116776624', 'name': 'O. Custodio'}, {'authorId': '88728462', 'name': 'C. D. Dalle Ore'}, {'authorId': '35228553', 'name': 'C. Deboy'}, {'authorId': '6685622', 'name': 'Z. Dischner'}, {'authorId': '2069078045', 'name': 'P. Dumont'}, {'authorId': '50377339', 'name': 'A. Earle'}, {'authorId': '49039752', 'name': 'H. Elliott'}, {'authorId': '115690081', 'name': 'J. Ercol'}, {'authorId': '33804631', 'name': 'C. Ernst'}, {'authorId': '48746614', 'name': 'T. Finley'}, {'authorId': '101653683', 'name': 'S. H. Flanigan'}, {'authorId': '3260212', 'name': 'G. Fountain'}, {'authorId': '1698527041', 'name': 'M. Freeze'}, {'authorId': '4681394', 'name': 'T. Greathouse'}, {'authorId': '30999144', 'name': 'J. Green'}, {'authorId': '2107822296', 'name': 'Y. Guo'}, {'authorId': '46686185', 'name': 'M. Hahn'}, {'authorId': '2228671193', 'name': 'D. Hamilton'}, {'authorId': '2069334378', 'name': 'S. Hamilton'}, {'authorId': '108780783', 'name': 'J. Hanley'}, {'authorId': '5123252', 'name': 'A. Harch'}, {'authorId': '103038277', 'name': 'H. Hart'}, {'authorId': '65987131', 'name': 'C. Hersman'}, {'authorId': '2072855102', 'name': 'A. Hill'}, {'authorId': '15621312', 'name': 'M. Hill'}, {'authorId': '5610936', 'name': 'D. Hinson'}, {'authorId': '51055187', 'name': 'M. Holdridge'}, {'authorId': '48736351', 'name': 'M. Horányi'}, {'authorId': '143842010', 'name': 'A. Howard'}, {'authorId': '144867456', 'name': 'C. Howett'}, {'authorId': '71251275', 'name': 'C. Jackman'}, {'authorId': '143826218', 'name': 'R. Jacobson'}, {'authorId': '3114626', 'name': 'D. Jennings'}, {'authorId': '40198370', 'name': 'J. Kammer'}, {'authorId': '2206975340', 'name': 'H. K. Kang'}, {'authorId': '13974518', 'name': 'D. Kaufmann'}, {'authorId': '3935671', 'name': 'P. Kollmann'}, {'authorId': '145312312', 'name': 'S. Krimigis'}, {'authorId': '4762243', 'name': 'D. Kusnierkiewicz'}, {'authorId': '2369234', 'name': 'T. Lauer'}, {'authorId': '2153272132', 'name': 'J. E. Lee'}, {'authorId': '117685933', 'name': 'K. L. Lindstrom'}, {'authorId': '2826609', 'name': 'I. Linscott'}, {'authorId': '52561266', 'name': 'C. Lisse'}, {'authorId': '145665823', 'name': 'A. Lunsford'}, {'authorId': '113643607', 'name': 'V. Mallder'}, {'authorId': '2068266177', 'name': 'N. Martin'}, {'authorId': '145390435', 'name': 'D. McComas'}, {'authorId': '2400562', 'name': 'R. McNutt'}, {'authorId': '6564626', 'name': 'D. Mehoke'}, {'authorId': '3109678', 'name': 'T. Mehoke'}, {'authorId': '2083034634', 'name': 'E. Melin'}, {'authorId': '50702037', 'name': 'M. Mutchler'}, {'authorId': '115517410', 'name': 'D. Nelson'}, {'authorId': '4977917', 'name': 'F. Nimmo'}, {'authorId': '46852290', 'name': 'J. Núñez'}, {'authorId': '40149438', 'name': 'A. Ocampo'}, {'authorId': '4780988', 'name': 'W. Owen'}, {'authorId': '152493032', 'name': 'M. Paetzold'}, {'authorId': '144330595', 'name': 'B. Page'}, {'authorId': '14641808', 'name': 'A. Parker'}, {'authorId': '40542088', 'name': 'J. Parker'}, {'authorId': '114342249', 'name': 'F. Pelletier'}, {'authorId': '50501427', 'name': 'J. Peterson'}, {'authorId': '6664207', 'name': 'N. Pinkine'}, {'authorId': '4203985', 'name': 'M. Piquette'}, {'authorId': '144697219', 'name': 'S. Porter'}, {'authorId': '5375164', 'name': 'S. Protopapa'}, {'authorId': '40485883', 'name': 'J. Redfern'}, {'authorId': '6467444', 'name': 'H. Reitsema'}, {'authorId': '6896152', 'name': 'D. Reuter'}, {'authorId': '2107414758', 'name': 'J. Roberts'}, {'authorId': '33221375', 'name': 'S. Robbins'}, {'authorId': '46267028', 'name': 'G. Rogers'}, {'authorId': '108196123', 'name': 'D. Rose'}, {'authorId': '4166509', 'name': 'K. Runyon'}, {'authorId': '4348335', 'name': 'K. Retherford'}, {'authorId': '94990717', 'name': 'M. Ryschkewitsch'}, {'authorId': '144956598', 'name': 'P. Schenk'}, {'authorId': '49278836', 'name': 'E. Schindhelm'}, {'authorId': '114943626', 'name': 'B. Sepan'}, {'authorId': '92434756', 'name': 'M. Showalter'}, {'authorId': '143953436', 'name': 'K. Singer'}, {'authorId': '104081149', 'name': 'M. Soluri'}, {'authorId': '100745778', 'name': 'D. Stanbridge'}, {'authorId': '4807386', 'name': 'A. Steffl'}, {'authorId': '32410104', 'name': 'D. Strobel'}, {'authorId': '5600908', 'name': 'T. Stryk'}, {'authorId': '50377583', 'name': 'M. Summers'}, {'authorId': '152916383', 'name': 'J. Szalay'}, {'authorId': '31784508', 'name': 'M. Tapley'}, {'authorId': '145592784', 'name': 'A. Taylor'}, {'authorId': '144173680', 'name': 'H. Taylor'}, {'authorId': '4232734', 'name': 'H. Throop'}, {'authorId': '31790664', 'name': 'C. Tsang'}, {'authorId': '2998858', 'name': 'G. L. Tyler'}, {'authorId': '5209100', 'name': 'O. Umurhan'}, {'authorId': '4377111', 'name': 'A. Verbiscer'}, {'authorId': '119939518', 'name': 'M. Versteeg'}, {'authorId': '153301791', 'name': 'M. Vincent'}, {'authorId': '115199046', 'name': 'R. Webbert'}, {'authorId': '104248787', 'name': 'S. Weidner'}, {'authorId': '48758610', 'name': 'G. Weigle'}, {'authorId': '1397577620', 'name': 'O. White'}, {'authorId': '51055218', 'name': 'K. Whittenburg'}, {'authorId': '48964745', 'name': 'B. Williams'}, {'authorId': '2118657719', 'name': 'K. Williams'}, {'authorId': '2107720028', 'name': 'S. Williams'}, {'authorId': '103414780', 'name': 'W. W. Woods'}, {'authorId': '48228254', 'name': 'A. Zangari'}, {'authorId': '152732919', 'name': 'E. Zirnstein'}]}, {'paperId': 'f443476b7c84200a3afb503dad5a67fd0d5a397e', 'title': 'Committing to ecological restoration', 'abstract': 'Efforts around the globe need legal and policy clarification At the September 2014 United Nations Climate Summit, governments rallied around an international agreement—the New York Declaration on Forests—that underscored restoration of degraded ecosystems as an auspicious solution to climate change. Ethiopia committed to restore more than one-sixth of its land. Uganda, the Democratic Republic of Congo, Guatemala, and Colombia pledged to restore huge areas within their borders. In total, parties committed to restore a staggering 350 million hectares by 2030.', 'year': 2015, 'citationCount': 371, 'authors': [{'authorId': '4723097', 'name': 'K. Suding'}, {'authorId': '50281294', 'name': 'E. Higgs'}, {'authorId': '2722335', 'name': 'M. Palmer'}, {'authorId': '8466219', 'name': 'J. B. Callicott'}, {'authorId': '46830133', 'name': 'C. Anderson'}, {'authorId': '2067798096', 'name': 'M. Baker'}, {'authorId': '6663611', 'name': 'J. Gutrich'}, {'authorId': '6180776', 'name': 'K. Hondula'}, {'authorId': '7302071', 'name': 'M. LaFevor'}, {'authorId': '144773504', 'name': 'B. Larson'}, {'authorId': '49125009', 'name': 'A. Randall'}, {'authorId': '123493788', 'name': 'Alan Randall'}, {'authorId': '98277763', 'name': 'J. Ruhl'}, {'authorId': '34264152', 'name': 'Katrina Z. S. Schwartz'}]}, {'paperId': 'fa9d10870a42106cf74cd3a6a97e44581ecef6c8', 'title': 'Asymmetric apportioning of aged mitochondria between daughter cells is required for stemness', 'abstract': 'Stem cells can sort mitochondria by age The renewal of tissues in aging organisms requires stem cells, which have the unusual ability to divide asymmetrically into one daughter cell that retains stem cell properties and another that differentiates into a particular tissue type. Katajisto et al. used photoactivated marker proteins to monitor the age of cell organelles in stemlike cells from human breast tissue and their distribution into daughter cells. Most organelles were evenly distributed, but daughter cells that maintained stem-cell properties received more newly produced mitochondria and fewer old ones. Science, this issue p. 340 Stem cells preferentially select new rather than old mitochondria as they divide. By dividing asymmetrically, stem cells can generate two daughter cells with distinct fates. However, evidence is limited in mammalian systems for the selective apportioning of subcellular contents between daughters. We followed the fates of old and young organelles during the division of human mammary stemlike cells and found that such cells apportion aged mitochondria asymmetrically between daughter cells. Daughter cells that received fewer old mitochondria maintained stem cell traits. Inhibition of mitochondrial fission disrupted both the age-dependent subcellular localization and segregation of mitochondria and caused loss of stem cell properties in the progeny cells. Hence, mechanisms exist for mammalian stemlike cells to asymmetrically sort aged and young mitochondria, and these are important for maintaining stemness properties.', 'year': 2015, 'citationCount': 428, 'authors': [{'authorId': '3581044', 'name': 'P. Katajisto'}, {'authorId': '8093966', 'name': 'Julia Döhla'}, {'authorId': '4870813', 'name': 'Christine L. Chaffer'}, {'authorId': '4301484', 'name': 'Nalle Pentinmikko'}, {'authorId': '34608185', 'name': 'N. Marjanovic'}, {'authorId': '7878948', 'name': 'Sharif Iqbal'}, {'authorId': '3034790', 'name': 'R. Zoncu'}, {'authorId': '47482597', 'name': 'Walter W. Chen'}, {'authorId': '34628374', 'name': 'R. Weinberg'}, {'authorId': '1843680', 'name': 'D. Sabatini'}]}, {'paperId': 'e68f8737cbabd33366c5d9af83279208b9d99d14', 'title': 'RIPK1 and NF-κB signaling in dying cells determines cross-priming of CD8+ T cells', 'abstract': 'Dying to impress the immune system Besides reacting to microbes, T cells can also mount immune responses to fragments of dying cells, which they encounter displayed on dendritic cells. Not all dying cells activate T cells, however, so what differentiates the dying cells that do? Yatim et al. studied two forms of programmed cells death: apoptosis and necroptosis. Using mouse cells in culture and mouse models of inflammatory cell death and anti-tumor immunity, they found that programmed cell death initiated T cell immunity only when the dying cells signaled through the enzyme RIPK1 and the transcription factor NF-κB. Science, this issue p. 328 Active signaling in dying cells determines their immunogenicity. Dying cells initiate adaptive immunity by providing both antigens and inflammatory stimuli for dendritic cells, which in turn activate CD8+ T cells through a process called antigen cross-priming. To define how different forms of programmed cell death influence immunity, we established models of necroptosis and apoptosis, in which dying cells are generated by receptor-interacting protein kinase-3 and caspase-8 dimerization, respectively. We found that the release of inflammatory mediators, such as damage-associated molecular patterns, by dying cells was not sufficient for CD8+ T cell cross-priming. Instead, robust cross-priming required receptor-interacting protein kinase-1 (RIPK1) signaling and nuclear factor κB (NF-κB)–induced transcription within dying cells. Decoupling NF-κB signaling from necroptosis or inflammatory apoptosis reduced priming efficiency and tumor immunity. Our results reveal that coordinated inflammatory and cell death signaling pathways within dying cells orchestrate adaptive immunity.', 'year': 2015, 'citationCount': 417, 'authors': [{'authorId': '115263635', 'name': 'N. Yatim'}, {'authorId': '1399772482', 'name': 'H. Jusforgues-Saklani'}, {'authorId': '46949224', 'name': 'S. Orozco'}, {'authorId': '34272292', 'name': 'O. Schulz'}, {'authorId': '39319949', 'name': 'R. Barreira da Silva'}, {'authorId': '5531706', 'name': 'C. Reis e Sousa'}, {'authorId': '3792271', 'name': 'D. Green'}, {'authorId': '6678698', 'name': 'A. Oberst'}, {'authorId': '21068591', 'name': 'M. Albert'}]}, {'paperId': 'bd795f6c19098963b32a38c52e7b4b91f9fc565b', 'title': 'Genomic signatures of evolutionary transitions from solitary to group living', 'abstract': 'For bees, many roads lead to social harmony Eusociality, where workers sacrifice their reproductive rights to support the colony, has evolved repeatedly and represents the most evolved form of social evolution in insects. Kapheim et al. looked across the genomes of 10 bee species with varying degrees of sociality to determine the underlying genomic contributions. No one genomic path led to eusociality, but similarities across genomes were seen in features such as increases in gene regulation and methylation. It also seems that selection pressures relaxed after the emergence of complex sociality. Science, this issue p. 1139 Social evolution in bees has followed diverse genomic paths but shares genomic patterns. The evolution of eusociality is one of the major transitions in evolution, but the underlying genomic changes are unknown. We compared the genomes of 10 bee species that vary in social complexity, representing multiple independent transitions in social evolution, and report three major findings. First, many important genes show evidence of neutral evolution as a consequence of relaxed selection with increasing social complexity. Second, there is no single road map to eusociality; independent evolutionary transitions in sociality have independent genetic underpinnings. Third, though clearly independent in detail, these transitions do have similar general features, including an increase in constrained protein evolution accompanied by increases in the potential for gene regulation and decreases in diversity and abundance of transposable elements. Eusociality may arise through different mechanisms each time, but would likely always involve an increase in the complexity of gene networks.', 'year': 2015, 'citationCount': 325, 'authors': [{'authorId': '6011535', 'name': 'K. Kapheim'}, {'authorId': '34884098', 'name': 'Hailin Pan'}, {'authorId': '46651780', 'name': 'Cai Li'}, {'authorId': '1744109', 'name': 'S. Salzberg'}, {'authorId': '1810831', 'name': 'D. Puiu'}, {'authorId': '49485917', 'name': 'Tanja Magoc'}, {'authorId': '2162158443', 'name': 'H. Robertson'}, {'authorId': '40148862', 'name': 'M. Hudson'}, {'authorId': '6228035', 'name': 'A. Venkat'}, {'authorId': '4482221', 'name': 'Brielle J. Fischman'}, {'authorId': '145338796', 'name': 'Alvaro G. Hernandez'}, {'authorId': '1968099', 'name': 'M. Yandell'}, {'authorId': '2021675', 'name': 'Daniel Ence'}, {'authorId': '144696702', 'name': 'C. Holt'}, {'authorId': '5174447', 'name': 'G. Yocum'}, {'authorId': '16489526', 'name': 'W. Kemp'}, {'authorId': '46786576', 'name': 'J. Bosch'}, {'authorId': '3008998', 'name': 'R. Waterhouse'}, {'authorId': '2182484', 'name': 'E. Zdobnov'}, {'authorId': '4791609', 'name': 'Eckart Stolle'}, {'authorId': '1783900', 'name': 'F. B. Kraus'}, {'authorId': '7004510', 'name': 'Sophie Helbing'}, {'authorId': '47046110', 'name': 'R. Moritz'}, {'authorId': '5303807', 'name': 'K. Glastad'}, {'authorId': '4745746', 'name': 'B. Hunt'}, {'authorId': '3887545', 'name': 'M. Goodisman'}, {'authorId': '46526157', 'name': 'F. Hauser'}, {'authorId': '4223070', 'name': 'C. Grimmelikhuijzen'}, {'authorId': '3119104', 'name': 'D. G. Pinheiro'}, {'authorId': '2083804782', 'name': 'F. M. Nunes'}, {'authorId': '4272557', 'name': 'M. P. Soares'}, {'authorId': '82826483', 'name': 'É. D. Tanaka'}, {'authorId': '90243733', 'name': 'Z. Simões'}, {'authorId': '6807475', 'name': 'K. Hartfelder'}, {'authorId': '2488538', 'name': 'J. Evans'}, {'authorId': '4959699', 'name': 'S. Barribeau'}, {'authorId': '2108887689', 'name': 'Reed M. Johnson'}, {'authorId': '87813874', 'name': 'J. Massey'}, {'authorId': '1795501', 'name': 'B. Southey'}, {'authorId': '47357198', 'name': 'M. Hasselmann'}, {'authorId': '2307512', 'name': 'D. Hamacher'}, {'authorId': '32447626', 'name': 'M. Biewer'}, {'authorId': '38221616', 'name': 'C. Kent'}, {'authorId': '37301230', 'name': 'A. Zayed'}, {'authorId': '3077932', 'name': 'C. Blatti'}, {'authorId': '2162164055', 'name': 'S. Sinha'}, {'authorId': '2228046255', 'name': 'J. Johnston'}, {'authorId': '4934449', 'name': 'Shawn J. Hanrahan'}, {'authorId': '4733082', 'name': 'Sarah D. Kocher'}, {'authorId': '40579687', 'name': 'Jun Wang'}, {'authorId': '2779588', 'name': 'G. Robinson'}, {'authorId': '2166645736', 'name': 'Guojie Zhang'}]}, {'paperId': '08815884a34551e292aa4b408500c201109bfd97', 'title': 'Slip pulse and resonance of the Kathmandu basin during the 2015 Gorkha earthquake, Nepal', 'abstract': 'The bigger they are, the harder they fall The magnitude 7.8 Gorkha earthquake hit Nepal on 25 April 2015. The earthquake killed thousands and caused great damage. Galetzka et al. determined how the fault that caused this earthquake ruptured. The rupture showed a smooth slip pulse 20 km wide that moved eastward along the fault over about 6 s. The nature of the rupture limited damage to regular dwellings but generated shaking that collapsed taller structures. Science, this issue p. 1091 Continuous GPS and InSAR measurements record slip on the fault responsible for the 2015 Mw 7.8 Gorkha earthquake in Nepal. Detailed geodetic imaging of earthquake ruptures enhances our understanding of earthquake physics and associated ground shaking. The 25 April 2015 moment magnitude 7.8 earthquake in Gorkha, Nepal was the first large continental megathrust rupture to have occurred beneath a high-rate (5-hertz) Global Positioning System (GPS) network. We used GPS and interferometric synthetic aperture radar data to model the earthquake rupture as a slip pulse ~20 kilometers in width, ~6 seconds in duration, and with a peak sliding velocity of 1.1 meters per second, which propagated toward the Kathmandu basin at ~3.3 kilometers per second over ~140 kilometers. The smooth slip onset, indicating a large (~5-meter) slip-weakening distance, caused moderate ground shaking at high frequencies (>1 hertz; peak ground acceleration, ~16% of Earth’s gravity) and minimized damage to vernacular dwellings. Whole-basin resonance at a period of 4 to 5 seconds caused the collapse of tall structures, including cultural artifacts.', 'year': 2015, 'citationCount': 292, 'authors': [{'authorId': '4424939', 'name': 'J. Galetzka'}, {'authorId': '97318651', 'name': 'D. Melgar'}, {'authorId': '6525824', 'name': 'J. Genrich'}, {'authorId': '2067256', 'name': 'J. Geng'}, {'authorId': '50335261', 'name': 'S. Owen'}, {'authorId': '35220108', 'name': 'E. Lindsey'}, {'authorId': '40511842', 'name': 'Xiaohua Xu'}, {'authorId': '8636234', 'name': 'Y. Bock'}, {'authorId': '2991483', 'name': 'J. Avouac'}, {'authorId': '2991483', 'name': 'J. Avouac'}, {'authorId': '31647886', 'name': 'L. Adhikari'}, {'authorId': '46256223', 'name': 'B. Upreti'}, {'authorId': '1404929646', 'name': 'B. Pratt-Sitaula'}, {'authorId': '72539811', 'name': 'T. Bhattarai'}, {'authorId': '41220903', 'name': 'B. P. Sitaula'}, {'authorId': '32767683', 'name': 'A. Moore'}, {'authorId': '5757269', 'name': 'K. Hudnut'}, {'authorId': '34772022', 'name': 'W. Szeliga'}, {'authorId': '12632658', 'name': 'J. Normandeau'}, {'authorId': '112908957', 'name': 'M. Fend'}, {'authorId': '5403146', 'name': 'M. Flouzat'}, {'authorId': '144290187', 'name': 'L. Bollinger'}, {'authorId': '2061030288', 'name': 'P. Shrestha'}, {'authorId': '31545592', 'name': 'B. P. Koirala'}, {'authorId': '13457014', 'name': 'U. Gautam'}, {'authorId': '8476221', 'name': 'M. Bhatterai'}, {'authorId': '94190859', 'name': 'R. Gupta'}, {'authorId': '104873910', 'name': 'T. Kandel'}, {'authorId': '8130744', 'name': 'C. Timsina'}, {'authorId': '5548401', 'name': 'S. Sapkota'}, {'authorId': '4446887', 'name': 'S. Rajaure'}, {'authorId': '2065905236', 'name': 'N. Maharjan'}]}, {'paperId': '26c414be169fcb7ec90f18696beb5729a3815347', 'title': 'Chemiexcitation of melanin derivatives induces DNA photoproducts long after UV exposure', 'abstract': \"The dark side of melanin exposed Sun worshippers may have more to worry about than the DNA damage that occurs while they're relaxing on the beach. It seems that the DNA photoproducts responsible for cancer-causing mutations in skin cells continue to be generated for hours after sunlight exposure. Premi et al. find that a key mediator of this delayed damage is melanin, a pigment thought to protect against cancer (see the Perspective by Taylor). They propose a “chemiexcitation” model in which reactive oxygen and nitrogen species induced by ultraviolet light excite an electron in melanin fragments. This energy is then transferred to DNA, inducing the same damage as ultraviolet light, but in the dark. Conceivably, this energy could be dissipated by adding quenchers to sunscreens. Science, this issue p. 842; see also p. 824 Melanin, a pigment thought to protect against skin cancer, contributes to DNA damage occurring after ultraviolet exposure. [Also see Perspective by Taylor] Mutations in sunlight-induced melanoma arise from cyclobutane pyrimidine dimers (CPDs), DNA photoproducts that are typically created picoseconds after an ultraviolet (UV) photon is absorbed at thymine or cytosine. We found that in melanocytes, CPDs are generated for >3 hours after exposure to UVA, a major component of the radiation in sunlight and in tanning beds. These “dark CPDs” constitute the majority of CPDs and include the cytosine-containing CPDs that initiate UV-signature C→T mutations. Dark CPDs arise when UV-induced reactive oxygen and nitrogen species combine to excite an electron in fragments of the pigment melanin. This creates a quantum triplet state that has the energy of a UV photon but induces CPDs by energy transfer to DNA in a radiation-independent manner. Melanin may thus be carcinogenic as well as protective against cancer. These findings also validate the long-standing suggestion that chemically generated excited electronic states are relevant to mammalian biology.\", 'year': 2015, 'citationCount': 386, 'authors': [{'authorId': '3672160', 'name': 'S. Premi'}, {'authorId': '14204647', 'name': 'S. Wallisch'}, {'authorId': '6987851', 'name': 'C. M. Mano'}, {'authorId': '3970174', 'name': 'A. Weiner'}, {'authorId': '4812673', 'name': 'Antonella Bacchiocchi'}, {'authorId': '4426667', 'name': 'K. Wakamatsu'}, {'authorId': '143665266', 'name': 'E. Bechara'}, {'authorId': '7741232', 'name': 'R. Halaban'}, {'authorId': '4130691', 'name': 'T. Douki'}, {'authorId': '1939904', 'name': 'D. Brash'}]}, {'paperId': '3cb46fad2a71ab3958eac67e0c0396524b8ac5b0', 'title': 'Nanomaterials in transistors: From high-performance to thin-film applications', 'abstract': 'Improving transistors with nanomaterials High-performance silicon transistors and thin-film transistors used in display technologies are fundamentally limited to miniaturization. Incorporating nanomaterials—such as carbon nanotubes, graphene, and related two-dimensional materials like molybdenum disulfide—into these devices as gate materials may circumvent some of these limitations. Franklin reviews the opportunities and challenges for incorporating nanomaterials into transistors to improve performance. Because high-performance transistors are distinct from thin-film transistors, incorporating them into flexible or transparent platforms raises new challenges. Science, this issue 10.1126/science.aab2750 BACKGROUND Transistors are one of the most enabling “hidden” technologies of all time and have facilitated the development of computers, the Internet, thin mobile displays, and much more. Silicon, which has been the material of choice for transistors in nearly every application for decades, is now reaching the fundamental limits to what it can offer for future transistor technologies. The newest display technologies are already turning to metal oxide materials, such as indium gallium zinc oxide (IGZO), for the improvements needed to drive organic light-emitting diodes. Ranging from applications such as display backplanes to high-performance microprocessors for servers, nanomaterials offer lasting advantages for the coming decades of transistor technologies. In this Review, the advantages of nanomaterials are discussed in the context of different transistor applications, along with the breakthroughs needed for nanomaterial transistors to enable the next generation of technological advancement. ADVANCES About 15 years ago, nanomaterials began receiving focused attention for transistors. Carbon nanotubes—molecules consisting of a cylindrical single layer of carbon atoms arranged in a hexagonal lattice—were the first to be given serious consideration, and their benefits quickly became widely acclaimed. Given their ability to transport electrical current with near-zero resistance, even at room temperature, the explosion of interest in nanotubes for electronics was understandable. Graphene, a related allotrope of carbon, benefited from the expansive interest carbon nanotubes had created for nanomaterial electronics. Although graphene transistors eventually proved less viable for digital applications, owing to the absence of an energy band gap, the excitement over graphene ushered in a complete revolution of interest in similar two-dimensional materials. Now, transition metal dichalcogenides and the so-called X-ene family of nanomaterials (e.g., silicence, phosphorene) dominate the attention of the nanoelectronics community. Hardly a day goes by without a paper being published on some advancement related to the use of nanomaterials in transistors. Hence, this Review focuses on how to keep such progress in the proper context with respect to the target transistor application, as well as the consideration of nanomaterials for completely new application spaces. OUTLOOK The benefits and practicality differ for each nanomaterial, and varied amounts of progress have been made in considering each of them for transistors. In just a few short years, thousands of papers have been published on improving synthesis or demonstrating simple functions of the newer nanomaterials. However, reflection on whether their newness translates to actual superiority over other options is warranted. Clearly, all of the nanomaterial possibilities offer certain advantages for future transistor technologies, but some do so with fewer caveats than others. Future research will benefit from keeping scientific advancement of nanomaterial transistors in line with end-goal deliverables. Overall, considering that only 15 years have elapsed since the study of nanomaterials for transistors began in earnest, the toolbox of available options and the developments toward overcoming challenges are promising. Technologies enabled by high-performance and thin-film transistors over the past 25 years. (Top) Silicon transistors have driven the microprocessors used in computational devices ranging from low-power gadgets to large servers. (Bottom) Various forms of cheaper silicon enabled the display revolution, now being shared by IGZO. (Right) Nanomaterials may be the next transistor material for enabling a new generation of technologies. CREDITS: Old computer: Wikimedia Commons/Creative Commons; Other computer: J.Dray/Flickr; Laptop: C. Berkeley/Flickr; Supercomputer: Wikimedia Commons/Creative Commons; Smartphone: Pixabay/Creative Commons; Camera: DAYJOY/Flickr; FLAT SCREEN TV: AV Hire London/Flickr; Curved TV: SamsungTomorrow/Flickr For more than 50 years, silicon transistors have been continuously shrunk to meet the projections of Moore’s law but are now reaching fundamental limits on speed and power use. With these limits at hand, nanomaterials offer great promise for improving transistor performance and adding new applications through the coming decades. With different transistors needed in everything from high-performance servers to thin-film display backplanes, it is important to understand the targeted application needs when considering new material options. Here the distinction between high-performance and thin-film transistors is reviewed, along with the benefits and challenges to using nanomaterials in such transistors. In particular, progress on carbon nanotubes, as well as graphene and related materials (including transition metal dichalcogenides and X-enes), outlines the advances and further research needed to enable their use in transistors for high-performance computing, thin films, or completely new technologies such as flexible and transparent devices.', 'year': 2015, 'citationCount': 449, 'authors': [{'authorId': '3752052', 'name': 'A. Franklin'}]}, {'paperId': 'a9b91fed8118feb8e75f6459812155f2e45f200b', 'title': 'Dynamic profiling of the protein life cycle in response to pathogens', 'abstract': 'How the immune system readies for battle Although gene expression is tightly controlled at both the RNA and protein levels, the quantitative contribution of each step, especially during dynamic responses, remains largely unknown. Indeed, there has been much debate whether changes in RNA level contribute substantially to protein-level regulation. Jovanovic et al. built a genome-scale model of the temporal dynamics of differential protein expression during the stimulation of immunological dendritic cells (see the Perspective by Li and Biggin). Newly stimulated functions involved the up-regulation of specific RNAs and concomitant increases in the levels of the proteins they encode, whereas housekeeping functions were regulated posttranscriptionally at the protein level. Science, this issue 10.1126/science.1259038; see also p. 1066 Levels of “housekeeping” proteins are maintained directly, but those of immune response proteins depend on more transcription. [Also see Perspective by Li and Biggin] INTRODUCTION Mammalian gene expression is tightly controlled through the interplay between the RNA and protein life cycles. Although studies of individual genes have shown that regulation of each of these processes is important for correct protein expression, the quantitative contribution of each step to changes in protein expression levels remains largely unknown and much debated. Many studies have attempted to address this question in the context of steady-state protein levels, and comparing steady-state RNA and protein abundances has indicated a considerable discrepancy between RNA and protein levels. In contrast, only a few studies have attempted to shed light on how changes in each of these processes determine differential protein expression—either relative (ratios) or absolute (differences)—during dynamic responses, and only one recent report has attempted to quantitate each process. Understanding these contributions to a dynamic response on a systems scale is essential both for deciphering how cells deploy regulatory processes to accomplish physiological changes and for discovering key molecular regulators controlling each process. RATIONALE We developed an integrated experimental and computational strategy to quantitatively assess how protein levels are maintained in the context of a dynamic response and applied it to the model response of mouse immune bone marrow–derived dendritic cells (DCs) to stimulation with lipopolysaccharide (LPS). We used a modified pulsed-SILAC (stable isotope labeling with amino acids in cell culture) approach to track newly synthesized and previously labeled proteins over the first 12 hours of the response. In addition, we independently measured replicate RNA-sequencing profiles under the same conditions. We devised a computational strategy to infer per-mRNA translation rates and protein degradation rates at each time point from the temporal transcriptional profiles and pulsed-SILAC proteomics data. This allowed us to build a genome-scale quantitative model of the temporal dynamics of differential protein expression in DCs responding to LPS. RESULTS We found that before stimulation, mRNA levels contribute to overall protein expression levels more than double the combined contribution of protein translation and degradation rates. Upon LPS stimulation, changes in mRNA abundance play an even more dominant role in dynamic changes in protein levels, especially in immune response genes. Nevertheless, several protein modules—especially the preexisting proteome of proteins performing basic cellular functions—are predominantly regulated in stimulated cells at the level of protein translation or degradation, accounting for over half of the absolute change in protein molecules in the cell. In particular, despite the repression of their transcripts, the level of many proteins in the translational machinery is up-regulated upon LPS stimulation because of significantly increased translation rates, and elevated protein degradation of mitochondrial proteins plays a central role in remodeling cellular energy metabolism. CONCLUSIONS Our results support a model in which the induction of novel cellular functions is primarily driven through transcriptional changes, whereas regulation of protein production or degradation updates the levels of preexisting functions as required for an activated state. Our approach for building quantitative genome-scale models of the temporal dynamics of protein expression is broadly applicable to other dynamic systems. Dynamic protein expression regulation in dendritic cells upon stimulation with LPS. We developed an integrated experimental and computational strategy to quantitatively assess how protein levels are maintained in the context of a dynamic response. Our results support a model in which the induction of novel cellular functions is primarily driven through transcriptional changes, whereas regulation of protein production or degradation updates the levels of preexisting functions. Protein expression is regulated by the production and degradation of messenger RNAs (mRNAs) and proteins, but their specific relationships remain unknown. We combine measurements of protein production and degradation and mRNA dynamics so as to build a quantitative genomic model of the differential regulation of gene expression in lipopolysaccharide-stimulated mouse dendritic cells. Changes in mRNA abundance play a dominant role in determining most dynamic fold changes in protein levels. Conversely, the preexisting proteome of proteins performing basic cellular functions is remodeled primarily through changes in protein production or degradation, accounting for more than half of the absolute change in protein molecules in the cell. Thus, the proteome is regulated by transcriptional induction for newly activated cellular functions and by protein life-cycle changes for remodeling of preexisting functions.', 'year': 2015, 'citationCount': 392, 'authors': [{'authorId': '48589378', 'name': 'M. Jovanović'}, {'authorId': '47454189', 'name': 'Michael S. Rooney'}, {'authorId': '4903346', 'name': 'Philipp Mertins'}, {'authorId': '1977398', 'name': 'Dariusz Przybylski'}, {'authorId': '152208347', 'name': 'Nicolas Chevrier'}, {'authorId': '50285728', 'name': 'R. Satija'}, {'authorId': '10273378', 'name': 'Edwin H. Rodriguez'}, {'authorId': '8356406', 'name': 'Alexander P. Fields'}, {'authorId': '40434210', 'name': 'Schraga Schwartz'}, {'authorId': '3556352', 'name': 'R. Raychowdhury'}, {'authorId': '4556276', 'name': 'Maxwell R Mumbach'}, {'authorId': '5974082', 'name': 'T. Eisenhaure'}, {'authorId': '5405703', 'name': 'M. Rabani'}, {'authorId': '4691526', 'name': 'David G Gennert'}, {'authorId': '5803380', 'name': 'Diana Lu'}, {'authorId': '47128726', 'name': 'T. Delorey'}, {'authorId': '3168827', 'name': 'J. Weissman'}, {'authorId': '144298908', 'name': 'S. Carr'}, {'authorId': '3860533', 'name': 'N. Hacohen'}, {'authorId': '144416712', 'name': 'A. Regev'}]}, {'paperId': '1585b76d97d4ee526471570cc92b582c8457ba4b', 'title': 'Nanophotonics: Shrinking light-based technology', 'abstract': 'The study of light at the nanoscale has become a vibrant field of research, as researchers now master the flow of light at length scales far below the optical wavelength, largely surpassing the classical limits imposed by diffraction. Using metallic and dielectric nanostructures precisely sculpted into two-dimensional (2D) and 3D nanoarchitectures, light can be scattered, refracted, confined, filtered, and processed in fascinating new ways that are impossible to achieve with natural materials and in conventional geometries. This control over light at the nanoscale has not only unveiled a plethora of new phenomena but has also led to a variety of relevant applications, including new venues for integrated circuitry, optical computing, solar, and medical technologies, setting high expectations for many novel discoveries in the years to come.', 'year': 2015, 'citationCount': 445, 'authors': [{'authorId': '6380838', 'name': 'A. Koenderink'}, {'authorId': '2186738926', 'name': 'A. Alú'}, {'authorId': '25343260', 'name': 'A. Polman'}]}, {'paperId': 'bb1081e7fe91e49d6d54b9eb42a25f08cc06b0ef', 'title': 'Group 3 innate lymphoid cells mediate intestinal selection of commensal bacteria–specific CD4+ T cells', 'abstract': 'Innate lymphoid cells keep gut T cells in check Trillions of bacteria inhabit our guts. So do many types of immune cells, including T cells, which might be expected to attack these bacteria. How, then, do our bodies manage to keep the peace? Working in mice, Hepworth et al. report one such mechanism. A population of immune cells, called innate lymphoid cells, directly killed CD4+ T cells that react to commensal gut microbes. Some of the specifics of this process parallel how the immune system keeps developing self-reactive T cells in check in the thymus. Furthermore, this peacekeeping process may be disrupted in children with inflammatory bowel disease. Science, this issue p. 1031 Innate lymphoid cells delete commensal bacteria–specific CD4+ T cells from the intestine in mice. Inflammatory CD4+ T cell responses to self or commensal bacteria underlie the pathogenesis of autoimmunity and inflammatory bowel disease (IBD), respectively. Although selection of self-specific T cells in the thymus limits responses to mammalian tissue antigens, the mechanisms that control selection of commensal bacteria–specific T cells remain poorly understood. Here, we demonstrate that group 3 innate lymphoid cell (ILC3)–intrinsic expression of major histocompatibility complex class II (MHCII) is regulated similarly to thymic epithelial cells and that MHCII+ ILC3s directly induce cell death of activated commensal bacteria–specific T cells. Further, MHCII on colonic ILC3s was reduced in pediatric IBD patients. Collectively, these results define a selection pathway for commensal bacteria–specific CD4+ T cells in the intestine and suggest that this process is dysregulated in human IBD.', 'year': 2015, 'citationCount': 390, 'authors': [{'authorId': '3015665', 'name': 'M. Hepworth'}, {'authorId': '3306458', 'name': 'Thomas C. Fung'}, {'authorId': '33150279', 'name': 'Samuel H. Masur'}, {'authorId': '5362885', 'name': 'J. Kelsen'}, {'authorId': '35639694', 'name': 'F. McConnell'}, {'authorId': '4204825', 'name': 'J. Dubrot'}, {'authorId': '2099622', 'name': 'D. Withers'}, {'authorId': '4533784', 'name': 'S. Hugues'}, {'authorId': '2689185', 'name': 'M. Farrar'}, {'authorId': '144538049', 'name': 'W. Reith'}, {'authorId': '5133043', 'name': 'G. Eberl'}, {'authorId': '5247138', 'name': 'R. Baldassano'}, {'authorId': '5284820', 'name': 'T. Laufer'}, {'authorId': '31931780', 'name': 'C. Elson'}, {'authorId': '5771393', 'name': 'Gregory F. Sonnenberg'}]}, {'paperId': '98ef072d4656ba9fa11b22900fc7e9c44c647706', 'title': 'The Symbiodinium kawagutii genome illuminates dinoflagellate gene expression and coral symbiosis', 'abstract': 'Symbionts are adapted to work with corals Many corals have formed mutualistic associations with dinoflagellate symbionts, which are thought to provide nutrients and other benefits. To examine the underlying genetics of this association, S. Lin et al. sequenced the genome of the endosymbiont dinoflagellate Symbiodinium kawagutii. The genome includes gene number expansions and encodes microRNAs that show complementarity to genes within the coral genome. Such microRNAs may be involved in regulating coral genes. Furthermore, coral and S. kawagutii appear to share homologs of genes encoding specific nutrient transporters. The findings shed light on how symbiosis is established and maintained between dinoflagellates and corals. Science, this issue p. 691 The genome of the coral symbiont Symbiodinium reveals fundamental aspects of the coral-alga symbiosis. Dinoflagellates are important components of marine ecosystems and essential coral symbionts, yet little is known about their genomes. We report here on the analysis of a high-quality assembly from the 1180-megabase genome of Symbiodinium kawagutii. We annotated protein-coding genes and identified Symbiodinium-specific gene families. No whole-genome duplication was observed, but instead we found active (retro)transposition and gene family expansion, especially in processes important for successful symbiosis with corals. We also documented genes potentially governing sexual reproduction and cyst formation, novel promoter elements, and a microRNA system potentially regulating gene expression in both symbiont and coral. We found biochemical complementarity between genomes of S. kawagutii and the anthozoan Acropora, indicative of host-symbiont coevolution, providing a resource for studying the molecular basis and evolution of coral symbiosis.', 'year': 2015, 'citationCount': 358, 'authors': [{'authorId': '1720742921', 'name': 'Senjie Lin'}, {'authorId': '38165443', 'name': 'Shifeng Cheng'}, {'authorId': '1398290474', 'name': 'Bo Song'}, {'authorId': '144137667', 'name': 'Xiao Zhong'}, {'authorId': '145122731', 'name': 'Xin Lin'}, {'authorId': '48624658', 'name': 'Wujiao Li'}, {'authorId': '1410384558', 'name': 'Ling Li'}, {'authorId': '121310673', 'name': 'Yaqun Zhang'}, {'authorId': '51505924', 'name': 'Huan Zhang'}, {'authorId': '89744571', 'name': 'Zhiliang Ji'}, {'authorId': '48629710', 'name': 'M. Cai'}, {'authorId': '38109162', 'name': 'Yunyun Zhuang'}, {'authorId': '5962195', 'name': 'Xinguo Shi'}, {'authorId': '5795772', 'name': 'Lingxiao Lin'}, {'authorId': '2153513358', 'name': 'Lu Wang'}, {'authorId': '48708968', 'name': 'Zhaobao Wang'}, {'authorId': '2230105370', 'name': 'Xin Liu'}, {'authorId': '2112274376', 'name': 'Sheng-li Yu'}, {'authorId': '47614770', 'name': 'Peng Zeng'}, {'authorId': '2065511559', 'name': 'H. Hao'}, {'authorId': '2054839914', 'name': 'Q. Zou'}, {'authorId': '80042641', 'name': 'Chengxuan Chen'}, {'authorId': '152998435', 'name': 'Yanjun Li'}, {'authorId': '2155514105', 'name': 'Ying Wang'}, {'authorId': '2115783836', 'name': 'Chunyan Xu'}, {'authorId': '2144278725', 'name': 'S. Meng'}, {'authorId': '2162171547', 'name': 'Xun Xu'}, {'authorId': '40579687', 'name': 'Jun Wang'}, {'authorId': '40566506', 'name': 'Huanming Yang'}, {'authorId': '1875961', 'name': 'D. Campbell'}, {'authorId': '6722875', 'name': 'N. Sturm'}, {'authorId': '1398757639', 'name': 'S. Dagenais-Bellefeuille'}, {'authorId': '145233479', 'name': 'D. Morse'}]}, {'paperId': 'a490336788276bb1cd0e0b18f62f72010e1f43f5', 'title': 'Antimicrobial peptide resistance mediates resilience of prominent gut commensals during inflammation', 'abstract': \"Gut microbes resist inflammation It is vital to human well-being that our gut microbiota can be distinguished from harmful, but often very similar, organisms. Cullen et al. begin to analyze how one dominant symbiont, Bacteroidetes thetaiotaomicron, does this. Our guts release potent antimicrobial peptides when we become infected with pathogenic bacteria such as salmonella, but these symbionts make an outer lipopolysaccharide coat that differs from those of pathogens by only one phosphate molecule. Enzymatic removal of this group is enough to confer resistance to the host's immune response and allow the symbionts to escape damage. Science, this issue p. 170 Gut commensals avoid elimination during inflammation via a single phosphate in lipopolysaccharide. Resilience to host inflammation and other perturbations is a fundamental property of gut microbial communities, yet the underlying mechanisms are not well understood. We have found that human gut microbes from all dominant phyla are resistant to high levels of inflammation-associated antimicrobial peptides (AMPs) and have identified a mechanism for lipopolysaccharide (LPS) modification in the phylum Bacteroidetes that increases AMP resistance by four orders of magnitude. Bacteroides thetaiotaomicron mutants that fail to remove a single phosphate group from their LPS were displaced from the microbiota during inflammation triggered by pathogen infection. These findings establish a mechanism that determines the stability of prominent members of a healthy microbiota during perturbation.\", 'year': 2015, 'citationCount': 302, 'authors': [{'authorId': '4531670', 'name': 'T. W. Cullen'}, {'authorId': '3963577', 'name': 'W. B. Schofield'}, {'authorId': '34717499', 'name': 'N. A. Barry'}, {'authorId': '11998377', 'name': 'E. E. Putnam'}, {'authorId': '5594197', 'name': 'E. Rundell'}, {'authorId': '144623783', 'name': 'M. Trent'}, {'authorId': '5781125', 'name': 'P. Degnan'}, {'authorId': '32574524', 'name': 'C. Booth'}, {'authorId': '2219967279', 'name': 'H. Yu'}, {'authorId': '144958203', 'name': 'A. Goodman'}]}, {'paperId': '1469da4446aaa4785b0c76ca1527f76efbdd34ec', 'title': 'Stable inhibitory activity of regulatory T cells requires the transcription factor Helios', 'abstract': 'How T cells maintain their identity Although best known for their pathogen-fighting prowess, T lymphocytes also ensure that the immune response does not run amok. A subset of T cells called regulatory T cells (Tregs) performs this function by, for example, making sure T cells only attack pathogens and not self. T cells can exhibit plasticity in their functions in the face of an inflammatory stimulus. Kim et al. sought to identify the molecules that ensure the stable maintenance of Tregs. Using genetically modified mice, they found that both CD4+ and CD8+ Tregs require the transcription factor Helios to stably maintain their identity. Science, this issue p. 334 Regulatory T cells require the transcription factor Helios to stably maintain their identity. The maintenance of immune homeostasis requires regulatory T cells (Tregs). Given their intrinsic self-reactivity, Tregs must stably maintain a suppressive phenotype to avoid autoimmunity. We report that impaired expression of the transcription factor (TF) Helios by FoxP3+ CD4 and Qa-1–restricted CD8 Tregs results in defective regulatory activity and autoimmunity in mice. Helios-deficient Tregs develop an unstable phenotype during inflammatory responses characterized by reduced FoxP3 expression and increased effector cytokine expression secondary to diminished activation of the STAT5 pathway. CD8 Tregs also require Helios-dependent STAT5 activation for survival and to prevent terminal T cell differentiation. The definition of Helios as a key transcription factor that stabilizes Tregs in the face of inflammatory responses provides a genetic explanation for a core property of Tregs.', 'year': 2015, 'citationCount': 304, 'authors': [{'authorId': '2155254510', 'name': 'Hye-Jung Kim'}, {'authorId': '6202837', 'name': 'R. A. Barnitz'}, {'authorId': '5853769', 'name': 'Taras Kreslavsky'}, {'authorId': '7644721', 'name': 'Flavian D. Brown'}, {'authorId': '5976390', 'name': 'H. Moffett'}, {'authorId': '2566050', 'name': 'M. Lemieux'}, {'authorId': '5914812', 'name': 'Yasemin Kaygusuz'}, {'authorId': '34151365', 'name': 'T. Meissner'}, {'authorId': '6835064', 'name': 'T. Holderried'}, {'authorId': '2148763858', 'name': 'Susan Chan'}, {'authorId': '38486988', 'name': 'P. Kastner'}, {'authorId': '144414260', 'name': 'W. Haining'}, {'authorId': '145472731', 'name': 'H. Cantor'}]}]}\n",
      "total amount {'total': 250, 'offset': 0, 'next': 100, 'data': [{'paperId': 'ef160aee837c415ee3168efb0fe515ec2df5b2e8', 'title': 'Estimating the reproducibility of psychological science', 'abstract': 'Empirically analyzing empirical evidence One of the central goals in any scientific endeavor is to understand causality. Experiments that seek to demonstrate a cause/effect relation most often manipulate the postulated causal factor. Aarts et al. describe the replication of 100 experiments reported in papers published in 2008 in three high-ranking psychology journals. Assessing whether the replication and the original experiment yielded the same result according to several criteria, they find that about one-third to one-half of the original findings were also observed in the replication study. Science, this issue 10.1126/science.aac4716 A large-scale assessment suggests that experimental reproducibility in psychology leaves a lot to be desired. INTRODUCTION Reproducibility is a defining feature of science, but the extent to which it characterizes current research is unknown. Scientific claims should not gain credence because of the status or authority of their originator but by the replicability of their supporting evidence. Even research of exemplary quality may have irreproducible empirical findings because of random or systematic error. RATIONALE There is concern about the rate and predictors of reproducibility, but limited evidence. Potentially problematic practices include selective reporting, selective analysis, and insufficient specification of the conditions necessary or sufficient to obtain the results. Direct replication is the attempt to recreate the conditions believed sufficient for obtaining a previously observed finding and is the means of establishing reproducibility of a finding with new data. We conducted a large-scale, collaborative effort to obtain an initial estimate of the reproducibility of psychological science. RESULTS We conducted replications of 100 experimental and correlational studies published in three psychology journals using high-powered designs and original materials when available. There is no single standard for evaluating replication success. Here, we evaluated reproducibility using significance and P values, effect sizes, subjective assessments of replication teams, and meta-analysis of effect sizes. The mean effect size (r) of the replication effects (Mr = 0.197, SD = 0.257) was half the magnitude of the mean effect size of the original effects (Mr = 0.403, SD = 0.188), representing a substantial decline. Ninety-seven percent of original studies had significant results (P < .05). Thirty-six percent of replications had significant results; 47% of original effect sizes were in the 95% confidence interval of the replication effect size; 39% of effects were subjectively rated to have replicated the original result; and if no bias in original results is assumed, combining original and replication results left 68% with statistically significant effects. Correlational tests suggest that replication success was better predicted by the strength of original evidence than by characteristics of the original and replication teams. CONCLUSION No single indicator sufficiently describes replication success, and the five indicators examined here are not the only ways to evaluate reproducibility. Nonetheless, collectively these results offer a clear conclusion: A large portion of replications produced weaker evidence for the original findings despite using materials provided by the original authors, review in advance for methodological fidelity, and high statistical power to detect the original effect sizes. Moreover, correlational evidence is consistent with the conclusion that variation in the strength of initial evidence (such as original P value) was more predictive of replication success than variation in the characteristics of the teams conducting the research (such as experience and expertise). The latter factors certainly can influence replication success, but they did not appear to do so here. Reproducibility is not well understood because the incentives for individual scientists prioritize novelty over replication. Innovation is the engine of discovery and is vital for a productive, effective scientific enterprise. However, innovative ideas become old news fast. Journal reviewers and editors may dismiss a new test of a published idea as unoriginal. The claim that “we already know this” belies the uncertainty of scientific evidence. Innovation points out paths that are possible; replication points out paths that are likely; progress relies on both. Replication can increase certainty when findings are reproduced and promote innovation when they are not. This project provides accumulating evidence for many findings in psychological research and suggests that there is still more work to do to verify whether we know what we think we know. Original study effect size versus replication effect size (correlation coefficients). Diagonal line represents replication effect size equal to original effect size. Dotted line represents replication effect size of 0. Points below the dotted line were effects in the opposite direction of the original. Density plots are separated by significant (blue) and nonsignificant (red) effects. Reproducibility is a defining feature of science, but the extent to which it characterizes current research is unknown. We conducted replications of 100 experimental and correlational studies published in three psychology journals using high-powered designs and original materials when available. Replication effects were half the magnitude of original effects, representing a substantial decline. Ninety-seven percent of original studies had statistically significant results. Thirty-six percent of replications had statistically significant results; 47% of original effect sizes were in the 95% confidence interval of the replication effect size; 39% of effects were subjectively rated to have replicated the original result; and if no bias in original results is assumed, combining original and replication results left 68% with statistically significant effects. Correlational tests suggest that replication success was better predicted by the strength of original evidence than by characteristics of the original and replication teams.', 'year': 2015, 'citationCount': 5746, 'authors': [{'authorId': '2231439', 'name': 'Alexander A. Aarts'}, {'authorId': '1400206933', 'name': 'Joanna E. Anderson'}, {'authorId': '96413924', 'name': 'Christopher J. Anderson'}, {'authorId': '113650771', 'name': 'Peter Raymond Attridge'}, {'authorId': '1788796', 'name': 'A. Attwood'}, {'authorId': '152159926', 'name': 'Jordan R. Axt'}, {'authorId': '36417745', 'name': 'Molly Babel'}, {'authorId': '5252377', 'name': 'Š. Bahník'}, {'authorId': '6174975', 'name': 'E. Baranski'}, {'authorId': '1387458734', 'name': 'M. Barnett-Cowan'}, {'authorId': '4867423', 'name': 'E. Bartmess'}, {'authorId': '3039399', 'name': 'Jennifer S. Beer'}, {'authorId': '47431109', 'name': 'Raoul Bell'}, {'authorId': '2074992746', 'name': 'Heather Bentley'}, {'authorId': '114670541', 'name': 'Leah Beyan'}, {'authorId': '30661345', 'name': 'Grace Binion'}, {'authorId': '3237553', 'name': 'D. Borsboom'}, {'authorId': '116191440', 'name': 'Annick Bosch'}, {'authorId': '8597004', 'name': 'Frank Bosco'}, {'authorId': '72356050', 'name': 'Sara D. Bowman'}, {'authorId': '145470466', 'name': 'M. Brandt'}, {'authorId': '66335862', 'name': 'E. L. Braswell'}, {'authorId': '51904698', 'name': 'Hilmar Brohmer'}, {'authorId': '48120787', 'name': 'B. T. Brown'}, {'authorId': '4197188', 'name': 'Kristina Brown'}, {'authorId': '31412979', 'name': 'Jovita Brüning'}, {'authorId': '1400158480', 'name': 'Ann Calhoun-Sauls'}, {'authorId': '36027478', 'name': 'Shannon P. Callahan'}, {'authorId': '95354096', 'name': 'E. Chagnon'}, {'authorId': '48719411', 'name': 'Jesse J. Chandler'}, {'authorId': '38049428', 'name': 'Christopher R. Chartier'}, {'authorId': '145787104', 'name': 'F. Cheung'}, {'authorId': '51055585', 'name': 'Cody Daniel Christopherson'}, {'authorId': '4704887', 'name': 'Linda Cillessen'}, {'authorId': '48987469', 'name': 'R. Clay'}, {'authorId': '46550612', 'name': 'Hayley M. D. Cleary'}, {'authorId': '117283612', 'name': 'M. Cloud'}, {'authorId': '143947719', 'name': 'Michael Conn'}, {'authorId': '1795487', 'name': 'J. Cohoon'}, {'authorId': '7293849', 'name': 'Simon Columbus'}, {'authorId': '2052289610', 'name': 'A. Cordes'}, {'authorId': '49251371', 'name': 'G. Costantini'}, {'authorId': '15675192', 'name': 'Leslie D. Cramblet Alvarez'}, {'authorId': '51034966', 'name': 'E. Cremata'}, {'authorId': '104792844', 'name': 'J. Crusius'}, {'authorId': '2480852', 'name': 'J. DeCoster'}, {'authorId': '122378013', 'name': 'M. DeGaetano'}, {'authorId': '1665535543', 'name': 'Nicolás Delia Penna'}, {'authorId': '1665493156', 'name': 'Bobby Den Bezemer'}, {'authorId': '32368039', 'name': 'M. Deserno'}, {'authorId': '113076847', 'name': 'Olivia Devitt'}, {'authorId': '6629103', 'name': 'L. Dewitte'}, {'authorId': '6958962', 'name': 'David G. Dobolyi'}, {'authorId': '16313368', 'name': 'Geneva T. Dodson'}, {'authorId': '2467192', 'name': 'M. Donnellan'}, {'authorId': '2094220350', 'name': 'Ryan Donohue'}, {'authorId': '5663433', 'name': 'Rebecca A. Dore'}, {'authorId': '5021006', 'name': 'A. Dorrough'}, {'authorId': '3488772', 'name': 'Anna Dreber'}, {'authorId': '5818783', 'name': 'M. Dugas'}, {'authorId': '3100527', 'name': 'Elizabeth W. Dunn'}, {'authorId': '14421983', 'name': 'Kayleigh E. Easey'}, {'authorId': '117807982', 'name': 'Sylvia Eboigbe'}, {'authorId': '34959578', 'name': 'C. Eggleston'}, {'authorId': '123302634', 'name': 'Jo Embley'}, {'authorId': '3018477', 'name': 'S. Epskamp'}, {'authorId': '52004348', 'name': 'Timothy M. Errington'}, {'authorId': '117698833', 'name': 'Vivien Estel'}, {'authorId': '3392676', 'name': 'Frank J. Farach'}, {'authorId': '25452274', 'name': 'J. Feather'}, {'authorId': '2322314', 'name': 'A. Fedor'}, {'authorId': '1400158404', 'name': 'B. Fernández-Castilla'}, {'authorId': '38548340', 'name': 'S. Fiedler'}, {'authorId': '32576544', 'name': 'James G. Field'}, {'authorId': '2237526', 'name': 'Stanka A. Fitneva'}, {'authorId': '2387162', 'name': 'T. Flagan'}, {'authorId': '6543598', 'name': 'Amanda L. Forest'}, {'authorId': '4193992', 'name': 'Eskil Forsell'}, {'authorId': '153664010', 'name': 'J. D. Foster'}, {'authorId': '2536223', 'name': 'Michael C. Frank'}, {'authorId': '144106517', 'name': 'Rebecca S. Frazier'}, {'authorId': '2057921617', 'name': 'Heather M. Fuchs'}, {'authorId': '5807132', 'name': 'P. Gable'}, {'authorId': '4908329', 'name': 'Jeff Galak'}, {'authorId': '52169965', 'name': 'E. Galliani'}, {'authorId': '39007608', 'name': 'Anup Gampa'}, {'authorId': '2072604774', 'name': 'Sara Garcia'}, {'authorId': '4034033', 'name': 'Douglas Gazarian'}, {'authorId': '152595201', 'name': 'E. Gilbert'}, {'authorId': '1400843054', 'name': 'Roger Giner-Sorolla'}, {'authorId': '2098939', 'name': 'A. Glöckner'}, {'authorId': '35467332', 'name': 'Lars Goellner'}, {'authorId': '6760964', 'name': 'Jin X. Goh'}, {'authorId': '1402974706', 'name': 'R. Goldberg'}, {'authorId': '6439793', 'name': 'Patrick T. Goodbourn'}, {'authorId': '1400158332', 'name': 'S. Gordon-McKeon'}, {'authorId': '103288020', 'name': 'Bryan H. Gorges'}, {'authorId': '1677019678', 'name': 'Jessie Gorges'}, {'authorId': '113021959', 'name': 'J. Goss'}, {'authorId': '40433926', 'name': 'J. Graham'}, {'authorId': '50590791', 'name': 'J. Grange'}, {'authorId': '2556954', 'name': 'J. Gray'}, {'authorId': '72271933', 'name': 'C.H.J. Hartgerink'}, {'authorId': '27816717', 'name': 'Joshua K. Hartshorne'}, {'authorId': '3390722', 'name': 'F. Hasselman'}, {'authorId': '144616859', 'name': 'Timothy Hayes'}, {'authorId': '3650446', 'name': 'Emma Heikensten'}, {'authorId': '2196843', 'name': 'Felix Henninger'}, {'authorId': '6941720', 'name': 'J. Hodsoll'}, {'authorId': '3388157', 'name': 'Taylor Holubar'}, {'authorId': '92341957', 'name': 'G. Hoogendoorn'}, {'authorId': '1665486287', 'name': 'D. Humphries'}, {'authorId': '70247182', 'name': 'C. O. Hung'}, {'authorId': '87889800', 'name': 'N. Immelman'}, {'authorId': '6169028', 'name': 'Vanessa C. Irsik'}, {'authorId': '36524750', 'name': 'G. Jahn'}, {'authorId': '80488489', 'name': 'F. Jäkel'}, {'authorId': '2485703', 'name': 'Marc Jekel'}, {'authorId': '145281248', 'name': 'M. Johannesson'}, {'authorId': '117496518', 'name': 'L. Johnson'}, {'authorId': '2150444569', 'name': 'David J. Johnson'}, {'authorId': '10776195', 'name': 'Kate M. Johnson'}, {'authorId': '2071756595', 'name': 'William Johnston'}, {'authorId': '2640241', 'name': 'K. Jonas'}, {'authorId': '1400158286', 'name': 'Jennifer A. Joy-Gaba'}, {'authorId': '4049176', 'name': 'H. Kappes'}, {'authorId': '120498542', 'name': 'Kim Kelso'}, {'authorId': '8633058', 'name': 'Mallory C. Kidwell'}, {'authorId': '2109708345', 'name': 'Seung K. Kim'}, {'authorId': '6395769', 'name': 'M. Kirkhart'}, {'authorId': '6032930', 'name': 'Bennett Kleinberg'}, {'authorId': '50012037', 'name': 'G. Knežević'}, {'authorId': '38197330', 'name': 'F. Kolorz'}, {'authorId': '39916640', 'name': 'J. Kossakowski'}, {'authorId': '153875794', 'name': 'R. Krause'}, {'authorId': '123143083', 'name': 'J.M.T. Krijnen'}, {'authorId': '5199064', 'name': 'T. Kuhlmann'}, {'authorId': '90301659', 'name': 'Y. Kunkels'}, {'authorId': '123804984', 'name': 'Megan M. Kyc'}, {'authorId': '2148764895', 'name': 'Calvin K. Lai'}, {'authorId': '1665495033', 'name': 'Aamir Laique'}, {'authorId': '87792747', 'name': 'D. Lakens'}, {'authorId': '47951763', 'name': 'Kristin A. Lane'}, {'authorId': '6028786', 'name': 'Bethany Lassetter'}, {'authorId': '40526243', 'name': 'L. Lazarević'}, {'authorId': '1665487375', 'name': 'E. Bel'}, {'authorId': '2110236533', 'name': 'Key Jung Lee'}, {'authorId': '46293822', 'name': 'Minha Lee'}, {'authorId': '6560078', 'name': 'K. Lemm'}, {'authorId': '2248921', 'name': 'C. Levitan'}, {'authorId': '48575845', 'name': 'Melissa Lewis'}, {'authorId': '2117106185', 'name': 'Lin Lin'}, {'authorId': '2145198693', 'name': 'Stephanie C. Lin'}, {'authorId': '35780819', 'name': 'Matthias Lippold'}, {'authorId': '6767826', 'name': 'Darren Loureiro'}, {'authorId': '116957928', 'name': 'Ilse Luteijn'}, {'authorId': '35093146', 'name': 'S. Mackinnon'}, {'authorId': '120566087', 'name': 'Heather N. Mainard'}, {'authorId': '4526963', 'name': 'Denise C. Marigold'}, {'authorId': '150078826', 'name': 'D. P. Martin'}, {'authorId': '123396296', 'name': 'Tylar Martinez'}, {'authorId': '4362855', 'name': 'E. Masicampo'}, {'authorId': '84358116', 'name': 'Joshua J. Matacotta'}, {'authorId': '32716879', 'name': 'Maya B. Mathur'}, {'authorId': '2055522260', 'name': 'Michael May'}, {'authorId': '5739306', 'name': 'Nicole C Mechin'}, {'authorId': '2796734', 'name': 'P. Mehta'}, {'authorId': '3505739', 'name': 'Johannes M. Meixner'}, {'authorId': '2292034', 'name': 'Alissa Melinger'}, {'authorId': '2109507581', 'name': 'Jeremy K. Miller'}, {'authorId': '1664699522', 'name': 'Mallorie Miller'}, {'authorId': '3260369', 'name': 'K. Moore'}, {'authorId': '8173711', 'name': 'Marcus Möschl'}, {'authorId': '40632770', 'name': 'Matt Motyl'}, {'authorId': '49338794', 'name': 'S. Muller'}, {'authorId': '3057626', 'name': 'M. Munafo'}, {'authorId': '6699090', 'name': 'K. Neijenhuijs'}, {'authorId': '122669704', 'name': 'Taylor Nervi'}, {'authorId': '28866192', 'name': 'Gandalf Nicolas'}, {'authorId': '5120903', 'name': 'G. Nilsonne'}, {'authorId': '2862527', 'name': 'Brian A. Nosek'}, {'authorId': '4222695', 'name': 'Michèle B. Nuijten'}, {'authorId': '2061321863', 'name': 'Catherine Olsson'}, {'authorId': '21707288', 'name': 'C. Osborne'}, {'authorId': '15808420', 'name': 'Lutz Ostkamp'}, {'authorId': '1800603', 'name': 'M. Pavel'}, {'authorId': '1398036394', 'name': 'I. Penton-Voak'}, {'authorId': '123910286', 'name': 'O. Perna'}, {'authorId': '1733092', 'name': 'C. Pernet'}, {'authorId': '5070565', 'name': 'M. Perugini'}, {'authorId': '145798446', 'name': 'R. Pipitone'}, {'authorId': '7731005', 'name': 'M. Pitts'}, {'authorId': '3086561', 'name': 'F. Plessow'}, {'authorId': '5666302', 'name': 'J. Prenoveau'}, {'authorId': '114309502', 'name': 'R. Rahal'}, {'authorId': '4412299', 'name': 'Kate A. Ratliff'}, {'authorId': '46557202', 'name': 'David A. Reinhard'}, {'authorId': '4137233', 'name': 'Frank Renkewitz'}, {'authorId': '4977822', 'name': 'Ashley A. Ricker'}, {'authorId': '3911950', 'name': 'A. Rigney'}, {'authorId': '145836378', 'name': 'Andrew M Rivers'}, {'authorId': '51055550', 'name': 'Mark A. Roebke'}, {'authorId': '6431015', 'name': 'Abraham M. Rutchick'}, {'authorId': '2055552083', 'name': 'Robert S. Ryan'}, {'authorId': '48176355', 'name': 'O. Şahin'}, {'authorId': '7144751', 'name': 'Anondah Saide'}, {'authorId': '4225466', 'name': 'Gillian M. Sandstrom'}, {'authorId': '2113520139', 'name': 'David Santos'}, {'authorId': '2055715460', 'name': 'R. Saxe'}, {'authorId': '40063306', 'name': 'R. Schlegelmilch'}, {'authorId': '49371070', 'name': 'Kathleen Schmidt'}, {'authorId': '49715936', 'name': 'Sabine Scholz'}, {'authorId': '123478162', 'name': 'L. Seibel'}, {'authorId': '4067481', 'name': 'Dylan Selterman'}, {'authorId': '3397505', 'name': 'S. Shaki'}, {'authorId': '31665223', 'name': 'W. B. Simpson'}, {'authorId': '144232281', 'name': 'H. Sinclair'}, {'authorId': '143804040', 'name': 'Jeanine L. M. Skorinko'}, {'authorId': '2111978802', 'name': 'A. Slowik'}, {'authorId': '2500175', 'name': 'J. Snyder'}, {'authorId': '4110370', 'name': 'C. Soderberg'}, {'authorId': '32218314', 'name': 'Carina M. Sonnleitner'}, {'authorId': '2056814613', 'name': 'N. Spencer'}, {'authorId': '34782684', 'name': 'Jeffrey R. Spies'}, {'authorId': '2889085', 'name': 'S. Steegen'}, {'authorId': '3308834', 'name': 'S. Stieger'}, {'authorId': '3918197', 'name': 'Nina Strohminger'}, {'authorId': '3349937', 'name': 'G. Sullivan'}, {'authorId': '6998374', 'name': 'T. Talhelm'}, {'authorId': '123526524', 'name': 'M. Tapia'}, {'authorId': '120986210', 'name': 'A. T. Dorsthorst'}, {'authorId': '17886332', 'name': 'M. Thomae'}, {'authorId': '2108534776', 'name': 'Sarah L. Thomas'}, {'authorId': '5407737', 'name': 'Pia Tio'}, {'authorId': '26384240', 'name': 'Frits Traets'}, {'authorId': '50343359', 'name': 'Steve Tsang'}, {'authorId': '2109141', 'name': 'F. Tuerlinckx'}, {'authorId': '118528873', 'name': 'Paul Turchan'}, {'authorId': '3738838', 'name': 'Milan Valášek'}, {'authorId': '143861972', 'name': 'A. V. Veer'}, {'authorId': '81322766', 'name': 'R. V. Aert'}, {'authorId': '13333804', 'name': 'M. V. Assen'}, {'authorId': '41197124', 'name': 'R. V. Bork'}, {'authorId': '122303122', 'name': 'Mathijs van de Ven'}, {'authorId': '103097236', 'name': 'D. V. D. Bergh'}, {'authorId': '50093142', 'name': 'M. Hulst'}, {'authorId': '51240054', 'name': 'R. V. Dooren'}, {'authorId': '50233092', 'name': 'J. Doorn'}, {'authorId': '4155110', 'name': 'D. V. Renswoude'}, {'authorId': '2392419', 'name': 'H. Rijn'}, {'authorId': '2409652', 'name': 'W. Vanpaemel'}, {'authorId': '2078595605', 'name': 'Alejandro Echeverría'}, {'authorId': '2070303231', 'name': 'Melissa Vazquez'}, {'authorId': '6122250', 'name': 'Natalia Vélez'}, {'authorId': '35039893', 'name': 'Marieke Vermue'}, {'authorId': '88801524', 'name': 'M. Verschoor'}, {'authorId': '46989769', 'name': 'M. Vianello'}, {'authorId': '2176390', 'name': 'M. Voracek'}, {'authorId': '113898223', 'name': 'Gina Vuu'}, {'authorId': '2505272', 'name': 'E. Wagenmakers'}, {'authorId': '6348079', 'name': 'Joanneke Weerdmeester'}, {'authorId': '2058927069', 'name': 'A. Welsh'}, {'authorId': '5900056', 'name': 'Erin C. Westgate'}, {'authorId': '67239528', 'name': 'Joeri Wissink'}, {'authorId': '145322160', 'name': 'M. Wood'}, {'authorId': '48197426', 'name': 'A. Woods'}, {'authorId': '2058458367', 'name': 'Emily M. Wright'}, {'authorId': '151482749', 'name': 'Sining Wu'}, {'authorId': '145525838', 'name': 'M. Zeelenberg'}, {'authorId': '117597026', 'name': 'Kellylynn Zuni'}]}, {'paperId': 'b5f109c8d49a95f72313aff41cf8a0b2e8e1407a', 'title': 'The Genotype-Tissue Expression (GTEx) pilot analysis: Multitissue gene regulation in humans', 'abstract': 'Expression, genetic variation, and tissues Human genomes show extensive genetic variation across individuals, but we have only just started documenting the effects of this variation on the regulation of gene expression. Furthermore, only a few tissues have been examined per genetic variant. In order to examine how genetic expression varies among tissues within individuals, the Genotype-Tissue Expression (GTEx) Consortium collected 1641 postmortem samples covering 54 body sites from 175 individuals. They identified quantitative genetic traits that affect gene expression and determined which of these exhibit tissue-specific expression patterns. Melé et al. measured how transcription varies among tissues, and Rivas et al. looked at how truncated protein variants affect expression across tissues. Science, this issue p. 648, p. 660, p. 666; see also p. 640 Sampling RNA from multiple tissues across the human body illuminates how genetic variation affects gene expression. [Also see Perspective by Gibson] Understanding the functional consequences of genetic variation, and how it affects complex human disease and quantitative traits, remains a critical challenge for biomedicine. We present an analysis of RNA sequencing data from 1641 samples across 43 tissues from 175 individuals, generated as part of the pilot phase of the Genotype-Tissue Expression (GTEx) project. We describe the landscape of gene expression across tissues, catalog thousands of tissue-specific and shared regulatory expression quantitative trait loci (eQTL) variants, describe complex network relationships, and identify signals from genome-wide association studies explained by eQTLs. These findings provide a systematic understanding of the cellular and biological consequences of human genetic variation and of the heterogeneity of such effects among a diverse set of human tissues.', 'year': 2015, 'citationCount': 3980, 'authors': [{'authorId': '4351471', 'name': 'K. Ardlie'}, {'authorId': '46980374', 'name': 'D. DeLuca'}, {'authorId': '4155963', 'name': 'A. Segrè'}, {'authorId': '2068615525', 'name': 'T. Sullivan'}, {'authorId': '31774477', 'name': 'Taylor R. Young'}, {'authorId': '16172097', 'name': 'Ellen T. Gelfand'}, {'authorId': '11133741', 'name': 'Casandra A. Trowbridge'}, {'authorId': '3254016', 'name': 'J. Maller'}, {'authorId': '5173577', 'name': 'T. Tukiainen'}, {'authorId': '3015823', 'name': 'M. Lek'}, {'authorId': '36987543', 'name': 'L. D. Ward'}, {'authorId': '1957116', 'name': 'P. Kheradpour'}, {'authorId': '3768568', 'name': 'B. Iriarte'}, {'authorId': '2114020679', 'name': 'Y. Meng'}, {'authorId': '4128156', 'name': 'Cameron Palmer'}, {'authorId': '52200875', 'name': 'T. Esko'}, {'authorId': '2060379', 'name': 'W. Winckler'}, {'authorId': '3294928', 'name': 'J. Hirschhorn'}, {'authorId': '2809250', 'name': 'Manolis Kellis'}, {'authorId': '2029438', 'name': 'D. MacArthur'}, {'authorId': '2110594', 'name': 'G. Getz'}, {'authorId': '4407885', 'name': 'A. Shabalin'}, {'authorId': '2108549769', 'name': 'Gen Li'}, {'authorId': '34661060', 'name': 'Yi‐Hui Zhou'}, {'authorId': '1794435', 'name': 'A. Nobel'}, {'authorId': '2430148', 'name': 'I. Rusyn'}, {'authorId': '40250286', 'name': 'F. Wright'}, {'authorId': '144946488', 'name': 'T. Lappalainen'}, {'authorId': '145209580', 'name': 'Pedro G. Ferreira'}, {'authorId': '3410962', 'name': 'H. Ongen'}, {'authorId': '145764939', 'name': 'M. Rivas'}, {'authorId': '145086335', 'name': 'A. Battle'}, {'authorId': '47278080', 'name': 'S. Mostafavi'}, {'authorId': '6596913', 'name': 'Jean Monlong'}, {'authorId': '1971387', 'name': 'M. Sammeth'}, {'authorId': '2326247', 'name': 'Marta Melé'}, {'authorId': '1704215264', 'name': 'F. Reverter'}, {'authorId': '47824049', 'name': 'J. Goldmann'}, {'authorId': '1736370', 'name': 'D. Koller'}, {'authorId': '1853610', 'name': 'R. Guigó'}, {'authorId': '2235633839', 'name': 'M. McCarthy'}, {'authorId': '2939073', 'name': 'E. Dermitzakis'}, {'authorId': '3051203', 'name': 'E. Gamazon'}, {'authorId': '2427364', 'name': 'H. Im'}, {'authorId': '1784291', 'name': 'A. Konkashbaev'}, {'authorId': '1783950', 'name': 'D. Nicolae'}, {'authorId': '2173670631', 'name': 'N. Cox'}, {'authorId': '50045364', 'name': 'T. Flutre'}, {'authorId': '36836750', 'name': 'X. Wen'}, {'authorId': '145761702', 'name': 'M. Stephens'}, {'authorId': '50528060', 'name': 'J. Pritchard'}, {'authorId': '40501864', 'name': 'Zhidong Tu'}, {'authorId': '49846439', 'name': 'Bin Zhang'}, {'authorId': '2110812174', 'name': 'Tao Huang'}, {'authorId': '48548974', 'name': 'Q. Long'}, {'authorId': '34999215', 'name': 'Luan Lin'}, {'authorId': '2144559662', 'name': 'Jialiang Yang'}, {'authorId': '145254054', 'name': 'Jun Zhu'}, {'authorId': '2234974904', 'name': 'Jun S. Liu'}, {'authorId': '2110644715', 'name': 'A. Brown'}, {'authorId': '4721474', 'name': 'B. Mestichelli'}, {'authorId': '5632758', 'name': 'D. Tidwell'}, {'authorId': '4412904', 'name': 'E. Lo'}, {'authorId': '6900442', 'name': 'Mike Salvatore'}, {'authorId': '40353616', 'name': 'Saboor Shad'}, {'authorId': '2115544487', 'name': 'Jeffrey A. Thomas'}, {'authorId': '103811085', 'name': 'J. Lonsdale'}, {'authorId': '28868673', 'name': 'M. T. Moser'}, {'authorId': '13378769', 'name': 'Bryan M. Gillard'}, {'authorId': '5353188', 'name': 'E. Karasik'}, {'authorId': '40652641', 'name': 'K. Ramsey'}, {'authorId': '40155820', 'name': 'Christopher Choi'}, {'authorId': '1844728', 'name': 'B. Foster'}, {'authorId': '6752960', 'name': 'John Syron'}, {'authorId': '3717383', 'name': 'Johnelle Fleming'}, {'authorId': '48365408', 'name': 'Harold Magazine'}, {'authorId': '10396153', 'name': 'R. Hasz'}, {'authorId': '19099263', 'name': 'Gary Walters'}, {'authorId': '52213800', 'name': 'J. Bridge'}, {'authorId': '50062134', 'name': 'Mark Miklos'}, {'authorId': '2074770062', 'name': 'Susan Sullivan'}, {'authorId': '31821652', 'name': 'Laura K. Barker'}, {'authorId': '3633762', 'name': 'H. Traino'}, {'authorId': '3507395', 'name': 'M. Mosavel'}, {'authorId': '1581155106', 'name': 'L. Siminoff'}, {'authorId': '6011362', 'name': 'Dana R. Valley'}, {'authorId': '15828822', 'name': 'D. Rohrer'}, {'authorId': '4317180', 'name': 'S. Jewell'}, {'authorId': '39668238', 'name': 'P. Branton'}, {'authorId': '5477494', 'name': 'L. Sobin'}, {'authorId': '37365786', 'name': 'M. Barcus'}, {'authorId': '50741911', 'name': 'L. Qi'}, {'authorId': '46808203', 'name': 'J. McLean'}, {'authorId': '49119953', 'name': 'P. Hariharan'}, {'authorId': '40341965', 'name': 'K. S. Um'}, {'authorId': '2199287', 'name': 'Shenpei Wu'}, {'authorId': '4007205', 'name': 'D. Tabor'}, {'authorId': '6456063', 'name': 'C. Shive'}, {'authorId': '2116583787', 'name': 'Anna M. Smith'}, {'authorId': '6399876', 'name': 'Stephen A. Buia'}, {'authorId': '5664172', 'name': 'Anita H. Undale'}, {'authorId': '4735590', 'name': 'Karna L. Robinson'}, {'authorId': '39656869', 'name': 'Nancy V. Roche'}, {'authorId': '31606684', 'name': 'Kimberly M. Valentino'}, {'authorId': '49117155', 'name': 'A. Britton'}, {'authorId': '47562897', 'name': 'R. Burges'}, {'authorId': '33915352', 'name': 'Debra Bradbury'}, {'authorId': '3801585', 'name': 'Kenneth W. Hambright'}, {'authorId': '4354568', 'name': 'John Seleski'}, {'authorId': '4575963', 'name': 'G. Korzeniewski'}, {'authorId': '38190914', 'name': 'Kenyon Erickson'}, {'authorId': '40408502', 'name': 'Yvonne Marcus'}, {'authorId': '2006434778', 'name': 'J. Tejada'}, {'authorId': '1934316', 'name': 'M. Taherian'}, {'authorId': '3567450', 'name': 'Chunrong Lu'}, {'authorId': '40512923', 'name': 'Margaret J. Basile'}, {'authorId': '8055139', 'name': 'D. Mash'}, {'authorId': '35338611', 'name': 'S. Volpi'}, {'authorId': '143755016', 'name': 'J. Struewing'}, {'authorId': '6264745', 'name': 'Gary F. Temple'}, {'authorId': '40591554', 'name': 'Joy T. Boyer'}, {'authorId': '4618842', 'name': 'Deborah Colantuoni'}, {'authorId': '120055212', 'name': 'R. Little'}, {'authorId': '32043266', 'name': 'S. Koester'}, {'authorId': '6407402', 'name': 'Latarsha J. Carithers'}, {'authorId': '48294660', 'name': 'Helen M. Moore'}, {'authorId': '46344169', 'name': 'P. Guan'}, {'authorId': '3862532', 'name': 'C. Compton'}, {'authorId': '32536451', 'name': 'Sherilyn J. Sawyer'}, {'authorId': '4915811', 'name': 'J. Demchok'}, {'authorId': '48219890', 'name': 'J. Vaught'}, {'authorId': '47626774', 'name': 'C. Rabiner'}, {'authorId': '90929184', 'name': 'Lockhart'}]}, {'paperId': 'a798f8e9c58258bdbb996b8518b313fcc449be2c', 'title': 'Plastic waste inputs from land into the ocean', 'abstract': 'Dumping lots of plastics into our oceans Considerable progress has been made in determining the amount and location of plastic debris in our seas, but how much plastic actually enters them in the first place is more uncertain. Jambeck et al. combine available data on solid waste with a model that uses population density and economic status to estimate the amount of land-based plastic waste entering the ocean. Unless waste management practices are improved, the flux of plastics to the oceans could increase by an order of magnitude within the next decade. Science, this issue p. 768 Millions of tons of plastic waste end up in our oceans every year. Plastic debris in the marine environment is widely documented, but the quantity of plastic entering the ocean from waste generated on land is unknown. By linking worldwide data on solid waste, population density, and economic status, we estimated the mass of land-based plastic waste entering the ocean. We calculate that 275 million metric tons (MT) of plastic waste was generated in 192 coastal countries in 2010, with 4.8 to 12.7 million MT entering the ocean. Population size and the quality of waste management systems largely determine which countries contribute the greatest mass of uncaptured waste available to become plastic marine debris. Without waste management infrastructure improvements, the cumulative quantity of plastic waste available to enter the ocean from land is predicted to increase by an order of magnitude by 2025.', 'year': 2015, 'citationCount': 7170, 'authors': [{'authorId': '2105939', 'name': 'J. Jambeck'}, {'authorId': '47383913', 'name': 'R. Geyer'}, {'authorId': '143924375', 'name': 'C. Wilcox'}, {'authorId': '118664577', 'name': 'T. R. Siegler'}, {'authorId': '47308651', 'name': 'M. Perryman'}, {'authorId': '8417546', 'name': 'A. Andrady'}, {'authorId': '7745972', 'name': 'R. Narayan'}, {'authorId': '34861621', 'name': 'K. L. Law'}]}, {'paperId': 'ad032c039418ccfa07a9e3449a4007a6781c554a', 'title': 'Tissue-based map of the human proteome', 'abstract': 'Protein expression across human tissues Sequencing the human genome gave new insights into human biology and disease. However, the ultimate goal is to understand the dynamic expression of each of the approximately 20,000 protein-coding genes and the function of each protein. Uhlén et al. now present a map of protein expression across 32 human tissues. They not only measured expression at an RNA level, but also used antibody profiling to precisely localize the corresponding proteins. An interactive website allows exploration of expression patterns across the human body. Science, this issue 10.1126/science.1260419 Transcriptomics and immunohistochemistry map protein expression across 32 human tissues. INTRODUCTION Resolving the molecular details of proteome variation in the different tissues and organs of the human body would greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on quantitative transcriptomics on a tissue and organ level combined with protein profiling using microarray-based immunohistochemistry to achieve spatial localization of proteins down to the single-cell level. We provide a global analysis of the secreted and membrane proteins, as well as an analysis of the expression profiles for all proteins targeted by pharmaceutical drugs and proteins implicated in cancer. RATIONALE We have used an integrative omics approach to study the spatial human proteome. Samples representing all major tissues and organs (n = 44) in the human body have been analyzed based on 24,028 antibodies corresponding to 16,975 protein-encoding genes, complemented with RNA-sequencing data for 32 of the tissues. The antibodies have been used to produce more than 13 million tissue-based immunohistochemistry images, each annotated by pathologists for all sampled tissues. To facilitate integration with other biological resources, all data are available for download and cross-referencing. RESULTS We report a genome-wide analysis of the tissue specificity of RNA and protein expression covering more than 90% of the putative protein-coding genes, complemented with analyses of various subproteomes, such as predicted secreted proteins (n = 3171) and membrane-bound proteins (n = 5570). The analysis shows that almost half of the genes are expressed in all analyzed tissues, which suggests that the gene products are needed in all cells to maintain “housekeeping” functions such as cell growth, energy generation, and basic metabolism. Furthermore, there is enrichment in metabolism among these genes, as 60% of all metabolic enzymes are expressed in all analyzed tissues. The largest number of tissue-enriched genes is found in the testis, followed by the brain and the liver. Analysis of the 618 proteins targeted by clinically approved drugs unexpectedly showed that 30% are expressed in all analyzed tissues. An analysis of metabolic activity based on genome-scale metabolic models (GEMS) revealed liver as the most metabolically active tissue, followed by adipose tissue and skeletal muscle. CONCLUSIONS A freely available interactive resource is presented as part of the Human Protein Atlas portal (www.proteinatlas.org), offering the possibility to explore the tissue-elevated proteomes in tissues and organs and to analyze tissue profiles for specific protein classes. Comprehensive lists of proteins expressed at elevated levels in the different tissues have been compiled to provide a spatial context with localization of the proteins in the subcompartments of each tissue and organ down to the single-cell level. The human tissue–enriched proteins. All tissue-enriched proteins are shown for 13 representative tissues or groups of tissues, stratified according to their predicted subcellular localization. Enriched proteins are mainly intracellular in testis, mainly membrane bound in brain and kidney, and mainly secreted in pancreas and liver. Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.', 'year': 2015, 'citationCount': 9684, 'authors': [{'authorId': '2781789', 'name': 'M. Uhlén'}, {'authorId': '3841801', 'name': 'L. Fagerberg'}, {'authorId': '4787480', 'name': 'B. Hallström'}, {'authorId': '5710895', 'name': 'C. Lindskog'}, {'authorId': '3781249', 'name': 'P. Oksvold'}, {'authorId': '2567446', 'name': 'A. Mardinoğlu'}, {'authorId': '3521812', 'name': 'Åsa Sivertsson'}, {'authorId': '3919959', 'name': 'C. Kampf'}, {'authorId': '6961788', 'name': 'E. Sjöstedt'}, {'authorId': '5177460', 'name': 'A. Asplund'}, {'authorId': '49850546', 'name': 'Ingmarie Olsson'}, {'authorId': '2483844', 'name': 'K. Edlund'}, {'authorId': '89824313', 'name': 'E. Lundberg'}, {'authorId': '6505855', 'name': 'S. Navani'}, {'authorId': '50704448', 'name': 'C. Szigyarto'}, {'authorId': '1950314', 'name': 'J. Odeberg'}, {'authorId': '7484504', 'name': 'D. Djureinovic'}, {'authorId': '8300411', 'name': 'Jenny Ottosson Takanen'}, {'authorId': '6483536', 'name': 'S. Hober'}, {'authorId': '32692536', 'name': 'T. Alm'}, {'authorId': '6542432', 'name': 'P. Edqvist'}, {'authorId': '8049983', 'name': 'H. Berling'}, {'authorId': '3948842', 'name': 'Hanna Tegel'}, {'authorId': '144464470', 'name': 'J. Mulder'}, {'authorId': '4420951', 'name': 'Johan Rockberg'}, {'authorId': '48586383', 'name': 'P. Nilsson'}, {'authorId': '4136538', 'name': 'J. Schwenk'}, {'authorId': '8465156', 'name': 'M. Hamsten'}, {'authorId': '134273659', 'name': 'K. von Feilitzen'}, {'authorId': '2068107910', 'name': 'Mattias Forsberg'}, {'authorId': '46852754', 'name': 'L. Persson'}, {'authorId': '7369919', 'name': 'Fredric Johansson'}, {'authorId': '20374706', 'name': 'M. Zwahlen'}, {'authorId': '3300090', 'name': 'G. von Heijne'}, {'authorId': '145583597', 'name': 'J. Nielsen'}, {'authorId': '143810507', 'name': 'F. Pontén'}]}, {'paperId': 'd0a5bc5ff2c33c640e97009377304357fe8d2838', 'title': 'Mutational landscape determines sensitivity to PD-1 blockade in non–small cell lung cancer', 'abstract': 'Immune checkpoint inhibitors, which unleash a patient’s own T cells to kill tumors, are revolutionizing cancer treatment. To unravel the genomic determinants of response to this therapy, we used whole-exome sequencing of non–small cell lung cancers treated with pembrolizumab, an antibody targeting programmed cell death-1 (PD-1). In two independent cohorts, higher nonsynonymous mutation burden in tumors was associated with improved objective response, durable clinical benefit, and progression-free survival. Efficacy also correlated with the molecular smoking signature, higher neoantigen burden, and DNA repair pathway mutations; each factor was also associated with mutation burden. In one responder, neoantigen-specific CD8+ T cell responses paralleled tumor regression, suggesting that anti–PD-1 therapy enhances neoantigen-specific T cell reactivity. Our results suggest that the genomic landscape of lung cancers shapes response to anti–PD-1 therapy. An anticancer drug is more effective against tumors that carry more mutations. More mutations predict better efficacy Despite the remarkable success of cancer immunotherapies, many patients do not respond to treatment. Rizvi et al. studied the tumors of patients with non–small-cell lung cancer undergoing immunotherapy. In two independent cohorts, treatment efficacy was associated with a higher number of mutations in the tumors. In one patient, a tumor-specific T cell response paralleled tumor regression. Science, this issue p. 124', 'year': 2015, 'citationCount': 5728, 'authors': [{'authorId': '5226159', 'name': 'N. Rizvi'}, {'authorId': '4727109', 'name': 'M. Hellmann'}, {'authorId': '144304933', 'name': 'A. Snyder'}, {'authorId': '5411400', 'name': 'P. Kvistborg'}, {'authorId': '144252051', 'name': 'V. Makarov'}, {'authorId': '6057547', 'name': 'J. Havel'}, {'authorId': '15006569', 'name': 'William Lee'}, {'authorId': '39171288', 'name': 'Jianda Yuan'}, {'authorId': '47181234', 'name': 'P. Wong'}, {'authorId': '153046030', 'name': 'Teresa S. Ho'}, {'authorId': '31957119', 'name': 'Martin L. Miller'}, {'authorId': '5514453', 'name': 'N. Rekhtman'}, {'authorId': '144849915', 'name': 'A. Moreira'}, {'authorId': '46673012', 'name': 'Fawzia Ibrahim'}, {'authorId': '152679701', 'name': 'Cameron Bruggeman'}, {'authorId': '8452615', 'name': 'Billel Gasmi'}, {'authorId': '6951535', 'name': 'R. Zappasodi'}, {'authorId': '48187107', 'name': 'Y. Maeda'}, {'authorId': '144882390', 'name': 'C. Sander'}, {'authorId': '2570491', 'name': 'E. Garon'}, {'authorId': '6949756', 'name': 'T. Merghoub'}, {'authorId': '6892960', 'name': 'J. Wolchok'}, {'authorId': '144394800', 'name': 'T. Schumacher'}, {'authorId': '2234300666', 'name': 'T. Chan'}]}, {'paperId': '36d7fd7b34bb276f35928a4cef30940564b2e276', 'title': 'Cell types in the mouse cortex and hippocampus revealed by single-cell RNA-seq', 'abstract': 'Cellular diversity in the brain revealed The mammalian brain has an extraordinarily large number of cells. Although there are quite a few different cell types, many cells in any one category tend to look alike. Zeisel et al. analyzed the transcriptomes of mouse brain cells to reveal more than meets the eye. Interneurons of similar type were found in dissimilar regions of the brain. Oligodendrocytes that seemed to be all of one class were differentiated by their molecular signatures into a half-dozen classes. Microglia associated with blood vessels were distinguished from look-alike perivascular macrophages. Thus, the complex microanatomy of the brain can be revealed by the RNAs expressed in its cells. Science, this issue p. 1138 A close look at the genes expressed by cells in the brain reveals hidden and coordinated cellular complexity. The mammalian cerebral cortex supports cognitive functions such as sensorimotor integration, memory, and social behaviors. Normal brain function relies on a diverse set of differentiated cell types, including neurons, glia, and vasculature. Here, we have used large-scale single-cell RNA sequencing (RNA-seq) to classify cells in the mouse somatosensory cortex and hippocampal CA1 region. We found 47 molecularly distinct subclasses, comprising all known major cell types in the cortex. We identified numerous marker genes, which allowed alignment with known cell types, morphology, and location. We found a layer I interneuron expressing Pax6 and a distinct postmitotic oligodendrocyte subclass marked by Itpr2. Across the diversity of cortical cell types, transcription factors formed a complex, layered regulatory code, suggesting a mechanism for the maintenance of adult cell type identity.', 'year': 2015, 'citationCount': 2354, 'authors': [{'authorId': '2565412', 'name': 'Amit Zeisel'}, {'authorId': '1399254562', 'name': 'A. B. Muñoz-Manchado'}, {'authorId': '6427264', 'name': 'S. Codeluppi'}, {'authorId': '5211583', 'name': 'P. Lönnerberg'}, {'authorId': '134014902', 'name': 'Gioele La Manno'}, {'authorId': '10271546', 'name': 'Anna Juréus'}, {'authorId': '47123084', 'name': 'Sueli C Marques'}, {'authorId': '5124778', 'name': 'H. Munguba'}, {'authorId': '49159823', 'name': 'Liqun He'}, {'authorId': '5077801', 'name': 'C. Betsholtz'}, {'authorId': '6779735', 'name': 'Charlotte Rolny'}, {'authorId': '1384303053', 'name': 'G. Castelo-Branco'}, {'authorId': '1398819597', 'name': 'J. Hjerling-Leffler'}, {'authorId': '2500111', 'name': 'S. Linnarsson'}]}, {'paperId': 'c8665198592b52ba122e5cf84032a9bc1c61eade', 'title': 'Exposure to ideologically diverse news and opinion on Facebook', 'abstract': \"Not getting all sides of the news? People are increasingly turning away from mass media to social media as a way of learning news and civic information. Bakshy et al. examined the news that millions of Facebook users' peers shared, what information these users were presented with, and what they ultimately consumed (see the Perspective by Lazer). Friends shared substantially less cross-cutting news from sources aligned with an opposing ideology. People encountered roughly 15% less cross-cutting content in news feeds due to algorithmic ranking and clicked through to 70% less of this cross-cutting content. Within the domain of political news encountered in social media, selective exposure appears to drive attention. Science, this issue p. 1130; see also p. 1090 Despite the diversity of information available, people still pay attention to a limited range of opinions. [Also see Perspective by Lazer] Exposure to news, opinion, and civic information increasingly occurs through social media. How do these online networks influence exposure to perspectives that cut across ideological lines? Using deidentified data, we examined how 10.1 million U.S. Facebook users interact with socially shared news. We directly measured ideological homophily in friend networks and examined the extent to which heterogeneous friends could potentially expose individuals to cross-cutting content. We then quantified the extent to which individuals encounter comparatively more or less diverse content while interacting via Facebook’s algorithmically ranked News Feed and further studied users’ choices to click through to ideologically discordant content. Compared with algorithmic ranking, individuals’ choices played a stronger role in limiting exposure to cross-cutting content.\", 'year': 2015, 'citationCount': 2099, 'authors': [{'authorId': '3071703', 'name': 'E. Bakshy'}, {'authorId': '50252567', 'name': 'Solomon Messing'}, {'authorId': '1778398', 'name': 'Lada A. Adamic'}]}, {'paperId': '20ace79f735dfa350fa6c5eb0fa431c68093b19a', 'title': 'Genomic correlates of response to CTLA-4 blockade in metastatic melanoma', 'abstract': 'Is cancer immunotherapy a private affair? Immune checkpoint blockade, a relatively new cancer treatment, substantially extends the survival of a subset of patients. Previous work has shown that patients whose tumors harbor the largest number of mutations—and thus produce a large number of “neoantigens” recognized as foreign by the immune system—are most likely to benefit. Expanding on these earlier studies, Van Allen et al. studied over 100 patients with melanoma and found a similar correlation (see the Perspective by Gubin and Schreiber). There was no evidence, however, that specific neoantigen sequences were shared by patients who responded. Science, this issue p. 207, see also p. 158 Melanoma patients who respond to immunotherapy do not appear to share common tumor neoantigens. [Also see Perspective by Gubin and Schreiber ] Monoclonal antibodies directed against cytotoxic T lymphocyte–associated antigen-4 (CTLA-4), such as ipilimumab, yield considerable clinical benefit for patients with metastatic melanoma by inhibiting immune checkpoint activity, but clinical predictors of response to these therapies remain incompletely characterized. To investigate the roles of tumor-specific neoantigens and alterations in the tumor microenvironment in the response to ipilimumab, we analyzed whole exomes from pretreatment melanoma tumor biopsies and matching germline tissue samples from 110 patients. For 40 of these patients, we also obtained and analyzed transcriptome data from the pretreatment tumor samples. Overall mutational load, neoantigen load, and expression of cytolytic markers in the immune microenvironment were significantly associated with clinical benefit. However, no recurrent neoantigen peptide sequences predicted responder patient populations. Thus, detailed integrated molecular characterization of large patient cohorts may be needed to identify robust determinants of response and resistance to immune checkpoint inhibitors.', 'year': 2015, 'citationCount': 2114, 'authors': [{'authorId': '3459785', 'name': 'E. V. Van Allen'}, {'authorId': '36477474', 'name': 'Diana Miao'}, {'authorId': '145830928', 'name': 'B. Schilling'}, {'authorId': '2150905503', 'name': 'S. Shukla'}, {'authorId': '40616063', 'name': 'C. Blank'}, {'authorId': '32670858', 'name': 'L. Zimmer'}, {'authorId': '3864817', 'name': 'A. Sucker'}, {'authorId': '5005890', 'name': 'U. Hillen'}, {'authorId': '89064804', 'name': 'M. G. Geukes Foppen'}, {'authorId': '1974776', 'name': 'S. Goldinger'}, {'authorId': '6070014', 'name': 'J. Utikal'}, {'authorId': '144237883', 'name': 'J. Hassel'}, {'authorId': '3631833', 'name': 'B. Weide'}, {'authorId': '1390736503', 'name': 'K. Kaehler'}, {'authorId': '6815231', 'name': 'C. Loquai'}, {'authorId': '144577542', 'name': 'P. Mohr'}, {'authorId': '152420720', 'name': 'R. Gutzmer'}, {'authorId': '144139860', 'name': 'R. Dummer'}, {'authorId': '34428249', 'name': 'S. Gabriel'}, {'authorId': '1920712896', 'name': 'Catherine J. Wu'}, {'authorId': '144267265', 'name': 'D. Schadendorf'}, {'authorId': '3003287', 'name': 'L. Garraway'}]}, {'paperId': '514e52838f7513a8eab75a51814d2d06acca1baf', 'title': 'Commensal Bifidobacterium promotes antitumor immunity and facilitates anti–PD-L1 efficacy', 'abstract': 'Gut microbes affect immunotherapy The unleashing of antitumor T cell responses has ushered in a new era of cancer treatment. Although these therapies can cause dramatic tumor regressions in some patients, many patients inexplicably see no benefit. Mice have been used in two studies to investigate what might be happening. Specific members of the gut microbiota influence the efficacy of this type of immunotherapy (see the Perspective by Snyder et al.). Vétizou et al. found that optimal responses to anticytotoxic T lymphocyte antigen blockade required specific Bacteroides spp. Similarly, Sivan et al. discovered that Bifidobacterium spp. enhanced the efficacy of antiprogrammed cell death ligand 1 therapy. Science, this issue, p. 1079 and p. 1084; see also p. 1031 Gut microbes modulate the effectiveness of cancer immunotherapies in mice. [Also see Perspective by Snyder et al.] T cell infiltration of solid tumors is associated with favorable patient outcomes, yet the mechanisms underlying variable immune responses between individuals are not well understood. One possible modulator could be the intestinal microbiota. We compared melanoma growth in mice harboring distinct commensal microbiota and observed differences in spontaneous antitumor immunity, which were eliminated upon cohousing or after fecal transfer. Sequencing of the 16S ribosomal RNA identified Bifidobacterium as associated with the antitumor effects. Oral administration of Bifidobacterium alone improved tumor control to the same degree as programmed cell death protein 1 ligand 1 (PD-L1)–specific antibody therapy (checkpoint blockade), and combination treatment nearly abolished tumor outgrowth. Augmented dendritic cell function leading to enhanced CD8+ T cell priming and accumulation in the tumor microenvironment mediated the effect. Our data suggest that manipulating the microbiota may modulate cancer immunotherapy.', 'year': 2015, 'citationCount': 2492, 'authors': [{'authorId': '6010657', 'name': 'A. Sivan'}, {'authorId': '1899112', 'name': 'L. Corrales'}, {'authorId': '47063763', 'name': 'N. Hubert'}, {'authorId': '2111787256', 'name': 'Jason B. Williams'}, {'authorId': '1398500939', 'name': 'K. Aquino-Michaels'}, {'authorId': '3718121', 'name': 'Z. Earley'}, {'authorId': '1392752860', 'name': 'Franco W. Benyamin'}, {'authorId': '1768772854', 'name': 'Yuk Man Lei'}, {'authorId': '4216729', 'name': 'B. Jabri'}, {'authorId': '82924398', 'name': 'M. Alegre'}, {'authorId': '1702329', 'name': 'E. Chang'}, {'authorId': '3283308', 'name': 'T. Gajewski'}]}, {'paperId': '3c6e60e52c6a9f8b0998174030ebdb6a9fde5aa8', 'title': 'Rationally engineered Cas9 nucleases with improved specificity', 'abstract': 'Making the correct cut The CRISPR/Cas system is a prokaryotic immune system that targets and cuts out foreign DNA in bacteria. It has been adopted for gene editing because it can be designed to recognize and cut specific locations in the genome. A challenge in developing clinical applications is the potential for off-target effects that could result in DNA cleavage at the wrong locations. Slaymaker et al. used structure-guided engineering to improve the specificity of Streptococcus pyogenes Cas9 (SpCas9). They identified enhanced-specificity variants (eSpCas9) that display reduced off-target cleavage while maintaining robust on-target activity Science, this issue p. 84 Structure-guided engineering improves the genome editing specificity of the CRISPR-associated endonuclease Cas9. The RNA-guided endonuclease Cas9 is a versatile genome-editing tool with a broad range of applications from therapeutics to functional annotation of genes. Cas9 creates double-strand breaks (DSBs) at targeted genomic loci complementary to a short RNA guide. However, Cas9 can cleave off-target sites that are not fully complementary to the guide, which poses a major challenge for genome editing. Here, we use structure-guided protein engineering to improve the specificity of Streptococcus pyogenes Cas9 (SpCas9). Using targeted deep sequencing and unbiased whole-genome off-target analysis to assess Cas9-mediated DNA cleavage in human cells, we demonstrate that “enhanced specificity” SpCas9 (eSpCas9) variants reduce off-target effects and maintain robust on-target cleavage. Thus, eSpCas9 could be broadly useful for genome-editing applications requiring a high level of specificity.', 'year': 2015, 'citationCount': 1857, 'authors': [{'authorId': '4201929', 'name': 'I. Slaymaker'}, {'authorId': '3391994', 'name': 'L. Gao'}, {'authorId': '4627929', 'name': 'Bernd Zetsche'}, {'authorId': '153167506', 'name': 'David A. Scott'}, {'authorId': '5851153', 'name': 'Winston X. Yan'}, {'authorId': '145126988', 'name': 'Feng Zhang'}]}, {'paperId': '927a32a1b1c69a6041a586b71f52be1b957e5030', 'title': 'Promoting an open research culture', 'abstract': 'Author guidelines for journals could help to promote transparency, openness, and reproducibility Transparency, openness, and reproducibility are readily recognized as vital features of science (1, 2). When asked, most scientists embrace these features as disciplinary norms and values (3). Therefore, one might expect that these valued features would be routine in daily practice. Yet, a growing body of evidence suggests that this is not the case (4–6).', 'year': 2015, 'citationCount': 1653, 'authors': [{'authorId': '2862527', 'name': 'Brian A. Nosek'}, {'authorId': '2139865103', 'name': 'G. Alter'}, {'authorId': '8611013', 'name': 'G. Banks'}, {'authorId': '3237553', 'name': 'D. Borsboom'}, {'authorId': '72356049', 'name': 'S. Bowman'}, {'authorId': '6240718', 'name': 'S. Breckler'}, {'authorId': '2073872541', 'name': 'S. Buck'}, {'authorId': '46975669', 'name': 'C. Chambers'}, {'authorId': '2058346280', 'name': 'G. Chin'}, {'authorId': '2054652938', 'name': 'G. Christensen'}, {'authorId': '50230722', 'name': 'M. Contestabile'}, {'authorId': '3198576', 'name': 'A. Dafoe'}, {'authorId': '34840029', 'name': 'E. Eich'}, {'authorId': '2364509', 'name': 'J. Freese'}, {'authorId': '4249203', 'name': 'R. Glennerster'}, {'authorId': '13065618', 'name': 'D. Goroff'}, {'authorId': '48997800', 'name': 'D. P. Green'}, {'authorId': '119365490', 'name': 'B. Hesse'}, {'authorId': '32272151', 'name': 'M. Humphreys'}, {'authorId': '46636130', 'name': 'J. Ishiyama'}, {'authorId': '2124080032', 'name': 'D. Karlan'}, {'authorId': '47733218', 'name': 'A. Kraut'}, {'authorId': '47975256', 'name': 'A. Lupia'}, {'authorId': '3154113', 'name': 'P. Mabry'}, {'authorId': '4133783', 'name': 'T. Madon'}, {'authorId': '118685494', 'name': 'N. Malhotra'}, {'authorId': '1390039913', 'name': 'E. Mayo-Wilson'}, {'authorId': '114856072', 'name': 'M. McNutt'}, {'authorId': '144624624', 'name': 'E. Miguel'}, {'authorId': '4659343', 'name': 'E. Paluck'}, {'authorId': '2130902', 'name': 'U. Simonsohn'}, {'authorId': '4110370', 'name': 'C. Soderberg'}, {'authorId': '101916661', 'name': 'B. Spellman'}, {'authorId': '1398590331', 'name': 'J. Turitto'}, {'authorId': '80222152', 'name': 'G. VandenBos'}, {'authorId': '2809726', 'name': 'S. Vazire'}, {'authorId': '2505272', 'name': 'E. Wagenmakers'}, {'authorId': '153160363', 'name': 'R. Wilson'}, {'authorId': '2075675', 'name': 'T. Yarkoni'}]}, {'paperId': '0e166790fb686d43bb6983800d05c38766c79d39', 'title': 'The human transcriptome across tissues and individuals', 'abstract': 'Expression, genetic variation, and tissues Human genomes show extensive genetic variation across individuals, but we have only just started documenting the effects of this variation on the regulation of gene expression. Furthermore, only a few tissues have been examined per genetic variant. In order to examine how genetic expression varies among tissues within individuals, the Genotype-Tissue Expression (GTEx) Consortium collected 1641 postmortem samples covering 54 body sites from 175 individuals. They identified quantitative genetic traits that affect gene expression and determined which of these exhibit tissue-specific expression patterns. Melé et al. measured how transcription varies among tissues, and Rivas et al. looked at how truncated protein variants affect expression across tissues. Science, this issue p. 648, p. 660, p. 666; see also p. 640 RNA expression documents patterns of human transcriptome variation across individuals and tissues. [Also see Perspective by Gibson] Transcriptional regulation and posttranscriptional processing underlie many cellular and organismal phenotypes. We used RNA sequence data generated by Genotype-Tissue Expression (GTEx) project to investigate the patterns of transcriptome variation across individuals and tissues. Tissues exhibit characteristic transcriptional signatures that show stability in postmortem samples. These signatures are dominated by a relatively small number of genes—which is most clearly seen in blood—though few are exclusive to a particular tissue and vary more across tissues than individuals. Genes exhibiting high interindividual expression variation include disease candidates associated with sex, ethnicity, and age. Primary transcription is the major driver of cellular specificity, with splicing playing mostly a complementary role; except for the brain, which exhibits a more divergent splicing program. Variation in splicing, despite its stochasticity, may play in contrast a comparatively greater role in defining individual phenotypes.', 'year': 2015, 'citationCount': 1054, 'authors': [{'authorId': '2326247', 'name': 'Marta Melé'}, {'authorId': '145209580', 'name': 'Pedro G. Ferreira'}, {'authorId': '1704215264', 'name': 'F. Reverter'}, {'authorId': '46980374', 'name': 'D. DeLuca'}, {'authorId': '6596913', 'name': 'Jean Monlong'}, {'authorId': '1971387', 'name': 'M. Sammeth'}, {'authorId': '31774477', 'name': 'Taylor R. Young'}, {'authorId': '47824049', 'name': 'J. Goldmann'}, {'authorId': '3288223', 'name': 'D. Pervouchine'}, {'authorId': '2068615525', 'name': 'T. Sullivan'}, {'authorId': '1405949033', 'name': 'Rory Johnson'}, {'authorId': '4155963', 'name': 'A. Segrè'}, {'authorId': '3434507', 'name': 'S. Djebali'}, {'authorId': '4501384', 'name': 'Anastasia Niarchou'}, {'authorId': '2122677498', 'name': 'The GTEx Consortium'}, {'authorId': '2056823854', 'name': 'Fred A. Wright'}, {'authorId': '144946488', 'name': 'T. Lappalainen'}, {'authorId': '2060334738', 'name': 'Miquel Calvo'}, {'authorId': '2110594', 'name': 'G. Getz'}, {'authorId': '2939073', 'name': 'E. Dermitzakis'}, {'authorId': '4351471', 'name': 'K. Ardlie'}, {'authorId': '1853610', 'name': 'R. Guigó'}]}, {'paperId': 'd7b42a977e9c09654b5a4e65538fb56bc9252927', 'title': 'Spatially resolved, highly multiplexed RNA profiling in single cells', 'abstract': 'Multiplexed RNA imaging in single cells The basis of cellular function is where and when proteins are expressed and in what quantities. Single-molecule fluorescence in situ hybridization (smFISH) experiments quantify the copy number and location of mRNA molecules; however, the numbers of RNA species that can be simultaneously measured by smFISH has been limited. Using combinatorial labeling with error-robust encoding schemes, Chen et al. simultaneously imaged 100 to 1000 RNA species in a single cell. Such large-scale detection allows regulatory interactions to be analyzed at the transcriptome scale. Science, this issue p. 10.1126/science.aaa6090 A single-molecule imaging method allows simultaneous measurement of 1000 RNA species in single cells. INTRODUCTION The copy number and intracellular localization of RNA are important regulators of gene expression. Measurement of these properties at the transcriptome scale in single cells will give answers to many questions related to gene expression and regulation. Single-molecule RNA imaging approaches, such as single-molecule fluorescence in situ hybridization (smFISH), are powerful tools for counting and mapping RNA; however, the number of RNA species that can be simultaneously imaged in individual cells has been limited. This makes it challenging to perform transcriptomic analysis of single cells in a spatially resolved manner. Here, we report multiplexed error-robust FISH (MERFISH), a single-molecule imaging method that allows thousands of RNA species to be imaged in single cells by using combinatorial FISH labeling with encoding schemes capable of detecting and/or correcting errors. RATIONALE We labeled each cellular RNA with a set of encoding probes, which contain targeting sequences that bind the RNA and readout sequences that bind fluorescently labeled readout probes. Each RNA species is encoded with a particular combination of readout sequences. We used successive rounds of hybridization and imaging, each with a different readout probe, to identify the readout sequences bound to each RNA and to decode the RNA. In principle, combinatorial labeling allows the number of detectable RNA species to grow exponentially with the number of imaging rounds, but the detection errors also increase exponentially. To combat such accumulating errors, we exploited error-robust encoding schemes used in digital electronics, such as the extended Hamming code, in the design of our encoding probes but modified these schemes in order to account for the error properties in FISH measurements. We assigned each RNA a binary word in our modified Hamming code and encoded the RNA with a combination of readout sequences according to this binary word. RESULTS We first imaged 140 RNA species in human fibroblast cells using MERFISH with 16 rounds of hybridization and a modified Hamming code capable of both error detection and correction. We obtained ~80% detection efficiency and observed excellent correlation of RNA copy numbers determined with MERFISH with both bulk RNA sequencing data and conventional smFISH measurements of individual genes. Next, we used an alternative MERFISH encoding scheme, which is capable of detecting but not correcting errors, to image 1001 RNA species in individual cells using only 14 rounds of hybridization. The observed RNA copy numbers again correlate well with bulk sequencing data. However, the detection efficiency is only one-third that of the error-correcting encoding scheme. We performed correlation analysis of the 104 to 106 pairs of measured genes and identified many covarying gene groups that share common regulatory elements. Such grouping allowed us to hypothesize potential functions of ~100 unannotated or partially annotated genes of unknown functions. We further analyzed correlations in the spatial distributions of different RNA species and identified groups of RNAs with different distribution patterns in the cell. DISCUSSION This highly multiplexed imaging approach enables analyses based on the variation and correlation of copy numbers and spatial distributions of a large number of RNA species within single cells. Such analyses should facilitate the delineation of regulatory networks and in situ identification of cell types. We envision that this approach will allow spatially resolved transcriptomes to be determined for single cells. MERFISH for transcriptome imaging. Numerous RNA species can be identified, counted, and localized in a single cell by using MERFISH, a single-molecule imaging approach that uses combinatorial labeling and sequential imaging with encoding schemes capable of detection and/or correction of errors. This highly multiplexed measurement of individual RNAs can be used to compute the gene expression profile and noise, covariation in expression among different genes, and spatial distribution of RNAs within single cells. Knowledge of the expression profile and spatial landscape of the transcriptome in individual cells is essential for understanding the rich repertoire of cellular behaviors. Here, we report multiplexed error-robust fluorescence in situ hybridization (MERFISH), a single-molecule imaging approach that allows the copy numbers and spatial localizations of thousands of RNA species to be determined in single cells. Using error-robust encoding schemes to combat single-molecule labeling and detection errors, we demonstrated the imaging of 100 to 1000 distinct RNA species in hundreds of individual cells. Correlation analysis of the ~104 to 106 pairs of genes allowed us to constrain gene regulatory networks, predict novel functions for many unannotated genes, and identify distinct spatial distribution patterns of RNAs that correlate with properties of the encoded proteins.', 'year': 2015, 'citationCount': 1428, 'authors': [{'authorId': '50053851', 'name': 'Kok Hao Chen'}, {'authorId': '2291340', 'name': 'A. Boettiger'}, {'authorId': '2786991', 'name': 'J. Moffitt'}, {'authorId': '2116420386', 'name': 'Siyuan S Wang'}, {'authorId': '1837985', 'name': 'X. Zhuang'}]}, {'paperId': '2db355f222fc6b05df2e36033d1b1d24acb799ab', 'title': 'm6A mRNA methylation facilitates resolution of naïve pluripotency toward differentiation', 'abstract': 'mRNA modification regulates pluripotency When stem cells progress from an embryonic pluripotent state toward a particular lineage, molecular switches dismantle the transcription factor network that keeps the cell pluripotent. Geula et al. now show that N6-methyladenosine (m6A), a messenger RNA (mRNA) modification present on transcripts of pluripotency factors, drives this transition. Methylation destabilized mRNA transcripts and limited their translation efficiency, which promoted the timely decay of naïve pluripotency. This m6A methylation was also critical for mammalian development. Science, this issue p. 1002 A messenger RNA epigenetic modification regulates stem cell progression from the pluripotent to the differentiated state. Naïve and primed pluripotent states retain distinct molecular properties, yet limited knowledge exists on how their state transitions are regulated. Here, we identify Mettl3, an N6-methyladenosine (m6A) transferase, as a regulator for terminating murine naïve pluripotency. Mettl3 knockout preimplantation epiblasts and naïve embryonic stem cells are depleted for m6A in mRNAs, yet are viable. However, they fail to adequately terminate their naïve state and, subsequently, undergo aberrant and restricted lineage priming at the postimplantation stage, which leads to early embryonic lethality. m6A predominantly and directly reduces mRNA stability, including that of key naïve pluripotency-promoting transcripts. This study highlights a critical role for an mRNA epigenetic modification in vivo and identifies regulatory modules that functionally influence naïve and primed pluripotency in an opposing manner.', 'year': 2015, 'citationCount': 1173, 'authors': [{'authorId': '6819175', 'name': 'Shay Geula'}, {'authorId': '1398554907', 'name': 'S. Moshitch-Moshkovitz'}, {'authorId': '4119568', 'name': 'D. Dominissini'}, {'authorId': '31155224', 'name': 'A. Mansour'}, {'authorId': '2336656', 'name': 'N. Kol'}, {'authorId': '1398554898', 'name': 'M. Salmon-Divon'}, {'authorId': '6481645', 'name': 'Vera Hershkovitz'}, {'authorId': '49925624', 'name': 'Eyal Peer'}, {'authorId': '7998267', 'name': 'Nofar Mor'}, {'authorId': '47428977', 'name': 'Y. Manor'}, {'authorId': '1398555041', 'name': 'Moshe Shay Ben-Haim'}, {'authorId': '3184215', 'name': 'E. Eyal'}, {'authorId': '6895327', 'name': 'Sharon Yunger'}, {'authorId': '2735516', 'name': 'Y. Pinto'}, {'authorId': '5425973', 'name': 'Diego Jaitin'}, {'authorId': '4322985', 'name': 'Sergey V. Viukov'}, {'authorId': '6581218', 'name': 'Y. Rais'}, {'authorId': '4992332', 'name': 'V. Krupalnik'}, {'authorId': '5125935', 'name': 'Elad Chomsky'}, {'authorId': '48950453', 'name': 'Mirie Zerbib'}, {'authorId': '32317040', 'name': 'I. Maza'}, {'authorId': '8244084', 'name': 'Yoav Rechavi'}, {'authorId': '7526795', 'name': \"R'ada Massarwa\"}, {'authorId': '40379130', 'name': 'Suhair Hanna'}, {'authorId': '2960101', 'name': 'I. Amit'}, {'authorId': '2672226', 'name': 'E. Levanon'}, {'authorId': '3432577', 'name': 'N. Amariglio'}, {'authorId': '1397179745', 'name': 'Noam Stern-Ginossar'}, {'authorId': '2376964', 'name': 'Noa Novershtern'}, {'authorId': '1786579', 'name': 'G. Rechavi'}, {'authorId': '1820962', 'name': 'J. Hanna'}]}, {'paperId': 'b3634280958e617fbedad078961f816538da1944', 'title': 'Continuous liquid interface production of 3D objects', 'abstract': 'Fast, continuous, 3D printing Although three-dimensional (3D) printing is now possible using relatively small and low-cost machines, it is still a fairly slow process. This is because 3D printers require a series of steps to cure, replenish, and reposition themselves for each additive cycle. Tumbleston et al. devised a process to effectively grow solid structures out of a liquid bath. The key to the process is the creation of an oxygen-containing “dead zone” between the solid part and the liquid precursor where solidification cannot occur. The precursor liquid is then renewed by the upward movement of the growing solid part. This approach made structures tens of centimeters in size that could contain features with a resolution below 100 µm. Science, this issue p. 1349 Solid parts are elevated from a liquid resin pool at a speed of hundreds of millimeters per hour. Additive manufacturing processes such as 3D printing use time-consuming, stepwise layer-by-layer approaches to object fabrication. We demonstrate the continuous generation of monolithic polymeric parts up to tens of centimeters in size with feature resolution below 100 micrometers. Continuous liquid interface production is achieved with an oxygen-permeable window below the ultraviolet image projection plane, which creates a “dead zone” (persistent liquid interface) where photopolymerization is inhibited between the window and the polymerizing part. We delineate critical control parameters and show that complex solid parts can be drawn out of the resin at rates of hundreds of millimeters per hour. These print speeds allow parts to be produced in minutes instead of hours.', 'year': 2015, 'citationCount': 1456, 'authors': [{'authorId': '3856850', 'name': 'John R. Tumbleston'}, {'authorId': '2888642', 'name': 'D. Shirvanyants'}, {'authorId': '4737061', 'name': 'N. Ermoshkin'}, {'authorId': '5768526', 'name': 'R. Janusziewicz'}, {'authorId': '25711869', 'name': 'Ashley R. Johnson'}, {'authorId': '2070074460', 'name': 'David Kelly'}, {'authorId': '2118439764', 'name': 'Kai Chen'}, {'authorId': '69923437', 'name': 'R. Pinschmidt'}, {'authorId': '1850765', 'name': 'J. Rolland'}, {'authorId': '2661307', 'name': 'A. Ermoshkin'}, {'authorId': '49805613', 'name': 'E. Samulski'}, {'authorId': '6880755', 'name': 'J. Desimone'}]}, {'paperId': '667be50b805b2ac510150821c68e9f911cbe0d1b', 'title': 'Principles of connectivity among morphologically defined cell types in adult neocortex', 'abstract': \"A census of neocortical neurons Despite the importance of the brain's neocortex, we still do not completely understand the diversity and functional connections of its cell types. Jiang et al. recorded, labeled, and classified over 1200 interneurons and more than 400 pyramidal neurons in the mature mouse visual cortex. Fifteen major classes of interneurons fell into three types: some connect to all neurons, some connect to other interneurons, and some form synapses with pyramidal neurons. Science, this issue p. 10.1126/science.aac9462 The connections between more than 10,000 pairs of individually classified neurons in the visual cortex of adult mice are mapped. INTRODUCTION The intricate microcircuitry of the cerebral cortex is thought to be a critical substrate from which arise the impressive capabilities of the mammalian brain. Until now, our knowledge of the stereotypical connectivity in neocortical microcircuits has been pieced together from individual studies of the connectivity between small numbers of neuronal cell types. Here, we provide unbiased, large-scale profiling of neuronal cell types and connections to reveal the essential building blocks of the cortex and the principles governing their assembly into cortical circuits. Using advanced techniques for tissue slicing, multiple simultaneous whole-cell recording, and morphological reconstruction, we are able to provide a comprehensive view of the connectivity between diverse types of neurons, particularly among types of γ-aminobutyric acid–releasing (GABAergic) interneurons, in the adult animal. RATIONALE We took advantage of a method for preparing high-quality slices of adult tissue and combined this technique with octuple simultaneous, whole-cell recordings followed by an improved staining method that allowed detailed recovery of axonal and dendritic arbor morphology. These data allowed us to perform a census of morphologically and electrophysiologically defined neuronal types (primarily GABAergic interneurons) in neocortical layers 1, 2/3, and 5 (L1, L23, and L5, respectively) and to observe their connectivity patterns in adult animals. RESULTS Our large-scale, comprehensive profiling of neocortical neurons differentiated 15 major types of interneurons, in addition to two lamina-defined types of pyramidal neurons (L23 and L5). Cortical interneurons comprise two types in L1 (eNGC and SBC-like), seven in L23 (L23MC, L23NGC, BTC, BPC, DBC, L23BC, and ChC), and six in L5 (L5MC, L5NGC, L5BC, SC, HEC, and DC) (see the figure). Each type has stereotypical electrophysiological properties and morphological features and can be differentiated from all others by cell type–specific axonal geometry and axonal projection patterns. Importantly, each type of neuron has its own characteristic input-output connectivity profile, connecting with other constituent neuronal types with varying degrees of specificity in postsynaptic targets, laminar location, and synaptic characteristics. Despite specific connection patterns for each cell type, we found that a small number of simple connectivity motifs are repeated across layers and cell types defining a canonical cortical microcircuit. CONCLUSION Our comprehensive profiling of neuronal cell types and connections in adult neocortex provides the most complete wiring diagram of neocortical microcircuits to date. Compared with current genetic labels for cell class, which paint the cortex in broad strokes, our analysis of morphological and electrophysiological properties revealed new cell classes and allowed us to derive a small number of simple connectivity rules that were repeated across layers and cell types. This detailed blueprint of cortical wiring should aid efforts to identify specific circuit abnormalities in animal models of brain disease and may eventually provide a path toward the development of comprehensive circuit-based, cell type–specific interventions. Connectivity among morphologically defined cell types in adult neocortex. (A) Simultaneous octuple whole-cell recording to study connectivity followed by morphological reconstruction. (B) Synaptic connectivity between morphologically distinct types of neurons, including pyramidal (P) neurons. (C) Connectivity from NGCs to other cell types. This connectivity is believed to be nonsynaptic and mediated by volume transmission. Martinotti cell, MC; neurogliaform cell, NGC; basket cell, BC; single-bouquet cell-like cell, SBC-like; bitufted cell, BTC; bipolar cell, BPC, double-bouquet cell, DBC; chandelier cell, ChC; shrub cell, SC; horizontally elongated cell, HEC; deep-projecting cell, DC. Since the work of Ramón y Cajal in the late 19th and early 20th centuries, neuroscientists have speculated that a complete understanding of neuronal cell types and their connections is key to explaining complex brain functions. However, a complete census of the constituent cell types and their wiring diagram in mature neocortex remains elusive. By combining octuple whole-cell recordings with an optimized avidin-biotin-peroxidase staining technique, we carried out a morphological and electrophysiological census of neuronal types in layers 1, 2/3, and 5 of mature neocortex and mapped the connectivity between more than 11,000 pairs of identified neurons. We categorized 15 types of interneurons, and each exhibited a characteristic pattern of connectivity with other interneuron types and pyramidal cells. The essential connectivity structure of the neocortical microcircuit could be captured by only a few connectivity motifs.\", 'year': 2015, 'citationCount': 696, 'authors': [{'authorId': '46812938', 'name': 'Xiaolong Jiang'}, {'authorId': '2072819148', 'name': 'Shan Shen'}, {'authorId': '5686630', 'name': 'C. Cadwell'}, {'authorId': '1689077', 'name': 'Philipp Berens'}, {'authorId': '50095217', 'name': 'Fabian H Sinz'}, {'authorId': '1746183', 'name': 'Alexander S. Ecker'}, {'authorId': '4330752', 'name': 'Saumil S. Patel'}, {'authorId': '1739838', 'name': 'A. Tolias'}]}, {'paperId': '4414a1e13595c8ff7bdc861c5725836bc11d76e0', 'title': 'Exome sequencing in amyotrophic lateral sclerosis identifies risk genes and pathways', 'abstract': \"New players in Lou Gehrig's disease Amyotrophic lateral sclerosis (ALS), often referred to as “Lou Gehrig's disease,” is a progressive neurodegenerative disease that affects nerve cells in the brain and the spinal cord. Cirulli et al. sequenced the expressed genes of nearly 3000 ALS patients and compared them with those of more than 6000 controls (see the Perspective by Singleton and Traynor). They identified several proteins that were linked to disease in patients. One such protein, TBK1, is implicated in innate immunity and autophagy and may represent a therapeutic target. Science, this issue p. 1436; see also p. 1422 Analysis of the expressed genes of nearly 2900 patients with amyotrophic lateral sclerosis and about 6400 controls reveals a disease predisposition–associated gene. [Also see Perspective by Singleton and Traynor] Amyotrophic lateral sclerosis (ALS) is a devastating neurological disease with no effective treatment. We report the results of a moderate-scale sequencing study aimed at increasing the number of genes known to contribute to predisposition for ALS. We performed whole-exome sequencing of 2869 ALS patients and 6405 controls. Several known ALS genes were found to be associated, and TBK1 (the gene encoding TANK-binding kinase 1) was identified as an ALS gene. TBK1 is known to bind to and phosphorylate a number of proteins involved in innate immunity and autophagy, including optineurin (OPTN) and p62 (SQSTM1/sequestosome), both of which have also been implicated in ALS. These observations reveal a key role of the autophagic pathway in ALS and suggest specific targets for therapeutic intervention.\", 'year': 2015, 'citationCount': 827, 'authors': [{'authorId': '2853875', 'name': 'E. Cirulli'}, {'authorId': '6055820', 'name': 'Brittany N. Lasseigne'}, {'authorId': '2024504', 'name': 'S. Petrovski'}, {'authorId': '5755914', 'name': 'P. Sapp'}, {'authorId': '2546787', 'name': 'P. Dion'}, {'authorId': '3641674', 'name': 'C. Leblond'}, {'authorId': '6713179', 'name': 'Julien Couthouis'}, {'authorId': '2256947', 'name': 'Yi-Fan Lu'}, {'authorId': '2145348474', 'name': 'Quanli Wang'}, {'authorId': '34252586', 'name': 'Brian J. Krueger'}, {'authorId': '144653215', 'name': 'Z. Ren'}, {'authorId': '32045858', 'name': 'Jonathan E. M. Keebler'}, {'authorId': '48386356', 'name': 'Yujun Han'}, {'authorId': '33853109', 'name': 'S. Levy'}, {'authorId': '1795313', 'name': 'B. Boone'}, {'authorId': '5113017', 'name': 'Jack R. Wimbish'}, {'authorId': '31952392', 'name': 'L. Waite'}, {'authorId': '39049918', 'name': 'Angela L. Jones'}, {'authorId': '1695735', 'name': 'J. Carulli'}, {'authorId': '1380548039', 'name': 'A. Day-Williams'}, {'authorId': '5295959', 'name': 'J. Staropoli'}, {'authorId': '3573620', 'name': 'W. Xin'}, {'authorId': '4939557', 'name': 'A. Chesi'}, {'authorId': '144284776', 'name': 'Alya R. Raphael'}, {'authorId': '1412594994', 'name': 'D. McKenna-Yasek'}, {'authorId': '144166444', 'name': 'J. Cady'}, {'authorId': '8360115', 'name': 'J. M. B. Vianney de Jong'}, {'authorId': '4292338', 'name': 'K. Kenna'}, {'authorId': '5849128', 'name': 'Bradley N. Smith'}, {'authorId': '32677588', 'name': 'S. Topp'}, {'authorId': '2118099706', 'name': 'Jack W. Miller'}, {'authorId': '7790207', 'name': 'A. Gkazi'}, {'authorId': '87486204', 'name': 'A. Al-Chalabi'}, {'authorId': '4703755', 'name': 'L. H. van den Berg'}, {'authorId': '50075325', 'name': 'J. Veldink'}, {'authorId': '2989411', 'name': 'V. Silani'}, {'authorId': '5230585', 'name': 'N. Ticozzi'}, {'authorId': '144756102', 'name': 'C. Shaw'}, {'authorId': '6466995', 'name': 'R. Baloh'}, {'authorId': '1992098', 'name': 'S. Appel'}, {'authorId': '40023500', 'name': 'E. Simpson'}, {'authorId': '1398469732', 'name': 'C. Lagier-Tourenne'}, {'authorId': '34857044', 'name': 'S. Pulst'}, {'authorId': '34924662', 'name': 'S. Gibson'}, {'authorId': '2410201', 'name': 'J. Trojanowski'}, {'authorId': '143893347', 'name': 'L. Elman'}, {'authorId': '145315316', 'name': 'L. McCluskey'}, {'authorId': '144535321', 'name': 'M. Grossman'}, {'authorId': '3750137', 'name': 'N. Shneider'}, {'authorId': '145062232', 'name': 'W. Chung'}, {'authorId': '5606258', 'name': 'J. Ravits'}, {'authorId': '2069522', 'name': 'J. Glass'}, {'authorId': '4127881', 'name': 'K. Sims'}, {'authorId': '5159642', 'name': 'V. V. van Deerlin'}, {'authorId': '2551763', 'name': 'T. Maniatis'}, {'authorId': '5715284', 'name': 'Sebastian Hayes'}, {'authorId': '4096885', 'name': 'A. Ordureau'}, {'authorId': '21152395', 'name': 'S. Swarup'}, {'authorId': '4754915', 'name': 'J. Landers'}, {'authorId': '1995886', 'name': 'F. Baas'}, {'authorId': '144643055', 'name': 'A. Allen'}, {'authorId': '6241427', 'name': 'R. Bedlack'}, {'authorId': '143851342', 'name': 'J. Harper'}, {'authorId': '2102239', 'name': 'A. Gitler'}, {'authorId': '2153485', 'name': 'G. Rouleau'}, {'authorId': '2235964004', 'name': 'Robert H. Brown'}, {'authorId': '22104678', 'name': 'M. Harms'}, {'authorId': '31273465', 'name': 'G. Cooper'}, {'authorId': '2067989942', 'name': 'Tim Harris'}, {'authorId': '34819901', 'name': 'R. Myers'}, {'authorId': '50252657', 'name': 'D. Goldstein'}]}, {'paperId': '96511e6b1a6e0a231f08e933becd20687ca9c41d', 'title': 'High burden and pervasive positive selection of somatic mutations in normal human skin', 'abstract': \"Normal skin's curiously abnormal genome Within every tumor, a battle is being waged. As individual tumor cells acquire new mutations that promote their survival and growth, they clonally expand at the expense of tumor cells that are “less fit.” Martincorena et al. sequenced 234 biopsies of sun-exposed but physiologically normal skin from four individuals (see the Perspective by Brash). They found a surprisingly high burden of mutations, higher than that of many tumors. Many of the mutations known to drive the growth of cutaneous squamous cell carcinomas were already under strong positive selection. More than a quarter of normal skin cells carried a driver mutation, and every square centimeter of skin contained hundreds of competing mutant clones. Science, this issue p. 880; see also p. 867 Sun-exposed but physiologically normal human skin harbors an unexpectedly high number of cancer-causing mutations. [Also see Perspective by Brash] How somatic mutations accumulate in normal cells is central to understanding cancer development but is poorly understood. We performed ultradeep sequencing of 74 cancer genes in small (0.8 to 4.7 square millimeters) biopsies of normal skin. Across 234 biopsies of sun-exposed eyelid epidermis from four individuals, the burden of somatic mutations averaged two to six mutations per megabase per cell, similar to that seen in many cancers, and exhibited characteristic signatures of exposure to ultraviolet light. Remarkably, multiple cancer genes are under strong positive selection even in physiologically normal skin, including most of the key drivers of cutaneous squamous cell carcinomas. Positively selected mutations were found in 18 to 32% of normal skin cells at a density of ~140 driver mutations per square centimeter. We observed variability in the driver landscape among individuals and variability in the sizes of clonal expansions across genes. Thus, aged sun-exposed skin is a patchwork of thousands of evolving clones with over a quarter of cells carrying cancer-causing mutations while maintaining the physiological functions of epidermis.\", 'year': 2015, 'citationCount': 1159, 'authors': [{'authorId': '2748985', 'name': 'I. Martincorena'}, {'authorId': '145503694', 'name': 'A. Roshan'}, {'authorId': '2420343', 'name': 'M. Gerstung'}, {'authorId': '2059629181', 'name': 'P. Ellis'}, {'authorId': '2358657', 'name': 'P. Van Loo'}, {'authorId': '3846300', 'name': 'Stuart McLaren'}, {'authorId': '2824722', 'name': 'D. Wedge'}, {'authorId': '8037157', 'name': 'A. Fullam'}, {'authorId': '2757728', 'name': 'L. Alexandrov'}, {'authorId': '143863567', 'name': 'J. Tubío'}, {'authorId': '2933842', 'name': 'L. Stebbings'}, {'authorId': '143864634', 'name': 'A. Menzies'}, {'authorId': '152218260', 'name': 'S. Widaa'}, {'authorId': '144693778', 'name': 'M. Stratton'}, {'authorId': '145090665', 'name': 'P. Jones'}, {'authorId': '145761328', 'name': 'P. Campbell'}]}, {'paperId': 'abbf1e4d346fd561d60a46bafd0a1de6c2609a98', 'title': 'Slow adaptation in the face of rapid warming leads to collapse of the Gulf of Maine cod fishery', 'abstract': 'Several studies have documented fish populations changing in response to long-term warming. Over the past decade, sea surface temperatures in the Gulf of Maine increased faster than 99% of the global ocean. The warming, which was related to a northward shift in the Gulf Stream and to changes in the Atlantic Multidecadal Oscillation and Pacific Decadal Oscillation, led to reduced recruitment and increased mortality in the region’s Atlantic cod (Gadus morhua) stock. Failure to recognize the impact of warming on cod contributed to overfishing. Recovery of this fishery depends on sound management, but the size of the stock depends on future temperature conditions. The experience in the Gulf of Maine highlights the need to incorporate environmental factors into resource management. Warming waters prevented cod recovery in the North Atlantic. Double jeopardy In the best of worlds, exploited fish stocks are monitored so that harvest quotas protect the reproductive ability of the population. Climate change is likely to complicate this process substantially. Pershing et al. found that cod stocks declined continuously during intense warming in the North Atlantic. Fisheries quotas, even though they were responsibly set and followed by fishers, decreased the reproductive rate. Thus, managing fisheries in a warming world is going to be increasingly problematic. Science, this issue p. 809', 'year': 2015, 'citationCount': 645, 'authors': [{'authorId': '4666755', 'name': 'A. Pershing'}, {'authorId': '47099873', 'name': 'M. Alexander'}, {'authorId': '32328272', 'name': 'Christina M. Hernández'}, {'authorId': '46781149', 'name': 'L. Kerr'}, {'authorId': '48082771', 'name': 'A. Le Bris'}, {'authorId': '5046907', 'name': 'Katherine E. Mills'}, {'authorId': '31020656', 'name': 'J. Nye'}, {'authorId': '2558603', 'name': 'N. Record'}, {'authorId': '7688646', 'name': 'Hillary A. Scannell'}, {'authorId': '32999974', 'name': 'James D. Scott'}, {'authorId': '16789099', 'name': 'G. Sherwood'}, {'authorId': '145250501', 'name': 'A. Thomas'}]}, {'paperId': '81faacd7a521fd2e923227ab1dd01f199520e3df', 'title': 'Marine defaunation: Animal loss in the global ocean', 'abstract': 'Marine animals are disappearing, too The loss of animal species in terrestrial environments has been well documented and is continuing. Loss of species in marine environments has been slower than in terrestrial systems, but appears to be increasing rapidly. McCauley et al. review the recent patterns of species decline and loss in marine environments. Though they note many worrying declines, they also highlight approaches that might allow us to prevent the type of massive defaunation that has occurred on land. Science, this issue 10.1126/science.1255641 BACKGROUND Comparing patterns of terrestrial and marine defaunation helps to place human impacts on marine fauna in context and to navigate toward recovery. Defaunation began in earnest tens of thousands of years later in the oceans than it did on land. Although defaunation has been less severe in the oceans than on land, our effects on marine animals are increasing in pace and impact. Humans have caused few complete extinctions in the sea, but we are responsible for many ecological, commercial, and local extinctions. Despite our late start, humans have already powerfully changed virtually all major marine ecosystems. ADVANCES Humans have profoundly decreased the abundance of both large (e.g., whales) and small (e.g., anchovies) marine fauna. Such declines can generate waves of ecological change that travel both up and down marine food webs and can alter ocean ecosystem functioning. Human harvesters have also been a major force of evolutionary change in the oceans and have reshaped the genetic structure of marine animal populations. Climate change threatens to accelerate marine defaunation over the next century. The high mobility of many marine animals offers some increased, though limited, capacity for marine species to respond to climate stress, but it also exposes many species to increased risk from other stressors. Because humans are intensely reliant on ocean ecosystems for food and other ecosystem services, we are deeply affected by all of these forecasted changes. Three lessons emerge when comparing the marine and terrestrial defaunation experiences: (i) today’s low rates of marine extinction may be the prelude to a major extinction pulse, similar to that observed on land during the industrial revolution, as the footprint of human ocean use widens; (ii) effectively slowing ocean defaunation requires both protected areas and careful management of the intervening ocean matrix; and (iii) the terrestrial experience and current trends in ocean use suggest that habitat destruction is likely to become an increasingly dominant threat to ocean wildlife over the next 150 years. OUTLOOK Wildlife populations in the oceans have been badly damaged by human activity. Nevertheless, marine fauna generally are in better condition than terrestrial fauna: Fewer marine animal extinctions have occurred; many geographic ranges have shrunk less; and numerous ocean ecosystems remain more wild than terrestrial ecosystems. Consequently, meaningful rehabilitation of affected marine animal populations remains within the reach of managers. Human dependency on marine wildlife and the linked fate of marine and terrestrial fauna necessitate that we act quickly to slow the advance of marine defaunation. Timeline (log scale) of marine and terrestrial defaunation. The marine defaunation experience is much less advanced, even though humans have been harvesting ocean wildlife for thousands of years. The recent industrialization of this harvest, however, initiated an era of intense marine wildlife declines. If left unmanaged, we predict that marine habitat alteration, along with climate change (colored bar: IPCC warming), will exacerbate marine defaunation. Marine defaunation, or human-caused animal loss in the oceans, emerged forcefully only hundreds of years ago, whereas terrestrial defaunation has been occurring far longer. Though humans have caused few global marine extinctions, we have profoundly affected marine wildlife, altering the functioning and provisioning of services in every ocean. Current ocean trends, coupled with terrestrial defaunation lessons, suggest that marine defaunation rates will rapidly intensify as human use of the oceans industrializes. Though protected areas are a powerful tool to harness ocean productivity, especially when designed with future climate in mind, additional management strategies will be required. Overall, habitat degradation is likely to intensify as a major driver of marine wildlife loss. Proactive intervention can avert a marine defaunation disaster of the magnitude observed on land.', 'year': 2015, 'citationCount': 902, 'authors': [{'authorId': '35646245', 'name': 'D. McCauley'}, {'authorId': '27836011', 'name': 'M. Pinsky'}, {'authorId': '3857267', 'name': 'S. Palumbi'}, {'authorId': '34611309', 'name': 'J. Estes'}, {'authorId': '7591314', 'name': 'Francis H. Joyce'}, {'authorId': '32657386', 'name': 'R. R. Warner'}]}, {'paperId': '5fe8ed6e2c75a3b352be695f423cc2fac19446fa', 'title': 'Multiplex single-cell profiling of chromatin accessibility by combinatorial cellular indexing', 'abstract': 'Chromatin state and the single cell Identifying the chromatin state of any single cell, which may or may not have a different function or represent different stages relative to others collected within any single culture, experiment, or tissue, has been challenging. Cusanovitch et al. skirted previously identified technological limitations to identify regions of accessible chromatin at single-cell resolution. Combinatorial cellular indexing, a strategy for multiplex barcoding of thousands of single cells per experiment, was successfully used to investigate the genome-wide chromatin accessibility landscape in each of over 15,000 single cells. Science, this issue p. 910 Combinatorial indexing can identify chromatin states at single-cell resolution. Technical advances have enabled the collection of genome and transcriptome data sets with single-cell resolution. However, single-cell characterization of the epigenome has remained challenging. Furthermore, because cells must be physically separated before biochemical processing, conventional single-cell preparatory methods scale linearly. We applied combinatorial cellular indexing to measure chromatin accessibility in thousands of single cells per assay, circumventing the need for compartmentalization of individual cells. We report chromatin accessibility profiles from more than 15,000 single cells and use these data to cluster cells on the basis of chromatin accessibility landscapes. We identify modules of coordinately regulated chromatin accessibility at the level of single cells both between and within cell types, with a scalable method that may accelerate progress toward a human cell atlas.', 'year': 2015, 'citationCount': 892, 'authors': [{'authorId': '6639465', 'name': 'D. Cusanovich'}, {'authorId': '49491404', 'name': 'R. Daza'}, {'authorId': '5552754', 'name': 'Andrew C. Adey'}, {'authorId': '47777448', 'name': 'Hannah A. Pliner'}, {'authorId': '10803574', 'name': 'Lena Christiansen'}, {'authorId': '2147608', 'name': 'K. Gunderson'}, {'authorId': '6420805', 'name': 'F. Steemers'}, {'authorId': '2244164', 'name': 'C. Trapnell'}, {'authorId': '2431330', 'name': 'J. Shendure'}]}, {'paperId': '77d7aa4efa1f562a0749b104c77662562cc0f3c5', 'title': 'High-efficiency solution-processed perovskite solar cells with millimeter-scale grains', 'abstract': 'Large-crystal perovskite films The performance of organic-inorganic hybrid perovskite planar solar cells has steadily improved. One outstanding issue is that grain boundaries and defects in polycrystalline films degrade their output. Now, two studies report the growth of millimeter-scale single crystals. Nie et al. grew continuous, pinhole-free, thin iodochloride films with a hot-casting technique and report device efficiencies of 18%. Shi et al. used antisolvent vapor-assisted crystallization to grow millimeter-scale bromide and iodide cubic crystals with charge-carrier diffusion lengths exceeding 10 mm. Science, this issue p. 522, p. 519 Solution processing techniques enable the growth of high-quality, large-area perovskite crystals for solar cells. State-of-the-art photovoltaics use high-purity, large-area, wafer-scale single-crystalline semiconductors grown by sophisticated, high-temperature crystal growth processes. We demonstrate a solution-based hot-casting technique to grow continuous, pinhole-free thin films of organometallic perovskites with millimeter-scale crystalline grains. We fabricated planar solar cells with efficiencies approaching 18%, with little cell-to-cell variability. The devices show hysteresis-free photovoltaic response, which had been a fundamental bottleneck for the stable operation of perovskite devices. Characterization and modeling attribute the improved performance to reduced bulk defects and improved charge carrier mobility in large-grain devices. We anticipate that this technique will lead the field toward synthesis of wafer-scale crystalline perovskites, necessary for the fabrication of high-efficiency solar cells, and will be applicable to several other material systems plagued by polydispersity, defects, and grain boundary recombination in solution-processed thin films.', 'year': 2015, 'citationCount': 2824, 'authors': [{'authorId': '48638740', 'name': 'W. Nie'}, {'authorId': '3875114', 'name': 'H. Tsai'}, {'authorId': '2785212', 'name': 'R. Asadpour'}, {'authorId': '144223671', 'name': 'J. Blancon'}, {'authorId': '4541893', 'name': 'A. Neukirch'}, {'authorId': '144410378', 'name': 'Gautam Gupta'}, {'authorId': '88634292', 'name': 'J. Crochet'}, {'authorId': '5025124', 'name': 'M. Chhowalla'}, {'authorId': '6232252', 'name': 'S. Tretiak'}, {'authorId': '2110011191', 'name': 'M. Alam'}, {'authorId': '7643816', 'name': 'Hsing-lin Wang'}, {'authorId': '4788240', 'name': 'A. Mohite'}]}, {'paperId': '8595b62e8933f29f614d151a60d34fb228891b75', 'title': 'The dominant role of semi-arid ecosystems in the trend and variability of the land CO2 sink', 'abstract': 'The difference is found at the margins The terrestrial biosphere absorbs about a quarter of all anthropogenic carbon dioxide emissions, but the amount that they take up varies from year to year. Why? Combining models and observations, Ahlström et al. found that marginal ecosystems—semiarid savannas and low-latitude shrublands—are responsible for most of the variability. Biological productivity in these semiarid regions is water-limited and strongly associated with variations in precipitation, unlike wetter tropical areas. Understanding carbon uptake by these marginal lands may help to improve predictions of variations in the global carbon cycle. Science, this issue p. 895 Semi-arid regions cause most of the interannual variability of the terrestrial carbon dioxide sink. The growth rate of atmospheric carbon dioxide (CO2) concentrations since industrialization is characterized by large interannual variability, mostly resulting from variability in CO2 uptake by terrestrial ecosystems (typically termed carbon sink). However, the contributions of regional ecosystems to that variability are not well known. Using an ensemble of ecosystem and land-surface models and an empirical observation-based product of global gross primary production, we show that the mean sink, trend, and interannual variability in CO2 uptake by terrestrial ecosystems are dominated by distinct biogeographic regions. Whereas the mean sink is dominated by highly productive lands (mainly tropical forests), the trend and interannual variability of the sink are dominated by semi-arid ecosystems whose carbon balance is strongly associated with circulation-driven variations in both precipitation and temperature.', 'year': 2015, 'citationCount': 950, 'authors': [{'authorId': '3640646', 'name': 'A. Ahlström'}, {'authorId': '34992220', 'name': 'M. Raupach'}, {'authorId': '4156014', 'name': 'G. Schurgers'}, {'authorId': '1417792690', 'name': 'Benjamin Smith'}, {'authorId': '2792896', 'name': 'A. Arneth'}, {'authorId': '95799904', 'name': 'M. Jung'}, {'authorId': '2530948', 'name': 'M. Reichstein'}, {'authorId': '66251280', 'name': 'J. Canadell'}, {'authorId': '4213663', 'name': 'P. Friedlingstein'}, {'authorId': '145347290', 'name': 'Atul K. Jain'}, {'authorId': '46310681', 'name': 'Etsushi Kato'}, {'authorId': '144853616', 'name': 'B. Poulter'}, {'authorId': '3728155', 'name': 'S. Sitch'}, {'authorId': '35759515', 'name': 'B. Stocker'}, {'authorId': '2350727', 'name': 'N. Viovy'}, {'authorId': '49417469', 'name': 'Ying Ping Wang'}, {'authorId': '8407849', 'name': 'A. Wiltshire'}, {'authorId': '4640770', 'name': 'S. Zaehle'}, {'authorId': '2149303', 'name': 'N. Zeng'}]}, {'paperId': '1c8d59d11fcdf222bbe3773e00513f264d268be0', 'title': 'The DNA damage response induces inflammation and senescence by inhibiting autophagy of GATA4', 'abstract': 'Transcriptional control of cell senescence Senescent cells that have stopped proliferating secrete molecules that influence the cells around them. Prevention of this senescence-activated secretory phenotype seems to slow organismal aging. Kang et al. explored the regulatory process behind cell senescence and found that DNA damage led to stabilization of the transcription factor GATA4 (see the Perspective by Cassidy and Narita). Increased activity of GATA4 in senescent cells stimulated genes encoding secreted factors. GATA4 also accumulates in the brains of aging mice or humans. Science, this issue 10.1126/science.aaa5612; see also p. 1448 The transcription factor GATA4 promotes cell senescence. [Also see Perspective by Cassidy and Narita] INTRODUCTION Cellular senescence is a program of arrested proliferation and altered gene expression triggered by many stresses. Although it is a potent tumor-suppressive mechanism, senescence has been implicated in several pathological processes including aging, age-associated diseases, and (counterintuitively) tumorigenesis. One potential mechanism through which senescent cells exert such pleiotropic effects is the secretion of proinflammatory cytokines, chemokines, growth factors, and proteases, termed the senescence-associated secretory phenotype (SASP), which affects senescent cells and their microenvironment. The mechanism by which the SASP is initiated and maintained is not well characterized beyond the classical regulators of inflammation, including the transcription factors NF-κB and C/EBPβ. RATIONALE In senescence growth arrest, two core senescence-regulating pathways, p53 and p16INK4a/Rb, play a critical role. By contrast, the SASP does not depend on either p53 or p16INK4a, which suggests the existence of an independent senescence regulatory network that controls the SASP. Having observed high levels of induction of microRNA miR-146a during induced senescence in human fibroblasts, we developed a green fluorescent protein–tagged senescence reporter based on a miR-146a promoter fragment. This reporter responded to senescence-inducing stimuli, including replicative exhaustion, DNA damage, and oncogenic RAS activation—all of which activate the SASP. This system allowed us to identify additional regulators of senescence and the SASP. RESULTS Through miR-146a promoter analysis, we mapped the critical region for senescence-induced activity and identified the transcriptional regulator responsible for this regulation, GATA4, previously known as a regulator of embryonic development. Ectopic expression of GATA4 induced senescence, whereas disruption of GATA4 suppressed it, thus establishing GATA4 as a senescence regulator. GATA4 protein abundance, but not mRNA, increased during sene1scence, primarily as a result of increased protein stability. Under normal conditions, GATA4 binds the p62 autophagy adaptor and is degraded by selective autophagy. Upon senescence induction, however, this selective autophagy was suppressed through decreased interaction between GATA4 and p62, thereby stabilizing GATA4. GATA4 in turn induced TRAF3IP2 (tumor necrosis factor receptor–associated factor interacting protein 2) and IL1A (interleukin 1A), which activate NF-κB to initiate and maintain the SASP, thus facilitating senescence. GATA4 pathway activation depends on the key DNA damage response (DDR) kinases ATM (ataxia telangiectasia mutated) and ATR (ataxia telangiectasia and Rad3–related), as does senescence-associated activation of p53 and p16INK4a. However, the GATA4 pathway is independent of p53 and p16INK4a. Finally, GATA4 protein accumulated in multiple tissues in mice treated with senescence-inducing stimuli and during normal mouse and human aging, including many cell types in the brain; these findings raise the possibility that the GATA4 pathway drives age-dependent inflammation. CONCLUSION Our results indicate that GATA4 connects autophagy and the DDR to senescence and inflammation through TRAF3IP2 and IL1A activation of NF-κB. These findings establish GATA4 as a key switch activated by the DDR to regulate senescence, independently of p53 and p16INK4a. Our in vivo data indicate a potential role of GATA4 during aging and its associated inflammation. Because accumulation of senescent cells is thought to promote aging and aging-associated diseases through the resulting inflammatory response, inhibiting the GATA4 pathway may provide an avenue for therapeutic intervention. GATA4 functions as a key switch in the senescence regulatory network to activate the SASP. The nonsenescent state is maintained by inhibitory barriers that prevent cell cycle arrest and inflammation. Upon senescence-inducing signals, ATM and ATR relieve inhibition of the p53 and p16INK4a pathways to induce growth arrest and also block p62-dependent autophagic degradation of GATA4, resulting in NF-κB activation and SASP induction. Cellular senescence is a terminal stress-activated program controlled by the p53 and p16INK4a tumor suppressor proteins. A striking feature of senescence is the senescence-associated secretory phenotype (SASP), a pro-inflammatory response linked to tumor promotion and aging. We have identified the transcription factor GATA4 as a senescence and SASP regulator. GATA4 is stabilized in cells undergoing senescence and is required for the SASP. Normally, GATA4 is degraded by p62-mediated selective autophagy, but this regulation is suppressed during senescence, thereby stabilizing GATA4. GATA4 in turn activates the transcription factor NF-κB to initiate the SASP and facilitate senescence. GATA4 activation depends on the DNA damage response regulators ATM and ATR, but not on p53 or p16INK4a. GATA4 accumulates in multiple tissues, including the aging brain, and could contribute to aging and its associated inflammation.', 'year': 2015, 'citationCount': 631, 'authors': [{'authorId': '49206657', 'name': 'Chanhee Kang'}, {'authorId': '8285259', 'name': 'Qikai Xu'}, {'authorId': '144696763', 'name': 'Timothy D. Martin'}, {'authorId': '25857253', 'name': 'Mamie Z. Li'}, {'authorId': '9109024', 'name': 'M. Demaria'}, {'authorId': '30472762', 'name': 'Liviu Aron'}, {'authorId': '40629125', 'name': 'Tao Lu'}, {'authorId': '6749057', 'name': 'B. Yankner'}, {'authorId': '3050721', 'name': 'J. Campisi'}, {'authorId': '4339245', 'name': 'S. Elledge'}]}, {'paperId': 'd290f977586fac98c695346c9591e33763269987', 'title': 'Activation of proto-oncogenes by disruption of chromosome neighborhoods', 'abstract': 'The spread of bad neighborhoods Our genomes have complex three-dimensional (3D) arrangements that partition and regulate gene expression. Cancer cells frequently have their genomes grossly rearranged, disturbing this intricate 3D organization. Hnisz et al. show that the disruption of these 3D neighborhoods can bring oncogenes under the control of regulatory elements normally kept separate from them (see the Perspective by Wala and Beroukim). These novel juxtapositions can result in the inappropriate activation of oncogenes. Science, this issue p. 1454; see also p. 1398 Disrupting the boundaries between three-dimensional neighborhoods in the genome can activate cancer-promoting genes. [Also see Perspective by Wala and Beroukim] Oncogenes are activated through well-known chromosomal alterations such as gene fusion, translocation, and focal amplification. In light of recent evidence that the control of key genes depends on chromosome structures called insulated neighborhoods, we investigated whether proto-oncogenes occur within these structures and whether oncogene activation can occur via disruption of insulated neighborhood boundaries in cancer cells. We mapped insulated neighborhoods in T cell acute lymphoblastic leukemia (T-ALL) and found that tumor cell genomes contain recurrent microdeletions that eliminate the boundary sites of insulated neighborhoods containing prominent T-ALL proto-oncogenes. Perturbation of such boundaries in nonmalignant cells was sufficient to activate proto-oncogenes. Mutations affecting chromosome neighborhood boundaries were found in many types of cancer. Thus, oncogene activation can occur via genetic alterations that disrupt insulated neighborhoods in malignant cells.', 'year': 2015, 'citationCount': 793, 'authors': [{'authorId': '3787252', 'name': 'Denes Hnisz'}, {'authorId': '48537699', 'name': 'Abraham S. Weintraub'}, {'authorId': '5885383', 'name': 'Daniel S. Day'}, {'authorId': '7255958', 'name': 'Anne-Laure Valton'}, {'authorId': '3725971', 'name': 'R. Bak'}, {'authorId': '2109482219', 'name': 'Charles H. Li'}, {'authorId': '4188811', 'name': 'Johanna E. Goldmann'}, {'authorId': '2399615', 'name': 'B. Lajoie'}, {'authorId': '144073851', 'name': 'Z. Fan'}, {'authorId': '4066956', 'name': 'A. Sigova'}, {'authorId': '46291633', 'name': 'J. Reddy'}, {'authorId': '1398807023', 'name': 'Diego Borges-Rivera'}, {'authorId': '152240986', 'name': 'Tong Ihn Lee'}, {'authorId': '5203884', 'name': 'R. Jaenisch'}, {'authorId': '4046906', 'name': 'M. Porteus'}, {'authorId': '145696688', 'name': 'J. Dekker'}, {'authorId': '144787676', 'name': 'R. Young'}]}, {'paperId': 'e171cbf02cc7aa5a6f72d514799ca291523360e4', 'title': 'Low trap-state density and long carrier diffusion in organolead trihalide perovskite single crystals', 'abstract': 'Large-crystal perovskite films The performance of organic-inorganic hybrid perovskite planar solar cells has steadily improved. One outstanding issue is that grain boundaries and defects in polycrystalline films degrade their output. Now, two studies report the growth of millimeter-scale single crystals. Nie et al. grew continuous, pinhole-free, thin iodochloride films with a hot-casting technique and report device efficiencies of 18%. Shi et al. used antisolvent vapor-assisted crystallization to grow millimeter-scale bromide and iodide cubic crystals with charge-carrier diffusion lengths exceeding 10 mm. Science, this issue p. 522, p. 519 Solution processing techniques enable the growth of high-quality, large-area perovskite crystals for solar cells. The fundamental properties and ultimate performance limits of organolead trihalide MAPbX3 (MA = CH3NH3+; X = Br– or I–) perovskites remain obscured by extensive disorder in polycrystalline MAPbX3 films. We report an antisolvent vapor-assisted crystallization approach that enables us to create sizable crack-free MAPbX3 single crystals with volumes exceeding 100 cubic millimeters. These large single crystals enabled a detailed characterization of their optical and charge transport characteristics. We observed exceptionally low trap-state densities on the order of 109 to 1010 per cubic centimeter in MAPbX3 single crystals (comparable to the best photovoltaic-quality silicon) and charge carrier diffusion lengths exceeding 10 micrometers. These results were validated with density functional theory calculations.', 'year': 2015, 'citationCount': 3627, 'authors': [{'authorId': '144968738', 'name': 'D. Shi'}, {'authorId': '35620881', 'name': 'V. Adinolfi'}, {'authorId': '47486987', 'name': 'R. Comin'}, {'authorId': '1758918', 'name': 'M. Yuan'}, {'authorId': '6032563', 'name': 'E. Alarousu'}, {'authorId': '48368168', 'name': 'A. Buin'}, {'authorId': '2145266143', 'name': 'Yin Chen'}, {'authorId': '3891522', 'name': 'S. Hoogland'}, {'authorId': '6209438', 'name': 'A. Rothenberger'}, {'authorId': '7837348', 'name': 'K. Katsiev'}, {'authorId': '46368681', 'name': 'Y. Losovyj'}, {'authorId': '2149172030', 'name': 'Xin Zhang'}, {'authorId': '145977172', 'name': 'P. Dowben'}, {'authorId': '144348186', 'name': 'O. Mohammed'}, {'authorId': '1799992', 'name': 'E. Sargent'}, {'authorId': '3095691', 'name': 'O. Bakr'}]}, {'paperId': '26e7f0da2f0d586c60fc3a8bf8605d4b2a38ff78', 'title': 'RNA editing by ADAR1 prevents MDA5 sensing of endogenous dsRNA as nonself', 'abstract': 'RNA editing helps identify cellular RNAs Adenosine bases in messenger RNA (mRNAs) can be enzymatically modified and changed into inosine bases. This RNA “editing” is mediated by adenosine deaminase acting on RNA (ADAR) enzymes. Liddicoat et al. show that the in vivo targets of the principal editing enzyme, ADAR1, are long double-stranded RNA (dsRNA) structures in noncoding portions of cellular mRNAs. ADAR1-directed editing of these cellular targets is critical to avoid activation of an immune response to dsRNA in the cytoplasm, because dsRNA is also a marker of viral infection. Science, this issue p. 1115 The principal RNA-editing enzyme modifies cellular RNAs to prevent their erroneous identification as foreign RNA. Adenosine-to-inosine (A-to-I) editing is a highly prevalent posttranscriptional modification of RNA, mediated by ADAR (adenosine deaminase acting on RNA) enzymes. In addition to RNA editing, additional functions have been proposed for ADAR1. To determine the specific role of RNA editing by ADAR1, we generated mice with an editing-deficient knock-in mutation (Adar1E861A, where E861A denotes Glu861→Ala861). Adar1E861A/E861A embryos died at ~E13.5 (embryonic day 13.5), with activated interferon and double-stranded RNA (dsRNA)–sensing pathways. Genome-wide analysis of the in vivo substrates of ADAR1 identified clustered hyperediting within long dsRNA stem loops within 3′ untranslated regions of endogenous transcripts. Finally, embryonic death and phenotypes of Adar1E861A/E861A were rescued by concurrent deletion of the cytosolic sensor of dsRNA, MDA5. A-to-I editing of endogenous dsRNA is the essential function of ADAR1, preventing the activation of the cytosolic dsRNA response by endogenous transcripts.', 'year': 2015, 'citationCount': 578, 'authors': [{'authorId': '5785182', 'name': 'Brian J. Liddicoat'}, {'authorId': '5585182', 'name': 'Robert Piskol'}, {'authorId': '3027693', 'name': 'A. Chalk'}, {'authorId': '37579243', 'name': 'Gokul Ramaswami'}, {'authorId': '39661583', 'name': 'M. Higuchi'}, {'authorId': '5218250', 'name': 'Jochen C. Hartner'}, {'authorId': '1492117701', 'name': 'J. B. Li'}, {'authorId': '144870675', 'name': 'P. Seeburg'}, {'authorId': '4730920', 'name': 'Carl R Walkley'}]}, {'paperId': '08aa0f7272560b90d8cc4e49bd5defbdb20d1842', 'title': 'Boreal forest health and global change', 'abstract': 'The boreal forest, one of the largest biomes on Earth, provides ecosystem services that benefit society at levels ranging from local to global. Currently, about two-thirds of the area covered by this biome is under some form of management, mostly for wood production. Services such as climate regulation are also provided by both the unmanaged and managed boreal forests. Although most of the boreal forests have retained the resilience to cope with current disturbances, projected environmental changes of unprecedented speed and amplitude pose a substantial threat to their health. Management options to reduce these threats are available and could be implemented, but economic incentives and a greater focus on the boreal biome in international fora are needed to support further adaptation and mitigation actions.', 'year': 2015, 'citationCount': 683, 'authors': [{'authorId': '46813897', 'name': 'S. Gauthier'}, {'authorId': '33655263', 'name': 'P. Bernier'}, {'authorId': '5684568', 'name': 'T. Kuuluvainen'}, {'authorId': '2689353', 'name': 'A. Shvidenko'}, {'authorId': '1877912', 'name': 'D. Schepaschenko'}]}, {'paperId': '99a3c6c33c7c4470a550061f2aa25dcd406df9ce', 'title': 'Pervasive drought legacies in forest ecosystems and their implications for carbon cycle models', 'abstract': \"Drought effects on carbon cycling The response of forest ecosystems to drought is increasingly important in the context of a warming climate. Anderegg et al. studied a tree-ring database of 1338 forest sites from around the globe. They found that forests exhibit a drought “legacy effect” with 3 to 4 years' reduced growth following drought. During this postdrought delay, forests will be less able to act as a sink for carbon. Incorporating forest legacy effects into Earth system models will provide more accurate predictions of the effects of drought on the global carbon cycle. Science, this issue p. 528 Tree-ring data from 1338 forest sites show that after a drought tree growth is reduced for a further 3 to 4 years. The impacts of climate extremes on terrestrial ecosystems are poorly understood but important for predicting carbon cycle feedbacks to climate change. Coupled climate–carbon cycle models typically assume that vegetation recovery from extreme drought is immediate and complete, which conflicts with the understanding of basic plant physiology. We examined the recovery of stem growth in trees after severe drought at 1338 forest sites across the globe, comprising 49,339 site-years, and compared the results with simulated recovery in climate-vegetation models. We found pervasive and substantial “legacy effects” of reduced growth and incomplete recovery for 1 to 4 years after severe drought. Legacy effects were most prevalent in dry ecosystems, among Pinaceae, and among species with low hydraulic safety margins. In contrast, limited or no legacy effects after drought were simulated by current climate-vegetation models. Our results highlight hysteresis in ecosystem-level carbon cycling and delayed recovery from climate extremes.\", 'year': 2015, 'citationCount': 728, 'authors': [{'authorId': '3736387', 'name': 'W. Anderegg'}, {'authorId': '3736387', 'name': 'W. Anderegg'}, {'authorId': '2139023', 'name': 'C. Schwalm'}, {'authorId': '2939516', 'name': 'F. Biondi'}, {'authorId': '2359553', 'name': 'J. Camarero'}, {'authorId': '153749410', 'name': 'G. Koch'}, {'authorId': '3739443', 'name': 'M. Litvak'}, {'authorId': '2563825', 'name': 'K. Ogle'}, {'authorId': '144827840', 'name': 'J. Shaw'}, {'authorId': '3773288', 'name': 'E. Shevliakova'}, {'authorId': '1518746020', 'name': 'A. P. Williams'}, {'authorId': '2073843', 'name': 'A. Wolf'}, {'authorId': '6513992', 'name': 'E. Ziaco'}, {'authorId': '4955260', 'name': 'S. Pacala'}]}, {'paperId': 'e964eec826f48f6bec6e62319655e34764a13335', 'title': 'Gene essentiality and synthetic lethality in haploid human cells', 'abstract': \"Zeroing in on essential human genes More powerful genetic techniques are helping to define the list of genes required for the life of a human cell. Two papers used the CRISPR genome editing system and a gene trap method in haploid human cells to screen for essential genes (see the Perspective by Boone and Andrews). Wang et al.'s analysis of multiple cell lines indicates that it may be possible to find tumor-specific dependencies on particular genes. Blomen et al. investigate the phenomenon in which nonessential genes are required for fitness in the absence of another gene. Hence, complexity rather than robustness is the human strategy. Science, this issue p. 1096 and p. 1092; see also p. 1028 Systematic mutagenesis reveals essential genetic interactions required for human cells to keep growing. [Also see Perspective by Boone and Andrews] Although the genes essential for life have been identified in less complex model organisms, their elucidation in human cells has been hindered by technical barriers. We used extensive mutagenesis in haploid human cells to identify approximately 2000 genes required for optimal fitness under culture conditions. To study the principles of genetic interactions in human cells, we created a synthetic lethality network focused on the secretory pathway based exclusively on mutations. This revealed a genetic cross-talk governing Golgi homeostasis, an additional subunit of the human oligosaccharyltransferase complex, and a phosphatidylinositol 4-kinase β adaptor hijacked by viruses. The synthetic lethality map parallels observations made in yeast and projects a route forward to reveal genetic networks in diverse aspects of human cell biology.\", 'year': 2015, 'citationCount': 677, 'authors': [{'authorId': '4368790', 'name': 'V. Blomen'}, {'authorId': '34025797', 'name': 'P. Májek'}, {'authorId': '3994560', 'name': 'L. T. Jae'}, {'authorId': '4217436', 'name': 'J. Bigenzahn'}, {'authorId': '2676853', 'name': 'Joppe Nieuwenhuis'}, {'authorId': '8518116', 'name': 'J. Staring'}, {'authorId': '153530312', 'name': 'R. Sacco'}, {'authorId': '102248379', 'name': 'Ferdy R. van Diemen'}, {'authorId': '1820863502', 'name': 'N. Olk'}, {'authorId': '3697345', 'name': 'A. Stukalov'}, {'authorId': '4686719', 'name': 'C. Marceau'}, {'authorId': '144985777', 'name': 'H. Janssen'}, {'authorId': '16762589', 'name': 'J. Carette'}, {'authorId': '2502649', 'name': 'K. Bennett'}, {'authorId': '2248107', 'name': 'J. Colinge'}, {'authorId': '1387445843', 'name': 'G. Superti-Furga'}, {'authorId': '3658300', 'name': 'T. Brummelkamp'}]}, {'paperId': 'acfea18276e393b6e147bed8f04b0ce8bccc9aaf', 'title': 'Soil and human security in the 21st century', 'abstract': \"Global soil resources under stress The future of humanity is intertwined with the future of Earth's soil resources. Soil provides for agriculture, improves water quality, and buffers greenhouse gases in the atmosphere. Yet human activities, including agricultural soil erosion, are rapidly degrading soil faster than it is naturally replenished. At this rate, human security over the next century will be severely threatened by unsustainable soil management practices. Amundson et al. review recent advances in understanding global soil resources, including how carbon stored in soil responds to anthropogenic warming. Translating this knowledge into practice is the biggest challenge remaining. Science, this issue 10.1126/science.1261071 BACKGROUND Earth’s soil has formed by processes that have maintained a persistent and expansive global soil mantle, one that in turn provided the stage for the evolution of the vast diversity of life on land. The underlying stability of soil systems is controlled by their inherent balance between inputs and losses of nutrients and carbon. Human exploitation of these soil resources, beginning a few thousand years ago, allowed agriculture to become an enormous success. The vastness of the planet and its soil resources allowed agriculture to expand, with growing populations, or to move, when soil resources were depleted. However, the practice of farming greatly accelerated rates of erosion relative to soil production, and soil has been and continues to be lost at rates that are orders of magnitude greater than mechanisms that replenish soil. Additionally, agricultural practices greatly altered natural soil carbon balances and feedbacks. Cultivation thus began an ongoing slow ignition of Earth’s largest surficial reservoir of carbon—one that, when combined with the anthropogenic warming of many biomes, is capable of driving large positive feedbacks that will further increase the accumulation of atmospheric greenhouse gases and exacerbate associated climate change. ADVANCES The study of soil is now the domain of diverse schools of physical and biological science. Rapid advances in empirical and theoretical understanding of soil processes are occurring. These advances have brought an international, and global, perspective to the study of soil processes and focused the implications of soil stewardship for societal well-being. Major advances in the past decade include our first quantitative understanding of the natural rates of soil production, derived from isotopic methods developed by collaboration of geochemists and geomorphologists. Proliferation of research by soil and ecological scientists in the northern latitudes continues to illuminate and improve estimates of the magnitude of soil carbon storage in these regions and its sensitivity and response to warming. The role of soil processes in global carbon and climate models is entering a period of growing attention and increasing maturity. These activities in turn reveal the severity of soil-related issues at stake for the remainder of this century—the need to rapidly regain a balance to the physical and biological processes that drive and maintain soil properties, and the societal implications that will result if we do not. OUTLOOK Both great challenges and opportunities exist in regards to maintaining soil’s role in food, climate, and human security. Erosion continues to exceed natural rates of soil renewal even in highly developed countries. The recent focus by economists and natural scientists on potential future shortages of phosphorus fertilizer offers opportunities for novel partnerships to develop efficient methods of nutrient recycling and redistribution systems in urban settings. Possibly the most challenging issues will be to better understand the magnitude of global soil carbon feedbacks to climate change and to mitigating climate change in a timely fashion. The net results of human impacts on soil resources this century will be global in scale and will have direct impacts on human security for centuries to come. Large-scale erosion forming a gully system in the watershed of Lake Bogoria, Kenya. Accelerated soil erosion here is due to both overgrazing and improper agricultural management, which are partially due to political-social impacts of past colonization and inadequate resources and infrastructure. The erosion additionally affects the long-term future of Lake Bogoria because of rapid sedimentation. This example illustrates the disruption of the natural balance of soil production and erosion over geological time scales by human activity and the rapidity of the consequences of this imbalance. CREDIT: BRENT STIRTON/GETTY IMAGES Human security has and will continue to rely on Earth’s diverse soil resources. Yet we have now exploited the planet’s most productive soils. Soil erosion greatly exceeds rates of production in many agricultural regions. Nitrogen produced by fossil fuel and geological reservoirs of other fertilizers are headed toward possible scarcity, increased cost, and/or geopolitical conflict. Climate change is accelerating the microbial release of greenhouse gases from soil organic matter and will likely play a large role in our near-term climate future. In this Review, we highlight challenges facing Earth’s soil resources in the coming century. The direct and indirect response of soils to past and future human activities will play a major role in human prosperity and survival.\", 'year': 2015, 'citationCount': 832, 'authors': [{'authorId': '7249227', 'name': 'R. Amundson'}, {'authorId': '4885197', 'name': 'A. Berhe'}, {'authorId': '5631731', 'name': 'J. Hopmans'}, {'authorId': '82933112', 'name': 'C. Olson'}, {'authorId': '8311470', 'name': 'A. Sztein'}, {'authorId': '1870724', 'name': 'D. Sparks'}]}, {'paperId': '2846ecd7a2eb14e2262e7cd53296170ba49b76e2', 'title': 'Neutrophil extracellular traps license macrophages for cytokine production in atherosclerosis', 'abstract': 'Neutrophil NETs drive atherosclerosis The buildup of fats, cholesterol, and other substances in arteries causes atherosclerosis, which restricts blood flow and can lead to heart attacks and stroke. Inflammation contributes to the pathogenesis of atherosclerosis, but exactly how is not fully understood. Warnatsch et al. now show that immune cells called neutrophils release NETs (neutrophil extracellular traps) (see the Perspective by Nahrendorf and Swirski). These NETs are composed of DNA and antimicrobial proteins, and in the setting of atherosclerosis they activate innate immune signaling pathways in macrophages. This causes the macrophages to secrete proinflammatory cytokines, exacerbating the disease. Indirectly, NETS also attract a specialized subset of T cells that further amplify the proinflammatory response. Science, this issue p. 316; see also p. 237 Neutrophils release networks of extracellular fibers that contribute to the pathogenesis of atherosclerosis. [Also see Perspective by Nahrendorf and Swirski] Secretion of the cytokine interleukin-1β (IL-1β) by macrophages, a major driver of pathogenesis in atherosclerosis, requires two steps: Priming signals promote transcription of immature IL-1β, and then endogenous “danger” signals activate innate immune signaling complexes called inflammasomes to process IL-1β for secretion. Although cholesterol crystals are known to act as danger signals in atherosclerosis, what primes IL-1β transcription remains elusive. Using a murine model of atherosclerosis, we found that cholesterol crystals acted both as priming and danger signals for IL-1β production. Cholesterol crystals triggered neutrophils to release neutrophil extracellular traps (NETs). NETs primed macrophages for cytokine release, activating T helper 17 (TH17) cells that amplify immune cell recruitment in atherosclerotic plaques. Therefore, danger signals may drive sterile inflammation, such as that seen in atherosclerosis, through their interactions with neutrophils.', 'year': 2015, 'citationCount': 799, 'authors': [{'authorId': '3739915', 'name': 'A. Warnatsch'}, {'authorId': '145300176', 'name': 'M. Ioannou'}, {'authorId': '2153283114', 'name': 'Qian Wang'}, {'authorId': '49579944', 'name': 'V. Papayannopoulos'}]}, {'paperId': 'c68ca1258410b583cfdbe5a3733be5dbce0e03e3', 'title': 'Sea-level rise due to polar ice-sheet mass loss during past warm periods', 'abstract': 'Warming climate, melting ice, rising seas We know that the sea level will rise as climate warms. Nevertheless, accurate projections of how much sea-level rise will occur are difficult to make based solely on modern observations. Determining how ice sheets and sea level have varied in past warm periods can help us better understand how sensitive ice sheets are to higher temperatures. Dutton et al. review recent interdisciplinary progress in understanding this issue, based on data from four different warm intervals over the past 3 million years. Their synthesis provides a clear picture of the progress we have made and the hurdles that still exist. Science, this issue 10.1126/science.aaa4019 Reconstructing past magnitudes, rates, and sources of sea-level rise can help project what our warmer future may hold. BACKGROUND Although thermal expansion of seawater and melting of mountain glaciers have dominated global mean sea level (GMSL) rise over the last century, mass loss from the Greenland and Antarctic ice sheets is expected to exceed other contributions to GMSL rise under future warming. To better constrain polar ice-sheet response to warmer temperatures, we draw on evidence from interglacial periods in the geologic record that experienced warmer polar temperatures and higher GMSLs than present. Coastal records of sea level from these previous warm periods demonstrate geographic variability because of the influence of several geophysical processes that operate across a range of magnitudes and time scales. Inferring GMSL and ice-volume changes from these reconstructions is nontrivial and generally requires the use of geophysical models. ADVANCES Interdisciplinary studies of geologic archives have ushered in a new era of deciphering magnitudes, rates, and sources of sea-level rise. Advances in our understanding of polar ice-sheet response to warmer climates have been made through an increase in the number and geographic distribution of sea-level reconstructions, better ice-sheet constraints, and the recognition that several geophysical processes cause spatially complex patterns in sea level. In particular, accounting for glacial isostatic processes helps to decipher spatial variability in coastal sea-level records and has reconciled a number of site-specific sea-level reconstructions for warm periods that have occurred within the past several hundred thousand years. This enables us to infer that during recent interglacial periods, small increases in global mean temperature and just a few degrees of polar warming relative to the preindustrial period resulted in ≥6 m of GMSL rise. Mantle-driven dynamic topography introduces large uncertainties on longer time scales, affecting reconstructions for time periods such as the Pliocene (~3 million years ago), when atmospheric CO2 was ~400 parts per million (ppm), similar to that of the present. Both modeling and field evidence suggest that polar ice sheets were smaller during this time period, but because dynamic topography can cause tens of meters of vertical displacement at Earth’s surface on million-year time scales and uncertainty in model predictions of this signal are large, it is currently not possible to make a precise estimate of peak GMSL during the Pliocene. OUTLOOK Our present climate is warming to a level associated with significant polar ice-sheet loss in the past, but a number of challenges remain to further constrain ice-sheet sensitivity to climate change using paleo–sea level records. Improving our understanding of rates of GMSL rise due to polar ice-mass loss is perhaps the most societally relevant information the paleorecord can provide, yet robust estimates of rates of GMSL rise associated with polar ice-sheet retreat and/or collapse remain a weakness in existing sea-level reconstructions. Improving existing magnitudes, rates, and sources of GMSL rise will require a better (global) distribution of sea-level reconstructions with high temporal resolution and precise elevations and should include sites close to present and former ice sheets. Translating such sea-level data into a robust GMSL signal demands integration with geophysical models, which in turn can be tested through improved spatial and temporal sampling of coastal records. Further development is needed to refine estimates of past sea level from geochemical proxies. In particular, paired oxygen isotope and Mg/Ca data are currently unable to provide confident, quantitative estimates of peak sea level during these past warm periods. In some GMSL reconstructions, polar ice-sheet retreat is inferred from the total GMSL budget, but identifying the specific ice-sheet sources is currently hindered by limited field evidence at high latitudes. Given the paucity of such data, emerging geochemical and geophysical techniques show promise for identifying the sectors of the ice sheets that were most vulnerable to collapse in the past and perhaps will be again in the future. Peak global mean temperature, atmospheric CO2, maximum global mean sea level (GMSL), and source(s) of meltwater. Light blue shading indicates uncertainty of GMSL maximum. Red pie charts over Greenland and Antarctica denote fraction (not location) of ice retreat. Interdisciplinary studies of geologic archives have ushered in a new era of deciphering magnitudes, rates, and sources of sea-level rise from polar ice-sheet loss during past warm periods. Accounting for glacial isostatic processes helps to reconcile spatial variability in peak sea level during marine isotope stages 5e and 11, when the global mean reached 6 to 9 meters and 6 to 13 meters higher than present, respectively. Dynamic topography introduces large uncertainties on longer time scales, precluding robust sea-level estimates for intervals such as the Pliocene. Present climate is warming to a level associated with significant polar ice-sheet loss in the past. Here, we outline advances and challenges involved in constraining ice-sheet sensitivity to climate change with use of paleo–sea level records.', 'year': 2015, 'citationCount': 556, 'authors': [{'authorId': '47081678', 'name': 'A. Dutton'}, {'authorId': '104123258', 'name': 'A. Carlson'}, {'authorId': '40981288', 'name': 'A. Long'}, {'authorId': '29862195', 'name': 'G. Milne'}, {'authorId': '4367088', 'name': 'P. Clark'}, {'authorId': '104830825', 'name': 'R. DeConto'}, {'authorId': '3522204', 'name': 'B. Horton'}, {'authorId': '3522204', 'name': 'B. Horton'}, {'authorId': '1720524', 'name': 'S. Rahmstorf'}, {'authorId': '2226849', 'name': 'M. Raymo'}]}, {'paperId': '48b5bad16c3a98ccd22bea9965d821454be0fb65', 'title': 'Tissue residency of innate lymphoid cells in lymphoid and nonlymphoid organs', 'abstract': 'Establishing a longtime residency Innate lymphoid cells (ILCs) are a subset of immune cells that promote barrier immunity in tissues such as the gut and lungs and help to maintain immune homeostasis. Gasteiger et al. investigated how the body maintains its pools of ILCs in such peripheral tissues, as well as in immune tissues such as the lymph nodes and the spleen. In mice surgically joined to share their bloodstreams, unlike lymphocytes, most ILCs did not circulate through the blood. Instead, ILCs resided long term in tissues, even in the face of inflammation or infection. Science, this issue p. 981 Most innate lymphoid cells are locally maintained in tissues and do not circulate through the bloodstream. Innate lymphoid cells (ILCs) contribute to barrier immunity, tissue homeostasis, and immune regulation at various anatomical sites throughout the body. How ILCs maintain their presence in lymphoid and peripheral tissues thus far has been unclear. We found that in the lymphoid and nonlymphoid organs of adult mice, ILCs are tissue-resident cells that were maintained and expanded locally under physiologic conditions, upon systemic perturbation of immune homeostasis and during acute helminth infection. However, at later time points after infection, cells from hematogenous sources helped to partially replenish the pool of resident ILCs. Thus, ILCs are maintained by self-renewal in broadly different microenvironments and physiological settings. Such an extreme “sedentary” lifestyle is consistent with the proposed roles of ILCs as sentinels and local keepers of tissue function.', 'year': 2015, 'citationCount': 606, 'authors': [{'authorId': '5916140', 'name': 'G. Gasteiger'}, {'authorId': '46515943', 'name': 'Xiying Fan'}, {'authorId': '5348681', 'name': 'Stanislav Dikiy'}, {'authorId': '2107987539', 'name': 'Sue Y. Lee'}, {'authorId': '5146760', 'name': 'A. Rudensky'}]}, {'paperId': 'ee4bb392f3264abb684a43169240e20f05683e5a', 'title': 'RNA-Seq of single prostate CTCs implicates noncanonical Wnt signaling in antiandrogen resistance', 'abstract': 'Circulating signals of drug resistance Cancer drugs often lose their effectiveness because tumors acquire genetic changes that confer drug resistance. Ideally, patients would be switched to a different drug before tumor growth resumes, but this requires early knowledge of how resistance arose. Miyamoto et al. have developed a non-invasive method to spot resistance by sequencing RNA transcripts in single circulating tumor cells (CTCs) (see the Perspective by Nanus and Giannakakou). For example, in prostate cancer patients, drug resistance was triggered by activation of the Wnt signaling pathway. But CTCs are rare and fragile, and the technology needs further development before it is used in clinical practice. Science, this issue p. 1351; see also p. 1283 Analysis of circulating tumor cells from prostate cancer patients reveals a mechanism that contributes to treatment failure. [Also see Perspective by Nanus and Giannakakou] Prostate cancer is initially responsive to androgen deprivation, but the effectiveness of androgen receptor (AR) inhibitors in recurrent disease is variable. Biopsy of bone metastases is challenging; hence, sampling circulating tumor cells (CTCs) may reveal drug-resistance mechanisms. We established single-cell RNA-sequencing (RNA-Seq) profiles of 77 intact CTCs isolated from 13 patients (mean six CTCs per patient), by using microfluidic enrichment. Single CTCs from each individual display considerable heterogeneity, including expression of AR gene mutations and splicing variants. Retrospective analysis of CTCs from patients progressing under treatment with an AR inhibitor, compared with untreated cases, indicates activation of noncanonical Wnt signaling (P = 0.0064). Ectopic expression of Wnt5a in prostate cancer cells attenuates the antiproliferative effect of AR inhibition, whereas its suppression in drug-resistant cells restores partial sensitivity, a correlation also evident in an established mouse model. Thus, single-cell analysis of prostate CTCs reveals heterogeneity in signaling pathways that could contribute to treatment failure.', 'year': 2015, 'citationCount': 601, 'authors': [{'authorId': '3968473', 'name': 'D. Miyamoto'}, {'authorId': '145143275', 'name': 'Yu Zheng'}, {'authorId': '1726209', 'name': 'B. Wittner'}, {'authorId': '48723610', 'name': 'Richard J Lee'}, {'authorId': '50319431', 'name': 'Huili Zhu'}, {'authorId': '40267957', 'name': 'Katherine T. Broderick'}, {'authorId': '49277771', 'name': 'R. Desai'}, {'authorId': '50541900', 'name': 'Douglas B. Fox'}, {'authorId': '4577591', 'name': 'Brian W. Brannigan'}, {'authorId': '4257990', 'name': 'Julie Trautwein'}, {'authorId': '1985674', 'name': 'Kshitij S Arora'}, {'authorId': '6449022', 'name': 'Niyati Desai'}, {'authorId': '50331509', 'name': 'D. Dahl'}, {'authorId': '144346741', 'name': 'L. Sequist'}, {'authorId': '2209887632', 'name': 'Matthew R. Smith'}, {'authorId': '40250521', 'name': 'R. Kapur'}, {'authorId': '48251343', 'name': 'Chin-Lee Wu'}, {'authorId': '2281109', 'name': 'T. Shioda'}, {'authorId': '34631654', 'name': 'S. Ramaswamy'}, {'authorId': '2049483', 'name': 'D. Ting'}, {'authorId': '145527638', 'name': 'M. Toner'}, {'authorId': '144782680', 'name': 'S. Maheswaran'}, {'authorId': '2253481', 'name': 'D. Haber'}]}, {'paperId': '1ab87ca03b4fabcd691516a7a95d6ed325c809dd', 'title': 'Electrical switching of an antiferromagnet', 'abstract': \"Manipulating a stubborn magnet Spintronics is an alternative to conventional electronics, based on using the electron's spin rather than its charge. Spintronic devices, such as magnetic memory, have traditionally used ferromagnetic materials to encode the 1's and 0's of the binary code. A weakness of this approach—that strong magnetic fields can erase the encoded information—could be avoided by using antiferromagnets instead of ferromagnets. But manipulating the magnetic ordering of antiferromagnets is tricky. Now, Wadley et al. have found a way (see the Perspective by Marrows). Running currents along specific directions in the thin films of the antiferromagnetic compound CuMnAs reoriented the magnetic domains in the material. Science, this issue p. 587; see also p. 558 Transport and optical measurements are used to demonstrate the switching of domains in the antiferromagnetic compound CuMnAs. [Also see Perspective by Marrows] Antiferromagnets are hard to control by external magnetic fields because of the alternating directions of magnetic moments on individual atoms and the resulting zero net magnetization. However, relativistic quantum mechanics allows for generating current-induced internal fields whose sign alternates with the periodicity of the antiferromagnetic lattice. Using these fields, which couple strongly to the antiferromagnetic order, we demonstrate room-temperature electrical switching between stable configurations in antiferromagnetic CuMnAs thin-film devices by applied current with magnitudes of order 106 ampere per square centimeter. Electrical writing is combined in our solid-state memory with electrical readout and the stored magnetic state is insensitive to and produces no external magnetic field perturbations, which illustrates the unique merits of antiferromagnets for spintronics.\", 'year': 2015, 'citationCount': 971, 'authors': [{'authorId': '6529129', 'name': 'P. Wadley'}, {'authorId': '92662262', 'name': 'B. Howells'}, {'authorId': '7183375', 'name': 'J. Železný'}, {'authorId': '7183375', 'name': 'J. Železný'}, {'authorId': '122532770', 'name': 'C. Andrews'}, {'authorId': '116923455', 'name': 'V. Hills'}, {'authorId': '34810725', 'name': 'R. Campion'}, {'authorId': '153319812', 'name': 'V. Novák'}, {'authorId': '4373201', 'name': 'K. Olejník'}, {'authorId': '5090120', 'name': 'F. Maccherozzi'}, {'authorId': '4838511', 'name': 'S. Dhesi'}, {'authorId': '2107491459', 'name': 'S. Martin'}, {'authorId': '47426511', 'name': 'T. Wagner'}, {'authorId': '47426511', 'name': 'T. Wagner'}, {'authorId': '2678034', 'name': 'J. Wunderlich'}, {'authorId': '2678034', 'name': 'J. Wunderlich'}, {'authorId': '6559483', 'name': 'F. Freimuth'}, {'authorId': '6402565', 'name': 'Y. Mokrousov'}, {'authorId': '50241568', 'name': 'J. Kuneš'}, {'authorId': '1630347158', 'name': 'J. Chauhan'}, {'authorId': '50419899', 'name': 'M. Grzybowski'}, {'authorId': '50419899', 'name': 'M. Grzybowski'}, {'authorId': '144765908', 'name': 'A. Rushforth'}, {'authorId': '48918214', 'name': 'K. Edmonds'}, {'authorId': '46346797', 'name': 'B. Gallagher'}, {'authorId': '6620058', 'name': 'T. Jungwirth'}, {'authorId': '6620058', 'name': 'T. Jungwirth'}]}, {'paperId': '69de421683a59ea687cfbb8280b264793e41316d', 'title': 'Graphene, related two-dimensional crystals, and hybrid systems for energy conversion and storage', 'abstract': 'Background The integration of graphene in photovoltaic modules, fuel cells, batteries, supercapacitors, and devices for hydrogen generation offers opportunities to tackle challenges driven by the increasing global energy demand. Graphene’s two-dimensional (2D) nature leads to a theoretical surface-to-mass ratio of ~2600 m2/g, which combined with its high electrical conductivity and flexibility, gives it the potential to store electric charge, ions, or hydrogen. Other 2D crystals, such as transition metal chalcogenides (TMDs) and transition metal oxides, are also promising and are now gaining increasing attention for energy applications. The advantage of using such 2D crystals is linked to the possibility of creating and designing layered artificial structures with “on-demand” properties by means of spin-on processes, or layer-by-layer assembly. This approach exploits the availability of materials with metallic, semiconducting, and insulating properties. Advances The success of graphene and related materials (GRMs) for energy applications crucially depends on the development and optimization of production methods. High-volume liquid-phase exfoliation is being developed for a wide variety of layered materials. This technique is being optimized to control the flake size and to increase the edge-to-surface ratio, which is crucial for optimizing electrode performance in fuel cells and batteries. Micro- or nanocrystal or flake edge control can also be achieved through chemical synthesis. This is an ideal route for functionalization, in order to improve storage capacity. Large-area growth via chemical vapor deposition (CVD) has been demonstrated, producing material with high structural and electronic quality for the preparation of transparent conducting electrodes for displays and touch-screens, and is being evaluated for photovoltaic applications. CVD growth of other multicomponent layered materials is less mature and needs further development. Although many transfer techniques have been developed successfully, further improvement of high-volume manufacturing and transfer processes for multilayered heterostructures is needed. In this context, layer-by-layer assembly may enable the realization of devices with on-demand properties for targeted applications, such as photovoltaic devices in which photon absorption in TMDs is combined with charge transport in graphene. Outlook Substantial progress has been made on the preparation of GRMs at the laboratory level. However, cost-effective production of GRMs on an industrial scale is needed to create the future energy value chain. Applications that could benefit the most from GRMs include flexible electronics, batteries with efficient anodes and cathodes, supercapacitors with high energy density, and solar cells. The realization of GRMs with specific transport and insulating properties on demand is an important goal. Additional energy applications of GRMs comprise water splitting and hydrogen production. As an example, the edges of MoS2 single layers can oxidize fuels—such as hydrogen, methanol, and ethanol—in fuel cells, and GRM membranes can be used in fuel cells to improve proton exchange. Functionalized graphene can be exploited for water splitting and hydrogen production. Flexible and wearable devices and membranes incorporating GRMs can also generate electricity from motion, as well as from water and gas flows. Tailored GRMs for energy applications. The ability to produce GRMs with desired specific properties paves the way to their integration in a variety of energy devices. Solution processing and chemical vapor deposition are the ideal means to produce thin films that can be used as electrodes in energy devices (such as solar panels, batteries, fuel cells, or in hydrogen storage). Chemical synthesis is an attractive route to produce “active” elements in solar cell or thermoelectric devices. Graphene and related two-dimensional crystals and hybrid systems showcase several key properties that can address emerging energy needs, in particular for the ever growing market of portable and wearable energy conversion and storage devices. Graphene’s flexibility, large surface area, and chemical stability, combined with its excellent electrical and thermal conductivity, make it promising as a catalyst in fuel and dye-sensitized solar cells. Chemically functionalized graphene can also improve storage and diffusion of ionic species and electric charge in batteries and supercapacitors. Two-dimensional crystals provide optoelectronic and photocatalytic properties complementing those of graphene, enabling the realization of ultrathin-film photovoltaic devices or systems for hydrogen production. Here, we review the use of graphene and related materials for energy conversion and storage, outlining the roadmap for future applications. Layered materials power the cause Methods for storing and converting energy, including fuel cells, solar cells, and water splitting, often benefit from having materials with a large surface area. When combined with a high surface reactivity, high conductivity, or useful optical properties, two-dimensional layered materials become of notable interest for a range of applications. Bonaccorso et al. review the progress that has been made using graphene and other layered or two-dimensional materials at laboratory scales and the challenges in producing these materials in industrially relevant quantities. Science, this issue 10.1126/science.1246501', 'year': 2015, 'citationCount': 2686, 'authors': [{'authorId': '3040167', 'name': 'F. Bonaccorso'}, {'authorId': '50997465', 'name': 'L. Colombo'}, {'authorId': '5964001', 'name': 'Guihua Yu'}, {'authorId': '3992446', 'name': 'M. Stoller'}, {'authorId': '2359160', 'name': 'V. Tozzini'}, {'authorId': '153332598', 'name': 'A. Ferrari'}, {'authorId': '3197462', 'name': 'R. Ruoff'}, {'authorId': '2003758', 'name': 'V. Pellegrini'}]}, {'paperId': '894805e7984e34d93fbc5753c5e3c28facdb189d', 'title': 'K13-propeller mutations confer artemisinin resistance in Plasmodium falciparum clinical isolates', 'abstract': \"Mechanisms propelling drug resistance If it were to spread, resistance to the drug artemisinin would seriously derail the recent gains of global malaria control programs (see the Perspective by Sibley). Mutations in a region called the K13-propeller are predictive for artemisinin resistance in Southeast Asia. Mok et al. looked at the patterns of gene expression in parasites isolated from more than 1000 patients sampled in Africa, Bangladesh, and the Mekong region. A range of mutations that alter protein repair pathways and the timing of the parasite's developmental cycle were only found in parasites from the Mekong region. Straimer et al. genetically engineered the K13 region of parasites obtained from recent clinical isolates. Mutations in this region were indeed responsible for the resistance phenotypes. Science, this issue p. 431, p. 428; see also p. 373 Resistance to the primary antimalarial drug lies in mutations in protein repair and developmental pathways. [Also see Perspective by Sibley] The emergence of artemisinin resistance in Southeast Asia imperils efforts to reduce the global malaria burden. We genetically modified the Plasmodium falciparum K13 locus using zinc-finger nucleases and measured ring-stage survival rates after drug exposure in vitro; these rates correlate with parasite clearance half-lives in artemisinin-treated patients. With isolates from Cambodia, where resistance first emerged, survival rates decreased from 13 to 49% to 0.3 to 2.4% after the removal of K13 mutations. Conversely, survival rates in wild-type parasites increased from ≤0.6% to 2 to 29% after the insertion of K13 mutations. These mutations conferred elevated resistance to recent Cambodian isolates compared with that of reference lines, suggesting a contemporary contribution of additional genetic factors. Our data provide a conclusive rationale for worldwide K13-propeller sequencing to identify and eliminate artemisinin-resistant parasites.\", 'year': 2015, 'citationCount': 541, 'authors': [{'authorId': '4235059', 'name': 'J. Straimer'}, {'authorId': '3805330', 'name': 'Nina F. Gnädig'}, {'authorId': '3606394', 'name': 'B. Witkowski'}, {'authorId': '4086075', 'name': 'C. Amaratunga'}, {'authorId': '5151228', 'name': 'V. Duru'}, {'authorId': '3614601', 'name': 'A. P. Ramadani'}, {'authorId': '1694386255', 'name': 'Mélanie A. Dacheux'}, {'authorId': '4109824', 'name': 'N. Khim'}, {'authorId': '2152827305', 'name': 'Lei Zhang'}, {'authorId': '2056427851', 'name': 'Stephen C. Lam'}, {'authorId': '6635031', 'name': 'P. Gregory'}, {'authorId': '3571589', 'name': 'F. Urnov'}, {'authorId': '1398403785', 'name': 'O. Mercereau‐Puijalon'}, {'authorId': '1398426454', 'name': 'F. Benoit‐Vical'}, {'authorId': '87397838', 'name': 'R. Fairhurst'}, {'authorId': '144182939', 'name': 'D. Ménard'}, {'authorId': '2735233', 'name': 'D. Fidock'}]}, {'paperId': 'd73aca5a4e3a9b3dcde7a21e995efc67a820cd30', 'title': 'A dendritic cell vaccine increases the breadth and diversity of melanoma neoantigen-specific T cells', 'abstract': \"Giving antitumor T cells a boost Mutations allow tumors to divide, escape death, and resist treatment. But mutations can also cause tumors to express mutant proteins, which could potentially be exploited to drive antitumor T cell responses. Carreno et al. report the results of a small phase I trial seeking to do just this (see the Perspective by Delamarre et al.). They vaccinated three patients with advanced melanoma with personalized dendritic cell–based vaccines designed to activate T cells specific for mutations in the patients' cancer. T cells specific for mutant peptides did indeed expand. A next step will be to determine whether this promising strategy improves patient outcomes. Science, this issue p. 803; see also p. 760 Passenger mutations encode target melanoma antigens for human cancer immunotherapy. [Also see Perspective by Delamarre et al.] T cell immunity directed against tumor-encoded amino acid substitutions occurs in some melanoma patients. This implicates missense mutations as a source of patient-specific neoantigens. However, a systematic evaluation of these putative neoantigens as targets of antitumor immunity is lacking. Moreover, it remains unknown whether vaccination can augment such responses. We found that a dendritic cell vaccine led to an increase in naturally occurring neoantigen-specific immunity and revealed previously undetected human leukocyte antigen (HLA) class I–restricted neoantigens in patients with advanced melanoma. The presentation of neoantigens by HLA-A*02:01 in human melanoma was confirmed by mass spectrometry. Vaccination promoted a diverse neoantigen-specific T cell receptor (TCR) repertoire in terms of both TCR-β usage and clonal composition. Our results demonstrate that vaccination directed at tumor-encoded amino acid substitutions broadens the antigenic breadth and clonal diversity of antitumor immunity.\", 'year': 2015, 'citationCount': 904, 'authors': [{'authorId': '5194861', 'name': 'B. Carreno'}, {'authorId': '46540039', 'name': 'V. Magrini'}, {'authorId': '1398701413', 'name': 'M. Becker-Hapak'}, {'authorId': '5303952', 'name': 'S. Kaabinejadian'}, {'authorId': '3081783', 'name': 'J. Hundal'}, {'authorId': '3062663', 'name': 'A. Petti'}, {'authorId': '46762618', 'name': 'Amy Ly'}, {'authorId': '120626088', 'name': 'W. Lie'}, {'authorId': '3220399', 'name': 'W. Hildebrand'}, {'authorId': '3286001', 'name': 'E. Mardis'}, {'authorId': '4622371', 'name': 'G. Linette'}]}, {'paperId': 'adfc3c88d887818149b8aa672e96cd11313bbcd6', 'title': 'Patterns and ecological drivers of ocean viral communities', 'abstract': 'Viruses influence ecosystems by modulating microbial population size, diversity, metabolic outputs, and gene flow. Here, we use quantitative double-stranded DNA (dsDNA) viral-fraction metagenomes (viromes) and whole viral community morphological data sets from 43 Tara Oceans expedition samples to assess viral community patterns and structure in the upper ocean. Protein cluster cataloging defined pelagic upper-ocean viral community pan and core gene sets and suggested that this sequence space is well-sampled. Analyses of viral protein clusters, populations, and morphology revealed biogeographic patterns whereby viral communities were passively transported on oceanic currents and locally structured by environmental conditions that affect host community structure. Together, these investigations establish a global ocean dsDNA viromic data set with analyses supporting the seed-bank hypothesis to explain how oceanic viral communities maintain high local diversity.', 'year': 2015, 'citationCount': 554, 'authors': [{'authorId': '31932986', 'name': 'Jennifer R. Brum'}, {'authorId': '1398943556', 'name': 'J. C. Ignacio-Espinoza'}, {'authorId': '52161096', 'name': 'S. Roux'}, {'authorId': '4330984', 'name': 'Guilhem Doulcier'}, {'authorId': '6119340', 'name': 'S. Acinas'}, {'authorId': '6794377', 'name': 'A. Alberti'}, {'authorId': '2045252', 'name': 'Samuel Chaffron'}, {'authorId': '3529067', 'name': 'C. Cruaud'}, {'authorId': '2126901', 'name': 'C. de Vargas'}, {'authorId': '5262671', 'name': 'J. Gasol'}, {'authorId': '34168606', 'name': 'G. Gorsky'}, {'authorId': '49882697', 'name': 'A. Gregory'}, {'authorId': '10650860', 'name': 'L. Guidi'}, {'authorId': '2536107', 'name': 'P. Hingamp'}, {'authorId': '4206582', 'name': 'D. Iudicone'}, {'authorId': '2158864', 'name': 'F. Not'}, {'authorId': '2272089', 'name': 'H. Ogata'}, {'authorId': '5907934', 'name': 'S. Pesant'}, {'authorId': '4763837', 'name': 'B. Poulos'}, {'authorId': '6649875', 'name': 'S. Schwenck'}, {'authorId': '2154189', 'name': 'S. Speich'}, {'authorId': '6434378', 'name': 'Céline Dimier'}, {'authorId': '1396183973', 'name': 'S. Kandels-Lewis'}, {'authorId': '4157832', 'name': 'M. Picheral'}, {'authorId': '4309482', 'name': 'S. Searson'}, {'authorId': '3534315', 'name': 'P. Bork'}, {'authorId': '48244603', 'name': 'C. Bowler'}, {'authorId': '2067568', 'name': 'S. Sunagawa'}, {'authorId': '2839934', 'name': 'P. Wincker'}, {'authorId': '4192764', 'name': 'E. Karsenti'}, {'authorId': '5518928', 'name': 'M. Sullivan'}]}, {'paperId': '5f53968d110eb8413882b9c730fa7ac846b0e5e7', 'title': 'Lysosomal amino acid transporter SLC38A9 signals arginine sufficiency to mTORC1', 'abstract': 'Sensing amino acids at the lysosome The mTORC1 protein kinase is a complex of proteins that functions to regulate growth and metabolism. Activity of mTORC1 is sensitive to the abundance of amino acids, but how the sensing of amino acids is coupled to the control of mTORC1 has been unclear. Wang et al. searched for predicted membrane proteins that interacted with regulators of mTORC1. They identified a protein currently known only as SLC38A9. Interaction of SLC38A9 with mTORC1 regulators was sensitive to the presence of amino acids. SLC38A9 has sequence similarity to amino acid transporters. Effects of modulation of SLC38A9 in cultured human cells indicate that it may be the sensor that connects the abundance of arginine and leucine to mTORC1 activity. Science, this issue p. 188 A possible sensor for detecting and signaling amino acid abundance is identified. [Also see Perspective by Abraham] The mechanistic target of rapamycin complex 1 (mTORC1) protein kinase is a master growth regulator that responds to multiple environmental cues. Amino acids stimulate, in a Rag-, Ragulator-, and vacuolar adenosine triphosphatase–dependent fashion, the translocation of mTORC1 to the lysosomal surface, where it interacts with its activator Rheb. Here, we identify SLC38A9, an uncharacterized protein with sequence similarity to amino acid transporters, as a lysosomal transmembrane protein that interacts with the Rag guanosine triphosphatases (GTPases) and Ragulator in an amino acid–sensitive fashion. SLC38A9 transports arginine with a high Michaelis constant, and loss of SLC38A9 represses mTORC1 activation by amino acids, particularly arginine. Overexpression of SLC38A9 or just its Ragulator-binding domain makes mTORC1 signaling insensitive to amino acid starvation but not to Rag activity. Thus, SLC38A9 functions upstream of the Rag GTPases and is an excellent candidate for being an arginine sensor for the mTORC1 pathway.', 'year': 2015, 'citationCount': 529, 'authors': [{'authorId': '2155437961', 'name': 'Shuyu Wang'}, {'authorId': '4099866', 'name': 'Z. Tsun'}, {'authorId': '6862301', 'name': 'Rachel L. Wolfson'}, {'authorId': '2705753', 'name': 'K. Shen'}, {'authorId': '4587526', 'name': 'Gregory A. Wyant'}, {'authorId': '4606345', 'name': 'Molly E Plovanich'}, {'authorId': '143966902', 'name': 'Elizabeth D. Yuan'}, {'authorId': '10064557', 'name': 'Tony D. Jones'}, {'authorId': '3054115', 'name': 'Lynne Chantranupong'}, {'authorId': '4037235', 'name': 'W. Comb'}, {'authorId': '49981123', 'name': 'Tim Wang'}, {'authorId': '1397944980', 'name': 'L. Bar-Peled'}, {'authorId': '3034790', 'name': 'R. Zoncu'}, {'authorId': '48449774', 'name': 'Christoph Straub'}, {'authorId': '1787267307', 'name': 'Choah Kim'}, {'authorId': '2115930814', 'name': 'Jiwon Park'}, {'authorId': '2845457', 'name': 'B. Sabatini'}, {'authorId': '1843680', 'name': 'D. Sabatini'}]}, {'paperId': '009e98d679ea67ae8b7a71f0ccd62b9b0eaae3ef', 'title': 'Universal linear optics', 'abstract': \"Complex quantum optical circuitry Encoding and manipulating information in the states of single photons provides a potential platform for quantum computing and communication. Carolan et al. developed a reconfigurable integrated waveguide device fabricated in a glass chip (see the Perspective by Rohde and Dowling). The device allowed for universal linear optics transformations on six wave-guides using 15 integrated Mach-Zehnder interferometers, each of which was individually programmable. Functional performance in a number of applications in optics and quantum optics demonstrates the versatility of the device's reprogrammable architecture. Science, this issue p. 711; see also p. 696 A reconfigurable optical circuit provides a platform for a photonically-based quantum computer. [Also see Perspective by Rohde and Dowling] Linear optics underpins fundamental tests of quantum mechanics and quantum technologies. We demonstrate a single reprogrammable optical circuit that is sufficient to implement all possible linear optical protocols up to the size of that circuit. Our six-mode universal system consists of a cascade of 15 Mach-Zehnder interferometers with 30 thermo-optic phase shifters integrated into a single photonic chip that is electrically and optically interfaced for arbitrary setting of all phase shifters, input of up to six photons, and their measurement with a 12-single-photon detector system. We programmed this system to implement heralded quantum logic and entangling gates, boson sampling with verification tests, and six-dimensional complex Hadamards. We implemented 100 Haar random unitaries with an average fidelity of 0.999 ± 0.001. Our system can be rapidly reprogrammed to implement these and any other linear optical protocol, pointing the way to applications across fundamental science and quantum technologies.\", 'year': 2015, 'citationCount': 819, 'authors': [{'authorId': '48228088', 'name': 'J. Carolan'}, {'authorId': '144731834', 'name': 'C. Harrold'}, {'authorId': '40062159', 'name': 'Chris Sparrow'}, {'authorId': '1404394297', 'name': 'E. Martin-Lopez'}, {'authorId': '48070560', 'name': 'N. Russell'}, {'authorId': '14244862', 'name': 'J. Silverstone'}, {'authorId': '2990311', 'name': 'P. Shadbolt'}, {'authorId': '40184662', 'name': 'N. Matsuda'}, {'authorId': '3134178', 'name': 'M. Oguma'}, {'authorId': '2011983', 'name': 'M. Itoh'}, {'authorId': '48258157', 'name': 'G. Marshall'}, {'authorId': '1690351', 'name': 'M. Thompson'}, {'authorId': '33859373', 'name': 'J. Matthews'}, {'authorId': '32256564', 'name': 'T. Hashimoto'}, {'authorId': '8187633', 'name': \"J. O'Brien\"}, {'authorId': '39220451', 'name': 'A. Laing'}]}, {'paperId': 'e77709623a4c7824be6200933353883097ccd170', 'title': 'Function-led design of new porous materials', 'abstract': \"It's all about the holes From kitchen sieves and strainers to coffee filters, porous materials have a wide range of uses. On an industrial scale, they are used as sorbents, filters, membranes, and catalysts. Slater and Cooper review how each application will limit the materials that can be used, and also the size and connectivity of the pores required. They go on to compare and contrast a growing range of porous materials that are finding increasing use in academic and industrial applications. Science, this issue 10.1126/science.aaa8075 BACKGROUND Porous materials are important in established processes such as catalysis and molecular separations and in emerging technologies for energy and health. Porous zeolites have made the largest contribution to society so far, and that field is still developing rapidly. Other porous solids have also entered the scene in the past two decades, such as metal-organic frameworks (MOFs), covalent organic frameworks (COFs), and porous organic polymers. No single class of porous material is ideal for all purposes. For example, crystallinity and long-range order might enhance selectivity for a molecular separation while also reducing mechanical stability or processability with respect to less ordered structures. To have an impact on real applications, porous materials must be scalable and must satisfy multiple functional criteria such as long-term stability, selectivity, adsorption kinetics, and processability, all within a viable cost envelope. This presents a broad design challenge, and it requires us to be able to control structure and to understand multiple structure-property relationships at a detailed level. ADVANCES In addition to MOFs, COFs, and porous polymer networks, other classes of molecular porous solids have emerged in the past 10 years, such as polymers of intrinsic microporosity and porous organic cages. The range of possible functions for porous solids is thus much broader than before. For example, conjugated microporous polymers and some COFs have extended, conjugated structures that are not present in zeolites or MOFs and have led to porous organic photocatalysts and electronic materials. The crystal engineering approaches developed for zeolites, MOFs, and COFs cannot be applied directly to amorphous solids such as porous polymers, but analogous modular strategies have allowed functions such as porosity and electronic band gap to be controlled by choosing the appropriate molecular building blocks. Rapid advances in the computational prediction of structure and function offer a strategy for identifying the best porous materials for specific applications, for example, via large-scale screening of gas adsorption in hypothetical MOFs. OUTLOOK Advances in synthesis have produced new classes of functional porous solids as well as fundamental breakthroughs in areas such as selective carbon dioxide capture, molecular separations, and catalysis. As yet, these rapid developments in basic understanding are unmatched by large-scale commercial implementation, but enhanced functions (such as enzyme-like CO2 selectivity) and new processing options (such as soluble porous solids) present exciting opportunities. A general challenge will be to reengineer porous materials where scale-up is prohibited by cost, retaining the advanced function but using cheaper and more sustainable building blocks. It is therefore important to develop structure-property relationships to understand how promising materials work. Not all future opportunities for porous solids involve improving on existing materials or the development of more scalable preparation routes. For example, porous photocatalysts that can perform direct solar water splitting might provide a completely new platform for energy production. As we seek increasingly complex functions for porous materials, the use of in silico computational design to guide experiment will become more important. Porous materials can be defined by type or by function, but it is function that will determine the scope for practical applications. Our ability to design functions in porous solids has advanced markedly in the past two decades as a result of developments in modular synthesis, materials characterization, and (more recently) computational structure-property predictions. This figure is based on the pore channels, shown in yellow, for an organic cage molecule, a new type of solution-processable porous solid developed over the past 6 years. Porous solids are important as membranes, adsorbents, catalysts, and in other chemical applications. But for these materials to find greater use at an industrial scale, it is necessary to optimize multiple functions in addition to pore structure and surface area, such as stability, sorption kinetics, processability, mechanical properties, and thermal properties. Several different classes of porous solids exist, and there is no one-size-fits-all solution; it can therefore be challenging to choose the right type of porous material for a given job. Computational prediction of structure and properties has growing potential to complement experiment to identify the best porous materials for specific applications.\", 'year': 2015, 'citationCount': 1058, 'authors': [{'authorId': '47387234', 'name': 'A. Slater'}, {'authorId': '1754626', 'name': 'A. Cooper'}]}, {'paperId': 'f0f09a65a5924b61ab14c5d1197d444a983a656e', 'title': 'Near-unity photoluminescence quantum yield in MoS2', 'abstract': 'Brighter molybdenum layers The confined layers of molybdenum disulphide (MoS2) exhibit photoluminescence that is attractive for optolectronic applications. In practice, efficiencies are low, presumably because defects trap excitons before they can recombine and radiate light. Amani et al. show that treatment of monolayer MoS2 with a nonoxidizing organic superacid, bis(trifluoromethane) sulfonimide, increased luminescence efficiency in excess of 95%. The enhancement mechanism may be related to the shielding of defects, such as sulfur vacancies. Science, this issue p. 1065 Superacid treatment enhances the luminescence efficiency of monolayer molybdenum disulfide from 1% to >95%. Two-dimensional (2D) transition metal dichalcogenides have emerged as a promising material system for optoelectronic applications, but their primary figure of merit, the room-temperature photoluminescence quantum yield (QY), is extremely low. The prototypical 2D material molybdenum disulfide (MoS2) is reported to have a maximum QY of 0.6%, which indicates a considerable defect density. Here we report on an air-stable, solution-based chemical treatment by an organic superacid, which uniformly enhances the photoluminescence and minority carrier lifetime of MoS2 monolayers by more than two orders of magnitude. The treatment eliminates defect-mediated nonradiative recombination, thus resulting in a final QY of more than 95%, with a longest-observed lifetime of 10.8 ± 0.6 nanoseconds. Our ability to obtain optoelectronic monolayers with near-perfect properties opens the door for the development of highly efficient light-emitting diodes, lasers, and solar cells based on 2D materials.', 'year': 2015, 'citationCount': 704, 'authors': [{'authorId': '4795241', 'name': 'M. Amani'}, {'authorId': '2029675', 'name': 'D. Lien'}, {'authorId': '51220328', 'name': 'D. Kiriya'}, {'authorId': '2111222656', 'name': 'Jun Xiao'}, {'authorId': '3986759', 'name': 'A. Azcatl'}, {'authorId': '2072788421', 'name': 'Jiyoung Noh'}, {'authorId': '5910856', 'name': 'S. Madhvapathy'}, {'authorId': '6009973', 'name': 'R. Addou'}, {'authorId': '2222022011', 'name': 'Santosh Kc'}, {'authorId': '34126382', 'name': 'M. Dubey'}, {'authorId': '2198231', 'name': 'Kyeongjae Cho'}, {'authorId': '2974186', 'name': 'R. Wallace'}, {'authorId': '2144375736', 'name': 'Si-Chen Lee'}, {'authorId': '49264345', 'name': 'Jr-hau He'}, {'authorId': '40544683', 'name': 'J. Ager'}, {'authorId': '2190290509', 'name': 'Xiang Zhang'}, {'authorId': '1823951', 'name': 'E. Yablonovitch'}, {'authorId': '2402712', 'name': 'A. Javey'}]}, {'paperId': 'a8674e93d1e4b9cdc471c81a1edfea76ab23ad6a', 'title': 'Extended-resolution structured illumination imaging of endocytic and cytoskeletal dynamics', 'abstract': 'Adding to the super-resolution arsenal Structured illumination microscopy (SIM) uses light intensities that are orders of magnitude lower than other super-resolution methods. SIM is also far faster over cellular-sized fields of view. Li et al. used two approaches to improve the resolution of SIM to allow live cell imaging of dynamic cellular processes, including endocytosis and cytoskeleton remodeling. The contrast in performance between SIM and other techniques is due to a few key differences. Defining the practical resolution at the limited signal-to-noise ratios necessary for live cell imaging will require better imaging metrics. Science, this issue 10.1126/science.aab3500 Super-resolution imaging of fast dynamic processes in living cells is facilitated by improvements to structured illumination microscopy. INTRODUCTION Various methods of super-resolution (SR) fluorescence microscopy have the potential to follow the dynamic nanoscale interactions of specific macromolecular assemblies in living cells. However, this potential is often left unfulfilled, either owing to the method’s inability to follow these processes at the speeds dictated by nature or because they require intense light that can substantially perturb the very physiology one hopes to study. An exception is structured illumination microscopy (SIM), which can image live cells far faster and with orders of magnitude less light than required for other SR approaches. However, SIM’s resolution is usually limited to only a twofold gain beyond conventional optical microscopes, or ~100 nm with visible light. RATIONALE We endeavored to find ways to extend SIM to the sub-100-nm regime while retaining, to the greatest extent possible, the advantages that make it the preferred SR method for live-cell imaging. Our first solution used an ultrahigh numerical aperture (NA) lens and total internal reflection fluorescence (TIRF) to achieve 84-nm resolution at subsecond acquisition speeds over hundreds of time points in multiple colors near the basal plasma membrane. Our second exploited the spatially patterned activation of a recently developed, reversibly photoswitchable fluorescent protein to reach 45- to 62-nm resolution, also at subsecond acquisition, over ∼10 to 40 time points. RESULTS We used high-NA TIRF-SIM to image the dynamic associations of cortical filamentous actin with myosin IIA, paxillin, or clathrin, as well as paxillin with vinculin and clathrin with transferrin receptors. Thanks to the combination of high spatial and temporal resolution, we were able to measure the sizes of individual clathrin-coated pits through their initiation, growth, and internalization. We were also able to relate pit size to lifetime, identify and characterize localized hot spots of pit generation, and describe the interaction of actin with clathrin and its role in accelerating endocytosis. With nonlinear SIM by use of patterned activation (PA NL-SIM), we monitored the remodeling of the actin cytoskeleton and the dynamics of caveolae at the cell surface. By combining TIRF-SIM and PA NL-SIM for two-color imaging, we followed the dynamic association of actin with α-actinin in expanding filopodia and membrane ruffles and characterized shape changes in and the transport of early endosomes. Last, by combining PA NL-SIM with lattice light sheet microscopy, we observed, in three dimensions and across the entire volume of whole cells, the dynamics of the actin cytoskeleton, the fusion and fission of mitochondria, and the trafficking of vesicles to and from the Golgi apparatus, each at axial resolution fivefold better than that of conventional widefield microscopy. In addition, through direct experimental comparisons, we demonstrated that the resolution for our methods is comparable with or better than other SR approaches yet allowed us to image at far higher speeds, and for far longer durations. To understand why this is so, we developed a detailed theoretical model showing that our methods transmit the information encoded in spatial frequencies beyond the diffraction limit with much greater strength than do other alternatives and hence require far fewer photons emitted from the specimen, using far less intense light. CONCLUSION High-NA TIRF-SIM and PA NL-SIM fill an unmet need for minimally invasive tools to image live cells in the gap between the 100-nm resolution traditionally associated with SIM and the sub-60-nm regime of protein-specific structural imaging served by single-molecule localization microscopy. Two approaches for improved live-cell imaging at sub-100-nm resolution. (Left) Association of cortical actin (purple) with clathrin-coated pits (green), the latter seen as rings (inset) at 84-nm resolution via a combination of total internal reflection fluorescence and structured illumination microscopy at ultrahigh numerical aperture (high-NA TIRF-SIM). (Right) Progression of resolution improvement across the actin cytoskeleton of a COS-7 cell, from conventional, diffraction-limited TIRF (220-nm resolution), to TIRF-SIM (97-nm resolution), and nonlinear SIM based on the patterned activation of a reversibly photoswitchable fluorescent protein (PA NL-SIM, 62 nm resolution). (Left and right represent single frames from time-lapse movies over 91 and 30 frames, respectively. Scale bars, 2 μm (left); 3 μm (right). Super-resolution fluorescence microscopy is distinct among nanoscale imaging tools in its ability to image protein dynamics in living cells. Structured illumination microscopy (SIM) stands out in this regard because of its high speed and low illumination intensities, but typically offers only a twofold resolution gain. We extended the resolution of live-cell SIM through two approaches: ultrahigh numerical aperture SIM at 84-nanometer lateral resolution for more than 100 multicolor frames, and nonlinear SIM with patterned activation at 45- to 62-nanometer resolution for approximately 20 to 40 frames. We applied these approaches to image dynamics near the plasma membrane of spatially resolved assemblies of clathrin and caveolin, Rab5a in early endosomes, and α-actinin, often in relationship to cortical actin. In addition, we examined mitochondria, actin, and the Golgi apparatus dynamics in three dimensions.', 'year': 2015, 'citationCount': 564, 'authors': [{'authorId': '2108819888', 'name': 'Dong Li'}, {'authorId': '49971595', 'name': 'L. Shao'}, {'authorId': '50461904', 'name': 'Bi-Chang Chen'}, {'authorId': '2108286623', 'name': 'Xi Zhang'}, {'authorId': '4144655', 'name': 'Mingshu Zhang'}, {'authorId': '2069911672', 'name': 'Brian Moses'}, {'authorId': '5119148', 'name': 'D. Milkie'}, {'authorId': '39875282', 'name': 'Jordan R. Beach'}, {'authorId': '144477660', 'name': 'J. Hammer'}, {'authorId': '4611850', 'name': 'Mithun Pasham'}, {'authorId': '4218577', 'name': 'T. Kirchhausen'}, {'authorId': '48179239', 'name': 'M. A. Baird'}, {'authorId': '48949259', 'name': 'M. Davidson'}, {'authorId': '145731493', 'name': 'P. Xu'}, {'authorId': '1774970', 'name': 'E. Betzig'}]}, {'paperId': '5e218166d49017d40c651669c9e130e6071e1be4', 'title': 'A prudent path forward for genomic engineering and germline gene modification', 'abstract': 'A framework for open discourse on the use of CRISPR-Cas9 technology to manipulate the human genome is urgently needed Genome engineering technology offers unparalleled potential for modifying human and nonhuman genomes. In humans, it holds the promise of curing genetic disease, while in other organisms it provides methods to reshape the biosphere for the benefit of the environment and human societies. However, with such enormous opportunities come unknown risks to human health and well-being. In January, a group of interested stakeholders met in Napa, California (1), to discuss the scientific, medical, legal, and ethical implications of these new prospects for genome biology. The goal was to initiate an informed discussion of the uses of genome engineering technology, and to identify those areas where action is essential to prepare for future developments. The meeting identified immediate steps to take toward ensuring that the application of genome engineering technology is performed safely and ethically.', 'year': 2015, 'citationCount': 562, 'authors': [{'authorId': '123871122', 'name': 'D. Baltimore'}, {'authorId': '145479891', 'name': 'P. Berg'}, {'authorId': '5754463', 'name': 'M. Botchan'}, {'authorId': '145108881', 'name': 'D. Carroll'}, {'authorId': '6809758', 'name': 'R. Charo'}, {'authorId': '145892667', 'name': 'G. Church'}, {'authorId': '5129546', 'name': 'J. Corn'}, {'authorId': '2028224', 'name': 'G. Daley'}, {'authorId': '4448238', 'name': 'J. Doudna'}, {'authorId': '41191823', 'name': 'M. Fenner'}, {'authorId': '3277528', 'name': 'H. Greely'}, {'authorId': '3693015', 'name': 'M. Jinek'}, {'authorId': '2013804594', 'name': 'G. S. Martin'}, {'authorId': '8047750', 'name': 'E. Penhoet'}, {'authorId': '4689670', 'name': 'J. Puck'}, {'authorId': '2993789', 'name': 'S. H. Sternberg'}, {'authorId': '3168827', 'name': 'J. Weissman'}, {'authorId': '115645782', 'name': 'K. Yamamoto'}]}, {'paperId': '259d28767d1a4f297788ac09b7cc9b8d4aa9d7b5', 'title': 'Macroscale superlubricity enabled by graphene nanoscroll formation', 'abstract': 'Slip sliding away Many applications would benefit from ultralow friction conditions to minimize wear on the moving parts such as in hard drives or engines. On the very small scale, ultralow friction has been observed with graphite as a lubricant. Berman et al. achieved superlubricity using graphene in combination with crystalline diamond nanoparticles and diamondlike carbon (see the Perspective by Hone and Carpick). Simulations showed that sliding of the graphene patches around the tiny nanodiamond particles led to nanoscrolls with reduced contact area that slide easily against the amorphous diamondlike carbon surface. Science, this issue p. 1118; see also p. 1087 Nanodiamonds wrapped with graphene sheets lead to ultralow friction against a diamondlike carbon surface. [Also see Perspective by Hone and Carpick] Friction and wear remain as the primary modes of mechanical energy dissipation in moving mechanical assemblies; thus, it is desirable to minimize friction in a number of applications. We demonstrate that superlubricity can be realized at engineering scale when graphene is used in combination with nanodiamond particles and diamondlike carbon (DLC). Macroscopic superlubricity originates because graphene patches at a sliding interface wrap around nanodiamonds to form nanoscrolls with reduced contact area that slide against the DLC surface, achieving an incommensurate contact and substantially reduced coefficient of friction (~0.004). Atomistic simulations elucidate the overall mechanism and mesoscopic link bridging the nanoscale mechanics and macroscopic experimental observations.', 'year': 2015, 'citationCount': 605, 'authors': [{'authorId': '20991272', 'name': 'D. Berman'}, {'authorId': '3887827', 'name': 'Sanket A. Deshmukh'}, {'authorId': '2929500', 'name': 'S. Sankaranarayanan'}, {'authorId': '48715178', 'name': 'A. Erdemir'}, {'authorId': '2149699', 'name': 'A. Sumant'}]}, {'paperId': 'd6b3d4b1e5b081af3226d5fdbbc3661113f331ee', 'title': 'Readily processed protonic ceramic fuel cells with high performance at low temperatures', 'abstract': 'Cooler ceramic fuel cells Ceramic ion conductors can be used as electrolytes in fuel cells using natural gas. One drawback of such solid-oxide fuel cells that conduct oxygen ions is their high operating temperatures (at least 600°C). Duan et al. have made a proton-conducting ceramic fuel cell with a modified cathode material that exhibits high performance on methane fuel at 500°C (see the Perspective by Gorte). Science, this issue p. 1321; see also p. 1290 A proton-conduction cathode and simpler fabrication enable lower-temperature operation of methane-fueled ceramic fuel cells. [Also see Perspective by Gorte] Because of the generally lower activation energy associated with proton conduction in oxides compared to oxygen ion conduction, protonic ceramic fuel cells (PCFCs) should be able to operate at lower temperatures than solid oxide fuel cells (250° to 550°C versus ≥600°C) on hydrogen and hydrocarbon fuels if fabrication challenges and suitable cathodes can be developed. We fabricated the complete sandwich structure of PCFCs directly from raw precursor oxides with only one moderate-temperature processing step through the use of sintering agents such as copper oxide. We also developed a proton-, oxygen-ion–, and electron-hole–conducting PCFC-compatible cathode material, BaCo0.4Fe0.4Zr0.1Y0.1O3-δ (BCFZY0.1), that greatly improved oxygen reduction reaction kinetics at intermediate to low temperatures. We demonstrated high performance from five different types of PCFC button cells without degradation after 1400 hours. Power densities as high as 455 milliwatts per square centimeter at 500°C on H2 and 142 milliwatts per square centimeter on CH4 were achieved, and operation was possible even at 350°C.', 'year': 2015, 'citationCount': 811, 'authors': [{'authorId': '7906816', 'name': 'Chuancheng Duan'}, {'authorId': '48620856', 'name': 'J. Tong'}, {'authorId': '2144460956', 'name': 'Meng Shang'}, {'authorId': '4406457', 'name': 'S. Nikodemski'}, {'authorId': '2004546422', 'name': 'M. Sanders'}, {'authorId': '8176738', 'name': 'S. Ricote'}, {'authorId': '34965842', 'name': 'A. Almansoori'}, {'authorId': '1398376191', 'name': \"R. O'Hayre\"}]}, {'paperId': 'a52e39e2cd8f98c002dafd84183661db682152bf', 'title': 'De novo mutations in congenital heart disease with neurodevelopmental and other congenital anomalies', 'abstract': 'Putting both heart and brain at risk For reasons that are unclear, newborns with congenital heart disease (CHD) have a high risk of neurodevelopmental disabilities. Homsy et al. performed exome sequence analysis of 1200 CHD patients and their parents to identify spontaneously arising (de novo) mutations. Patients with both CHD and neurodevelopmental disorders had a much higher burden of damaging de novo mutations, particularly in genes with likely roles in both heart and brain development. Thus, clinical genotyping of patients with CHD may help to identify those at greatest risk of neurodevelopmental disabilities, allowing surveillance and early intervention. Science, this issue p. 1262 Genotyping of children with congenital heart disease may identify those at high risk of neurodevelopmental disorders. Congenital heart disease (CHD) patients have an increased prevalence of extracardiac congenital anomalies (CAs) and risk of neurodevelopmental disabilities (NDDs). Exome sequencing of 1213 CHD parent-offspring trios identified an excess of protein-damaging de novo mutations, especially in genes highly expressed in the developing heart and brain. These mutations accounted for 20% of patients with CHD, NDD, and CA but only 2% of patients with isolated CHD. Mutations altered genes involved in morphogenesis, chromatin modification, and transcriptional regulation, including multiple mutations in RBFOX2, a regulator of mRNA splicing. Genes mutated in other cohorts examined for NDD were enriched in CHD cases, particularly those with coexisting NDD. These findings reveal shared genetic contributions to CHD, NDD, and CA and provide opportunities for improved prognostic assessment and early therapeutic intervention in CHD patients.', 'year': 2015, 'citationCount': 578, 'authors': [{'authorId': '36079066', 'name': 'Jason Homsy'}, {'authorId': '47885166', 'name': 'S. Zaidi'}, {'authorId': '37082667', 'name': 'Yufeng Shen'}, {'authorId': '48123382', 'name': 'J. Ware'}, {'authorId': '6255853', 'name': 'K. Samocha'}, {'authorId': '1813912', 'name': 'K. Karczewski'}, {'authorId': '5085779', 'name': 'S. DePalma'}, {'authorId': '36608677', 'name': 'D. McKean'}, {'authorId': '6344438', 'name': 'H. Wakimoto'}, {'authorId': '5444722', 'name': 'Joshua M. Gorham'}, {'authorId': '8130226', 'name': 'S. Jin'}, {'authorId': '2069206170', 'name': 'J. Deanfield'}, {'authorId': '3562597', 'name': 'A. Giardini'}, {'authorId': '38146876', 'name': 'G. Porter'}, {'authorId': '51178340', 'name': 'R. Kim'}, {'authorId': '4223946', 'name': 'K. Bilguvar'}, {'authorId': '1393617500', 'name': 'F. López-Giráldez'}, {'authorId': '36337379', 'name': 'Irina R. Tikhonova'}, {'authorId': '2666293', 'name': 'S. Mane'}, {'authorId': '1399293379', 'name': 'Angela Romano-Adesman'}, {'authorId': '145671097', 'name': 'Hongjian Qi'}, {'authorId': '3549190', 'name': 'B. Vardarajan'}, {'authorId': '5616092', 'name': 'Lijiang Ma'}, {'authorId': '144524355', 'name': 'M. Daly'}, {'authorId': '4419743', 'name': 'A. Roberts'}, {'authorId': '12217730', 'name': 'M. Russell'}, {'authorId': '144766484', 'name': 'S. Mital'}, {'authorId': '75082917', 'name': 'J. Newburger'}, {'authorId': '144442425', 'name': 'J. Gaynor'}, {'authorId': '3672289', 'name': 'R. Breitbart'}, {'authorId': '2154137', 'name': 'I. Iossifov'}, {'authorId': '4232362', 'name': 'M. Ronemus'}, {'authorId': '4493069', 'name': 'Stephan J Sanders'}, {'authorId': '4790942', 'name': 'J. Kaltman'}, {'authorId': '1703476', 'name': 'J. Seidman'}, {'authorId': '2141645', 'name': 'M. Brueckner'}, {'authorId': '5215835', 'name': 'B. Gelb'}, {'authorId': '4520908', 'name': 'E. Goldmuntz'}, {'authorId': '3038250', 'name': 'R. Lifton'}, {'authorId': '5088534', 'name': 'C. Seidman'}, {'authorId': '145062232', 'name': 'W. Chung'}]}, {'paperId': 'b7b9ab1eaf18a67f4aa620619d8f21211c7c5964', 'title': 'Extensive introgression in a malaria vector species complex revealed by phylogenomics', 'abstract': 'Introduction The notion that species boundaries can be porous to introgression is increasingly accepted. Yet the broader role of introgression in evolution remains contentious and poorly documented, partly because of the challenges involved in accurately identifying introgression in the very groups where it is most likely to occur. Recently diverged species often have incomplete reproductive barriers and may hybridize where they overlap. However, because of retention and stochastic sorting of ancestral polymorphisms, inference of the correct species branching order is notoriously challenging for recent speciation events, especially those closely spaced in time. Without knowledge of species relationships, it is impossible to identify instances of introgression. Rationale Since the discovery that the single mosquito taxon described in 1902 as Anopheles gambiae was actually a complex of several closely related and morphologically indistinguishable sibling species, the correct species branching order has remained controversial and unresolved. This Afrotropical complex contains the world’s most important vectors of human malaria, owing to their close association with humans, as well as minor vectors and species that do not bite humans. On the basis of ecology and behavior, one might predict phylogenetic clustering of the three highly anthropophilic vector species. However, previous phylogenetic analyses of the complex based on a limited number of markers strongly disagree about relationships between the major vectors, potentially because of historical introgression between them. To investigate the history of the species complex, we used whole-genome reference assemblies, as well as dozens of resequenced individuals from the field. Results We observed a large amount of phylogenetic discordance between trees generated from the autosomes and X chromosome. The autosomes, which make up the majority of the genome, overwhelmingly supported the grouping of the three major vectors of malaria, An. gambiae, An. coluzzii, and An. arabiensis. In stark contrast, the X chromosome strongly supported the grouping of An. arabiensis with a species that plays no role in malaria transmission, An. quadriannulatus. Although the whole-genome consensus phylogeny unequivocally agrees with the autosomal topology, we found that the topology most often located on the X chromosome follows the historical species branching order, with pervasive introgression on the autosomes producing relationships that group the three highly anthropophilic species together. With knowledge of the correct species branching order, we are further able to uncover introgression between another species pair, as well as a complex history of balancing selection, introgression, and local adaptation of a large autosomal inversion that confers aridity tolerance. Conclusion We identify the correct species branching order of the An. gambiae species complex, resolving a contentious phylogeny. Notably, lineages leading to the principal vectors of human malaria were among the first in the complex to radiate and are not most closely related to each other. Pervasive autosomal introgression between these human malaria vectors, including nonsister vector species, suggests that traits enhancing vectorial capacity can be acquired not only through de novo mutation but also through a more rapid process of interspecific genetic exchange. Time-lapse photographs of an adult anopheline mosquito emerging from its pupal case. RELATED ITEMS IN ScienceD. E. Neafsey et al., Science 347, 1258522 (2015) Introgressive hybridization is now recognized as a widespread phenomenon, but its role in evolution remains contested. Here, we use newly available reference genome assemblies to investigate phylogenetic relationships and introgression in a medically important group of Afrotropical mosquito sibling species. We have identified the correct species branching order to resolve a contentious phylogeny and show that lineages leading to the principal vectors of human malaria were among the first to split. Pervasive autosomal introgression between these malaria vectors means that only a small fraction of the genome, mainly on the X chromosome, has not crossed species boundaries. Our results suggest that traits enhancing vectorial capacity may be gained through interspecific gene flow, including between nonsister species. Mosquito adaptability across genomes Virtually everyone has first-hand experience with mosquitoes. Few recognize the subtle biological distinctions among these bloodsucking flies that render some bites mere nuisances and others the initiation of a potentially life-threatening infection. By sequencing the genomes of several mosquitoes in depth, Neafsey et al. and Fontaine et al. reveal clues that explain the mystery of why only some species of one genus of mosquitoes are capable of transmitting human malaria (see the Perspective by Clark and Messer). Science, this issue 10.1126/science.1258524 and 10.1126/science.1258522; see also p. 27 Comparison of several genomes reveals the genetic history of mosquitoes’ ability to vector malaria among humans. [Also see Perspective by Clark and Messer]', 'year': 2015, 'citationCount': 501, 'authors': [{'authorId': '144101105', 'name': 'M. Fontaine'}, {'authorId': '2497743', 'name': 'J. Pease'}, {'authorId': '47174215', 'name': 'Aaron Steele'}, {'authorId': '3008998', 'name': 'R. Waterhouse'}, {'authorId': '4303568', 'name': 'D. Neafsey'}, {'authorId': '2721351', 'name': 'I. Sharakhov'}, {'authorId': '143700507', 'name': 'Xiaofang Jiang'}, {'authorId': '144957945', 'name': 'A. B. Hall'}, {'authorId': '3795051', 'name': 'F. Catteruccia'}, {'authorId': '4003555', 'name': 'E. Kakani'}, {'authorId': '153336006', 'name': 'Sara N. Mitchell'}, {'authorId': '38870301', 'name': 'Yi-Chieh Wu'}, {'authorId': '153144270', 'name': 'Hilary A. Smith'}, {'authorId': '145966718', 'name': 'R. R. Love'}, {'authorId': '1688049', 'name': 'M. Lawniczak'}, {'authorId': '6198906', 'name': 'M. Slotman'}, {'authorId': '1760521', 'name': 'S. Emrich'}, {'authorId': '2235214549', 'name': 'M. Hahn'}, {'authorId': '2524799', 'name': 'N. Besansky'}]}, {'paperId': '2a0e218758c8ca8a2eba1db681d5dbbda81a43e2', 'title': 'Immunogenicity of somatic mutations in human gastrointestinal cancers', 'abstract': 'Low mutation rate okay for T cells Cancers that tend to have high numbers of mutations, such as melanoma and smoking-induced lung cancer, respond well to immunotherapies, whereas those with fewer mutations, such as pancreatic cancer, do not. Tran et al. searched for tumor mutation–reactive T cells in 10 patients with metastatic gastrointestinal cancers, which have relatively low mutation burdens, and discovered that 9 out of 10 harbored such cells. T cells from one patient recognized a mutation common to many types of cancers. Engineering T cells to express this particular mutation-reactive T cell receptor may extend adoptive cell immunotherapy to a larger pool of patients than previously anticipated. Science, this issue p. 1387 Individuals with cancers that have low mutation frequencies often harbor mutation-reactive T cells. It is unknown whether the human immune system frequently mounts a T cell response against mutations expressed by common epithelial cancers. Using a next-generation sequencing approach combined with high-throughput immunologic screening, we demonstrated that tumor-infiltrating lymphocytes (TILs) from 9 out of 10 patients with metastatic gastrointestinal cancers contained CD4+ and/or CD8+ T cells that recognized one to three neo-epitopes derived from somatic mutations expressed by the patient’s own tumor. There were no immunogenic epitopes shared between these patients. However, we identified in one patient a human leukocyte antigen–C*08:02–restricted T cell receptor from CD8+ TILs that targeted the KRASG12D hotspot driver mutation found in many human cancers. Thus, a high frequency of patients with common gastrointestinal cancers harbor immunogenic mutations that can potentially be exploited for the development of highly personalized immunotherapies.', 'year': 2015, 'citationCount': 585, 'authors': [{'authorId': '2280776', 'name': 'E. Tran'}, {'authorId': '38535581', 'name': 'M. Ahmadzadeh'}, {'authorId': '48518323', 'name': 'Yong-Chen Lu'}, {'authorId': '5168824', 'name': 'A. Gros'}, {'authorId': '3622148', 'name': 'S. Turcotte'}, {'authorId': '2083707', 'name': 'P. Robbins'}, {'authorId': '145772328', 'name': 'J. Gartner'}, {'authorId': '37322255', 'name': 'Z. Zheng'}, {'authorId': '47001537', 'name': 'Yong F Li'}, {'authorId': '8485007', 'name': 'Satyajit Ray'}, {'authorId': '32109109', 'name': 'J. Wunderlich'}, {'authorId': '5105081', 'name': 'R. Somerville'}, {'authorId': '144243541', 'name': 'S. Rosenberg'}]}, {'paperId': 'a50c931f46adc7bdaf8d3ce2f0cf7c9b87b35f89', 'title': 'Platinum-based nanocages with subnanometer-thick walls and well-defined, controllable facets', 'abstract': 'Etching platinum nanocage catalysts Although platinum is an excellent catalyst for the oxygen reduction reaction that occurs in fuel cells, its scarcity continues to drive efforts to improve its utilization. Zhang et al. made nanocages of platinum by coating palladium nanocrystals with only a few layers of platinum and then etching away the palladium core (see the Perspective by Strasser). Platinum nanocages made using nanoscale octahedra and cubes of palladium displayed different catalytic activity for the oxygen reduction reaction. Science, this issue p. 412; see also p. 379 Nanocage electrocatalysts can increase the utilization of platinum and improve activity by controlling surface structure. [Also see Perspective by Strasser] A cost-effective catalyst should have a high dispersion of the active atoms, together with a controllable surface structure for the optimization of activity, selectivity, or both. We fabricated nanocages by depositing a few atomic layers of platinum (Pt) as conformal shells on palladium (Pd) nanocrystals with well-defined facets and then etching away the Pd templates. Density functional theory calculations suggest that the etching is initiated via a mechanism that involves the formation of vacancies through the removal of Pd atoms incorporated into the outermost layer during the deposition of Pt. With the use of Pd nanoscale cubes and octahedra as templates, we obtained Pt cubic and octahedral nanocages enclosed by {100} and {111} facets, respectively, which exhibited distinctive catalytic activities toward oxygen reduction.', 'year': 2015, 'citationCount': 732, 'authors': [{'authorId': '2152832439', 'name': 'Lei Zhang'}, {'authorId': '4612075', 'name': 'L. T. Roling'}, {'authorId': '2144706211', 'name': 'Xue Wang'}, {'authorId': '2422767', 'name': 'M. Vara'}, {'authorId': '2232765', 'name': 'M. Chi'}, {'authorId': '6207126', 'name': 'J. Liu'}, {'authorId': '91669921', 'name': 'Sang‐Il Choi'}, {'authorId': '2109225880', 'name': 'Jinho Park'}, {'authorId': '2279071', 'name': 'J. Herron'}, {'authorId': '36620090', 'name': 'Zhaoxiong Xie'}, {'authorId': '47249637', 'name': 'M. Mavrikakis'}, {'authorId': '46267791', 'name': 'Younan Xia'}]}, {'paperId': '20bbc7c4c761d90f0f81c1c5928922caaaeef4af', 'title': 'Undergraduate research experiences: Impacts and opportunities', 'abstract': 'Assessing the value of undergraduate research Undergraduate research experiences often engender enthusiasm in the students involved, but how useful are they in terms of enhancing student learning? Linn et al. review studies that focus on the effectiveness of undergraduate research programs. Undergraduate research experiences in a class were distinguished from those involving individualized participation in a research program. Mentoring emerges as both an important component of a successful experience and a target for improvement. Science, this issue 10.1126/science.1261757 For any undergraduate contemplating a career in scientific research, participating in authentic research seems like a good opportunity. But what are authentic research experiences? How do they benefit undergraduates? What forms of mentoring are successful? What needs improvement? And how can these experiences meet the needs of interested students while at the same time be cost-effective in large research universities? We review the research tackling these questions and find few answers. While most undergraduates give high ratings to research experiences, specific benefits have not been documented. Of the 60 empirical studies published in the last 5 years, only 4 directly measured gains in research capabilities or conceptual understanding. Most studies draw conclusions from self-report surveys or interviews, notoriously poor methods for documenting impacts. These studies leave us with few insights into what works and little idea about how to make the experiences more effective. BACKGROUND Most colleges and universities offer Undergraduate Research Experiences (UREs) and/or Course-based Undergraduate Research Experiences (CUREs) (Fig. 1). Two large surveys, the 2004 Freshman Survey and the 2008 College Senior Survey, administered at over 200 institutions, generated data about the impact of undergraduate research experiences on persistence in science and intention to pursue graduate school. These studies document that students appreciate undergraduate research experiences. The surveys are unable, however, to distinguish between UREs and CUREs. In addition, the value that undergraduate research adds cannot be disentangled from precollege preparation, especially for students from groups that are underrepresented in science. ADVANCES Designers of UREs expect students to benefit from participating in a scientific laboratory but have not determined optimal ways to orient and guide participants. Students often expect the URE to mimic their college laboratory experiences with procedural guidance and planned outcomes. During the first year of a URE, students often report spending most of their effort on setting up and conducting experiments and limited effort on understanding the investigation or interpreting the results. Students would benefit from an orientation that integrates their beliefs and expectations with the realities of the research experience. The few studies that measure changes in understanding of scientific practices or relevant science concepts report little or no gains after 1 year in a URE. Students who spend over a year in a URE often learn new methodological techniques, collect their own data, interpret findings, and formulate new research questions. The slow enculturation into lab activities may make sense, especially when students join labs investigating questions that do not arise in undergraduate education. The time and resources needed, however, limit the scalability of UREs. Students encounter new ideas during their research experiences but often need guidance to integrate these ideas with their expectations. We discuss ways that designers of UREs can speed up enculturation and strengthen guidance. Individual mentoring emerges as an effective way to guide students and improve learning from research experiences. Activities that could help students benefit from research experiences include discussion with mentors, participation in group meetings where current research is discussed, guided opportunities to explore relevant research literature, reflection on observations in weekly journals, and synthesis of their insights by creating research proposals, reports, or posters. We discuss ways to prepare mentors so that they can efficiently guide students. OUTLOOK Undergraduate research experiences absorb a lot of time, money, and effort. The costs and benefits of research experiences for building human capital, benefitting undergraduates, improving workforce diversity, and strengthening educational outcomes need better understanding. Making the best use of extramural funds and the (often voluntary) contributions of faculty to improve undergraduate research experiences requires a strong research base. More rigorous research is needed, and the field could benefit by building on insights from the learning sciences. We use the knowledge integration framework to interpret the available findings and to identify gaps in the research base. We discuss ways to develop validated, generalizable assessments such as methods for measuring ability to locate and interpret primary literature. We suggest techniques for developing criteria for evaluating mentoring interactions. We identify ways to strengthen mentoring and to ensure that research experiences meet the needs of diverse students. URE in action. Biofuel research engages a Berkeley undergraduate researcher during a summer internship with the Synthetic Biology Engineering Research Center (funded by NSF grant 1132670). Most undergraduates give high ratings to research experiences. Studies report that these experiences improve participation and persistence, often by strengthening students’ views of themselves as scientists. Yet, the evidence for these claims is weak. More than half the 60 studies reviewed rely on self-report surveys or interviews. Rather than introducing new images of science, research experiences may reinforce flawed images especially of research practices and conceptual understanding. The most convincing studies show benefits for mentoring and for communicating the nature of science, but the ideas that students learn are often isolated or fragmented rather than integrated and coherent. Rigorous research is needed to identify ways to design research experiences so that they promote integrated understanding. These studies need powerful and generalizable assessments that can document student progress, help distinguish effective and ineffective aspects of the experiences, and illustrate how students interpret the research experiences they encounter. To create research experiences that meet the needs of interested students and make effective use of scarce resources, we encourage systematic, iterative studies with multiple indicators of success.', 'year': 2015, 'citationCount': 508, 'authors': [{'authorId': '48454219', 'name': 'M. Linn'}, {'authorId': '2059332914', 'name': 'Erin Palmer'}, {'authorId': '5029957', 'name': 'A. Baranger'}, {'authorId': '2075139616', 'name': 'Elizabeth Gerard'}, {'authorId': '39650728', 'name': 'E. Stone'}]}, {'paperId': 'ac44c78e8b2a938a71010fbc604500a8900ce521', 'title': 'Site-selective arene C-H amination via photoredox catalysis', 'abstract': 'Lighting the way to aryl C-N bonding Medicinal chemists like to add N bonds to the C atoms of aromatic rings to make bioactive compounds. By harnessing the energy in visible light, Romero et al. made these links and transformed C-H into C-N bonds. They used a blue-absorbing acridinium ion to activate a ring C for an incoming N partner. A nitroxyl radical co-catalyst (TEMPO) then choreographed the transfer of the H atom to O. The reaction worked for a broad range of substrates, including ammonium as a N source. Science, this issue p. 1326 Two catalysts leverage visible light and oxygen to form carbon-nitrogen bond motifs of interest in pharmaceutical research. Over the past several decades, organometallic cross-coupling chemistry has developed into one of the most reliable approaches to assemble complex aromatic compounds from preoxidized starting materials. More recently, transition metal–catalyzed carbon-hydrogen activation has circumvented the need for preoxidized starting materials, but this approach is limited by a lack of practical amination protocols. Here, we present a blueprint for aromatic carbon-hydrogen functionalization via photoredox catalysis and describe the utility of this strategy for arene amination. An organic photoredox-based catalyst system, consisting of an acridinium photooxidant and a nitroxyl radical, promotes site-selective amination of a variety of simple and complex aromatics with heteroaromatic azoles of interest in pharmaceutical research. We also describe the atom-economical use of ammonia to form anilines, without the need for prefunctionalization of the aromatic component.', 'year': 2015, 'citationCount': 532, 'authors': [{'authorId': '2775062', 'name': 'N. Romero'}, {'authorId': '7223629', 'name': 'Kaila A. Margrey'}, {'authorId': '3294517', 'name': 'Nicholas E. S. Tay'}, {'authorId': '4072096', 'name': 'David A. Nicewicz'}]}, {'paperId': '7a1a1f38ed470d814e2e92f3b976e99096ada4ad', 'title': 'Patients with LRBA deficiency show CTLA4 loss and immune dysregulation responsive to abatacept therapy', 'abstract': 'Trafficking from bedside to bench Typically in translational research, a discovery in cell or molecular biology is later exploited to improve patient care. Occasionally, information flows in the opposite direction. Lo et al. found that patients with an autoimmune disorder caused by deficiency of a protein called LRBA responded dramatically to the drug abatacept (see the Perspective by Sansom). Abatacept contains a segment of a potent inhibitory immune receptor, CTLA4. Experiments prompted by this observation revealed the relationship between the two proteins: LRBA controls the intracellular trafficking and degradation of CTLA4. This information may further improve patient care, because other clinically approved drugs have the desired mechanism of action with potentially fewer side effects. Science, this issue p. 436; see also p. 377 A rare autoimmune disorder is caused by aberrant degradation of a potent inhibitory immune receptor. [Also see Perspective by Sansom] Mutations in the LRBA gene (encoding the lipopolysaccharide-responsive and beige-like anchor protein) cause a syndrome of autoimmunity, lymphoproliferation, and humoral immune deficiency. The biological role of LRBA in immunologic disease is unknown. We found that patients with LRBA deficiency manifested a dramatic and sustained improvement in response to abatacept, a CTLA4 (cytotoxic T lymphocyte antigen-4)–immunoglobulin fusion drug. Clinical responses and homology of LRBA to proteins controlling intracellular trafficking led us to hypothesize that it regulates CTLA4, a potent inhibitory immune receptor. We found that LRBA colocalized with CTLA4 in endosomal vesicles and that LRBA deficiency or knockdown increased CTLA4 turnover, which resulted in reduced levels of CTLA4 protein in FoxP3+ regulatory and activated conventional T cells. In LRBA-deficient cells, inhibition of lysosome degradation with chloroquine prevented CTLA4 loss. These findings elucidate a mechanism for CTLA4 trafficking and control of immune responses and suggest therapies for diseases involving the CTLA4 pathway.', 'year': 2015, 'citationCount': 502, 'authors': [{'authorId': '35263862', 'name': 'Bernice Lo'}, {'authorId': '5170321', 'name': 'Kejian Zhang'}, {'authorId': '2198230', 'name': 'Wei-Yu Lu'}, {'authorId': '123584816', 'name': 'Lixin Zheng'}, {'authorId': '145261216', 'name': 'Qian Zhang'}, {'authorId': '40295074', 'name': 'C. Kanellopoulou'}, {'authorId': None, 'name': 'Yu Zhang'}, {'authorId': '2144292726', 'name': 'Zhiduo Liu'}, {'authorId': '39402782', 'name': 'Jill M. Fritz'}, {'authorId': '34743560', 'name': 'R. Marsh'}, {'authorId': '5997093', 'name': 'Ammar Husami'}, {'authorId': '6443598', 'name': 'D. Kissell'}, {'authorId': '8284739', 'name': 'Shannon Nortman'}, {'authorId': '7985432', 'name': 'Vijaya Chaturvedi'}, {'authorId': '50321247', 'name': 'H. Haines'}, {'authorId': '144059319', 'name': 'L. Young'}, {'authorId': '1431187881', 'name': 'J. Mo'}, {'authorId': '6579125', 'name': 'A. Filipovich'}, {'authorId': '77416037', 'name': 'J. Bleesing'}, {'authorId': '10355163', 'name': 'P. Mustillo'}, {'authorId': '49649865', 'name': 'Michael C. Stephens'}, {'authorId': '40506732', 'name': 'C. Rueda'}, {'authorId': '5879639', 'name': 'C. Chougnet'}, {'authorId': '6682787', 'name': 'K. Hoebe'}, {'authorId': '143964722', 'name': 'J. McElwee'}, {'authorId': '145930892', 'name': 'J. Hughes'}, {'authorId': '1401313922', 'name': 'E. Karakoc‐Aydiner'}, {'authorId': '4563707', 'name': 'H. Matthews'}, {'authorId': '48194986', 'name': 'S. Price'}, {'authorId': '144213500', 'name': 'H. Su'}, {'authorId': '144632738', 'name': 'V. Rao'}, {'authorId': '145001039', 'name': 'M. Lenardo'}, {'authorId': '31907435', 'name': 'M. Jordan'}]}, {'paperId': '02c24ef5736d9c2809da31ca658d8ced1d81350b', 'title': 'CTCF establishes discrete functional chromatin domains at the Hox clusters during differentiation', 'abstract': 'Keeping repressed genes repressed Hox genes confer positional identity to cells and tissues. Maintaining precise spatial patterns of Hox gene expression is vital during metazoan development. The transcriptional repressor CTCF is involved in the regulation of chromatin architecture. Narendra et al. show that a CTCF protein binding site insulates regions of active and repressed Hox gene expression from each other. This protects heterochromatin containing repressed Hox genes from the encroaching spread of active chromatin. The CTCF protein appears to organize the active and repressed chromatin regions into distinct architectural domains. Science, this issue p. 1017 A DNA binding protein insulates and protects repressed topological domains in the genome from adjacent active regions. Polycomb and Trithorax group proteins encode the epigenetic memory of cellular positional identity by establishing inheritable domains of repressive and active chromatin within the Hox clusters. Here we demonstrate that the CCCTC-binding factor (CTCF) functions to insulate these adjacent yet antagonistic chromatin domains during embryonic stem cell differentiation into cervical motor neurons. Deletion of CTCF binding sites within the Hox clusters results in the expansion of active chromatin into the repressive domain. CTCF functions as an insulator by organizing Hox clusters into spatially disjoint domains. Ablation of CTCF binding disrupts topological boundaries such that caudal Hox genes leave the repressed domain and become subject to transcriptional activation. Hence, CTCF is required to insulate facultative heterochromatin from impinging euchromatin to produce discrete positional identities.', 'year': 2015, 'citationCount': 472, 'authors': [{'authorId': '2098133', 'name': 'V. Narendra'}, {'authorId': '144949609', 'name': 'Pedro P. Rocha'}, {'authorId': '6659023', 'name': 'Disi An'}, {'authorId': '3382835', 'name': 'R. Raviram'}, {'authorId': '3382289', 'name': 'J. Skok'}, {'authorId': '2565403', 'name': 'E. Mazzoni'}, {'authorId': '5850010', 'name': 'D. Reinberg'}]}, {'paperId': '29e96ad5f757fb36b8d3f4ec2e5ea11e7121e518', 'title': 'Transcribed enhancers lead waves of coordinated transcription in transitioning mammalian cells', 'abstract': 'Uncaging promoter and enhancer dynamics In order to understand cellular differentiation, it is important to understand the timing of the regulation of gene expression. Arner et al. used cap analysis of gene expression (CAGE) to analyze gene enhancer and promoter activities in a number of human and mouse cell types. The RNA of enhancers was transcribed first, followed by that of transcription factors, and finally by genes that are not transcription factors. Science, this issue p. 1010 The RNAs of enhancers are expressed before the genes they regulate during human and mouse cell differentiation. Although it is generally accepted that cellular differentiation requires changes to transcriptional networks, dynamic regulation of promoters and enhancers at specific sets of genes has not been previously studied en masse. Exploiting the fact that active promoters and enhancers are transcribed, we simultaneously measured their activity in 19 human and 14 mouse time courses covering a wide range of cell types and biological stimuli. Enhancer RNAs, then messenger RNAs encoding transcription factors, dominated the earliest responses. Binding sites for key lineage transcription factors were simultaneously overrepresented in enhancers and promoters active in each cellular system. Our data support a highly generalizable model in which enhancer transcription is the earliest event in successive waves of transcriptional change during cellular differentiation or activation.', 'year': 2015, 'citationCount': 511, 'authors': [{'authorId': '23521489', 'name': 'E. Arner'}, {'authorId': '143917592', 'name': 'C. Daub'}, {'authorId': '1397636244', 'name': 'Kristoffer Vitting-Seerup'}, {'authorId': '40479623', 'name': 'R. Andersson'}, {'authorId': '4166525', 'name': 'B. Lilje'}, {'authorId': '114341708', 'name': 'F. Drabløs'}, {'authorId': '2688127', 'name': 'A. Lennartsson'}, {'authorId': '4886769', 'name': 'Michelle Rönnerblad'}, {'authorId': '47320354', 'name': 'Olga Hrydziuszko'}, {'authorId': '2532012', 'name': 'Morana Vitezic'}, {'authorId': '143803510', 'name': 'T. Freeman'}, {'authorId': '2653439', 'name': 'Ahmad M. N. Alhendi'}, {'authorId': '4212074', 'name': 'P. Arner'}, {'authorId': '6948294', 'name': 'R. Axton'}, {'authorId': '145337559', 'name': 'J. Baillie'}, {'authorId': '8528842', 'name': 'A. Beckhouse'}, {'authorId': '3696831', 'name': 'B. Bodega'}, {'authorId': '144968403', 'name': 'J. Briggs'}, {'authorId': '6018346', 'name': 'F. Brombacher'}, {'authorId': '30335332', 'name': 'M. Davis'}, {'authorId': '4516324', 'name': 'M. Detmar'}, {'authorId': '4699789', 'name': 'A. Ehrlund'}, {'authorId': '5356829', 'name': 'Mitsuhiro Endoh'}, {'authorId': '46939083', 'name': 'A. Eslami'}, {'authorId': '10935716', 'name': 'M. Fagiolini'}, {'authorId': '34034984', 'name': 'L. Fairbairn'}, {'authorId': '3076070', 'name': 'G. Faulkner'}, {'authorId': '46750409', 'name': 'Carmelo Ferrai'}, {'authorId': '153395152', 'name': 'M. Fisher'}, {'authorId': '5226420', 'name': 'L. Forrester'}, {'authorId': '2230666', 'name': 'D. Goldowitz'}, {'authorId': '3261646', 'name': 'R. Guler'}, {'authorId': '9655717', 'name': 'Thomas J. Ha'}, {'authorId': '9034129', 'name': 'Mitsuko Hara'}, {'authorId': '47789855', 'name': 'M. Herlyn'}, {'authorId': '4258998', 'name': 'T. Ikawa'}, {'authorId': '145708643', 'name': 'C. Kai'}, {'authorId': '2135221', 'name': 'H. Kawamoto'}, {'authorId': '3254387', 'name': 'L. Khachigian'}, {'authorId': '144555597', 'name': 'S. Klinken'}, {'authorId': '33518402', 'name': 'S. Kojima'}, {'authorId': '2777281', 'name': 'H. Koseki'}, {'authorId': '49705905', 'name': 'Sarah Klein'}, {'authorId': '6826212', 'name': 'N. Mejhert'}, {'authorId': '4971860', 'name': 'K. Miyaguchi'}, {'authorId': '50148706', 'name': 'Y. Mizuno'}, {'authorId': '34788980', 'name': 'M. Morimoto'}, {'authorId': '50525373', 'name': 'Kelly J. Morris'}, {'authorId': '3440534', 'name': 'C. Mummery'}, {'authorId': '2351194', 'name': 'Y. Nakachi'}, {'authorId': '1786708', 'name': 'S. Ogishima'}, {'authorId': '1397636249', 'name': 'M. Okada‐Hatakeyama'}, {'authorId': '50117403', 'name': 'Y. Okazaki'}, {'authorId': '37788422', 'name': 'V. Orlando'}, {'authorId': '3745403', 'name': 'D. Ovchinnikov'}, {'authorId': '3517695', 'name': 'R. Passier'}, {'authorId': '8576273', 'name': 'M. Patrikakis'}, {'authorId': '144656656', 'name': 'A. Pombo'}, {'authorId': '5128673', 'name': 'Xian-Yang Qin'}, {'authorId': '47115698', 'name': 'Sugata Roy'}, {'authorId': '145506513', 'name': 'Hiroki Sato'}, {'authorId': '5481867', 'name': 'Suzana Savvi'}, {'authorId': '49984299', 'name': 'A. Saxena'}, {'authorId': '2949707', 'name': 'Anita Schwegmann'}, {'authorId': '145080244', 'name': 'D. Sugiyama'}, {'authorId': '145354896', 'name': 'R. Swoboda'}, {'authorId': '143848108', 'name': 'Hiroshi Tanaka'}, {'authorId': '5779951', 'name': 'A. Tomoiu'}, {'authorId': '4853058', 'name': 'Louise N. Winteringham'}, {'authorId': '4431070', 'name': 'E. Wolvetang'}, {'authorId': '1400621142', 'name': 'Chiyo Yanagi-Mizuochi'}, {'authorId': '47848477', 'name': 'M. Yoneda'}, {'authorId': '4654219', 'name': 'S. Zabierowski'}, {'authorId': '1396200156', 'name': 'Peter G. Zhang'}, {'authorId': '145237763', 'name': 'I. Abugessaisa'}, {'authorId': '5942363', 'name': 'N. Bertin'}, {'authorId': '2389396', 'name': 'A. Diehl'}, {'authorId': '3245060', 'name': 'S. Fukuda'}, {'authorId': '47876805', 'name': 'M. Furuno'}, {'authorId': '3434398', 'name': 'J. Harshbarger'}, {'authorId': '145629751', 'name': 'A. Hasegawa'}, {'authorId': '27021163', 'name': 'Fumi Hori'}, {'authorId': '1400621118', 'name': 'Sachi Ishikawa-Kato'}, {'authorId': '11917019', 'name': 'Y. Ishizu'}, {'authorId': '144562623', 'name': 'M. Itoh'}, {'authorId': '6461494', 'name': 'T. Kawashima'}, {'authorId': '34833114', 'name': 'M. Kojima'}, {'authorId': '35970974', 'name': 'N. Kondo'}, {'authorId': '2124452', 'name': 'M. Lizio'}, {'authorId': '35466559', 'name': 'T. Meehan'}, {'authorId': '52038267', 'name': 'C. Mungall'}, {'authorId': '11168299', 'name': 'Mitsuyoshi Murata'}, {'authorId': '1400621107', 'name': 'Hiromi Nishiyori-Sueki'}, {'authorId': '48534111', 'name': 'S. Şahin'}, {'authorId': '1400621116', 'name': 'Sayaka Nagao-Sato'}, {'authorId': '3177339', 'name': 'J. Severin'}, {'authorId': '3207348', 'name': 'M. Hoon'}, {'authorId': '1695117', 'name': 'J. Kawai'}, {'authorId': '2572145', 'name': 'T. Kasukawa'}, {'authorId': '2775888', 'name': 'T. Lassmann'}, {'authorId': '2111430835', 'name': 'Harukazu Suzuki'}, {'authorId': '2657593', 'name': 'H. Kawaji'}, {'authorId': '1948404', 'name': 'K. Summers'}, {'authorId': '49078862', 'name': 'C. Wells'}, {'authorId': '2330321', 'name': 'D. Hume'}, {'authorId': '144627172', 'name': 'A. Forrest'}, {'authorId': '3275066', 'name': 'A. Sandelin'}, {'authorId': '2051569', 'name': 'Piero Carninci'}, {'authorId': '3345237', 'name': 'Y. Hayashizaki'}]}, {'paperId': '42df2ef9b0f3fa9a1d52fe29a2f09fea6d0a937c', 'title': 'Deposition, exhumation, and paleoclimate of an ancient lake deposit, Gale crater, Mars', 'abstract': 'Ancient lake system at Gale crater Since 2012, the Curiosity rover has been diligently studying rocky outcrops on Mars, looking for clues about past water, climate, and habitability. Grotzinger et al. describe the analysis of a huge section of sedimentary rocks near Gale crater, where Mount Sharp now stands (see the Perspective by Chan). The features within these sediments are reminiscent of delta, stream, and lake deposits on Earth. Although individual lakes were probably transient, it is likely that there was enough water to fill in low-lying depressions such as impact craters for up to 10,000 years. Wind-driven erosion removed many of these deposits, creating Mount Sharp. Science, this issue p.10.1126/science.aac7575, see also p. 167 Mount Sharp now stands where there was once a large intercrater lake system. [Also see Perspective by Chan] INTRODUCTION Remote observational data suggest that large bodies of standing water existed on the surface of Mars in its early history. This would have required a much wetter climate than that of the present, implying greater availability of water on a global basis and enhanced potential for global habitability. However, based on assumptions of a vast water inventory and models of atmospheric erosion, theoretical studies suggest a climate that was wetter but not by enough to sustain large lakes, even in depressions such as impact craters. RATIONALE The Mars Science Laboratory mission’s rover, Curiosity, provides the capability to test hypotheses about Mars’s past climate. The focus of the mission is the exploration of a ~5-km-high mountain, Aeolis Mons (informally known as Mount Sharp), located near the center of the ~140-km-wide Gale impact crater. Mount Sharp is underlain by hundreds of meters of sedimentary rock strata deposited ~3.6 billion to 3.2 billion years ago. These sediments accumulated in aqueous environments, recording the history of Mars’s ancient climate. Because of Curiosity’s ability to study these strata where they are exposed near the base of Mount Sharp, we can directly test the hypothesis that large impact craters were capable of accumulating and storing water as lakes for substantial periods of time. RESULTS Over the course of 2 years, Curiosity studied dozens of outcrops distributed along a ~9-km transect that also rose ~75 m in elevation. Image data were used to measure the geometry and grain sizes of strata and to survey the textures associated with sediment deposition and diagenesis. Erosion of Gale’s northern crater wall and rim generated gravel and sand that were transported southward in shallow streams. Over time, these stream deposits advanced toward the crater interior, transitioning downstream into finer-grained (sand-sized), southward-advancing delta deposits. These deltas marked the boundary of an ancient lake where the finest (mud-sized) sediments accumulated, infilling both the crater and its internal lake basin. After infilling of the crater, the sedimentary deposits in Gale crater were exhumed, probably by wind-driven erosion, creating Mount Sharp. The ancient stream and lake deposits are erosional remnants of superimposed depositional sequences that once extended at least 75 m, and perhaps several hundreds of meters, above the current elevation of the crater floor. Although the modern landscape dips northward away from Mount Sharp, the ancient sedimentary deposits were laid down along a profile that projected southward beneath Mount Sharp and indicate that a basin once existed where today there is a mountain. CONCLUSION Our observations suggest that individual lakes were stable on the ancient surface of Mars for 100 to 10,000 years, a minimum duration when each lake was stable both thermally (as liquid water) and in terms of mass balance (with inputs effectively matching evaporation and loss of water to colder regions). We estimate that the stratigraphy traversed thus far by Curiosity would have required 10,000 to 10,000,000 years to accumulate, and even longer if overlying strata are included. Though individual lakes may have come and gone, they were probably linked in time through a common groundwater table. Over the long term, this water table must have risen at least tens of meters to enable accumulation of the delta and lake deposits observed by Curiosity in Gale crater. Inclined strata in the foreground dip southward toward Mount Sharp and represent ancient delta deposits. These deposits transition into strata in the mid-field that were deposited in ancient lakes. The buttes and mesas in the background contain younger deposits that overlie and postdate the lake deposits beneath Mount Sharp. The outcrop in the foreground is about 6 m wide, and the buttes and mesas in the background are hundreds of meters wide and tens of meters high. The image has been white-balanced. [Credit: NASA/Caltech/JPL/MSSS] The landforms of northern Gale crater on Mars expose thick sequences of sedimentary rocks. Based on images obtained by the Curiosity rover, we interpret these outcrops as evidence for past fluvial, deltaic, and lacustrine environments. Degradation of the crater wall and rim probably supplied these sediments, which advanced inward from the wall, infilling both the crater and an internal lake basin to a thickness of at least 75 meters. This intracrater lake system probably existed intermittently for thousands to millions of years, implying a relatively wet climate that supplied moisture to the crater rim and transported sediment via streams into the lake basin. The deposits in Gale crater were then exhumed, probably by wind-driven erosion, creating Aeolis Mons (Mount Sharp).', 'year': 2015, 'citationCount': 507, 'authors': [{'authorId': '3844614', 'name': 'J. Grotzinger'}, {'authorId': '1747899867', 'name': 'Sanjeev Gupta'}, {'authorId': '2185806', 'name': 'M. Malin'}, {'authorId': '9216016', 'name': 'D. Rubin'}, {'authorId': '2167952988', 'name': 'J. Schieber'}, {'authorId': '3674499', 'name': 'K. Siebach'}, {'authorId': '2086032', 'name': 'D. Sumner'}, {'authorId': '3558841', 'name': 'K. Stack'}, {'authorId': '35070652', 'name': 'A. Vasavada'}, {'authorId': '2144646', 'name': 'R. Arvidson'}, {'authorId': '4373022', 'name': 'F. Calef'}, {'authorId': '46621475', 'name': 'L. Edgar'}, {'authorId': '2057396505', 'name': 'W. Fischer'}, {'authorId': '119605866', 'name': 'J. Grant'}, {'authorId': '3957803', 'name': 'J. Griffes'}, {'authorId': '4441973', 'name': 'L. Kah'}, {'authorId': '143785554', 'name': 'M. Lamb'}, {'authorId': '143671882', 'name': 'K. Lewis'}, {'authorId': '145544465', 'name': 'N. Mangold'}, {'authorId': '38372442', 'name': 'M. Minitti'}, {'authorId': '7001762', 'name': 'M. Palucis'}, {'authorId': '49565924', 'name': 'M. Rice'}, {'authorId': '2158137751', 'name': 'Rebecca M. E. Williams'}, {'authorId': '5962704', 'name': 'R. Yingst'}, {'authorId': '2068472632', 'name': 'D. Blake'}, {'authorId': '145681115', 'name': 'D. Blaney'}, {'authorId': '40098549', 'name': 'P. Conrad'}, {'authorId': '4047178', 'name': 'J. Crisp'}, {'authorId': '1406001779', 'name': 'W. Dietrich'}, {'authorId': '4718864', 'name': 'G. Dromart'}, {'authorId': '4759990', 'name': 'K. Edgett'}, {'authorId': '145657190', 'name': 'R. Ewing'}, {'authorId': '2714162', 'name': 'R. Gellert'}, {'authorId': '6882251', 'name': 'J. Hurowitz'}, {'authorId': '98112552', 'name': 'G. Kocurek'}, {'authorId': '4579249', 'name': 'P. Mahaffy'}, {'authorId': '41210735', 'name': 'M. McBride'}, {'authorId': '144183259', 'name': 'S. McLennan'}, {'authorId': '6507728', 'name': 'M. Mischna'}, {'authorId': '153607306', 'name': 'Doug Ming'}, {'authorId': '32008334', 'name': 'R. Milliken'}, {'authorId': '4763968', 'name': 'H. Newsom'}, {'authorId': '40335319', 'name': 'D. Oehler'}, {'authorId': '48123456', 'name': 'T. Parker'}, {'authorId': '4803566', 'name': 'D. Vaniman'}, {'authorId': '145952447', 'name': 'R. Wiens'}, {'authorId': '115224202', 'name': 'S. Wilson'}]}, {'paperId': '8e9776716ff99195a5b9b87474f2e4a48beba883', 'title': 'Farm dust and endotoxin protect against allergy through A20 induction in lung epithelial cells', 'abstract': 'How farming protects against allergies People who grow up on dairy farms only rarely develop asthma or allergies. This is probably because as children, they breathe air containing bacterial components, which reduce the overall reactivity of the immune system. Schuijs et al. chronically exposed mice to bacterial endotoxin before they received an allergic stimulus. The protocol indeed protected them from developing an allergic response. Protection relied on a particular enzyme: A20. In humans, a variant of A20 correlates with increased susceptibility to asthma and allergy in children growing up on farms. Science, this issue p. 1106 Chronic exposure to bacterial endotoxin protects against allergy through the enzyme A20. Growing up on a dairy farm protects children from allergy, hay fever, and asthma. A mechanism linking exposure to this endotoxin (bacterial lipopolysaccharide)–rich environment with protection has remained elusive. Here we show that chronic exposure to low-dose endotoxin or farm dust protects mice from developing house dust mite (HDM)–induced asthma. Endotoxin reduced epithelial cell cytokines that activate dendritic cells (DCs), thus suppressing type 2 immunity to HDMs. Loss of the ubiquitin-modifying enzyme A20 in lung epithelium abolished the protective effect. A single-nucleotide polymorphism in the gene encoding A20 was associated with allergy and asthma risk in children growing up on farms. Thus, the farming environment protects from allergy by modifying the communication between barrier epithelial cells and DCs through A20 induction.', 'year': 2015, 'citationCount': 477, 'authors': [{'authorId': '4314078', 'name': 'M. Schuijs'}, {'authorId': '4085503', 'name': 'M. Willart'}, {'authorId': '50071230', 'name': 'Karl Vergote'}, {'authorId': '2198038', 'name': 'D. Gras'}, {'authorId': '5935685', 'name': 'K. Deswarte'}, {'authorId': '6207885', 'name': 'M. Ege'}, {'authorId': '39913579', 'name': 'Filipe Branco Madeira'}, {'authorId': '79930198', 'name': 'R. Beyaert'}, {'authorId': '3941663', 'name': 'G. van Loo'}, {'authorId': '5868696', 'name': 'F. Bracher'}, {'authorId': '116242153', 'name': 'E. von Mutius'}, {'authorId': '145813672', 'name': 'P. Chanez'}, {'authorId': '50382649', 'name': 'B. Lambrecht'}, {'authorId': '143816987', 'name': 'H. Hammad'}]}, {'paperId': '5c7ba6c1f0902007eb5a3aedf889c6a67654361d', 'title': 'Titin mutations in iPS cells define sarcomere insufficiency as a cause of dilated cardiomyopathy', 'abstract': 'A giant disruption of the heart Certain forms of heart failure originate from genetic mutations. Understanding how the culprit mutant proteins alter normal heart function could lead to more effective treatments. One candidate is the giant protein tintin, which is mutated in a subset of patients with dilated cardiomyopathy. Through a combination of patient-derived stem cells, tissue engineering, and gene editing, Hinson et al. found that disease-associated titin mutations disrupt the function of the contractile unit in heart muscle. As a result, the heart does not respond properly to mechanical and other forms of stress. Science, this issue p. 982 Mutations in titin cause heart disease by disrupting the sarcomere, which normally helps the heart adapt to stress. Human mutations that truncate the massive sarcomere protein titin [TTN-truncating variants (TTNtvs)] are the most common genetic cause for dilated cardiomyopathy (DCM), a major cause of heart failure and premature death. Here we show that cardiac microtissues engineered from human induced pluripotent stem (iPS) cells are a powerful system for evaluating the pathogenicity of titin gene variants. We found that certain missense mutations, like TTNtvs, diminish contractile performance and are pathogenic. By combining functional analyses with RNA sequencing, we explain why truncations in the A-band domain of TTN cause DCM, whereas truncations in the I band are better tolerated. Finally, we demonstrate that mutant titin protein in iPS cell–derived cardiomyocytes results in sarcomere insufficiency, impaired responses to mechanical and β-adrenergic stress, and attenuated growth factor and cell signaling activation. Our findings indicate that titin mutations cause DCM by disrupting critical linkages between sarcomerogenesis and adaptive remodeling.', 'year': 2015, 'citationCount': 479, 'authors': [{'authorId': '40393910', 'name': 'J. T. Hinson'}, {'authorId': '4356579', 'name': 'Anant Chopra'}, {'authorId': '2234608653', 'name': 'N. Nafissi'}, {'authorId': '5719705', 'name': 'W. Polacheck'}, {'authorId': '3197049', 'name': 'Craig C. Benson'}, {'authorId': '6786388', 'name': 'Sandra Swist'}, {'authorId': '5444722', 'name': 'Joshua M. Gorham'}, {'authorId': '2967726', 'name': 'Luhan Yang'}, {'authorId': '48327688', 'name': 'S. Schafer'}, {'authorId': '50741905', 'name': 'C. Sheng'}, {'authorId': '2113192', 'name': 'A. Haghighi'}, {'authorId': '36079066', 'name': 'Jason Homsy'}, {'authorId': '5182815', 'name': 'N. Hubner'}, {'authorId': '145892667', 'name': 'G. Church'}, {'authorId': '145502615', 'name': 'S. Cook'}, {'authorId': '2125678', 'name': 'W. Linke'}, {'authorId': '2110195832', 'name': 'Christopher S. Chen'}, {'authorId': '1703476', 'name': 'J. Seidman'}, {'authorId': '5088534', 'name': 'C. Seidman'}, {'authorId': '5088534', 'name': 'C. Seidman'}, {'authorId': '5088534', 'name': 'C. Seidman'}]}, {'paperId': '33e32c7b267441671aab77ac9a0a13cc6e14f3ab', 'title': 'Somatic mutation in single human neurons tracks developmental and transcriptional history', 'abstract': 'Individualized neuronal mutations in the human brain The neurons of the human brain can last for decades, carrying out computational and signaling functions. Lodato et al. analyzed the DNA of individual neurons sampled from postmortem human brains and found that individual neurons acquired somatic mutations (see the Perspective by Linnarsson). The mechanism of mutation involved gene transcription rather than DNA replication. Thus, postmitotic neurons would seem to be their own worst enemy: Genes used for neuronal function are the very genes put most at risk of somatic mutation. Science, this issue p. 94; see also p. 37 Human brains are built from intermingled clones of cells that carry mutations linked to their use of particular neuronal genes. [Also see Perspective by Linnarsson] Neurons live for decades in a postmitotic state, their genomes susceptible to DNA damage. Here we survey the landscape of somatic single-nucleotide variants (SNVs) in the human brain. We identified thousands of somatic SNVs by single-cell sequencing of 36 neurons from the cerebral cortex of three normal individuals. Unlike germline and cancer SNVs, which are often caused by errors in DNA replication, neuronal mutations appear to reflect damage during active transcription. Somatic mutations create nested lineage trees, allowing them to be dated relative to developmental landmarks and revealing a polyclonal architecture of the human cerebral cortex. Thus, somatic mutations in the brain represent a durable and ongoing record of neuronal life history, from development through postmitotic function.', 'year': 2015, 'citationCount': 463, 'authors': [{'authorId': '4760282', 'name': 'Michael A. Lodato'}, {'authorId': '4878598', 'name': 'Mollie B. Woodworth'}, {'authorId': '153310467', 'name': 'Semin Lee'}, {'authorId': '6021345', 'name': 'G. Evrony'}, {'authorId': '48781169', 'name': 'Bhaven K. Mehta'}, {'authorId': '39868617', 'name': 'Amir Karger'}, {'authorId': '2117180651', 'name': 'Soohyun Lee'}, {'authorId': '3075784', 'name': 'T. Chittenden'}, {'authorId': '1398498130', 'name': 'A. D’Gama'}, {'authorId': '34370609', 'name': 'Xuyu Cai'}, {'authorId': '2403971', 'name': 'Lovelace J. Luquette'}, {'authorId': '143790536', 'name': 'E. Lee'}, {'authorId': '2068953915', 'name': 'Peter J. Park'}, {'authorId': '2067645419', 'name': 'C. Walsh'}]}, {'paperId': '4eb6a2628db47351f2d71f21fb6b1520a0082fa3', 'title': 'U-Pb geochronology of the Deccan Traps and relation to the end-Cretaceous mass extinction', 'abstract': 'Dating the influence of Deccan Traps eruptions The Deccan Traps flood basalts in India represent over a million cubic kilometers of erupted lava. These massive eruptions occurred around the same time as the end-Cretaceous mass extinction some 65 million years ago, which famously wiped out all nonavian dinosaurs. Schoene et al. determined the precise timing and duration of the main phase of the eruptions, which lasted over 750,000 years and occurred just 250,000 years before the Cretaceous-Paleogene boundary. The relative contribution of these eruptions and of the Chicxulub impact in Mexico to the mass extinction remains unclear, but both provide potential kill mechanisms. Science, this issue p. 182 The main phase of the Deccan Traps eruption began 250,000 years before the end-Cretaceous extinction and lasted 750,000 years. The Chicxulub asteroid impact (Mexico) and the eruption of the massive Deccan volcanic province (India) are two proposed causes of the end-Cretaceous mass extinction, which includes the demise of nonavian dinosaurs. Despite widespread acceptance of the impact hypothesis, the lack of a high-resolution eruption timeline for the Deccan basalts has prevented full assessment of their relationship to the mass extinction. Here we apply uranium-lead (U-Pb) zircon geochronology to Deccan rocks and show that the main phase of eruptions initiated ~250,000 years before the Cretaceous-Paleogene boundary and that >1.1 million cubic kilometers of basalt erupted in ~750,000 years. Our results are consistent with the hypothesis that the Deccan Traps contributed to the latest Cretaceous environmental change and biologic turnover that culminated in the marine and terrestrial mass extinctions.', 'year': 2015, 'citationCount': 391, 'authors': [{'authorId': '4761805', 'name': 'B. Schoene'}, {'authorId': '7981222', 'name': 'K. Samperton'}, {'authorId': '6210862', 'name': 'M. Eddy'}, {'authorId': '35423404', 'name': 'G. Keller'}, {'authorId': '7769845', 'name': 'T. Adatte'}, {'authorId': '4518129', 'name': 'S. Bowring'}, {'authorId': '15059237', 'name': 'S. Khadri'}, {'authorId': '7453609', 'name': 'B. Gertsch'}]}, {'paperId': '385514ac7a1908821811b6b662cca262bd047c78', 'title': 'Correlated gene expression supports synchronous activity in brain networks', 'abstract': 'Cooperating brain regions express similar genes When the brain is at rest, a number of distinct areas are functionally connected. They tend to be organized in networks. Richiardi et al. compared brain imaging and gene expression data to build computational models of these networks. These functional networks are underpinned by the correlated expression of a core set of 161 genes. In this set, genes coding for ion channels and other synaptic functions such as neurotransmitter release dominate. Science, this issue p. 1241 Gene expression is more similar than expected by chance in brain regions that are functionally connected. During rest, brain activity is synchronized between different regions widely distributed throughout the brain, forming functional networks. However, the molecular mechanisms supporting functional connectivity remain undefined. We show that functional brain networks defined with resting-state functional magnetic resonance imaging can be recapitulated by using measures of correlated gene expression in a post mortem brain tissue data set. The set of 136 genes we identify is significantly enriched for ion channels. Polymorphisms in this set of genes significantly affect resting-state functional connectivity in a large sample of healthy adolescents. Expression levels of these genes are also significantly associated with axonal connectivity in the mouse. The results provide convergent, multimodal evidence that resting-state functional networks correlate with the orchestrated activity of dozens of genes linked to ion channel activity and synaptic function.', 'year': 2015, 'citationCount': 488, 'authors': [{'authorId': '1994765', 'name': 'J. Richiardi'}, {'authorId': '1712994', 'name': 'A. Altmann'}, {'authorId': '50670546', 'name': 'A. Milazzo'}, {'authorId': '2157872630', 'name': 'Catie Chang'}, {'authorId': '2130783982', 'name': 'M. Chakravarty'}, {'authorId': '2131952', 'name': 'T. Banaschewski'}, {'authorId': '2117363497', 'name': 'G. Barker'}, {'authorId': '1932550', 'name': 'A. Bokde'}, {'authorId': '2531464', 'name': 'U. Bromberg'}, {'authorId': '2076663752', 'name': 'C. Büchel'}, {'authorId': '2683022', 'name': 'P. Conrod'}, {'authorId': '1398728945', 'name': 'M. Fauth-Bühler'}, {'authorId': '2072211482', 'name': 'Herta Flor'}, {'authorId': '2470189', 'name': 'V. Frouin'}, {'authorId': '2234167987', 'name': 'Jurgen Gallinat'}, {'authorId': '2943065', 'name': 'H. Garavan'}, {'authorId': '46384297', 'name': 'P. Gowland'}, {'authorId': '2064205252', 'name': 'A. Heinz'}, {'authorId': '2181399', 'name': 'H. Lemaître'}, {'authorId': '2227123074', 'name': 'Karl Mann'}, {'authorId': '2451948', 'name': 'J. Martinot'}, {'authorId': '2700120', 'name': 'F. Nees'}, {'authorId': '2328876', 'name': 'T. Paus'}, {'authorId': '3211950', 'name': 'Z. Pausova'}, {'authorId': '3100404', 'name': 'M. Rietschel'}, {'authorId': '2114808096', 'name': 'T. Robbins'}, {'authorId': '2272108', 'name': 'M. Smolka'}, {'authorId': '3226472', 'name': 'R. Spanagel'}, {'authorId': '2416099', 'name': 'A. Ströhle'}, {'authorId': '1904764', 'name': 'G. Schumann'}, {'authorId': '46760867', 'name': 'M. Hawrylycz'}, {'authorId': '144063147', 'name': 'J. Poline'}, {'authorId': '3266708', 'name': 'M. Greicius'}]}, {'paperId': '4071aac026357483b3a371f8416c0bc5c7e69720', 'title': 'HIV-1 neutralizing antibodies induced by native-like envelope trimers', 'abstract': \"Steps in the right direction HIV-1 mutates rapidly, making it difficult to design a vaccine that will protect people against all of the virus' iterations. A potential successful vaccine design might protect by eliciting broadly neutralizing antibodies (bNAbs), which target specific regions on HIV-1's trimeric envelope glycoprotein (Env) (see the Perspective by Mascola). Jardine et al. used mice engineered to express germline-reverted heavy chains of a particular bNAb and immunized them with an Env-based immunogen designed to bind to precursors of that bNAb. Sanders et al. compared rabbits and monkeys immunized with Env trimers that adopt a nativelike conformation. In both cases, immunized animals produced antibodies that shared similarities with bNAbs. Boosting these animals with other immunogens may drive these antibodies to further mutate into the longsought bNAbs. Chen et al. report that retaining the cytoplasmic domain of Env proteins may be important to attract bNAbs. Removing the cytoplasmic domain may distract the immune response and instead generate antibodies that target epitopes on Env that would not lead to protection. Science, this issue p. 139, 10.1126/science.aac4223, p. 156; see also p. 191 Recombinant, native-like, HIV-1 envelope trimers induce neutralizing antibody responses in animal models. [Also see Perspective by Mascola] INTRODUCTION A major goal of HIV-1 vaccine development is to identify immunogens capable of inducing protective titers of broadly neutralizing antibodies (bNAbs) against circulating, neutralization-resistant (tier 2) viruses. The envelope glycoprotein (Env) trimer on the virus surface is the only bNAb target and accordingly serves as the basis for recombinant protein immunogens intended to induce bNAbs. We have engineered soluble, recombinant trimers based on the BG505 clade A and B41 clade B tier 2 viruses. These proteolytically cleaved and stabilized trimers, designated BG505 and B41 SOSIP.664 gp140, display multiple bNAb epitopes and have antigenic properties that mimic viral Env. High-resolution structures of the BG505 trimers reinforce this perspective. In contrast, other trimer designs (uncleaved gp140) that lack the SOSIP modifications and are based on eliminating the intersubunit proteolytic cleavage site adopt non-native configurations. RATIONALE To determine the NAb response elicited by native-like trimers, we immunized rabbits and macaques with BG505 SOSIP.664 gp140 proteins, and rabbits with their B41 counterparts. For comparison with the BG505 trimers, we also tested gp120 monomers and uncleaved gp140 proteins of the same genotype. We characterized the resulting antibody responses by quantifying their ability to neutralize the autologous tier 2 viruses and multiple heterologous tier 1 (neutralization-sensitive) and tier 2 viruses. We mapped the BG505 NAb responses using various techniques, including a large panel of Env-pseudotyped virus mutants. RESULTS BG505 SOSIP.664 trimers consistently induced potent NAbs against the autologous tier 2 virus in rabbits and similar but weaker responses in macaques. Cross-reactive NAbs against the more sensitive tier 1 viruses were also induced in all the immunized animals, but heterologous neutralization of tier 2 viruses was seen only weakly and sporadically. The rabbit responses to B41 SOSIP.664 trimers were qualitatively similar to those seen in the BG505 trimer recipients. The autologous tier 2 NAb titers to both SOSIP.664 trimers were greater than any such responses previously observed using various uncleaved gp140 proteins, including the BG505 version used in this study. The rabbit tier 2 and tier 1 NAb responses to the BG505 SOSIP.664 trimers were uncorrelated. Mapping studies showed the tier 2 NAbs recognized conformational epitopes that differed between animals. In some cases, these NAbs targeted components of the glycan shield previously associated with bNAb epitopes. In contrast, the tier 1 NAbs targeted linear epitopes in the gp120 V3 region. CONCLUSIONS Although an autologous tier 2 NAb response is not sufficient for vaccine protection against HIV-1, it may be a necessary step in various strategies intended to induce bNAbs. Thus, our findings indicate that native-like trimers represent a promising starting point for the further development of recombinant Env immunogens intended to broaden the NAb response. The high-resolution structures of SOSIP.664 trimers allow improvements to be rationally designed. Relevant strategies include the further stabilization of trimer-associated bNAb epitopes while reducing the antigenicity of V3 and other non-neutralizing epitopes that may be distractive; immunizing with longitudinal series of SOSIP.664 trimers from infected individuals who generate bNAbs; using trimer cocktails (e.g., from clades A, B, and C); and priming with trimer variants that trigger desirable germline responses before boosting with wild-type trimers. Autologous tier 2 NAb responses in animals immunized with native-like (SOSIP.664) trimers or non-native uncleaved gp140s. (Left) Representative reference-free 2D class averages of negative-stain electron microscopy (EM) images of the uncleaved BG505 gp140 and the native-like BG505 SOSIP.664 trimers. The illustrations (orange) of the two categories of Env protein are based on negative-stain EM. (Right) Autologous tier 2 serum NAb titers (IC50) in 50 animals immunized with various uncleaved gp140 proteins, compared to titers in the 30 rabbits given the BG505 or B41 SOSIP.664 trimers described here (see legend of Fig. 2 in the full article for details; P < 0.0001, two-tailed Mann-Whitney test). A challenge for HIV-1 immunogen design is the difficulty of inducing neutralizing antibodies (NAbs) against neutralization-resistant (tier 2) viruses that dominate human transmissions. We show that a soluble recombinant HIV-1 envelope glycoprotein trimer that adopts a native conformation, BG505 SOSIP.664, induced NAbs potently against the sequence-matched tier 2 virus in rabbits and similar but weaker responses in macaques. The trimer also consistently induced cross-reactive NAbs against more sensitive (tier 1) viruses. Tier 2 NAbs recognized conformational epitopes that differed between animals and in some cases overlapped with those recognized by broadly neutralizing antibodies (bNAbs), whereas tier 1 responses targeted linear V3 epitopes. A second trimer, B41 SOSIP.664, also induced a strong autologous tier 2 NAb response in rabbits. Thus, native-like trimers represent a promising starting point for the development of HIV-1 vaccines aimed at inducing bNAbs.\", 'year': 2015, 'citationCount': 473, 'authors': [{'authorId': '11253121', 'name': 'R. Sanders'}, {'authorId': '1753122', 'name': 'M. V. van Gils'}, {'authorId': '6913448', 'name': 'R. Derking'}, {'authorId': '4498862', 'name': 'D. Sok'}, {'authorId': '3532587', 'name': 'T. Ketas'}, {'authorId': '4224266', 'name': 'J. Burger'}, {'authorId': '5403524', 'name': 'G. Ozorowski'}, {'authorId': '3720967', 'name': 'A. Cupo'}, {'authorId': '5582265', 'name': 'Cassandra Simonich'}, {'authorId': '49823402', 'name': 'Leslie Goo'}, {'authorId': '40190515', 'name': 'H. Arendt'}, {'authorId': '2109106668', 'name': 'Helen J. Kim'}, {'authorId': '2155633982', 'name': 'Jeong Hyun Lee'}, {'authorId': '5996931', 'name': 'P. Pugach'}, {'authorId': '47638142', 'name': 'Melissa Williams'}, {'authorId': '8115982', 'name': 'Gargi Debnath'}, {'authorId': '4251219', 'name': 'B. Moldt'}, {'authorId': '114511784', 'name': 'M. V. van Breemen'}, {'authorId': '3550089', 'name': 'G. Işık'}, {'authorId': '1398898273', 'name': 'M. Medina-Ramírez'}, {'authorId': '6295820', 'name': 'J. Back'}, {'authorId': '4282690', 'name': 'W. Koff'}, {'authorId': '143931879', 'name': 'J. Julien'}, {'authorId': '4012471', 'name': 'E. Rakasz'}, {'authorId': '2785079', 'name': 'M. Seaman'}, {'authorId': '34071496', 'name': 'M. Guttman'}, {'authorId': '1574068297', 'name': 'Kelly K. Lee'}, {'authorId': '4064627', 'name': 'P. Klasse'}, {'authorId': '4625245', 'name': 'C. Labranche'}, {'authorId': '2417923', 'name': 'W. Schief'}, {'authorId': '2069848127', 'name': 'I. Wilson'}, {'authorId': '5093157', 'name': 'J. Overbaugh'}, {'authorId': '144482154', 'name': 'D. Burton'}, {'authorId': '144621454', 'name': 'A. Ward'}, {'authorId': '144524278', 'name': 'D. Montefiori'}, {'authorId': '34969108', 'name': 'H. Dean'}, {'authorId': '1405933166', 'name': 'John P. Moore'}]}, {'paperId': '4b66f21a5d0bb8618f8cbb0eae90110d1cc05c0e', 'title': 'Negative local resistance caused by viscous electron backflow in graphene', 'abstract': 'Electrons that flow like a fluid Electrons inside a conductor are often described as flowing in response to an electric field. This flow rarely resembles anything like the familiar flow of water through a pipe, but three groups describe counterexamples (see the Perspective by Zaanen). Moll et al. found that the viscosity of the electron fluid in thin wires of PdCoO2 had a major effect on the flow, much like what happens in regular fluids. Bandurin et al. found evidence in graphene of electron whirlpools similar to those formed by viscous fluid flowing through a small opening. Finally, Crossno et al. observed a huge increase of thermal transport in graphene, a signature of so-called Dirac fluids. Science, this issue p. 1061, 1055, 1058; see also p. 1026 Proximity transport measurements find evidence of whirlpools of graphene’s viscous electronic fluid. [Also see Perspective by Zaanen] Graphene hosts a unique electron system in which electron-phonon scattering is extremely weak but electron-electron collisions are sufficiently frequent to provide local equilibrium above the temperature of liquid nitrogen. Under these conditions, electrons can behave as a viscous liquid and exhibit hydrodynamic phenomena similar to classical liquids. Here we report strong evidence for this transport regime. We found that doped graphene exhibits an anomalous (negative) voltage drop near current-injection contacts, which is attributed to the formation of submicrometer-size whirlpools in the electron flow. The viscosity of graphene’s electron liquid is found to be ~0.1 square meters per second, an order of magnitude higher than that of honey, in agreement with many-body theory. Our work demonstrates the possibility of studying electron hydrodynamics using high-quality graphene.', 'year': 2015, 'citationCount': 496, 'authors': [{'authorId': '8368425', 'name': 'D. Bandurin'}, {'authorId': '41205715', 'name': 'I. Torre'}, {'authorId': '31912825', 'name': 'R. K. Kumar'}, {'authorId': '102321688', 'name': 'M. Shalom'}, {'authorId': '4434710', 'name': 'A. Tomadin'}, {'authorId': '3992470', 'name': 'A. Principi'}, {'authorId': '6194428', 'name': 'G. Auton'}, {'authorId': '4352409', 'name': 'E. Khestanova'}, {'authorId': '49912197', 'name': 'K. Novoselov'}, {'authorId': '5224749', 'name': 'I. Grigorieva'}, {'authorId': '1994032', 'name': 'L. Ponomarenko'}, {'authorId': '5288936', 'name': 'SUPARNA DUTTASINHA'}, {'authorId': '4943878', 'name': 'M. Polini'}]}, {'paperId': 'ff04a23ed941f8829db26e23db81efb6e99423f1', 'title': 'SARM1 activation triggers axon degeneration locally via NAD+ destruction', 'abstract': 'SARM1-driven axon degeneration Axons, the long protrusions of nerve cells, are programmed to self-destruct under certain conditions that occur during development, stress, or disease states. Gerdts et al. outline a biochemical mechanism that controls such axon degeneration. The authors designed versions of SARM1 (sterile alpha and TIR motif—constraining 1) that could be activated or inhibited in cells. Their experiments showed that the activation of SARM1 was necessary and sufficient to cause axon destruction in cultured mouse neurons. SARM1-mediated destruction was associated with depletion of the metabolic cofactor NAD+ from cells. Science, this issue p. 453 A critical role is elucidated for destruction of a metabolic cofactor in neuronal degeneration. Axon degeneration is an intrinsic self-destruction program that underlies axon loss during injury and disease. Sterile alpha and TIR motif–containing 1 (SARM1) protein is an essential mediator of axon degeneration. We report that SARM1 initiates a local destruction program involving rapid breakdown of nicotinamide adenine dinucleotide (NAD+) after injury. We used an engineered protease-sensitized SARM1 to demonstrate that SARM1 activity is required after axon injury to induce axon degeneration. Dimerization of the Toll–interleukin receptor (TIR) domain of SARM1 alone was sufficient to induce locally mediated axon degeneration. Formation of the SARM1 TIR dimer triggered rapid breakdown of NAD+, whereas SARM1-induced axon destruction could be counteracted by increased NAD+ synthesis. SARM1-induced depletion of NAD+ may explain the potent axon protection in Wallerian degeneration slow (Wlds) mutant mice.', 'year': 2015, 'citationCount': 410, 'authors': [{'authorId': '40641371', 'name': 'Josiah Gerdts'}, {'authorId': '3987255', 'name': 'E. Brace'}, {'authorId': '48346296', 'name': 'Y. Sasaki'}, {'authorId': '4204986', 'name': 'A. Diantonio'}, {'authorId': '2178644', 'name': 'J. Milbrandt'}]}, {'paperId': '2649ec70d571c681cb2677f242d8445dfda69f32', 'title': 'Early Homo at 2.8 Ma from Ledi-Geraru, Afar, Ethiopia', 'abstract': 'Finding Homo nearly 3 million years ago The fossil record of humans is notoriously patchy and incomplete. Even so, skeletal remains and artifacts unearthed in Africa in recent decades have done much to illuminate human evolution. But what is the origin of the genus Homo? Villmoare et al. found a fossil mandible and teeth from the Afar region in Ethiopia. The find extends the record of recognizable Homo by at least half a million years, to almost 2.8 million years ago. The morphological traits of the fossil align more closely with Homo than with any other hominid genus. DiMaggio et al. confirm the ancient date of the site and suggest that these early humans lived in a setting that was more open and arid than previously thought. Science, this issue p. 1352, p. 1355 A fossil mandible and teeth from Ethiopia extend the record of the genus Homo by almost 0.5 million years. Our understanding of the origin of the genus Homo has been hampered by a limited fossil record in eastern Africa between 2.0 and 3.0 million years ago (Ma). Here we report the discovery of a partial hominin mandible with teeth from the Ledi-Geraru research area, Afar Regional State, Ethiopia, that establishes the presence of Homo at 2.80 to 2.75 Ma. This specimen combines primitive traits seen in early Australopithecus with derived morphology observed in later Homo, confirming that dentognathic departures from the australopith pattern occurred early in the Homo lineage. The Ledi-Geraru discovery has implications for hypotheses about the timing and place of origin of the genus Homo.', 'year': 2015, 'citationCount': 304, 'authors': [{'authorId': '3856850', 'name': 'John R. Tumbleston'}, {'authorId': '2888642', 'name': 'D. Shirvanyants'}, {'authorId': '4737061', 'name': 'N. Ermoshkin'}, {'authorId': '5768526', 'name': 'R. Janusziewicz'}, {'authorId': '25711869', 'name': 'Ashley R. Johnson'}, {'authorId': '2070074460', 'name': 'David Kelly'}, {'authorId': '2205902206', 'name': 'Kai-Feng Chen'}, {'authorId': '69923437', 'name': 'R. Pinschmidt'}, {'authorId': '1850765', 'name': 'J. Rolland'}, {'authorId': '2661307', 'name': 'A. Ermoshkin'}, {'authorId': '49805613', 'name': 'E. Samulski'}, {'authorId': '6880755', 'name': 'J. Desimone'}]}, {'paperId': 'da7c38b5db28486f6027ea7a4f142eb742a3df70', 'title': 'Genomic evidence for the Pleistocene and recent population history of Native Americans', 'abstract': 'Genetic history of Native Americans Several theories have been put forth as to the origin and timing of when Native American ancestors entered the Americas. To clarify this controversy, Raghavan et al. examined the genomic variation among ancient and modern individuals from Asia and the Americas. There is no evidence for multiple waves of entry or recurrent gene flow with Asians in northern populations. The earliest migrations occurred no earlier than 23,000 years ago from Siberian ancestors. Amerindians and Athabascans originated from a single population, splitting approximately 13,000 years ago. Science, this issue 10.1126/science.aab3884 Genetic variation within ancient and extant Native American populations informs on their migration into the Americas. INTRODUCTION The consensus view on the peopling of the Americas is that ancestors of modern Native Americans entered the Americas from Siberia via the Bering Land Bridge and that this occurred at least ~14.6 thousand years ago (ka). However, the number and timing of migrations into the Americas remain controversial, with conflicting interpretations based on anatomical and genetic evidence. RATIONALE In this study, we address four major unresolved issues regarding the Pleistocene and recent population history of Native Americans: (i) the timing of their divergence from their ancestral group, (ii) the number of migrations into the Americas, (iii) whether there was ~15,000 years of isolation of ancestral Native Americans in Beringia (Beringian Incubation Model), and (iv) whether there was post-Pleistocene survival of relict populations in the Americas related to Australo-Melanesians, as suggested by apparent differences in cranial morphologies between some early (“Paleoamerican”) remains and those of more recent Native Americans. We generated 31 high-coverage modern genomes from the Americas, Siberia, and Oceania; 23 ancient genomic sequences from the Americas dating between ~0.2 and 6 ka; and SNP chip genotype data from 79 present-day individuals belonging to 28 populations from the Americas and Siberia. The above data sets were analyzed together with published modern and ancient genomic data from worldwide populations, after masking some present-day Native Americans for recent European admixture. RESULTS Using three different methods, we determined the divergence time for all Native Americans (Athabascans and Amerindians) from their Siberian ancestors to be ~20 ka, and no earlier than ~23 ka. Furthermore, we dated the divergence between Athabascans (northern Native American branch, together with northern North American Amerindians) and southern North Americans and South and Central Americans (southern Native American branch) to be ~13 ka. Similar divergence times from East Asian populations and a divergence time between the two branches that is close in age to the earliest well-established archaeological sites in the Americas suggest that the split between the branches occurred within the Americas. We additionally found that several sequenced Holocene individuals from the Americas are related to present-day populations from the same geographical regions, implying genetic continuity of ancient and modern populations in some parts of the Americas over at least the past 8500 years. Moreover, our results suggest that there has been gene flow between some Native Americans from both North and South America and groups related to East Asians and Australo-Melanesians, the latter possibly through an East Asian route that might have included ancestors of modern Aleutian Islanders. Last, using both genomic and morphometric analyses, we found that historical Native American groups such as the Pericúes and Fuego-Patagonians were not “relicts” of Paleoamericans, and hence, our results do not support an early migration of populations directly related to Australo-Melanesians into the Americas. CONCLUSION Our results provide an upper bound of ~23 ka on the initial divergence of ancestral Native Americans from their East Asian ancestors, followed by a short isolation period of no more than ~8000 years, and subsequent entrance and spread across the Americas. The data presented are consistent with a single-migration model for all Native Americans, with later gene flow from sources related to East Asians and, indirectly, Australo-Melanesians. The single wave diversified ~13 ka, likely within the Americas, giving rise to the northern and southern branches of present-day Native Americans. Population history of present-day Native Americans. The ancestors of all Native Americans entered the Americas as a single migration wave from Siberia (purple) no earlier than ~23 ka, separate from the Inuit (green), and diversified into “northern” and “southern” Native American branches ~13 ka. There is evidence of post-divergence gene flow between some Native Americans and groups related to East Asians/Inuit and Australo-Melanesians (yellow). How and when the Americas were populated remains contentious. Using ancient and modern genome-wide data, we found that the ancestors of all present-day Native Americans, including Athabascans and Amerindians, entered the Americas as a single migration wave from Siberia no earlier than 23 thousand years ago (ka) and after no more than an 8000-year isolation period in Beringia. After their arrival to the Americas, ancestral Native Americans diversified into two basal genetic branches around 13 ka, one that is now dispersed across North and South America and the other restricted to North America. Subsequent gene flow resulted in some Native Americans sharing ancestry with present-day East Asians (including Siberians) and, more distantly, Australo-Melanesians. Putative “Paleoamerican” relict populations, including the historical Mexican Pericúes and South American Fuego-Patagonians, are not directly related to modern Australo-Melanesians as suggested by the Paleoamerican Model.', 'year': 2015, 'citationCount': 443, 'authors': [{'authorId': '40300212', 'name': 'M. Raghavan'}, {'authorId': '3376543', 'name': 'Matthias Steinrücken'}, {'authorId': '31712306', 'name': 'Kelley Harris'}, {'authorId': '6020526', 'name': 'S. Schiffels'}, {'authorId': '49743174', 'name': 'S. Rasmussen'}, {'authorId': '2087800', 'name': 'Michael Degiorgio'}, {'authorId': '2628454', 'name': 'A. Albrechtsen'}, {'authorId': '2249014', 'name': 'Cristina E. Valdiosera'}, {'authorId': '1404276333', 'name': 'María C. Ávila-Arcos'}, {'authorId': '2491508', 'name': 'Anna-Sapfo Malaspinas'}, {'authorId': '144213877', 'name': 'A. Eriksson'}, {'authorId': '2318765', 'name': 'Ida Moltke'}, {'authorId': '5369838', 'name': 'M. Metspalu'}, {'authorId': '5091885', 'name': 'J. Homburger'}, {'authorId': '152921524', 'name': 'J. Wall'}, {'authorId': '2588473', 'name': 'Omar E. Cornejo'}, {'authorId': '1398610269', 'name': 'J. V. Moreno-Mayar'}, {'authorId': '2806324', 'name': 'T. Korneliussen'}, {'authorId': '4268632', 'name': 'T. Pierre'}, {'authorId': '153154198', 'name': 'M. Rasmussen'}, {'authorId': '34783432', 'name': 'P. Campos'}, {'authorId': '21107284', 'name': 'P. Damgaard'}, {'authorId': '3878367', 'name': 'M. Allentoft'}, {'authorId': '4990987', 'name': 'J. Lindo'}, {'authorId': '6160843', 'name': 'E. Metspalu'}, {'authorId': '1398719145', 'name': 'Ricardo Rodríguez-Varela'}, {'authorId': '145539929', 'name': 'J. Mansilla'}, {'authorId': '17599631', 'name': 'Celeste N. Henrickson'}, {'authorId': '1398045956', 'name': 'Andaine Seguin-Orlando'}, {'authorId': '5283621', 'name': 'H. Malmström'}, {'authorId': '32687097', 'name': 'T. Stafford'}, {'authorId': '1472926470', 'name': 'S. Shringarpure'}, {'authorId': '1398672485', 'name': 'A. Moreno-Estrada'}, {'authorId': '2038457', 'name': 'Monika Karmin'}, {'authorId': '5549881', 'name': 'K. Tambets'}, {'authorId': '47269529', 'name': 'Anders Bergström'}, {'authorId': '33496542', 'name': 'Yali Xue'}, {'authorId': '3584737', 'name': 'Vera M. Warmuth'}, {'authorId': '104308183', 'name': 'A. Friend'}, {'authorId': '4443483', 'name': 'J. Singarayer'}, {'authorId': '2948218', 'name': 'P. Valdes'}, {'authorId': '2056491', 'name': 'F. Balloux'}, {'authorId': '6924143', 'name': 'I. Leboreiro'}, {'authorId': '30012904', 'name': 'J. Vera'}, {'authorId': '1398119682', 'name': 'H. Rangel-Villalobos'}, {'authorId': '4674234', 'name': 'D. Pettener'}, {'authorId': '6904055', 'name': 'D. Luiselli'}, {'authorId': '104380079', 'name': 'L. Davis'}, {'authorId': '145027324', 'name': 'E. Heyer'}, {'authorId': '35251830', 'name': 'C. Zollikofer'}, {'authorId': '4389705', 'name': 'Marcia S. Ponce de León'}, {'authorId': '2135808573', 'name': 'Colin Smith'}, {'authorId': '4384505', 'name': 'V. Grimes'}, {'authorId': '46634137', 'name': 'Kelly A. Pike'}, {'authorId': '81782215', 'name': 'M. Deal'}, {'authorId': '47655297', 'name': 'B. Fuller'}, {'authorId': '50603710', 'name': 'B. Arriaza'}, {'authorId': '4607947', 'name': 'V. Standen'}, {'authorId': '120016142', 'name': 'Maria F. Luz'}, {'authorId': '47193911', 'name': 'F. Ricaut'}, {'authorId': '8307737', 'name': 'N. Guidon'}, {'authorId': '3529431', 'name': 'L. Osipova'}, {'authorId': '6297765', 'name': 'M. Voevoda'}, {'authorId': '5537046', 'name': 'O. Posukh'}, {'authorId': '144071337', 'name': 'O. Balanovsky'}, {'authorId': '3755286', 'name': 'M. Lavryashina'}, {'authorId': '4400741', 'name': 'Y. Bogunov'}, {'authorId': '145568845', 'name': 'E. Khusnutdinova'}, {'authorId': '144314334', 'name': 'M. Gubina'}, {'authorId': '3869128', 'name': 'E. Balanovska'}, {'authorId': '144749924', 'name': 'S. Fedorova'}, {'authorId': '144633425', 'name': 'S. Litvinov'}, {'authorId': '2640106', 'name': 'B. Malyarchuk'}, {'authorId': '6149225', 'name': 'M. Derenko'}, {'authorId': '34533721', 'name': 'M. Mosher'}, {'authorId': '2056167967', 'name': 'David R. Archer'}, {'authorId': '5639635', 'name': 'J. Cybulski'}, {'authorId': '4819572', 'name': 'Barbara Petzelt'}, {'authorId': '40215642', 'name': 'Joycelynn Mitchell'}, {'authorId': '6246512', 'name': 'Rosita Worl'}, {'authorId': '2754682', 'name': 'P. Norman'}, {'authorId': '31740540', 'name': 'P. Parham'}, {'authorId': '5324079', 'name': 'B. Kemp'}, {'authorId': '8019318', 'name': 'T. Kivisild'}, {'authorId': '1396409355', 'name': 'C. Tyler-Smith'}, {'authorId': '2664570', 'name': 'M. Sandhu'}, {'authorId': '2066467070', 'name': 'Michael Crawford'}, {'authorId': '6637782', 'name': 'R. Villems'}, {'authorId': '38129056', 'name': 'D. G. Smith'}, {'authorId': '47743436', 'name': 'M. Waters'}, {'authorId': '46860177', 'name': 'T. Goebel'}, {'authorId': '145368911', 'name': 'John R. Johnson'}, {'authorId': '4155430', 'name': 'R. Malhi'}, {'authorId': '2827710', 'name': 'M. Jakobsson'}, {'authorId': '47324013', 'name': 'D. Meltzer'}, {'authorId': '3277575', 'name': 'A. Manica'}, {'authorId': '144187514', 'name': 'R. Durbin'}, {'authorId': '33858917', 'name': 'C. Bustamante'}, {'authorId': '145229109', 'name': 'Yun S. Song'}, {'authorId': '144471919', 'name': 'R. Nielsen'}, {'authorId': '1812266', 'name': 'E. Willerslev'}]}, {'paperId': '4a0a3f003a75b908f1ae7933a0a0d8fce6c60969', 'title': '2.2 Å resolution cryo-EM structure of β-galactosidase in complex with a cell-permeant inhibitor', 'abstract': 'Pushing the limits of electron microscopy Recent advances in cryo–electron microscopy (cryo-EM) allow structures of large macromolecules to be determined at near-atomic resolution. So far, though, resolutions approaching 2 Å, where features key to drug design are revealed, remain the province of x-ray crystallography. Bartesaghi et al. achieved a resolution of 2.2 Å for a 465-kD ligand-bound protein complex using cryo-EM. The density map is detailed enough to show close to 800 water molecules, magnesium and sodium ions, and precise side-chain conformations. These results bring routine use of cryo-EM in rational drug design a step closer. Science, this issue p. 1147 Advances in electron microscopy allow protein structure determination at resolutions useful in drug discovery. Cryo–electron microscopy (cryo-EM) is rapidly emerging as a powerful tool for protein structure determination at high resolution. Here we report the structure of a complex between Escherichia coli β-galactosidase and the cell-permeant inhibitor phenylethyl β-d-thiogalactopyranoside (PETG), determined by cryo-EM at an average resolution of ~2.2 angstroms (Å). Besides the PETG ligand, we identified densities in the map for ~800 water molecules and for magnesium and sodium ions. Although it is likely that continued advances in detector technology may further enhance resolution, our findings demonstrate that preparation of specimens of adequate quality and intrinsic protein flexibility, rather than imaging or image-processing technologies, now represent the major bottlenecks to routinely achieving resolutions close to 2 Å using single-particle cryo-EM.', 'year': 2015, 'citationCount': 439, 'authors': [{'authorId': '1920144', 'name': 'A. Bartesaghi'}, {'authorId': '6278602', 'name': 'A. Merk'}, {'authorId': '153564608', 'name': 'Soojay Banerjee'}, {'authorId': '31544014', 'name': 'D. Matthies'}, {'authorId': '2944941', 'name': 'Xiongwu Wu'}, {'authorId': '48705866', 'name': 'J. Milne'}, {'authorId': '145690187', 'name': 'S. Subramaniam'}]}, {'paperId': 'bc4ff07b48d8033a1a00e6c33753dcf90cde9ea2', 'title': 'Gravitational waves from binary supermassive black holes missing in pulsar observations', 'abstract': 'Placing bounds on gravitational wave detection Gravitational waves are expected to be generated by the interaction of the massive bodies in black-hole binary systems. As gravitational waves distort spacetime, it should be possible to verify their existence as they interfere with the pulses emitted by millisecond pulsars. However, after monitoring 24 pulsars with the Parkes radio telescope for 12 years, Shannon et al. found no detectable variation in pulsar records. This nondetection result indicates that a new detection strategy for gravitational waves is needed. Science, this issue p. 1522 A lack of observed variations in the timing of pulsars places constraints on the detection of gravitational waves. Gravitational waves are expected to be radiated by supermassive black hole binaries formed during galaxy mergers. A stochastic superposition of gravitational waves from all such binary systems would modulate the arrival times of pulses from radio pulsars. Using observations of millisecond pulsars obtained with the Parkes radio telescope, we constrained the characteristic amplitude of this background, Ac,yr, to be <1.0 × 10−15 with 95% confidence. This limit excludes predicted ranges for Ac,yr from current models with 91 to 99.7% probability. We conclude that binary evolution is either stalled or dramatically accelerated by galactic-center environments and that higher-cadence and shorter-wavelength observations would be more sensitive to gravitational waves.', 'year': 2015, 'citationCount': 358, 'authors': [{'authorId': '49037918', 'name': 'R. Shannon'}, {'authorId': '2089367767', 'name': 'Ravi'}, {'authorId': '5872642', 'name': 'L. Lentati'}, {'authorId': '1381303595', 'name': 'P. Lasky'}, {'authorId': '145913658', 'name': 'G. Hobbs'}, {'authorId': '39343524', 'name': 'M. Kerr'}, {'authorId': '4701856', 'name': 'R. Manchester'}, {'authorId': '4919284', 'name': 'W. Coles'}, {'authorId': '123786554', 'name': 'Y. Levin'}, {'authorId': '5903194', 'name': 'M. Bailes'}, {'authorId': '2229714046', 'name': 'Ndr Bhat'}, {'authorId': '1388067048', 'name': 'S. Burke-Spolaor'}, {'authorId': '2087720656', 'name': 'Shi Dai'}, {'authorId': '49918437', 'name': 'M. Keith'}, {'authorId': '97646607', 'name': 'S. Osłowski'}, {'authorId': '47314221', 'name': 'D. Reardon'}, {'authorId': '2229712953', 'name': 'W. V. Strateni'}, {'authorId': '39700583', 'name': 'L. Toomey'}, {'authorId': '97773615', 'name': 'J. Wang'}, {'authorId': '144731348', 'name': 'L. Wen'}, {'authorId': '93456160', 'name': 'J. Wyithe'}, {'authorId': '6248124', 'name': 'Xiaoshuai Zhu'}]}, {'paperId': 'a2eb1a77dd5233286045e8bafeffbc6d158f15cb', 'title': 'Two-pore channels control Ebola virus host cell entry and are drug targets for disease treatment', 'abstract': \"Channeling Ebola virus entry into the cell The current outbreak of Ebola virus in West Africa highlights the need for antiviral therapies. One strategy would be to block the Ebola virus's ability to enter host cells. Cells engulf Ebola virus particles, which then traffic into the cell in structures called endosomes. Sakurai et al. now report that the Ebola virus requires calcium channels called two-pore channels (TPCs) in endosomal membranes for successful entry (see the Perspective by Falzarano and Feldmann). The Ebola virus could not enter cells lacking TPCs or cells treated with a TPC inhibitor. Blocking TPCs therapeutically allowed 50% of mice to survive an ordinarily lethal Ebola virus infection. Science, this issue p. 995; see also p. 947 Ebola virus requires endosomal two-pore calcium channels to enter host cells. [Also see Perspective by Falzarano and Feldmann] Ebola virus causes sporadic outbreaks of lethal hemorrhagic fever in humans, but there is no currently approved therapy. Cells take up Ebola virus by macropinocytosis, followed by trafficking through endosomal vesicles. However, few factors controlling endosomal virus movement are known. Here we find that Ebola virus entry into host cells requires the endosomal calcium channels called two-pore channels (TPCs). Disrupting TPC function by gene knockout, small interfering RNAs, or small-molecule inhibitors halted virus trafficking and prevented infection. Tetrandrine, the most potent small molecule that we tested, inhibited infection of human macrophages, the primary target of Ebola virus in vivo, and also showed therapeutic efficacy in mice. Therefore, TPC proteins play a key role in Ebola virus infection and may be effective targets for antiviral therapy.\", 'year': 2015, 'citationCount': 409, 'authors': [{'authorId': '6471111', 'name': 'Y. Sakurai'}, {'authorId': '4367895', 'name': 'A. Kolokoltsov'}, {'authorId': '48239565', 'name': 'Cheng-Chang Chen'}, {'authorId': '46319361', 'name': 'Michael W Tidwell'}, {'authorId': '8610145', 'name': 'William Bauta'}, {'authorId': '4158787', 'name': 'N. Klugbauer'}, {'authorId': '2075378825', 'name': 'C. Grimm'}, {'authorId': '1398097274', 'name': 'C. Wahl-Schott'}, {'authorId': '144893002', 'name': 'M. Biel'}, {'authorId': '144524415', 'name': 'R. Davey'}]}, {'paperId': '16db92f38e2c33f5902d30a233d57e12bce69bda', 'title': 'The structure of the dynactin complex and its interaction with dynein', 'abstract': 'Making a molecular motor fit for purpose Dynactin is an essential cofactor of the microtubule motor, cytoplasmic dynein. Dynactin contains 23 subunits built around a short filament of an actin-related protein (Arp1). How dynactin is assembled, how it functions with dynein, and why it is built around an actin-like filament is unclear. Urnavicius et al. combined cryo–electron microscopy structural studies and a crystal structure to determine the three-dimensional architecture of dynactin and how it interacts with dynein. Science, this issue p. 1441 Cryo–electron microscopy structures reveal how the 23-subunit dynactin complex is assembled and binds to the motor protein cytoplasmic dynein-1. Dynactin is an essential cofactor for the microtubule motor cytoplasmic dynein-1. We report the structure of the 23-subunit dynactin complex by cryo-electron microscopy to 4.0 angstroms. Our reconstruction reveals how dynactin is built around a filament containing eight copies of the actin-related protein Arp1 and one of β-actin. The filament is capped at each end by distinct protein complexes, and its length is defined by elongated peptides that emerge from the α-helical shoulder domain. A further 8.2 angstrom structure of the complex between dynein, dynactin, and the motility-inducing cargo adaptor Bicaudal-D2 shows how the translational symmetry of the dynein tail matches that of the dynactin filament. The Bicaudal-D2 coiled coil runs between dynein and dynactin to stabilize the mutually dependent interactions between all three components.', 'year': 2015, 'citationCount': 344, 'authors': [{'authorId': '5205544', 'name': 'L. Urnavicius'}, {'authorId': '2158519970', 'name': 'Kai Zhang'}, {'authorId': '3639450', 'name': 'A. Diamant'}, {'authorId': '46504814', 'name': 'C. Motz'}, {'authorId': '6344076', 'name': 'Max A. Schlager'}, {'authorId': '47730624', 'name': 'Minmin Yu'}, {'authorId': '91998801', 'name': 'Nisha A. Patel'}, {'authorId': '144199447', 'name': 'C. Robinson'}, {'authorId': '31454155', 'name': 'A. Carter'}]}, {'paperId': '05fdf5bd17f6bebf0e478311d2874ef8012c47d0', 'title': 'On the nucleus structure and activity of comet 67P/Churyumov-Gerasimenko', 'abstract': 'Images from the OSIRIS scientific imaging system onboard Rosetta show that the nucleus of 67P/Churyumov-Gerasimenko consists of two lobes connected by a short neck. The nucleus has a bulk density less than half that of water. Activity at a distance from the Sun of >3 astronomical units is predominantly from the neck, where jets have been seen consistently. The nucleus rotates about the principal axis of momentum. The surface morphology suggests that the removal of larger volumes of material, possibly via explosive release of subsurface pressure or via creation of overhangs by sublimation, may be a major mass loss process. The shape raises the question of whether the two lobes represent a contact binary formed 4.5 billion years ago, or a single body where a gap has evolved via mass loss.', 'year': 2015, 'citationCount': 386, 'authors': [{'authorId': '50246256', 'name': 'H. Sierks'}, {'authorId': '39419736', 'name': 'C. Barbieri'}, {'authorId': '50403997', 'name': 'P. Lamy'}, {'authorId': '91031514', 'name': 'R. Rodrigo'}, {'authorId': '69029744', 'name': 'D. Koschny'}, {'authorId': '6973900', 'name': 'H. Rickman'}, {'authorId': '144446632', 'name': 'H. Keller'}, {'authorId': '145969939', 'name': 'J. Agarwal'}, {'authorId': '1398024389', 'name': 'M. A’Hearn'}, {'authorId': '48286249', 'name': 'F. Angrilli'}, {'authorId': '102857838', 'name': 'A. Auger'}, {'authorId': '2070612652', 'name': 'M. Barucci'}, {'authorId': '102414410', 'name': 'J. Bertaux'}, {'authorId': '47679133', 'name': 'I. Bertini'}, {'authorId': '47071682', 'name': 'S. Besse'}, {'authorId': '6691955', 'name': 'D. Bodewits'}, {'authorId': '2721300', 'name': 'C. Capanna'}, {'authorId': '3939528', 'name': 'G. Cremonese'}, {'authorId': '2896324', 'name': 'V. Da Deppo'}, {'authorId': '3984918', 'name': 'B. Davidsson'}, {'authorId': '2317823', 'name': 'S. Debei'}, {'authorId': '50129229', 'name': 'M. De Cecco'}, {'authorId': '46341917', 'name': 'F. Ferri'}, {'authorId': '2564082', 'name': 'S. Fornasier'}, {'authorId': '14100462', 'name': 'M. Fulle'}, {'authorId': '4755964', 'name': 'R. Gaskell'}, {'authorId': '2215595', 'name': 'L. Giacomini'}, {'authorId': '3155836', 'name': 'O. Groussin'}, {'authorId': '1403005524', 'name': 'P. Gutierrez-Marques'}, {'authorId': '49672000', 'name': 'P. Gutiérrez'}, {'authorId': '3354765', 'name': 'C. Güttler'}, {'authorId': '4257859', 'name': 'N. Hoekzema'}, {'authorId': '5303188', 'name': 'S. Hviid'}, {'authorId': '145155207', 'name': 'W. Ip'}, {'authorId': '3240472', 'name': 'L. Jorda'}, {'authorId': '102231574', 'name': 'J. Knollenberg'}, {'authorId': '134014421', 'name': 'G. Kovacs'}, {'authorId': '143828408', 'name': 'J. Kramm'}, {'authorId': '2250203', 'name': 'E. Kührt'}, {'authorId': '48974673', 'name': 'M. Küppers'}, {'authorId': '133961675', 'name': 'F. La Forgia'}, {'authorId': '145621605', 'name': 'L. Lara'}, {'authorId': '144479967', 'name': 'M. Lazzarin'}, {'authorId': '5077738', 'name': 'C. Leyrat'}, {'authorId': '114954861', 'name': 'J. Lopez Moreno'}, {'authorId': '6016807', 'name': 'S. Magrin'}, {'authorId': '50786404', 'name': 'S. Marchi'}, {'authorId': '144372011', 'name': 'F. Marzari'}, {'authorId': '2761573', 'name': 'M. Massironi'}, {'authorId': '144921681', 'name': 'H. Michalik'}, {'authorId': '5261364', 'name': 'R. Moissl'}, {'authorId': '144381122', 'name': 'S. Mottola'}, {'authorId': '2704601', 'name': 'G. Naletto'}, {'authorId': '5300286', 'name': 'N. Oklay'}, {'authorId': '1398024053', 'name': 'M. Pajola'}, {'authorId': '35197843', 'name': 'M. Pertile'}, {'authorId': '4614024', 'name': 'F. Preusker'}, {'authorId': '143930936', 'name': 'L. Sabau'}, {'authorId': '143715006', 'name': 'F. Scholten'}, {'authorId': '2155760', 'name': 'C. Snodgrass'}, {'authorId': '144570575', 'name': 'N. Thomas'}, {'authorId': '5653933', 'name': 'C. Tubiana'}, {'authorId': '12647806', 'name': 'J. Vincent'}, {'authorId': '114386622', 'name': 'K. Wenzel'}, {'authorId': '2592622', 'name': 'M. Zaccariotto'}, {'authorId': '2632184', 'name': 'M. Pätzold'}]}, {'paperId': 'a62607e79840a2ca96508794944b864beb9a166d', 'title': 'Adaptive synergy between catechol and lysine promotes wet adhesion by surface salt displacement', 'abstract': 'Keeping it sticky when wet Some biological molecules are remarkably sticky, even to surfaces submerged in water. Mussel adhesion, for example, is based on the overproduction of dihydroxyphenylalanine (DOPA) and proteins with a high abundance of cationic amine residues such as lysine. Using bacterial iron chelators consisting of paired DOPA and lysine groups as analogs for the mussel proteins, Maier et al. show that these two functional groups synergistically enhance interfacial adhesion (see the Perspective by Wilker). The lysine appears to displace hydrated cations from the surface, thus giving a dry patch for better adhesion. Science, this issue p. 628; see also p. 582 Biologically inspired adhesives reveal a synergistic interplay that enables wet adhesion to mineral surfaces. [Also see Perspective by Wilker] In physiological fluids and seawater, adhesion of synthetic polymers to solid surfaces is severely limited by high salt, pH, and hydration, yet these conditions have not deterred the evolution of effective adhesion by mussels. Mussel foot proteins provide insights about adhesive adaptations: Notably, the abundance and proximity of catecholic Dopa (3,4-dihydroxyphenylalanine) and lysine residues hint at a synergistic interplay in adhesion. Certain siderophores—bacterial iron chelators—consist of paired catechol and lysine functionalities, thereby providing a convenient experimental platform to explore molecular synergies in bioadhesion. These siderophores and synthetic analogs exhibit robust adhesion energies (Ead ≥−15 millijoules per square meter) to mica in saline pH 3.5 to 7.5 and resist oxidation. The adjacent catechol-lysine placement provides a “one-two punch,” whereby lysine evicts hydrated cations from the mineral surface, allowing catechol binding to underlying oxides.', 'year': 2015, 'citationCount': 485, 'authors': [{'authorId': '7560283', 'name': 'G. Maier'}, {'authorId': '38478808', 'name': 'Michael V. Rapp'}, {'authorId': '143955060', 'name': 'J. Waite'}, {'authorId': '6637150', 'name': 'J. Israelachvili'}, {'authorId': '144237978', 'name': 'A. Butler'}]}, {'paperId': '5f1febdd44de7d6819f180fe1e5c02a5d58dcb2e', 'title': 'Quantum squeezing of motion in a mechanical resonator', 'abstract': 'Manipulation of a quantum squeeze The uncertainty principle of quantum mechanics dictates that even when a system is cooled to its ground state, there are still fluctuations. This zero-point motion is unavoidable but can be manipulated. Wollman et al. demonstrate such manipulation with the motion of a micrometer-sized mechanical system. By driving up the fluctuations in one of the variables of the system, they are able to squeeze the other related variable below the expected zero-point limit. Quantum squeezing will be important for realizing ultrasensitive sensors and detectors. Science, this issue p. 952 The fluctuating motion of a mechanical system can be squeezed below the zero-point limit. According to quantum mechanics, a harmonic oscillator can never be completely at rest. Even in the ground state, its position will always have fluctuations, called the zero-point motion. Although the zero-point fluctuations are unavoidable, they can be manipulated. Using microwave frequency radiation pressure, we have manipulated the thermal fluctuations of a micrometer-scale mechanical resonator to produce a stationary quadrature-squeezed state with a minimum variance of 0.80 times that of the ground state. We also performed phase-sensitive, back-action evading measurements of a thermal state squeezed to 1.09 times the zero-point level. Our results are relevant to the quantum engineering of states of matter at large length scales, the study of decoherence of large quantum systems, and for the realization of ultrasensitive sensing of force and motion.', 'year': 2015, 'citationCount': 494, 'authors': [{'authorId': '40638497', 'name': 'E. Wollman'}, {'authorId': '39769515', 'name': 'C. Lei'}, {'authorId': '11079370', 'name': 'A. Weinstein'}, {'authorId': '2091526', 'name': 'J. Suh'}, {'authorId': '4040239', 'name': 'Andreas Kronwald'}, {'authorId': '48057862', 'name': 'F. Marquardt'}, {'authorId': '38315299', 'name': 'A. Clerk'}, {'authorId': '145384249', 'name': 'K. Schwab'}]}, {'paperId': '04fa534903823fbe7c63c3c4e07402ac2f03d767', 'title': 'Discovery and spectroscopy of the young jovian planet 51 Eri b with the Gemini Planet Imager', 'abstract': 'An exoplanet extracted from the bright Direct imaging of Jupiter-like exoplanets around young stars provides a glimpse into how our solar system formed. The brightness of young stars requires the use of next-generation devices such as the Gemini Planet Imager (GPI). Using the GPI, Macintosh et al. discovered a Jupiter-like planet orbiting a young star, 51 Eridani (see the Perspective by Mawet). The planet, 51 Eri b, has a methane signature and is probably the smallest exoplanet that has been directly imaged. These findings open the door to understanding solar system origins and herald the dawn of a new era in next-generation planetary imaging. Science, this issue p. 64; see also p. 39 The Gemini Planet Imager detects a Jupiter-like exoplanet orbiting the young star 51 Eridani. [Also see Perspective by Mawet] Directly detecting thermal emission from young extrasolar planets allows measurement of their atmospheric compositions and luminosities, which are influenced by their formation mechanisms. Using the Gemini Planet Imager, we discovered a planet orbiting the ~20-million-year-old star 51 Eridani at a projected separation of 13 astronomical units. Near-infrared observations show a spectrum with strong methane and water-vapor absorption. Modeling of the spectra and photometry yields a luminosity (normalized by the luminosity of the Sun) of 1.6 to 4.0 × 10−6 and an effective temperature of 600 to 750 kelvin. For this age and luminosity, “hot-start” formation models indicate a mass twice that of Jupiter. This planet also has a sufficiently low luminosity to be consistent with the “cold-start” core-accretion process that may have formed Jupiter.', 'year': 2015, 'citationCount': 419, 'authors': [{'authorId': '34346058', 'name': 'B. Macintosh'}, {'authorId': '34346058', 'name': 'B. Macintosh'}, {'authorId': '47581492', 'name': 'J. Graham'}, {'authorId': '32298861', 'name': 'T. Barman'}, {'authorId': '51056549', 'name': 'R. D. Rosa'}, {'authorId': '3510736', 'name': 'Q. Konopacky'}, {'authorId': '143790271', 'name': 'M. Marley'}, {'authorId': '5818645', 'name': 'C. Marois'}, {'authorId': '5818645', 'name': 'C. Marois'}, {'authorId': '144710553', 'name': 'E. Nielsen'}, {'authorId': '48643723', 'name': 'L. Pueyo'}, {'authorId': '145203539', 'name': 'A. Rajan'}, {'authorId': '21741189', 'name': 'J. Rameau'}, {'authorId': '5297870', 'name': 'D. Saumon'}, {'authorId': '49605872', 'name': 'Jason J. Wang'}, {'authorId': '49036366', 'name': 'J. Patience'}, {'authorId': '100858086', 'name': 'M. Ammons'}, {'authorId': '49782357', 'name': 'P. Arriaga'}, {'authorId': '5709961', 'name': 'É. Artigau'}, {'authorId': '144615444', 'name': 'S. Beckwith'}, {'authorId': '1387963762', 'name': 'J. Brewster'}, {'authorId': '46189399', 'name': 'S. Bruzzone'}, {'authorId': '41130114', 'name': 'J. Bulger'}, {'authorId': '41130114', 'name': 'J. Bulger'}, {'authorId': '145248197', 'name': 'B. Burningham'}, {'authorId': '145248197', 'name': 'B. Burningham'}, {'authorId': '144174723', 'name': 'A. Burrows'}, {'authorId': '48240220', 'name': 'Christine H. Chen'}, {'authorId': '3423834', 'name': 'E. Chiang'}, {'authorId': '5530835', 'name': 'J. Chilcote'}, {'authorId': '144597346', 'name': 'R. Dawson'}, {'authorId': '1710497627', 'name': 'Ruobing Dong'}, {'authorId': '48813871', 'name': 'R. Doyon'}, {'authorId': '46229682', 'name': 'Z. Draper'}, {'authorId': '46229682', 'name': 'Z. Draper'}, {'authorId': '49002627', 'name': 'G. Duchêne'}, {'authorId': '49002627', 'name': 'G. Duchêne'}, {'authorId': '49584561', 'name': 'T. Esposito'}, {'authorId': '6032941', 'name': 'D. Fabrycky'}, {'authorId': '2971243', 'name': 'M. Fitzgerald'}, {'authorId': '77166014', 'name': 'K. Follette'}, {'authorId': '2568738', 'name': 'J. Fortney'}, {'authorId': '50520275', 'name': 'B. Gerard'}, {'authorId': '50520275', 'name': 'B. Gerard'}, {'authorId': '3702656', 'name': 'S. Goodsell'}, {'authorId': '38989004', 'name': 'A. Greenbaum'}, {'authorId': '38989004', 'name': 'A. Greenbaum'}, {'authorId': '6482776', 'name': 'P. Hibon'}, {'authorId': '3297911', 'name': 'S. Hinkley'}, {'authorId': '4375416', 'name': 'T. Cotten'}, {'authorId': '145466741', 'name': 'L. Hung'}, {'authorId': '7195935', 'name': 'P. Ingraham'}, {'authorId': '1403781545', 'name': 'Mara Johnson-Groh'}, {'authorId': '1403781545', 'name': 'Mara Johnson-Groh'}, {'authorId': '5189125', 'name': 'P. Kalas'}, {'authorId': '32630686', 'name': 'D. Lafreniére'}, {'authorId': '38566478', 'name': 'J. Larkin'}, {'authorId': '2108937331', 'name': 'Jung-eun Lee'}, {'authorId': '3067786', 'name': 'M. Line'}, {'authorId': '96525405', 'name': 'D. Long'}, {'authorId': '51073120', 'name': 'J. Maire'}, {'authorId': '3771660', 'name': 'F. Marchis'}, {'authorId': '48140967', 'name': 'B. Matthews'}, {'authorId': '48140967', 'name': 'B. Matthews'}, {'authorId': '6825250', 'name': 'C. Max'}, {'authorId': '144630381', 'name': 'S. Metchev'}, {'authorId': '144630381', 'name': 'S. Metchev'}, {'authorId': '1380251886', 'name': 'M. Millar-Blanchaer'}, {'authorId': '46818187', 'name': 'T. Mittal'}, {'authorId': '46205305', 'name': 'C. Morley'}, {'authorId': '5432844', 'name': 'K. Morzinski'}, {'authorId': '1390047812', 'name': 'R. Murray-Clay'}, {'authorId': '2337669', 'name': 'R. Oppenheimer'}, {'authorId': '39555776', 'name': 'D. Palmer'}, {'authorId': '49833435', 'name': 'R. Patel'}, {'authorId': '5095283', 'name': 'M. Perrin'}, {'authorId': '5335319', 'name': 'L. Poyneer'}, {'authorId': '46187849', 'name': 'R. Rafikov'}, {'authorId': '3822852', 'name': 'F. Rantakyrö'}, {'authorId': '3483952', 'name': 'E. Rice'}, {'authorId': '3483952', 'name': 'E. Rice'}, {'authorId': '40865906', 'name': 'P. Rojo'}, {'authorId': '41222111', 'name': 'A. Rudy'}, {'authorId': '97341200', 'name': 'J. Ruffio'}, {'authorId': '145937717', 'name': 'M. Ruiz'}, {'authorId': '3503160', 'name': 'N. Sadakuni'}, {'authorId': '6058233', 'name': 'L. Saddlemyer'}, {'authorId': '51373537', 'name': 'M. Salama'}, {'authorId': '46834135', 'name': 'D. Savransky'}, {'authorId': '14819041', 'name': 'A. Schneider'}, {'authorId': '3004825', 'name': 'A. Sivaramakrishnan'}, {'authorId': '4965490', 'name': 'I. Song'}, {'authorId': '3392617', 'name': 'R. Soummer'}, {'authorId': '39334803', 'name': 'S. Thomas'}, {'authorId': '4255387', 'name': 'G. Vasisht'}, {'authorId': '145535282', 'name': 'J. Wallace'}, {'authorId': '1380251779', 'name': 'K. Ward-Duong'}, {'authorId': '5131976', 'name': 'S. Wiktorowicz'}, {'authorId': '32886400', 'name': 'S. Wolff'}, {'authorId': '32886400', 'name': 'S. Wolff'}, {'authorId': '47522098', 'name': 'B. Zuckerman'}]}, {'paperId': '056c4b46e774c75351b9c53880ec31510644d67e', 'title': 'A stable trimeric influenza hemagglutinin stem as a broadly protective immunogen', 'abstract': 'Flu vaccine candidate STEMs the tide Every year we need a new flu vaccine, because influenza virus constantly mutates the major target of antibodies to flu: the “head” region of the viral hemagglutinin (HA) protein. Avoiding the problem of mutation requires a vaccine that elicits antibodies against the more conserved “stem” region of HA. During infection, antibodies are occasionally produced that recognize the stem and that neutralize a broad range of influenza virus strains. Impagliazzo et al. engineered an HA stem–only vaccine candidate that elicited broadly neutralizing antibodies in mice and nonhuman primates and that protected mice against multiple influenza strains. Science, this issue p. 1301 An engineered stem-only hemagglutinin vaccine candidate provides broad protection in an animal model of influenza infection. The identification of human broadly neutralizing antibodies (bnAbs) targeting the hemagglutinin (HA) stem revitalized hopes of developing a universal influenza vaccine. Using a rational design and library approach, we engineered stable HA stem antigens (“mini-HAs”) based on an H1 subtype sequence. Our most advanced candidate exhibits structural and bnAb binding properties comparable to those of full-length HA, completely protects mice in lethal heterologous and heterosubtypic challenge models, and reduces fever after sublethal challenge in cynomolgus monkeys. Antibodies elicited by this mini-HA in mice and nonhuman primates bound a wide range of HAs, competed with human bnAbs for HA stem binding, neutralized H5N1 viruses, and mediated antibody-dependent effector activity. These results represent a proof of concept for the design of HA stem mimics that elicit bnAbs against influenza A group 1 viruses.', 'year': 2015, 'citationCount': 448, 'authors': [{'authorId': '6029922', 'name': 'A. Impagliazzo'}, {'authorId': '5223281', 'name': 'F. Milder'}, {'authorId': '6919469', 'name': 'H. Kuipers'}, {'authorId': '20651034', 'name': 'M. Wagner'}, {'authorId': '2303500', 'name': 'Xueyong Zhu'}, {'authorId': '33864211', 'name': 'Ryan M. B. Hoffman'}, {'authorId': '1768780382', 'name': 'Ruud van Meersbergen'}, {'authorId': '7656722', 'name': 'Jeroen Huizingh'}, {'authorId': '47810501', 'name': 'P. Wanningen'}, {'authorId': '8383421', 'name': 'Johan W. A. Verspuij'}, {'authorId': '2059170515', 'name': 'M. de Man'}, {'authorId': '3650958', 'name': 'Zhaoqing Ding'}, {'authorId': '47618166', 'name': 'Adrian Apetri'}, {'authorId': '5864616', 'name': 'Başak Kükrer'}, {'authorId': '1398950001', 'name': 'Eveline Sneekes-Vriese'}, {'authorId': '50309292', 'name': 'Danuta Tomkiewicz'}, {'authorId': '8558947', 'name': 'N. S. Laursen'}, {'authorId': '1406042252', 'name': 'Peter S. Lee'}, {'authorId': '46278598', 'name': 'A. Zakrzewska'}, {'authorId': '6556142', 'name': 'Liesbeth Dekking'}, {'authorId': '3193831', 'name': 'J. Tolboom'}, {'authorId': '7424762', 'name': 'Lisanne Tettero'}, {'authorId': '1768782361', 'name': 'Sander van Meerten'}, {'authorId': '7985846', 'name': 'Wenli Yu'}, {'authorId': '4754966', 'name': 'W. Koudstaal'}, {'authorId': '145531112', 'name': 'J. Goudsmit'}, {'authorId': '144621454', 'name': 'A. Ward'}, {'authorId': '6759290', 'name': 'W. Meijberg'}, {'authorId': '143791278', 'name': 'I. Wilson'}, {'authorId': '33475599', 'name': 'K. Radošević'}]}, {'paperId': 'c0abb175c6c78964ac056acd699722206183c251', 'title': 'Alternative lengthening of telomeres renders cancer cells hypersensitive to ATR inhibitors.', 'abstract': 'Cancer cells rely on telomerase or the alternative lengthening of telomeres (ALT) pathway to overcome replicative mortality. ALT is mediated by recombination and is prevalent in a subset of human cancers, yet whether it can be exploited therapeutically remains unknown. Loss of the chromatin-remodeling protein ATRX associates with ALT in cancers. Here, we show that ATRX loss compromises cell-cycle regulation of the telomeric noncoding RNA TERRA and leads to persistent association of replication protein A (RPA) with telomeres after DNA replication, creating a recombinogenic nucleoprotein structure. Inhibition of the protein kinase ATR, a critical regulator of recombination recruited by RPA, disrupts ALT and triggers chromosome fragmentation and apoptosis in ALT cells. The cell death induced by ATR inhibitors is highly selective for cancer cells that rely on ALT, suggesting that such inhibitors may be useful for treatment of ALT-positive cancers.', 'year': 2015, 'citationCount': 383, 'authors': [{'authorId': '144325445', 'name': 'R. L. Flynn'}, {'authorId': '6625525', 'name': 'Kelli E. Cox'}, {'authorId': '6612562', 'name': 'M. Jeitany'}, {'authorId': '34972222', 'name': 'H. Wakimoto'}, {'authorId': '7681457', 'name': 'Alysia R. Bryll'}, {'authorId': '6933996', 'name': 'Neil J. Ganem'}, {'authorId': '38161554', 'name': 'F. Bersani'}, {'authorId': '13583738', 'name': 'J. R. Pineda'}, {'authorId': '3728246', 'name': 'M. Suvà'}, {'authorId': '1714039', 'name': 'C. Benes'}, {'authorId': '2253481', 'name': 'D. Haber'}, {'authorId': '3726742', 'name': 'F. Boussin'}, {'authorId': '2234244492', 'name': 'L. Zou'}]}, {'paperId': 'cc510777bf01063e970efc8c9b24bff1c434e2d0', 'title': '67P/Churyumov-Gerasimenko, a Jupiter family comet with a high D/H ratio', 'abstract': 'The provenance of water and organic compounds on Earth and other terrestrial planets has been discussed for a long time without reaching a consensus. One of the best means to distinguish between different scenarios is by determining the deuterium-to-hydrogen (D/H) ratios in the reservoirs for comets and Earth’s oceans. Here, we report the direct in situ measurement of the D/H ratio in the Jupiter family comet 67P/Churyumov-Gerasimenko by the ROSINA mass spectrometer aboard the European Space Agency’s Rosetta spacecraft, which is found to be (5.3 ± 0.7) × 10−4—that is, approximately three times the terrestrial value. Previous cometary measurements and our new finding suggest a wide range of D/H ratios in the water within Jupiter family objects and preclude the idea that this reservoir is solely composed of Earth ocean–like water.', 'year': 2015, 'citationCount': 384, 'authors': [{'authorId': '4512323', 'name': 'K. Altwegg'}, {'authorId': '3783380', 'name': 'H. Balsiger'}, {'authorId': '1403746229', 'name': 'A. Bar-Nun'}, {'authorId': '2282158', 'name': 'J. Berthelier'}, {'authorId': '6862522', 'name': 'A. Bieler'}, {'authorId': '6862522', 'name': 'A. Bieler'}, {'authorId': '4019296', 'name': 'P. Bochsler'}, {'authorId': '100535580', 'name': 'C. Briois'}, {'authorId': '4612766', 'name': 'U. Calmonte'}, {'authorId': '2894238', 'name': 'M. Combi'}, {'authorId': '144087995', 'name': 'J. Keyser'}, {'authorId': '4793465', 'name': 'P. Eberhardt'}, {'authorId': '1694493', 'name': 'B. Fiethe'}, {'authorId': '47853914', 'name': 'S. Fuselier'}, {'authorId': '69455147', 'name': 'S. Gasc'}, {'authorId': '2358371', 'name': 'T. Gombosi'}, {'authorId': '34594942', 'name': 'K. Hansen'}, {'authorId': '115165858', 'name': 'M. Hässig'}, {'authorId': '115165858', 'name': 'M. Hässig'}, {'authorId': '145555577', 'name': 'A. Jäckel'}, {'authorId': '144047212', 'name': 'E. Kopp'}, {'authorId': '2881282', 'name': 'A. Korth'}, {'authorId': '39947404', 'name': 'L. Leroy'}, {'authorId': '1931940', 'name': 'U. Mall'}, {'authorId': '2490322', 'name': 'B. Marty'}, {'authorId': '5055989', 'name': 'O. Mousis'}, {'authorId': '2376017', 'name': 'E. Neefs'}, {'authorId': '6309276', 'name': 'T. Owen'}, {'authorId': '3172579', 'name': 'H. Rème'}, {'authorId': '3172579', 'name': 'H. Rème'}, {'authorId': '145337044', 'name': 'M. Rubin'}, {'authorId': '2094410987', 'name': 'T. Sémon'}, {'authorId': '143664095', 'name': 'C. Tzou'}, {'authorId': '28411223', 'name': 'H. Waite'}, {'authorId': '3101090', 'name': 'P. Wurz'}]}, {'paperId': 'ae861867a6693094db5af7a9ae45b204407426e2', 'title': 'Transcription factor trapping by RNA in gene regulatory elements', 'abstract': 'Noncoding RNA helps protein binding Besides reading the coding regions of genes, RNA polymerase generates RNA at promoter-proximal and -distal DNA elements, but the function of these molecules is largely unknown. Sigova et al. show that these RNAs facilitate interactions between gene regulators and the regulatory elements they occupy. Nascent RNA associates with the transcription factor YY1 and increases its ability to bind DNA. Thus, transcription at active regulatory elements may provide a positive feedback loop that reinforces regulatory elements contributing to the stability of gene expression programs. Science, this issue p. 978 Nascent RNAs facilitate interactions between gene regulators and regulatory elements. Transcription factors (TFs) bind specific sequences in promoter-proximal and -distal DNA elements to regulate gene transcription. RNA is transcribed from both of these DNA elements, and some DNA binding TFs bind RNA. Hence, RNA transcribed from regulatory elements may contribute to stable TF occupancy at these sites. We show that the ubiquitously expressed TF Yin-Yang 1 (YY1) binds to both gene regulatory elements and their associated RNA species across the entire genome. Reduced transcription of regulatory elements diminishes YY1 occupancy, whereas artificial tethering of RNA enhances YY1 occupancy at these elements. We propose that RNA makes a modest but important contribution to the maintenance of certain TFs at gene regulatory elements and suggest that transcription of regulatory elements produces a positive-feedback loop that contributes to the stability of gene expression programs.', 'year': 2015, 'citationCount': 367, 'authors': [{'authorId': '4066956', 'name': 'A. Sigova'}, {'authorId': '41330093', 'name': 'B. Abraham'}, {'authorId': '48552586', 'name': 'X. Ji'}, {'authorId': '3622415', 'name': 'B. Molinie'}, {'authorId': '3607792', 'name': 'N. Hannett'}, {'authorId': '49813841', 'name': 'Y. Guo'}, {'authorId': '4029700', 'name': 'Mohini Jangi'}, {'authorId': '3649962', 'name': 'Cosmas C. Giallourakis'}, {'authorId': '2065901', 'name': 'P. Sharp'}, {'authorId': '144787676', 'name': 'R. Young'}]}, {'paperId': '5036e687656efa470275eb6813d09137670511b6', 'title': 'The structure of the human mitochondrial ribosome', 'abstract': 'The whole mitoribosome at high resolution Mitochondria are thought to be the descendents of a prokaryotic cell that took up residence in a protoeukaryotic cell. Mitochondria retain a few genes involved in oxidative phosphorylation. To translate these genes, mitochondria contain highly divergent mitochondrial ribosomes, or mitoribosomes. Amunts et al. determined the high-resolution structures of complete mammalian mitoribosomes using cryoelectron microscopy. Mitoribosomes include an unusual mRNA binding channel. The findings elucidate how aminoglycoside antibiotics can inadvertently inhibit mitoribosomes and how mutations in mitoribosomes can lead to disease. Science, this issue p. 95 Structures of mammalian mitoribosomes reveal details underlying the function of this divergent class of ribosome. The highly divergent ribosomes of human mitochondria (mitoribosomes) synthesize 13 essential proteins of oxidative phosphorylation complexes. We have determined the structure of the intact mitoribosome to 3.5 angstrom resolution by means of single-particle electron cryogenic microscopy. It reveals 80 extensively interconnected proteins, 36 of which are specific to mitochondria, and three ribosomal RNA molecules. The head domain of the small subunit, particularly the messenger (mRNA) channel, is highly remodeled. Many intersubunit bridges are specific to the mitoribosome, which adopts conformations involving ratcheting or rolling of the small subunit that are distinct from those seen in bacteria or eukaryotes. An intrinsic guanosine triphosphatase mediates a contact between the head and central protuberance. The structure provides a reference for analysis of mutations that cause severe pathologies and for future drug design.', 'year': 2015, 'citationCount': 401, 'authors': [{'authorId': '3681289', 'name': 'A. Amunts'}, {'authorId': '144686935', 'name': 'Alan Brown'}, {'authorId': '95552201', 'name': 'J. Toots'}, {'authorId': '46489599', 'name': 'S. Scheres'}, {'authorId': '145575131', 'name': 'V. Ramakrishnan'}]}, {'paperId': '074fecabdd0f93f3ed501a37cf9f69e6bf151504', 'title': 'Dust measurements in the coma of comet 67P/Churyumov-Gerasimenko inbound to the Sun', 'abstract': 'Critical measurements for understanding accretion and the dust/gas ratio in the solar nebula, where planets were forming 4.5 billion years ago, are being obtained by the GIADA (Grain Impact Analyser and Dust Accumulator) experiment on the European Space Agency’s Rosetta spacecraft orbiting comet 67P/Churyumov-Gerasimenko. Between 3.6 and 3.4 astronomical units inbound, GIADA and OSIRIS (Optical, Spectroscopic, and Infrared Remote Imaging System) detected 35 outflowing grains of mass 10−10 to 10−7 kilograms, and 48 grains of mass 10−5 to 10−2 kilograms, respectively. Combined with gas data from the MIRO (Microwave Instrument for the Rosetta Orbiter) and ROSINA (Rosetta Orbiter Spectrometer for Ion and Neutral Analysis) instruments, we find a dust/gas mass ratio of 4 ± 2 averaged over the sunlit nucleus surface. A cloud of larger grains also encircles the nucleus in bound orbits from the previous perihelion. The largest orbiting clumps are meter-sized, confirming the dust/gas ratio of 3 inferred at perihelion from models of dust comae and trails.', 'year': 2015, 'citationCount': 334, 'authors': [{'authorId': '5193844', 'name': 'A. Rotundi'}, {'authorId': '50246256', 'name': 'H. Sierks'}, {'authorId': '113142748', 'name': 'V. Della Corte'}, {'authorId': '14100462', 'name': 'M. Fulle'}, {'authorId': '49672000', 'name': 'P. Gutiérrez'}, {'authorId': '145621605', 'name': 'L. Lara'}, {'authorId': '39419736', 'name': 'C. Barbieri'}, {'authorId': '50403997', 'name': 'P. Lamy'}, {'authorId': '91031514', 'name': 'R. Rodrigo'}, {'authorId': '69029744', 'name': 'D. Koschny'}, {'authorId': '6973900', 'name': 'H. Rickman'}, {'authorId': '144446632', 'name': 'H. Keller'}, {'authorId': '1393618617', 'name': 'J. Lopez‐Moreno'}, {'authorId': '48649617', 'name': 'M. Accolla'}, {'authorId': '145969939', 'name': 'J. Agarwal'}, {'authorId': '1398024389', 'name': 'M. A’Hearn'}, {'authorId': '1888631', 'name': 'N. Altobelli'}, {'authorId': '48286249', 'name': 'F. Angrilli'}, {'authorId': '145885511', 'name': 'M. Barucci'}, {'authorId': '102414410', 'name': 'J. Bertaux'}, {'authorId': '47679133', 'name': 'I. Bertini'}, {'authorId': '6691955', 'name': 'D. Bodewits'}, {'authorId': '48366207', 'name': 'E. Bussoletti'}, {'authorId': '5058495', 'name': 'L. Colangeli'}, {'authorId': '2762299', 'name': 'M. Cosi'}, {'authorId': '3939528', 'name': 'G. Cremonese'}, {'authorId': '31844775', 'name': 'J. Crifo'}, {'authorId': '2896324', 'name': 'V. Da Deppo'}, {'authorId': '3984918', 'name': 'B. Davidsson'}, {'authorId': '2317823', 'name': 'S. Debei'}, {'authorId': '50129229', 'name': 'M. De Cecco'}, {'authorId': '40416093', 'name': 'F. Esposito'}, {'authorId': '145605911', 'name': 'M. Ferrari'}, {'authorId': '2564082', 'name': 'S. Fornasier'}, {'authorId': '8294277', 'name': 'F. Giovane'}, {'authorId': '5047888', 'name': 'B. Gustafson'}, {'authorId': '4616264', 'name': 'S. Green'}, {'authorId': '3155836', 'name': 'O. Groussin'}, {'authorId': '144003691', 'name': 'E. Grün'}, {'authorId': '3354765', 'name': 'C. Güttler'}, {'authorId': '143947538', 'name': 'M. Herranz'}, {'authorId': '5303188', 'name': 'S. Hviid'}, {'authorId': '2095291385', 'name': 'W. Ip'}, {'authorId': '27392662', 'name': 'S. Ivanovski'}, {'authorId': '31930502', 'name': 'J. M. Jerónimo'}, {'authorId': '3240472', 'name': 'L. Jorda'}, {'authorId': '152615704', 'name': 'J. Knollenberg'}, {'authorId': '8510904', 'name': 'R. Kramm'}, {'authorId': '2250203', 'name': 'E. Kührt'}, {'authorId': '48974673', 'name': 'M. Küppers'}, {'authorId': '144479967', 'name': 'M. Lazzarin'}, {'authorId': '34670298', 'name': 'M. Leese'}, {'authorId': '1398306080', 'name': 'A. López-Jiménez'}, {'authorId': '2083713398', 'name': 'F. Lucarelli'}, {'authorId': '2891649', 'name': 'S. Lowry'}, {'authorId': '144372011', 'name': 'F. Marzari'}, {'authorId': '7642818', 'name': 'E. Epifani'}, {'authorId': '144186953', 'name': 'J. McDonnell'}, {'authorId': '4105305', 'name': 'V. Mennella'}, {'authorId': '144921681', 'name': 'H. Michalik'}, {'authorId': '143666794', 'name': 'A. Molina'}, {'authorId': '153285004', 'name': 'R. Morales'}, {'authorId': '1572187912', 'name': 'F. Moreno'}, {'authorId': '144381122', 'name': 'S. Mottola'}, {'authorId': '2704601', 'name': 'G. Naletto'}, {'authorId': '5300286', 'name': 'N. Oklay'}, {'authorId': '145786944', 'name': 'J. Ortiz'}, {'authorId': '2885404', 'name': 'E. Palomba'}, {'authorId': '144685886', 'name': 'P. Palumbo'}, {'authorId': '122587024', 'name': 'J. Perrin'}, {'authorId': '2116901131', 'name': 'Julio Rodríguez'}, {'authorId': '143930936', 'name': 'L. Sabau'}, {'authorId': '2155760', 'name': 'C. Snodgrass'}, {'authorId': '8154401', 'name': 'R. Sordini'}, {'authorId': '144570575', 'name': 'N. Thomas'}, {'authorId': '5653933', 'name': 'C. Tubiana'}, {'authorId': '12647806', 'name': 'J. Vincent'}, {'authorId': '4905966', 'name': 'P. Weissman'}, {'authorId': '114386622', 'name': 'K. Wenzel'}, {'authorId': '152222796', 'name': 'V. Zakharov'}, {'authorId': '5101497', 'name': 'J. Zarnecki'}]}, {'paperId': 'e08354cc53e6b3b2b10233777989f2c28d1d1911', 'title': 'The transcription factor GABP selectively binds and activates the mutant TERT promoter in cancer', 'abstract': \"A mutant promoter's partner in crime Telomerase is an enzyme that maintains the ends of chromosomes. TERT, the gene coding for the enzyme's catalytic subunit, is not expressed in healthy somatic cells, but its expression is reactivated in the majority of human cancers. The resultant high levels of telomerase help cancer cells survive and multiply. Recurrent mutations in the promoter region of TERT are associated with high telomerase levels in multiple cancer types. Bell et al. show that a specific transcription factor called GABP is selectively recruited to the mutant form of the TERT promoter, which activates TERT gene expression Science, this issue p. 1036 Cancer-associated mutations in the promoter of the telomerase gene allow increased activation by transcription factor binding. Reactivation of telomerase reverse transcriptase (TERT) expression enables cells to overcome replicative senescence and escape apoptosis, which are fundamental steps in the initiation of human cancer. Multiple cancer types, including up to 83% of glioblastomas (GBMs), harbor highly recurrent TERT promoter mutations of unknown function but specific to two nucleotide positions. We identified the functional consequence of these mutations in GBMs to be recruitment of the multimeric GA-binding protein (GABP) transcription factor specifically to the mutant promoter. Allelic recruitment of GABP is consistently observed across four cancer types, highlighting a shared mechanism underlying TERT reactivation. Tandem flanking native E26 transformation-specific motifs critically cooperate with these mutations to activate TERT, probably by facilitating GABP heterotetramer binding. GABP thus directly links TERT promoter mutations to aberrant expression in multiple cancers.\", 'year': 2015, 'citationCount': 421, 'authors': [{'authorId': '144450987', 'name': 'Robert J. A. Bell'}, {'authorId': '4281133', 'name': 'H. T. Rube'}, {'authorId': '4338966', 'name': 'Alex Kreig'}, {'authorId': '38557296', 'name': 'A. Mancini'}, {'authorId': '30033687', 'name': 'S. Fouse'}, {'authorId': '31430164', 'name': 'R. Nagarajan'}, {'authorId': '4630388', 'name': 'Serah Choi'}, {'authorId': '4824870', 'name': 'C. Hong'}, {'authorId': '3957980', 'name': 'Daniel He'}, {'authorId': '4798798', 'name': 'M. Pekmezci'}, {'authorId': '2704556', 'name': 'J. Wiencke'}, {'authorId': '2074070', 'name': 'M. Wrensch'}, {'authorId': '18014955', 'name': 'Susan M. Chang'}, {'authorId': '2196573', 'name': 'K. Walsh'}, {'authorId': '5862538', 'name': 'S. Myong'}, {'authorId': '143729167', 'name': 'Jun S. Song'}, {'authorId': '2552233', 'name': 'J. Costello'}]}, {'paperId': 'c2f0f097f803926fedb9d4f7c71002f7ca32ba95', 'title': 'Mars methane detection and variability at Gale crater', 'abstract': 'Of water and methane on Mars The Curiosity rover has been collecting data for the past 2 years, since its delivery to Mars (see the Perspective by Zahnle). Many studies now suggest that many millions of years ago, Mars was warmer and wetter than it is today. But those conditions required an atmosphere that seems to have vanished. Using the Curiosity rover, Mahaffy et al. measured the ratio of deuterium to hydrogen in clays that were formed 3.0 to 3.7 billion years ago. Hydrogen escapes more readily than deuterium, so this ratio offers a snapshot measure of the ancient atmosphere that can help constrain when and how it disappeared. Most methane on Earth has a biological origin, so planetary scientists have keenly pursued its detection in the martian atmosphere as well. Now, Webster et al. have precisely confirmed the presence of methane in the martian atmosphere with the instruments aboard the Curiosity rover at Gale crater. Science, this issue p. 412, p. 415; see also p. 370 Curiosity confirms the presence and variability of atmospheric methane, implying episodic production from an unknown source. [Also see Perspective by Zahnle] Reports of plumes or patches of methane in the martian atmosphere that vary over monthly time scales have defied explanation to date. From in situ measurements made over a 20-month period by the tunable laser spectrometer of the Sample Analysis at Mars instrument suite on Curiosity at Gale crater, we report detection of background levels of atmospheric methane of mean value 0.69 ± 0.25 parts per billion by volume (ppbv) at the 95% confidence interval (CI). This abundance is lower than model estimates of ultraviolet degradation of accreted interplanetary dust particles or carbonaceous chondrite material. Additionally, in four sequential measurements spanning a 60-sol period (where 1 sol is a martian day), we observed elevated levels of methane of 7.2 ± 2.1 ppbv (95% CI), implying that Mars is episodically producing methane from an additional unknown source.', 'year': 2015, 'citationCount': 408, 'authors': [{'authorId': '1997113', 'name': 'C. Webster'}, {'authorId': '4579249', 'name': 'P. Mahaffy'}, {'authorId': '1934029', 'name': 'S. Atreya'}, {'authorId': '39051505', 'name': 'G. Flesch'}, {'authorId': '6507728', 'name': 'M. Mischna'}, {'authorId': '7166207', 'name': 'P. Meslin'}, {'authorId': '37318312', 'name': 'K. Farley'}, {'authorId': '40098549', 'name': 'P. Conrad'}, {'authorId': '36072221', 'name': 'L. Christensen'}, {'authorId': '47907676', 'name': 'A. Pavlov'}, {'authorId': '1400376253', 'name': 'J. Martín‐Torres'}, {'authorId': '70175219', 'name': 'M. Zorzano'}, {'authorId': '7924487', 'name': 'T. McConnochie'}, {'authorId': '6309276', 'name': 'T. Owen'}, {'authorId': '5905992', 'name': 'J. Eigenbrode'}, {'authorId': '6766468', 'name': 'D. Glavin'}, {'authorId': '145255661', 'name': 'A. Steele'}, {'authorId': '4289200', 'name': 'C. Malespin'}, {'authorId': '1927274', 'name': 'P. Archer'}, {'authorId': '2193015', 'name': 'B. Sutter'}, {'authorId': '46708768', 'name': 'P. Coll'}, {'authorId': '34580847', 'name': 'C. Freissinet'}, {'authorId': '87435846', 'name': 'C. Mckay'}, {'authorId': '35101299', 'name': 'J. Moores'}, {'authorId': '6827709', 'name': 'S. Schwenzer'}, {'authorId': '4386694', 'name': 'J. Bridges'}, {'authorId': '1388991208', 'name': 'R. Navarro‐González'}, {'authorId': '2714162', 'name': 'R. Gellert'}, {'authorId': '32600129', 'name': 'M. Lemmon'}]}, {'paperId': '6ea6bdab5a986a06160615e48cf385f4784f973e', 'title': 'The Atlantic Multidecadal Oscillation without a role for ocean circulation', 'abstract': 'Ocean circulation changes not needed What causes the pattern of sea surface temperature change that is seen in the North Atlantic Ocean? This naturally occurring quasi-cyclical variation, known as the Atlantic Multidecadal Oscillation (AMO), affects weather and climate. Some have suggested that the AMO is a consequence of variable large-scale ocean circulation. Clement et al. suggest otherwise. They find that the pattern of AMO variability can be produced in a model that does not include ocean circulation changes, but only the effects of changes in air temperatures and winds. Science, this issue p. 320 The Atlantic Multidecadal Oscillation does not depend on variable whole-ocean circulation. The Atlantic Multidecadal Oscillation (AMO) is a major mode of climate variability with important societal impacts. Most previous explanations identify the driver of the AMO as the ocean circulation, specifically the Atlantic Meridional Overturning Circulation (AMOC). Here we show that the main features of the observed AMO are reproduced in models where the ocean heat transport is prescribed and thus cannot be the driver. Allowing the ocean circulation to interact with the atmosphere does not significantly alter the characteristics of the AMO in the current generation of climate models. These results suggest that the AMO is the response to stochastic forcing from the mid-latitude atmospheric circulation, with thermal coupling playing a role in the tropics. In this view, the AMOC and other ocean circulation changes would be largely a response to, not a cause of, the AMO.', 'year': 2015, 'citationCount': 303, 'authors': [{'authorId': '38500930', 'name': 'A. Clement'}, {'authorId': '48858437', 'name': 'K. Bellomo'}, {'authorId': '48395861', 'name': 'L. Murphy'}, {'authorId': '2409880', 'name': 'M. Cane'}, {'authorId': '4969561', 'name': 'T. Mauritsen'}, {'authorId': '152424253', 'name': 'G. Rädel'}, {'authorId': '121696502', 'name': 'B. Stevens'}]}, {'paperId': 'be652a4dc3d9217f054013a31c3ab081205a1c7a', 'title': 'Preventing proteostasis diseases by selective inhibition of a phosphatase regulatory subunit', 'abstract': 'Giving protein folding a helping hand The reversible phosphorylation of proteins controls virtually all aspects of cell and organismal function. Targeting phosphorylation offers a broad range of therapeutic opportunities, and thus kinases have become important therapeutic targets. As targets, phosphatases should be as attractive, but in fact they are more challenging to manipulate. Das et al. have found a safe and specific inhibitor, called Sephin1, that targets a regulatory subunit of protein phosphatase 1 in vivo. Sephin1 binds and inhibits PPP1R15A, but not the related regulatory phosphatase PPP1R15B. In mice, Sephin1 prolonged a stress-induced phospho-signaling pathway to prevent the pathological defects of the unrelated protein-misfolding diseases Charcot-Marie-Tooth 1B and amyotrophic lateral sclerosis. Science, this issue p. 239 Sephin1 selectively inhibits a protein phosphatase to prevent two protein misfolding diseases in mice. Protein phosphorylation regulates virtually all biological processes. Although protein kinases are popular drug targets, targeting protein phosphatases remains a challenge. Here, we describe Sephin1 (selective inhibitor of a holophosphatase), a small molecule that safely and selectively inhibited a regulatory subunit of protein phosphatase 1 in vivo. Sephin1 selectively bound and inhibited the stress-induced PPP1R15A, but not the related and constitutive PPP1R15B, to prolong the benefit of an adaptive phospho-signaling pathway, protecting cells from otherwise lethal protein misfolding stress. In vivo, Sephin1 safely prevented the motor, morphological, and molecular defects of two otherwise unrelated protein-misfolding diseases in mice, Charcot-Marie-Tooth 1B, and amyotrophic lateral sclerosis. Thus, regulatory subunits of phosphatases are drug targets, a property exploited here to safely prevent two protein misfolding diseases.', 'year': 2015, 'citationCount': 331, 'authors': [{'authorId': '153163469', 'name': 'I. Das'}, {'authorId': '5619187', 'name': 'Agnieszka Krzyżosiak'}, {'authorId': '8020826', 'name': 'Kim Schneider'}, {'authorId': '6452817', 'name': 'L. Wrabetz'}, {'authorId': '1401066945', 'name': 'M. D’Antonio'}, {'authorId': '40226529', 'name': 'N. Barry'}, {'authorId': '12051959', 'name': 'A. Sigurdardottir'}, {'authorId': '35866732', 'name': 'A. Bertolotti'}]}, {'paperId': 'd66b125cf23f2b06d7ba717fdbb00e73d620cf08', 'title': 'The Pluto system: Initial results from its exploration by New Horizons', 'abstract': \"New Horizons' views of Pluto The flyby of Pluto and its moon Charon by the New Horizons spacecraft generated news coverage around the world. Now Stern et al. report the first scientific results from the high-speed encounter. The surface of Pluto is surprisingly diverse, with large regions of differing brightness and composition. There is ample evidence for ongoing rich geological processes that act to sculpt its surface. Charon's surface is similarly complex, with numerous relief structures and varied coloration. Pluto's atmosphere is extensive but less dense than expected, whereas Charon has no detectable atmosphere. Science, this issue p. 10.1126/science.aad1815 The first scientific results from New Horizon’s flyby of Pluto reveal a rich and diverse geology. INTRODUCTION Pluto was discovered in 1930 and was long thought to be a misfit or anomaly in the solar system. However, the 1992 discovery of the Kuiper Belt—a torus-shaped region beyond Neptune’s orbit, and the largest structure in our three-zoned planetary system—provided new context, showing Pluto to be the largest of a new class of small planets formed in the outer solar system during the ancient era of planetary accretion ~4.5 billion years ago. NASA’s New Horizons spacecraft made the first exploration of Pluto, culminating on 14 July 2015; it collected numerous remote sensing and in situ measurements of Pluto and its system of five moons. We report the first scientific results and interpretations of that flyby. RATIONALE The New Horizons spacecraft completed a close approach to the Pluto system at a distance of 13,691 km from Pluto’s center. The spacecraft carries a sophisticated suite of scientific instruments, including the Ralph multicolor/panchromatic mapper and mapping infrared composition spectrometer; the LORRI long-focal-length panchromatic visible imager; the Alice extreme/far ultraviolet mapping spectrograph; twin REX radio science experiments; the SWAP solar wind detector; the PEPSSI high-energy charged particle spectrometer; and VBSDC, a dust impact detector. Together these instruments collected more than 50 gigabits of data on the Pluto system near the time of the spacecraft’s closest approach. RESULTS We found that Pluto’s surface displays a wide variety of landforms and terrain ages, as well as substantial albedo, color, and compositional variation. Evidence was also found for a water ice–rich crust, geologically young surface units, tectonic extension, surface volatile ice convection, possible wind streaks, volatile transport, and glacial flow. Pluto’s atmosphere is highly extended, with trace hydrocarbons, a global haze layer, and a surface pressure near 10 microbars. The bulk densities of Pluto and Charon were found to differ by less than 10%, which is consistent with bulk rock contents for the two bodies that are likewise similar. This could imply that both precursor bodies were undifferentiated (or only modestly differentiated) prior to their collision—which would have profound implications for the timing, the duration, and even the mechanism of accretion in the ancestral Kuiper Belt. Pluto’s large moon Charon displays extensional tectonics and extensive resurfacing, as well as possible evidence for a heterogeneous crustal composition; its north pole displays puzzling dark terrain. The sizes of Pluto’s small satellites Nix and Hydra were measured for the first time, as were their surface reflectivities, which are puzzlingly higher than Charon’s. No new satellites were detected. CONCLUSION The New Horizons encounter revealed that Pluto displays a surprisingly wide variety of geological landforms, including those resulting from glaciological and surface-atmosphere interactions as well as impact, tectonic, possible cryovolcanic, and mass-wasting processes. This suggests that other small planets of the Kuiper Belt, such as Eris, Makemake, and Haumea, could express similarly complex histories that rival those of terrestrial planets. Pluto’s diverse surface geology and long-term activity also raise fundamental questions about how it has remained active many billions of years after its formation. Pluto mosaic made from New Horizons LORRI images taken 14 July 2015 from a distance of 80,000 km. This view is projected from a point 1800 km above Pluto’s equator, looking northeast over the dark, cratered, informally named Cthulhu Regio toward the bright, smooth expanse of icy plains informally called Sputnik Planum. Pluto’s north pole is off the image to the left. This image mosaic was produced with panchromatic images from the New Horizons LORRI camera, with color overlaid from the Ralph color mapper onboard New Horizons. The Pluto system was recently explored by NASA’s New Horizons spacecraft, making closest approach on 14 July 2015. Pluto’s surface displays diverse landforms, terrain ages, albedos, colors, and composition gradients. Evidence is found for a water-ice crust, geologically young surface units, surface ice convection, wind streaks, volatile transport, and glacial flow. Pluto’s atmosphere is highly extended, with trace hydrocarbons, a global haze layer, and a surface pressure near 10 microbars. Pluto’s diverse surface geology and long-term activity raise fundamental questions about how small planets remain active many billions of years after formation. Pluto’s large moon Charon displays tectonics and evidence for a heterogeneous crustal composition; its north pole displays puzzling dark terrain. Small satellites Hydra and Nix have higher albedos than expected.\", 'year': 2015, 'citationCount': 366, 'authors': [{'authorId': '145615615', 'name': 'S. Stern'}, {'authorId': '2596024', 'name': 'F. Bagenal'}, {'authorId': '144810318', 'name': 'K. Ennico'}, {'authorId': '144398834', 'name': 'G. Gladstone'}, {'authorId': '144079258', 'name': 'W. Grundy'}, {'authorId': '145741171', 'name': 'W. McKinnon'}, {'authorId': '38254933', 'name': 'J. Moore'}, {'authorId': '5752279', 'name': 'C. Olkin'}, {'authorId': '145837796', 'name': 'J. Spencer'}, {'authorId': '4612630', 'name': 'H. Weaver'}, {'authorId': '144667642', 'name': 'L. Young'}, {'authorId': '35561159', 'name': 'T. Andert'}, {'authorId': '48457533', 'name': 'J. Andrews'}, {'authorId': '36631430', 'name': 'M. Banks'}, {'authorId': '122579257', 'name': 'B. Bauer'}, {'authorId': '101412277', 'name': 'J. Bauman'}, {'authorId': '6895822', 'name': 'O. Barnouin'}, {'authorId': '5484323', 'name': 'P. Bedini'}, {'authorId': '3733800', 'name': 'K. Beisser'}, {'authorId': '49503161', 'name': 'R. Beyer'}, {'authorId': '1813837', 'name': 'S. Bhaskaran'}, {'authorId': '6270608', 'name': 'R. Binzel'}, {'authorId': '4234897', 'name': 'E. Birath'}, {'authorId': '34057526', 'name': 'M. Bird'}, {'authorId': '90120170', 'name': 'D. Bogan'}, {'authorId': '48789119', 'name': 'A. Bowman'}, {'authorId': '47621602', 'name': 'V. Bray'}, {'authorId': '49509606', 'name': 'M. Brozović'}, {'authorId': '146338036', 'name': 'C. Bryan'}, {'authorId': '2106296101', 'name': 'M. R. Buckley'}, {'authorId': '2823138', 'name': 'M. Buie'}, {'authorId': '6498124', 'name': 'B. Buratti'}, {'authorId': '93963585', 'name': 'S. Bushman'}, {'authorId': '70707164', 'name': 'A. Calloway'}, {'authorId': '47589465', 'name': 'B. Carcich'}, {'authorId': '32584940', 'name': 'A. Cheng'}, {'authorId': '8189338', 'name': 'S. Conard'}, {'authorId': '91092942', 'name': 'C. A. Conrad'}, {'authorId': '89089970', 'name': 'J. Cook'}, {'authorId': '144051326', 'name': 'D. Cruikshank'}, {'authorId': '116776624', 'name': 'O. Custodio'}, {'authorId': '88728462', 'name': 'C. D. Dalle Ore'}, {'authorId': '35228553', 'name': 'C. Deboy'}, {'authorId': '6685622', 'name': 'Z. Dischner'}, {'authorId': '2069078045', 'name': 'P. Dumont'}, {'authorId': '50377339', 'name': 'A. Earle'}, {'authorId': '49039752', 'name': 'H. Elliott'}, {'authorId': '115690081', 'name': 'J. Ercol'}, {'authorId': '33804631', 'name': 'C. Ernst'}, {'authorId': '48746614', 'name': 'T. Finley'}, {'authorId': '101653683', 'name': 'S. H. Flanigan'}, {'authorId': '3260212', 'name': 'G. Fountain'}, {'authorId': '1698527041', 'name': 'M. Freeze'}, {'authorId': '4681394', 'name': 'T. Greathouse'}, {'authorId': '30999144', 'name': 'J. Green'}, {'authorId': '2107822296', 'name': 'Y. Guo'}, {'authorId': '46686185', 'name': 'M. Hahn'}, {'authorId': '2228671193', 'name': 'D. Hamilton'}, {'authorId': '2069334378', 'name': 'S. Hamilton'}, {'authorId': '108780783', 'name': 'J. Hanley'}, {'authorId': '5123252', 'name': 'A. Harch'}, {'authorId': '103038277', 'name': 'H. Hart'}, {'authorId': '65987131', 'name': 'C. Hersman'}, {'authorId': '2072855102', 'name': 'A. Hill'}, {'authorId': '15621312', 'name': 'M. Hill'}, {'authorId': '5610936', 'name': 'D. Hinson'}, {'authorId': '51055187', 'name': 'M. Holdridge'}, {'authorId': '48736351', 'name': 'M. Horányi'}, {'authorId': '143842010', 'name': 'A. Howard'}, {'authorId': '144867456', 'name': 'C. Howett'}, {'authorId': '71251275', 'name': 'C. Jackman'}, {'authorId': '143826218', 'name': 'R. Jacobson'}, {'authorId': '3114626', 'name': 'D. Jennings'}, {'authorId': '40198370', 'name': 'J. Kammer'}, {'authorId': '2206975340', 'name': 'H. K. Kang'}, {'authorId': '13974518', 'name': 'D. Kaufmann'}, {'authorId': '3935671', 'name': 'P. Kollmann'}, {'authorId': '145312312', 'name': 'S. Krimigis'}, {'authorId': '4762243', 'name': 'D. Kusnierkiewicz'}, {'authorId': '2369234', 'name': 'T. Lauer'}, {'authorId': '2153272132', 'name': 'J. E. Lee'}, {'authorId': '117685933', 'name': 'K. L. Lindstrom'}, {'authorId': '2826609', 'name': 'I. Linscott'}, {'authorId': '52561266', 'name': 'C. Lisse'}, {'authorId': '145665823', 'name': 'A. Lunsford'}, {'authorId': '113643607', 'name': 'V. Mallder'}, {'authorId': '2068266177', 'name': 'N. Martin'}, {'authorId': '145390435', 'name': 'D. McComas'}, {'authorId': '2400562', 'name': 'R. McNutt'}, {'authorId': '6564626', 'name': 'D. Mehoke'}, {'authorId': '3109678', 'name': 'T. Mehoke'}, {'authorId': '2083034634', 'name': 'E. Melin'}, {'authorId': '50702037', 'name': 'M. Mutchler'}, {'authorId': '115517410', 'name': 'D. Nelson'}, {'authorId': '4977917', 'name': 'F. Nimmo'}, {'authorId': '46852290', 'name': 'J. Núñez'}, {'authorId': '40149438', 'name': 'A. Ocampo'}, {'authorId': '4780988', 'name': 'W. Owen'}, {'authorId': '152493032', 'name': 'M. Paetzold'}, {'authorId': '144330595', 'name': 'B. Page'}, {'authorId': '14641808', 'name': 'A. Parker'}, {'authorId': '40542088', 'name': 'J. Parker'}, {'authorId': '114342249', 'name': 'F. Pelletier'}, {'authorId': '50501427', 'name': 'J. Peterson'}, {'authorId': '6664207', 'name': 'N. Pinkine'}, {'authorId': '4203985', 'name': 'M. Piquette'}, {'authorId': '144697219', 'name': 'S. Porter'}, {'authorId': '5375164', 'name': 'S. Protopapa'}, {'authorId': '40485883', 'name': 'J. Redfern'}, {'authorId': '6467444', 'name': 'H. Reitsema'}, {'authorId': '6896152', 'name': 'D. Reuter'}, {'authorId': '2107414758', 'name': 'J. Roberts'}, {'authorId': '33221375', 'name': 'S. Robbins'}, {'authorId': '46267028', 'name': 'G. Rogers'}, {'authorId': '108196123', 'name': 'D. Rose'}, {'authorId': '4166509', 'name': 'K. Runyon'}, {'authorId': '4348335', 'name': 'K. Retherford'}, {'authorId': '94990717', 'name': 'M. Ryschkewitsch'}, {'authorId': '144956598', 'name': 'P. Schenk'}, {'authorId': '49278836', 'name': 'E. Schindhelm'}, {'authorId': '114943626', 'name': 'B. Sepan'}, {'authorId': '92434756', 'name': 'M. Showalter'}, {'authorId': '143953436', 'name': 'K. Singer'}, {'authorId': '104081149', 'name': 'M. Soluri'}, {'authorId': '100745778', 'name': 'D. Stanbridge'}, {'authorId': '4807386', 'name': 'A. Steffl'}, {'authorId': '32410104', 'name': 'D. Strobel'}, {'authorId': '5600908', 'name': 'T. Stryk'}, {'authorId': '50377583', 'name': 'M. Summers'}, {'authorId': '152916383', 'name': 'J. Szalay'}, {'authorId': '31784508', 'name': 'M. Tapley'}, {'authorId': '145592784', 'name': 'A. Taylor'}, {'authorId': '144173680', 'name': 'H. Taylor'}, {'authorId': '4232734', 'name': 'H. Throop'}, {'authorId': '31790664', 'name': 'C. Tsang'}, {'authorId': '2998858', 'name': 'G. L. Tyler'}, {'authorId': '5209100', 'name': 'O. Umurhan'}, {'authorId': '4377111', 'name': 'A. Verbiscer'}, {'authorId': '119939518', 'name': 'M. Versteeg'}, {'authorId': '153301791', 'name': 'M. Vincent'}, {'authorId': '115199046', 'name': 'R. Webbert'}, {'authorId': '104248787', 'name': 'S. Weidner'}, {'authorId': '48758610', 'name': 'G. Weigle'}, {'authorId': '1397577620', 'name': 'O. White'}, {'authorId': '51055218', 'name': 'K. Whittenburg'}, {'authorId': '48964745', 'name': 'B. Williams'}, {'authorId': '2118657719', 'name': 'K. Williams'}, {'authorId': '2107720028', 'name': 'S. Williams'}, {'authorId': '103414780', 'name': 'W. W. Woods'}, {'authorId': '48228254', 'name': 'A. Zangari'}, {'authorId': '152732919', 'name': 'E. Zirnstein'}]}, {'paperId': 'f443476b7c84200a3afb503dad5a67fd0d5a397e', 'title': 'Committing to ecological restoration', 'abstract': 'Efforts around the globe need legal and policy clarification At the September 2014 United Nations Climate Summit, governments rallied around an international agreement—the New York Declaration on Forests—that underscored restoration of degraded ecosystems as an auspicious solution to climate change. Ethiopia committed to restore more than one-sixth of its land. Uganda, the Democratic Republic of Congo, Guatemala, and Colombia pledged to restore huge areas within their borders. In total, parties committed to restore a staggering 350 million hectares by 2030.', 'year': 2015, 'citationCount': 371, 'authors': [{'authorId': '4723097', 'name': 'K. Suding'}, {'authorId': '50281294', 'name': 'E. Higgs'}, {'authorId': '2722335', 'name': 'M. Palmer'}, {'authorId': '8466219', 'name': 'J. B. Callicott'}, {'authorId': '46830133', 'name': 'C. Anderson'}, {'authorId': '2067798096', 'name': 'M. Baker'}, {'authorId': '6663611', 'name': 'J. Gutrich'}, {'authorId': '6180776', 'name': 'K. Hondula'}, {'authorId': '7302071', 'name': 'M. LaFevor'}, {'authorId': '144773504', 'name': 'B. Larson'}, {'authorId': '49125009', 'name': 'A. Randall'}, {'authorId': '123493788', 'name': 'Alan Randall'}, {'authorId': '98277763', 'name': 'J. Ruhl'}, {'authorId': '34264152', 'name': 'Katrina Z. S. Schwartz'}]}, {'paperId': 'fa9d10870a42106cf74cd3a6a97e44581ecef6c8', 'title': 'Asymmetric apportioning of aged mitochondria between daughter cells is required for stemness', 'abstract': 'Stem cells can sort mitochondria by age The renewal of tissues in aging organisms requires stem cells, which have the unusual ability to divide asymmetrically into one daughter cell that retains stem cell properties and another that differentiates into a particular tissue type. Katajisto et al. used photoactivated marker proteins to monitor the age of cell organelles in stemlike cells from human breast tissue and their distribution into daughter cells. Most organelles were evenly distributed, but daughter cells that maintained stem-cell properties received more newly produced mitochondria and fewer old ones. Science, this issue p. 340 Stem cells preferentially select new rather than old mitochondria as they divide. By dividing asymmetrically, stem cells can generate two daughter cells with distinct fates. However, evidence is limited in mammalian systems for the selective apportioning of subcellular contents between daughters. We followed the fates of old and young organelles during the division of human mammary stemlike cells and found that such cells apportion aged mitochondria asymmetrically between daughter cells. Daughter cells that received fewer old mitochondria maintained stem cell traits. Inhibition of mitochondrial fission disrupted both the age-dependent subcellular localization and segregation of mitochondria and caused loss of stem cell properties in the progeny cells. Hence, mechanisms exist for mammalian stemlike cells to asymmetrically sort aged and young mitochondria, and these are important for maintaining stemness properties.', 'year': 2015, 'citationCount': 428, 'authors': [{'authorId': '3581044', 'name': 'P. Katajisto'}, {'authorId': '8093966', 'name': 'Julia Döhla'}, {'authorId': '4870813', 'name': 'Christine L. Chaffer'}, {'authorId': '4301484', 'name': 'Nalle Pentinmikko'}, {'authorId': '34608185', 'name': 'N. Marjanovic'}, {'authorId': '7878948', 'name': 'Sharif Iqbal'}, {'authorId': '3034790', 'name': 'R. Zoncu'}, {'authorId': '47482597', 'name': 'Walter W. Chen'}, {'authorId': '34628374', 'name': 'R. Weinberg'}, {'authorId': '1843680', 'name': 'D. Sabatini'}]}, {'paperId': 'e68f8737cbabd33366c5d9af83279208b9d99d14', 'title': 'RIPK1 and NF-κB signaling in dying cells determines cross-priming of CD8+ T cells', 'abstract': 'Dying to impress the immune system Besides reacting to microbes, T cells can also mount immune responses to fragments of dying cells, which they encounter displayed on dendritic cells. Not all dying cells activate T cells, however, so what differentiates the dying cells that do? Yatim et al. studied two forms of programmed cells death: apoptosis and necroptosis. Using mouse cells in culture and mouse models of inflammatory cell death and anti-tumor immunity, they found that programmed cell death initiated T cell immunity only when the dying cells signaled through the enzyme RIPK1 and the transcription factor NF-κB. Science, this issue p. 328 Active signaling in dying cells determines their immunogenicity. Dying cells initiate adaptive immunity by providing both antigens and inflammatory stimuli for dendritic cells, which in turn activate CD8+ T cells through a process called antigen cross-priming. To define how different forms of programmed cell death influence immunity, we established models of necroptosis and apoptosis, in which dying cells are generated by receptor-interacting protein kinase-3 and caspase-8 dimerization, respectively. We found that the release of inflammatory mediators, such as damage-associated molecular patterns, by dying cells was not sufficient for CD8+ T cell cross-priming. Instead, robust cross-priming required receptor-interacting protein kinase-1 (RIPK1) signaling and nuclear factor κB (NF-κB)–induced transcription within dying cells. Decoupling NF-κB signaling from necroptosis or inflammatory apoptosis reduced priming efficiency and tumor immunity. Our results reveal that coordinated inflammatory and cell death signaling pathways within dying cells orchestrate adaptive immunity.', 'year': 2015, 'citationCount': 417, 'authors': [{'authorId': '115263635', 'name': 'N. Yatim'}, {'authorId': '1399772482', 'name': 'H. Jusforgues-Saklani'}, {'authorId': '46949224', 'name': 'S. Orozco'}, {'authorId': '34272292', 'name': 'O. Schulz'}, {'authorId': '39319949', 'name': 'R. Barreira da Silva'}, {'authorId': '5531706', 'name': 'C. Reis e Sousa'}, {'authorId': '3792271', 'name': 'D. Green'}, {'authorId': '6678698', 'name': 'A. Oberst'}, {'authorId': '21068591', 'name': 'M. Albert'}]}, {'paperId': 'bd795f6c19098963b32a38c52e7b4b91f9fc565b', 'title': 'Genomic signatures of evolutionary transitions from solitary to group living', 'abstract': 'For bees, many roads lead to social harmony Eusociality, where workers sacrifice their reproductive rights to support the colony, has evolved repeatedly and represents the most evolved form of social evolution in insects. Kapheim et al. looked across the genomes of 10 bee species with varying degrees of sociality to determine the underlying genomic contributions. No one genomic path led to eusociality, but similarities across genomes were seen in features such as increases in gene regulation and methylation. It also seems that selection pressures relaxed after the emergence of complex sociality. Science, this issue p. 1139 Social evolution in bees has followed diverse genomic paths but shares genomic patterns. The evolution of eusociality is one of the major transitions in evolution, but the underlying genomic changes are unknown. We compared the genomes of 10 bee species that vary in social complexity, representing multiple independent transitions in social evolution, and report three major findings. First, many important genes show evidence of neutral evolution as a consequence of relaxed selection with increasing social complexity. Second, there is no single road map to eusociality; independent evolutionary transitions in sociality have independent genetic underpinnings. Third, though clearly independent in detail, these transitions do have similar general features, including an increase in constrained protein evolution accompanied by increases in the potential for gene regulation and decreases in diversity and abundance of transposable elements. Eusociality may arise through different mechanisms each time, but would likely always involve an increase in the complexity of gene networks.', 'year': 2015, 'citationCount': 325, 'authors': [{'authorId': '6011535', 'name': 'K. Kapheim'}, {'authorId': '34884098', 'name': 'Hailin Pan'}, {'authorId': '46651780', 'name': 'Cai Li'}, {'authorId': '1744109', 'name': 'S. Salzberg'}, {'authorId': '1810831', 'name': 'D. Puiu'}, {'authorId': '49485917', 'name': 'Tanja Magoc'}, {'authorId': '2162158443', 'name': 'H. Robertson'}, {'authorId': '40148862', 'name': 'M. Hudson'}, {'authorId': '6228035', 'name': 'A. Venkat'}, {'authorId': '4482221', 'name': 'Brielle J. Fischman'}, {'authorId': '145338796', 'name': 'Alvaro G. Hernandez'}, {'authorId': '1968099', 'name': 'M. Yandell'}, {'authorId': '2021675', 'name': 'Daniel Ence'}, {'authorId': '144696702', 'name': 'C. Holt'}, {'authorId': '5174447', 'name': 'G. Yocum'}, {'authorId': '16489526', 'name': 'W. Kemp'}, {'authorId': '46786576', 'name': 'J. Bosch'}, {'authorId': '3008998', 'name': 'R. Waterhouse'}, {'authorId': '2182484', 'name': 'E. Zdobnov'}, {'authorId': '4791609', 'name': 'Eckart Stolle'}, {'authorId': '1783900', 'name': 'F. B. Kraus'}, {'authorId': '7004510', 'name': 'Sophie Helbing'}, {'authorId': '47046110', 'name': 'R. Moritz'}, {'authorId': '5303807', 'name': 'K. Glastad'}, {'authorId': '4745746', 'name': 'B. Hunt'}, {'authorId': '3887545', 'name': 'M. Goodisman'}, {'authorId': '46526157', 'name': 'F. Hauser'}, {'authorId': '4223070', 'name': 'C. Grimmelikhuijzen'}, {'authorId': '3119104', 'name': 'D. G. Pinheiro'}, {'authorId': '2083804782', 'name': 'F. M. Nunes'}, {'authorId': '4272557', 'name': 'M. P. Soares'}, {'authorId': '82826483', 'name': 'É. D. Tanaka'}, {'authorId': '90243733', 'name': 'Z. Simões'}, {'authorId': '6807475', 'name': 'K. Hartfelder'}, {'authorId': '2488538', 'name': 'J. Evans'}, {'authorId': '4959699', 'name': 'S. Barribeau'}, {'authorId': '2108887689', 'name': 'Reed M. Johnson'}, {'authorId': '87813874', 'name': 'J. Massey'}, {'authorId': '1795501', 'name': 'B. Southey'}, {'authorId': '47357198', 'name': 'M. Hasselmann'}, {'authorId': '2307512', 'name': 'D. Hamacher'}, {'authorId': '32447626', 'name': 'M. Biewer'}, {'authorId': '38221616', 'name': 'C. Kent'}, {'authorId': '37301230', 'name': 'A. Zayed'}, {'authorId': '3077932', 'name': 'C. Blatti'}, {'authorId': '2162164055', 'name': 'S. Sinha'}, {'authorId': '2228046255', 'name': 'J. Johnston'}, {'authorId': '4934449', 'name': 'Shawn J. Hanrahan'}, {'authorId': '4733082', 'name': 'Sarah D. Kocher'}, {'authorId': '40579687', 'name': 'Jun Wang'}, {'authorId': '2779588', 'name': 'G. Robinson'}, {'authorId': '2166645736', 'name': 'Guojie Zhang'}]}, {'paperId': '08815884a34551e292aa4b408500c201109bfd97', 'title': 'Slip pulse and resonance of the Kathmandu basin during the 2015 Gorkha earthquake, Nepal', 'abstract': 'The bigger they are, the harder they fall The magnitude 7.8 Gorkha earthquake hit Nepal on 25 April 2015. The earthquake killed thousands and caused great damage. Galetzka et al. determined how the fault that caused this earthquake ruptured. The rupture showed a smooth slip pulse 20 km wide that moved eastward along the fault over about 6 s. The nature of the rupture limited damage to regular dwellings but generated shaking that collapsed taller structures. Science, this issue p. 1091 Continuous GPS and InSAR measurements record slip on the fault responsible for the 2015 Mw 7.8 Gorkha earthquake in Nepal. Detailed geodetic imaging of earthquake ruptures enhances our understanding of earthquake physics and associated ground shaking. The 25 April 2015 moment magnitude 7.8 earthquake in Gorkha, Nepal was the first large continental megathrust rupture to have occurred beneath a high-rate (5-hertz) Global Positioning System (GPS) network. We used GPS and interferometric synthetic aperture radar data to model the earthquake rupture as a slip pulse ~20 kilometers in width, ~6 seconds in duration, and with a peak sliding velocity of 1.1 meters per second, which propagated toward the Kathmandu basin at ~3.3 kilometers per second over ~140 kilometers. The smooth slip onset, indicating a large (~5-meter) slip-weakening distance, caused moderate ground shaking at high frequencies (>1 hertz; peak ground acceleration, ~16% of Earth’s gravity) and minimized damage to vernacular dwellings. Whole-basin resonance at a period of 4 to 5 seconds caused the collapse of tall structures, including cultural artifacts.', 'year': 2015, 'citationCount': 292, 'authors': [{'authorId': '4424939', 'name': 'J. Galetzka'}, {'authorId': '97318651', 'name': 'D. Melgar'}, {'authorId': '6525824', 'name': 'J. Genrich'}, {'authorId': '2067256', 'name': 'J. Geng'}, {'authorId': '50335261', 'name': 'S. Owen'}, {'authorId': '35220108', 'name': 'E. Lindsey'}, {'authorId': '40511842', 'name': 'Xiaohua Xu'}, {'authorId': '8636234', 'name': 'Y. Bock'}, {'authorId': '2991483', 'name': 'J. Avouac'}, {'authorId': '2991483', 'name': 'J. Avouac'}, {'authorId': '31647886', 'name': 'L. Adhikari'}, {'authorId': '46256223', 'name': 'B. Upreti'}, {'authorId': '1404929646', 'name': 'B. Pratt-Sitaula'}, {'authorId': '72539811', 'name': 'T. Bhattarai'}, {'authorId': '41220903', 'name': 'B. P. Sitaula'}, {'authorId': '32767683', 'name': 'A. Moore'}, {'authorId': '5757269', 'name': 'K. Hudnut'}, {'authorId': '34772022', 'name': 'W. Szeliga'}, {'authorId': '12632658', 'name': 'J. Normandeau'}, {'authorId': '112908957', 'name': 'M. Fend'}, {'authorId': '5403146', 'name': 'M. Flouzat'}, {'authorId': '144290187', 'name': 'L. Bollinger'}, {'authorId': '2061030288', 'name': 'P. Shrestha'}, {'authorId': '31545592', 'name': 'B. P. Koirala'}, {'authorId': '13457014', 'name': 'U. Gautam'}, {'authorId': '8476221', 'name': 'M. Bhatterai'}, {'authorId': '94190859', 'name': 'R. Gupta'}, {'authorId': '104873910', 'name': 'T. Kandel'}, {'authorId': '8130744', 'name': 'C. Timsina'}, {'authorId': '5548401', 'name': 'S. Sapkota'}, {'authorId': '4446887', 'name': 'S. Rajaure'}, {'authorId': '2065905236', 'name': 'N. Maharjan'}]}, {'paperId': '26c414be169fcb7ec90f18696beb5729a3815347', 'title': 'Chemiexcitation of melanin derivatives induces DNA photoproducts long after UV exposure', 'abstract': \"The dark side of melanin exposed Sun worshippers may have more to worry about than the DNA damage that occurs while they're relaxing on the beach. It seems that the DNA photoproducts responsible for cancer-causing mutations in skin cells continue to be generated for hours after sunlight exposure. Premi et al. find that a key mediator of this delayed damage is melanin, a pigment thought to protect against cancer (see the Perspective by Taylor). They propose a “chemiexcitation” model in which reactive oxygen and nitrogen species induced by ultraviolet light excite an electron in melanin fragments. This energy is then transferred to DNA, inducing the same damage as ultraviolet light, but in the dark. Conceivably, this energy could be dissipated by adding quenchers to sunscreens. Science, this issue p. 842; see also p. 824 Melanin, a pigment thought to protect against skin cancer, contributes to DNA damage occurring after ultraviolet exposure. [Also see Perspective by Taylor] Mutations in sunlight-induced melanoma arise from cyclobutane pyrimidine dimers (CPDs), DNA photoproducts that are typically created picoseconds after an ultraviolet (UV) photon is absorbed at thymine or cytosine. We found that in melanocytes, CPDs are generated for >3 hours after exposure to UVA, a major component of the radiation in sunlight and in tanning beds. These “dark CPDs” constitute the majority of CPDs and include the cytosine-containing CPDs that initiate UV-signature C→T mutations. Dark CPDs arise when UV-induced reactive oxygen and nitrogen species combine to excite an electron in fragments of the pigment melanin. This creates a quantum triplet state that has the energy of a UV photon but induces CPDs by energy transfer to DNA in a radiation-independent manner. Melanin may thus be carcinogenic as well as protective against cancer. These findings also validate the long-standing suggestion that chemically generated excited electronic states are relevant to mammalian biology.\", 'year': 2015, 'citationCount': 386, 'authors': [{'authorId': '3672160', 'name': 'S. Premi'}, {'authorId': '14204647', 'name': 'S. Wallisch'}, {'authorId': '6987851', 'name': 'C. M. Mano'}, {'authorId': '3970174', 'name': 'A. Weiner'}, {'authorId': '4812673', 'name': 'Antonella Bacchiocchi'}, {'authorId': '4426667', 'name': 'K. Wakamatsu'}, {'authorId': '143665266', 'name': 'E. Bechara'}, {'authorId': '7741232', 'name': 'R. Halaban'}, {'authorId': '4130691', 'name': 'T. Douki'}, {'authorId': '1939904', 'name': 'D. Brash'}]}, {'paperId': '3cb46fad2a71ab3958eac67e0c0396524b8ac5b0', 'title': 'Nanomaterials in transistors: From high-performance to thin-film applications', 'abstract': 'Improving transistors with nanomaterials High-performance silicon transistors and thin-film transistors used in display technologies are fundamentally limited to miniaturization. Incorporating nanomaterials—such as carbon nanotubes, graphene, and related two-dimensional materials like molybdenum disulfide—into these devices as gate materials may circumvent some of these limitations. Franklin reviews the opportunities and challenges for incorporating nanomaterials into transistors to improve performance. Because high-performance transistors are distinct from thin-film transistors, incorporating them into flexible or transparent platforms raises new challenges. Science, this issue 10.1126/science.aab2750 BACKGROUND Transistors are one of the most enabling “hidden” technologies of all time and have facilitated the development of computers, the Internet, thin mobile displays, and much more. Silicon, which has been the material of choice for transistors in nearly every application for decades, is now reaching the fundamental limits to what it can offer for future transistor technologies. The newest display technologies are already turning to metal oxide materials, such as indium gallium zinc oxide (IGZO), for the improvements needed to drive organic light-emitting diodes. Ranging from applications such as display backplanes to high-performance microprocessors for servers, nanomaterials offer lasting advantages for the coming decades of transistor technologies. In this Review, the advantages of nanomaterials are discussed in the context of different transistor applications, along with the breakthroughs needed for nanomaterial transistors to enable the next generation of technological advancement. ADVANCES About 15 years ago, nanomaterials began receiving focused attention for transistors. Carbon nanotubes—molecules consisting of a cylindrical single layer of carbon atoms arranged in a hexagonal lattice—were the first to be given serious consideration, and their benefits quickly became widely acclaimed. Given their ability to transport electrical current with near-zero resistance, even at room temperature, the explosion of interest in nanotubes for electronics was understandable. Graphene, a related allotrope of carbon, benefited from the expansive interest carbon nanotubes had created for nanomaterial electronics. Although graphene transistors eventually proved less viable for digital applications, owing to the absence of an energy band gap, the excitement over graphene ushered in a complete revolution of interest in similar two-dimensional materials. Now, transition metal dichalcogenides and the so-called X-ene family of nanomaterials (e.g., silicence, phosphorene) dominate the attention of the nanoelectronics community. Hardly a day goes by without a paper being published on some advancement related to the use of nanomaterials in transistors. Hence, this Review focuses on how to keep such progress in the proper context with respect to the target transistor application, as well as the consideration of nanomaterials for completely new application spaces. OUTLOOK The benefits and practicality differ for each nanomaterial, and varied amounts of progress have been made in considering each of them for transistors. In just a few short years, thousands of papers have been published on improving synthesis or demonstrating simple functions of the newer nanomaterials. However, reflection on whether their newness translates to actual superiority over other options is warranted. Clearly, all of the nanomaterial possibilities offer certain advantages for future transistor technologies, but some do so with fewer caveats than others. Future research will benefit from keeping scientific advancement of nanomaterial transistors in line with end-goal deliverables. Overall, considering that only 15 years have elapsed since the study of nanomaterials for transistors began in earnest, the toolbox of available options and the developments toward overcoming challenges are promising. Technologies enabled by high-performance and thin-film transistors over the past 25 years. (Top) Silicon transistors have driven the microprocessors used in computational devices ranging from low-power gadgets to large servers. (Bottom) Various forms of cheaper silicon enabled the display revolution, now being shared by IGZO. (Right) Nanomaterials may be the next transistor material for enabling a new generation of technologies. CREDITS: Old computer: Wikimedia Commons/Creative Commons; Other computer: J.Dray/Flickr; Laptop: C. Berkeley/Flickr; Supercomputer: Wikimedia Commons/Creative Commons; Smartphone: Pixabay/Creative Commons; Camera: DAYJOY/Flickr; FLAT SCREEN TV: AV Hire London/Flickr; Curved TV: SamsungTomorrow/Flickr For more than 50 years, silicon transistors have been continuously shrunk to meet the projections of Moore’s law but are now reaching fundamental limits on speed and power use. With these limits at hand, nanomaterials offer great promise for improving transistor performance and adding new applications through the coming decades. With different transistors needed in everything from high-performance servers to thin-film display backplanes, it is important to understand the targeted application needs when considering new material options. Here the distinction between high-performance and thin-film transistors is reviewed, along with the benefits and challenges to using nanomaterials in such transistors. In particular, progress on carbon nanotubes, as well as graphene and related materials (including transition metal dichalcogenides and X-enes), outlines the advances and further research needed to enable their use in transistors for high-performance computing, thin films, or completely new technologies such as flexible and transparent devices.', 'year': 2015, 'citationCount': 449, 'authors': [{'authorId': '3752052', 'name': 'A. Franklin'}]}, {'paperId': 'a9b91fed8118feb8e75f6459812155f2e45f200b', 'title': 'Dynamic profiling of the protein life cycle in response to pathogens', 'abstract': 'How the immune system readies for battle Although gene expression is tightly controlled at both the RNA and protein levels, the quantitative contribution of each step, especially during dynamic responses, remains largely unknown. Indeed, there has been much debate whether changes in RNA level contribute substantially to protein-level regulation. Jovanovic et al. built a genome-scale model of the temporal dynamics of differential protein expression during the stimulation of immunological dendritic cells (see the Perspective by Li and Biggin). Newly stimulated functions involved the up-regulation of specific RNAs and concomitant increases in the levels of the proteins they encode, whereas housekeeping functions were regulated posttranscriptionally at the protein level. Science, this issue 10.1126/science.1259038; see also p. 1066 Levels of “housekeeping” proteins are maintained directly, but those of immune response proteins depend on more transcription. [Also see Perspective by Li and Biggin] INTRODUCTION Mammalian gene expression is tightly controlled through the interplay between the RNA and protein life cycles. Although studies of individual genes have shown that regulation of each of these processes is important for correct protein expression, the quantitative contribution of each step to changes in protein expression levels remains largely unknown and much debated. Many studies have attempted to address this question in the context of steady-state protein levels, and comparing steady-state RNA and protein abundances has indicated a considerable discrepancy between RNA and protein levels. In contrast, only a few studies have attempted to shed light on how changes in each of these processes determine differential protein expression—either relative (ratios) or absolute (differences)—during dynamic responses, and only one recent report has attempted to quantitate each process. Understanding these contributions to a dynamic response on a systems scale is essential both for deciphering how cells deploy regulatory processes to accomplish physiological changes and for discovering key molecular regulators controlling each process. RATIONALE We developed an integrated experimental and computational strategy to quantitatively assess how protein levels are maintained in the context of a dynamic response and applied it to the model response of mouse immune bone marrow–derived dendritic cells (DCs) to stimulation with lipopolysaccharide (LPS). We used a modified pulsed-SILAC (stable isotope labeling with amino acids in cell culture) approach to track newly synthesized and previously labeled proteins over the first 12 hours of the response. In addition, we independently measured replicate RNA-sequencing profiles under the same conditions. We devised a computational strategy to infer per-mRNA translation rates and protein degradation rates at each time point from the temporal transcriptional profiles and pulsed-SILAC proteomics data. This allowed us to build a genome-scale quantitative model of the temporal dynamics of differential protein expression in DCs responding to LPS. RESULTS We found that before stimulation, mRNA levels contribute to overall protein expression levels more than double the combined contribution of protein translation and degradation rates. Upon LPS stimulation, changes in mRNA abundance play an even more dominant role in dynamic changes in protein levels, especially in immune response genes. Nevertheless, several protein modules—especially the preexisting proteome of proteins performing basic cellular functions—are predominantly regulated in stimulated cells at the level of protein translation or degradation, accounting for over half of the absolute change in protein molecules in the cell. In particular, despite the repression of their transcripts, the level of many proteins in the translational machinery is up-regulated upon LPS stimulation because of significantly increased translation rates, and elevated protein degradation of mitochondrial proteins plays a central role in remodeling cellular energy metabolism. CONCLUSIONS Our results support a model in which the induction of novel cellular functions is primarily driven through transcriptional changes, whereas regulation of protein production or degradation updates the levels of preexisting functions as required for an activated state. Our approach for building quantitative genome-scale models of the temporal dynamics of protein expression is broadly applicable to other dynamic systems. Dynamic protein expression regulation in dendritic cells upon stimulation with LPS. We developed an integrated experimental and computational strategy to quantitatively assess how protein levels are maintained in the context of a dynamic response. Our results support a model in which the induction of novel cellular functions is primarily driven through transcriptional changes, whereas regulation of protein production or degradation updates the levels of preexisting functions. Protein expression is regulated by the production and degradation of messenger RNAs (mRNAs) and proteins, but their specific relationships remain unknown. We combine measurements of protein production and degradation and mRNA dynamics so as to build a quantitative genomic model of the differential regulation of gene expression in lipopolysaccharide-stimulated mouse dendritic cells. Changes in mRNA abundance play a dominant role in determining most dynamic fold changes in protein levels. Conversely, the preexisting proteome of proteins performing basic cellular functions is remodeled primarily through changes in protein production or degradation, accounting for more than half of the absolute change in protein molecules in the cell. Thus, the proteome is regulated by transcriptional induction for newly activated cellular functions and by protein life-cycle changes for remodeling of preexisting functions.', 'year': 2015, 'citationCount': 392, 'authors': [{'authorId': '48589378', 'name': 'M. Jovanović'}, {'authorId': '47454189', 'name': 'Michael S. Rooney'}, {'authorId': '4903346', 'name': 'Philipp Mertins'}, {'authorId': '1977398', 'name': 'Dariusz Przybylski'}, {'authorId': '152208347', 'name': 'Nicolas Chevrier'}, {'authorId': '50285728', 'name': 'R. Satija'}, {'authorId': '10273378', 'name': 'Edwin H. Rodriguez'}, {'authorId': '8356406', 'name': 'Alexander P. Fields'}, {'authorId': '40434210', 'name': 'Schraga Schwartz'}, {'authorId': '3556352', 'name': 'R. Raychowdhury'}, {'authorId': '4556276', 'name': 'Maxwell R Mumbach'}, {'authorId': '5974082', 'name': 'T. Eisenhaure'}, {'authorId': '5405703', 'name': 'M. Rabani'}, {'authorId': '4691526', 'name': 'David G Gennert'}, {'authorId': '5803380', 'name': 'Diana Lu'}, {'authorId': '47128726', 'name': 'T. Delorey'}, {'authorId': '3168827', 'name': 'J. Weissman'}, {'authorId': '144298908', 'name': 'S. Carr'}, {'authorId': '3860533', 'name': 'N. Hacohen'}, {'authorId': '144416712', 'name': 'A. Regev'}]}, {'paperId': '1585b76d97d4ee526471570cc92b582c8457ba4b', 'title': 'Nanophotonics: Shrinking light-based technology', 'abstract': 'The study of light at the nanoscale has become a vibrant field of research, as researchers now master the flow of light at length scales far below the optical wavelength, largely surpassing the classical limits imposed by diffraction. Using metallic and dielectric nanostructures precisely sculpted into two-dimensional (2D) and 3D nanoarchitectures, light can be scattered, refracted, confined, filtered, and processed in fascinating new ways that are impossible to achieve with natural materials and in conventional geometries. This control over light at the nanoscale has not only unveiled a plethora of new phenomena but has also led to a variety of relevant applications, including new venues for integrated circuitry, optical computing, solar, and medical technologies, setting high expectations for many novel discoveries in the years to come.', 'year': 2015, 'citationCount': 445, 'authors': [{'authorId': '6380838', 'name': 'A. Koenderink'}, {'authorId': '2186738926', 'name': 'A. Alú'}, {'authorId': '25343260', 'name': 'A. Polman'}]}, {'paperId': 'bb1081e7fe91e49d6d54b9eb42a25f08cc06b0ef', 'title': 'Group 3 innate lymphoid cells mediate intestinal selection of commensal bacteria–specific CD4+ T cells', 'abstract': 'Innate lymphoid cells keep gut T cells in check Trillions of bacteria inhabit our guts. So do many types of immune cells, including T cells, which might be expected to attack these bacteria. How, then, do our bodies manage to keep the peace? Working in mice, Hepworth et al. report one such mechanism. A population of immune cells, called innate lymphoid cells, directly killed CD4+ T cells that react to commensal gut microbes. Some of the specifics of this process parallel how the immune system keeps developing self-reactive T cells in check in the thymus. Furthermore, this peacekeeping process may be disrupted in children with inflammatory bowel disease. Science, this issue p. 1031 Innate lymphoid cells delete commensal bacteria–specific CD4+ T cells from the intestine in mice. Inflammatory CD4+ T cell responses to self or commensal bacteria underlie the pathogenesis of autoimmunity and inflammatory bowel disease (IBD), respectively. Although selection of self-specific T cells in the thymus limits responses to mammalian tissue antigens, the mechanisms that control selection of commensal bacteria–specific T cells remain poorly understood. Here, we demonstrate that group 3 innate lymphoid cell (ILC3)–intrinsic expression of major histocompatibility complex class II (MHCII) is regulated similarly to thymic epithelial cells and that MHCII+ ILC3s directly induce cell death of activated commensal bacteria–specific T cells. Further, MHCII on colonic ILC3s was reduced in pediatric IBD patients. Collectively, these results define a selection pathway for commensal bacteria–specific CD4+ T cells in the intestine and suggest that this process is dysregulated in human IBD.', 'year': 2015, 'citationCount': 390, 'authors': [{'authorId': '3015665', 'name': 'M. Hepworth'}, {'authorId': '3306458', 'name': 'Thomas C. Fung'}, {'authorId': '33150279', 'name': 'Samuel H. Masur'}, {'authorId': '5362885', 'name': 'J. Kelsen'}, {'authorId': '35639694', 'name': 'F. McConnell'}, {'authorId': '4204825', 'name': 'J. Dubrot'}, {'authorId': '2099622', 'name': 'D. Withers'}, {'authorId': '4533784', 'name': 'S. Hugues'}, {'authorId': '2689185', 'name': 'M. Farrar'}, {'authorId': '144538049', 'name': 'W. Reith'}, {'authorId': '5133043', 'name': 'G. Eberl'}, {'authorId': '5247138', 'name': 'R. Baldassano'}, {'authorId': '5284820', 'name': 'T. Laufer'}, {'authorId': '31931780', 'name': 'C. Elson'}, {'authorId': '5771393', 'name': 'Gregory F. Sonnenberg'}]}, {'paperId': '98ef072d4656ba9fa11b22900fc7e9c44c647706', 'title': 'The Symbiodinium kawagutii genome illuminates dinoflagellate gene expression and coral symbiosis', 'abstract': 'Symbionts are adapted to work with corals Many corals have formed mutualistic associations with dinoflagellate symbionts, which are thought to provide nutrients and other benefits. To examine the underlying genetics of this association, S. Lin et al. sequenced the genome of the endosymbiont dinoflagellate Symbiodinium kawagutii. The genome includes gene number expansions and encodes microRNAs that show complementarity to genes within the coral genome. Such microRNAs may be involved in regulating coral genes. Furthermore, coral and S. kawagutii appear to share homologs of genes encoding specific nutrient transporters. The findings shed light on how symbiosis is established and maintained between dinoflagellates and corals. Science, this issue p. 691 The genome of the coral symbiont Symbiodinium reveals fundamental aspects of the coral-alga symbiosis. Dinoflagellates are important components of marine ecosystems and essential coral symbionts, yet little is known about their genomes. We report here on the analysis of a high-quality assembly from the 1180-megabase genome of Symbiodinium kawagutii. We annotated protein-coding genes and identified Symbiodinium-specific gene families. No whole-genome duplication was observed, but instead we found active (retro)transposition and gene family expansion, especially in processes important for successful symbiosis with corals. We also documented genes potentially governing sexual reproduction and cyst formation, novel promoter elements, and a microRNA system potentially regulating gene expression in both symbiont and coral. We found biochemical complementarity between genomes of S. kawagutii and the anthozoan Acropora, indicative of host-symbiont coevolution, providing a resource for studying the molecular basis and evolution of coral symbiosis.', 'year': 2015, 'citationCount': 358, 'authors': [{'authorId': '1720742921', 'name': 'Senjie Lin'}, {'authorId': '38165443', 'name': 'Shifeng Cheng'}, {'authorId': '1398290474', 'name': 'Bo Song'}, {'authorId': '144137667', 'name': 'Xiao Zhong'}, {'authorId': '145122731', 'name': 'Xin Lin'}, {'authorId': '48624658', 'name': 'Wujiao Li'}, {'authorId': '1410384558', 'name': 'Ling Li'}, {'authorId': '121310673', 'name': 'Yaqun Zhang'}, {'authorId': '51505924', 'name': 'Huan Zhang'}, {'authorId': '89744571', 'name': 'Zhiliang Ji'}, {'authorId': '48629710', 'name': 'M. Cai'}, {'authorId': '38109162', 'name': 'Yunyun Zhuang'}, {'authorId': '5962195', 'name': 'Xinguo Shi'}, {'authorId': '5795772', 'name': 'Lingxiao Lin'}, {'authorId': '2153513358', 'name': 'Lu Wang'}, {'authorId': '48708968', 'name': 'Zhaobao Wang'}, {'authorId': '2230105370', 'name': 'Xin Liu'}, {'authorId': '2112274376', 'name': 'Sheng-li Yu'}, {'authorId': '47614770', 'name': 'Peng Zeng'}, {'authorId': '2065511559', 'name': 'H. Hao'}, {'authorId': '2054839914', 'name': 'Q. Zou'}, {'authorId': '80042641', 'name': 'Chengxuan Chen'}, {'authorId': '152998435', 'name': 'Yanjun Li'}, {'authorId': '2155514105', 'name': 'Ying Wang'}, {'authorId': '2115783836', 'name': 'Chunyan Xu'}, {'authorId': '2144278725', 'name': 'S. Meng'}, {'authorId': '2162171547', 'name': 'Xun Xu'}, {'authorId': '40579687', 'name': 'Jun Wang'}, {'authorId': '40566506', 'name': 'Huanming Yang'}, {'authorId': '1875961', 'name': 'D. Campbell'}, {'authorId': '6722875', 'name': 'N. Sturm'}, {'authorId': '1398757639', 'name': 'S. Dagenais-Bellefeuille'}, {'authorId': '145233479', 'name': 'D. Morse'}]}, {'paperId': 'a490336788276bb1cd0e0b18f62f72010e1f43f5', 'title': 'Antimicrobial peptide resistance mediates resilience of prominent gut commensals during inflammation', 'abstract': \"Gut microbes resist inflammation It is vital to human well-being that our gut microbiota can be distinguished from harmful, but often very similar, organisms. Cullen et al. begin to analyze how one dominant symbiont, Bacteroidetes thetaiotaomicron, does this. Our guts release potent antimicrobial peptides when we become infected with pathogenic bacteria such as salmonella, but these symbionts make an outer lipopolysaccharide coat that differs from those of pathogens by only one phosphate molecule. Enzymatic removal of this group is enough to confer resistance to the host's immune response and allow the symbionts to escape damage. Science, this issue p. 170 Gut commensals avoid elimination during inflammation via a single phosphate in lipopolysaccharide. Resilience to host inflammation and other perturbations is a fundamental property of gut microbial communities, yet the underlying mechanisms are not well understood. We have found that human gut microbes from all dominant phyla are resistant to high levels of inflammation-associated antimicrobial peptides (AMPs) and have identified a mechanism for lipopolysaccharide (LPS) modification in the phylum Bacteroidetes that increases AMP resistance by four orders of magnitude. Bacteroides thetaiotaomicron mutants that fail to remove a single phosphate group from their LPS were displaced from the microbiota during inflammation triggered by pathogen infection. These findings establish a mechanism that determines the stability of prominent members of a healthy microbiota during perturbation.\", 'year': 2015, 'citationCount': 302, 'authors': [{'authorId': '4531670', 'name': 'T. W. Cullen'}, {'authorId': '3963577', 'name': 'W. B. Schofield'}, {'authorId': '34717499', 'name': 'N. A. Barry'}, {'authorId': '11998377', 'name': 'E. E. Putnam'}, {'authorId': '5594197', 'name': 'E. Rundell'}, {'authorId': '144623783', 'name': 'M. Trent'}, {'authorId': '5781125', 'name': 'P. Degnan'}, {'authorId': '32574524', 'name': 'C. Booth'}, {'authorId': '2219967279', 'name': 'H. Yu'}, {'authorId': '144958203', 'name': 'A. Goodman'}]}, {'paperId': '1469da4446aaa4785b0c76ca1527f76efbdd34ec', 'title': 'Stable inhibitory activity of regulatory T cells requires the transcription factor Helios', 'abstract': 'How T cells maintain their identity Although best known for their pathogen-fighting prowess, T lymphocytes also ensure that the immune response does not run amok. A subset of T cells called regulatory T cells (Tregs) performs this function by, for example, making sure T cells only attack pathogens and not self. T cells can exhibit plasticity in their functions in the face of an inflammatory stimulus. Kim et al. sought to identify the molecules that ensure the stable maintenance of Tregs. Using genetically modified mice, they found that both CD4+ and CD8+ Tregs require the transcription factor Helios to stably maintain their identity. Science, this issue p. 334 Regulatory T cells require the transcription factor Helios to stably maintain their identity. The maintenance of immune homeostasis requires regulatory T cells (Tregs). Given their intrinsic self-reactivity, Tregs must stably maintain a suppressive phenotype to avoid autoimmunity. We report that impaired expression of the transcription factor (TF) Helios by FoxP3+ CD4 and Qa-1–restricted CD8 Tregs results in defective regulatory activity and autoimmunity in mice. Helios-deficient Tregs develop an unstable phenotype during inflammatory responses characterized by reduced FoxP3 expression and increased effector cytokine expression secondary to diminished activation of the STAT5 pathway. CD8 Tregs also require Helios-dependent STAT5 activation for survival and to prevent terminal T cell differentiation. The definition of Helios as a key transcription factor that stabilizes Tregs in the face of inflammatory responses provides a genetic explanation for a core property of Tregs.', 'year': 2015, 'citationCount': 304, 'authors': [{'authorId': '2155254510', 'name': 'Hye-Jung Kim'}, {'authorId': '6202837', 'name': 'R. A. Barnitz'}, {'authorId': '5853769', 'name': 'Taras Kreslavsky'}, {'authorId': '7644721', 'name': 'Flavian D. Brown'}, {'authorId': '5976390', 'name': 'H. Moffett'}, {'authorId': '2566050', 'name': 'M. Lemieux'}, {'authorId': '5914812', 'name': 'Yasemin Kaygusuz'}, {'authorId': '34151365', 'name': 'T. Meissner'}, {'authorId': '6835064', 'name': 'T. Holderried'}, {'authorId': '2148763858', 'name': 'Susan Chan'}, {'authorId': '38486988', 'name': 'P. Kastner'}, {'authorId': '144414260', 'name': 'W. Haining'}, {'authorId': '145472731', 'name': 'H. Cantor'}]}]}\n",
      "Research field A Iteration:  0\n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "\u001b[1;32m/home/linuxuser/spiced-academy/data_science_capstone/convergence_oracle/Playground_KT/fetching/0_Semantic_API_kt/0_Fetching_Data_From_SemanticScholar_alp.ipynb Cell 5\u001b[0m line \u001b[0;36m4\n\u001b[1;32m     <a href='vscode-notebook-cell://wsl%2Bubuntu/home/linuxuser/spiced-academy/data_science_capstone/convergence_oracle/Playground_KT/fetching/0_Semantic_API_kt/0_Fetching_Data_From_SemanticScholar_alp.ipynb#W3sdnNjb2RlLXJlbW90ZQ%3D%3D?line=35'>36</a>\u001b[0m params \u001b[39m=\u001b[39m {\n\u001b[1;32m     <a href='vscode-notebook-cell://wsl%2Bubuntu/home/linuxuser/spiced-academy/data_science_capstone/convergence_oracle/Playground_KT/fetching/0_Semantic_API_kt/0_Fetching_Data_From_SemanticScholar_alp.ipynb#W3sdnNjb2RlLXJlbW90ZQ%3D%3D?line=36'>37</a>\u001b[0m     \u001b[39m'\u001b[39m\u001b[39mquery\u001b[39m\u001b[39m'\u001b[39m: alphabet_letters[k],  \u001b[39m# Use the first research field from the list\u001b[39;00m\n\u001b[1;32m     <a href='vscode-notebook-cell://wsl%2Bubuntu/home/linuxuser/spiced-academy/data_science_capstone/convergence_oracle/Playground_KT/fetching/0_Semantic_API_kt/0_Fetching_Data_From_SemanticScholar_alp.ipynb#W3sdnNjb2RlLXJlbW90ZQ%3D%3D?line=37'>38</a>\u001b[0m     \u001b[39m'\u001b[39m\u001b[39mvenue\u001b[39m\u001b[39m'\u001b[39m: journal[\u001b[39m0\u001b[39m],\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m     <a href='vscode-notebook-cell://wsl%2Bubuntu/home/linuxuser/spiced-academy/data_science_capstone/convergence_oracle/Playground_KT/fetching/0_Semantic_API_kt/0_Fetching_Data_From_SemanticScholar_alp.ipynb#W3sdnNjb2RlLXJlbW90ZQ%3D%3D?line=41'>42</a>\u001b[0m     \u001b[39m'\u001b[39m\u001b[39mfields\u001b[39m\u001b[39m'\u001b[39m: \u001b[39m'\u001b[39m\u001b[39mtitle,authors,abstract,citationCount,year\u001b[39m\u001b[39m'\u001b[39m,\n\u001b[1;32m     <a href='vscode-notebook-cell://wsl%2Bubuntu/home/linuxuser/spiced-academy/data_science_capstone/convergence_oracle/Playground_KT/fetching/0_Semantic_API_kt/0_Fetching_Data_From_SemanticScholar_alp.ipynb#W3sdnNjb2RlLXJlbW90ZQ%3D%3D?line=42'>43</a>\u001b[0m }\n\u001b[1;32m     <a href='vscode-notebook-cell://wsl%2Bubuntu/home/linuxuser/spiced-academy/data_science_capstone/convergence_oracle/Playground_KT/fetching/0_Semantic_API_kt/0_Fetching_Data_From_SemanticScholar_alp.ipynb#W3sdnNjb2RlLXJlbW90ZQ%3D%3D?line=44'>45</a>\u001b[0m \u001b[39m# Make the GET request\u001b[39;00m\n\u001b[0;32m---> <a href='vscode-notebook-cell://wsl%2Bubuntu/home/linuxuser/spiced-academy/data_science_capstone/convergence_oracle/Playground_KT/fetching/0_Semantic_API_kt/0_Fetching_Data_From_SemanticScholar_alp.ipynb#W3sdnNjb2RlLXJlbW90ZQ%3D%3D?line=45'>46</a>\u001b[0m response \u001b[39m=\u001b[39m requests\u001b[39m.\u001b[39;49mget(api_url, params\u001b[39m=\u001b[39;49mparams)\n\u001b[1;32m     <a href='vscode-notebook-cell://wsl%2Bubuntu/home/linuxuser/spiced-academy/data_science_capstone/convergence_oracle/Playground_KT/fetching/0_Semantic_API_kt/0_Fetching_Data_From_SemanticScholar_alp.ipynb#W3sdnNjb2RlLXJlbW90ZQ%3D%3D?line=46'>47</a>\u001b[0m \u001b[39mprint\u001b[39m(\u001b[39m\"\u001b[39m\u001b[39mtotal amount\u001b[39m\u001b[39m\"\u001b[39m,response\u001b[39m.\u001b[39mjson())\n\u001b[1;32m     <a href='vscode-notebook-cell://wsl%2Bubuntu/home/linuxuser/spiced-academy/data_science_capstone/convergence_oracle/Playground_KT/fetching/0_Semantic_API_kt/0_Fetching_Data_From_SemanticScholar_alp.ipynb#W3sdnNjb2RlLXJlbW90ZQ%3D%3D?line=48'>49</a>\u001b[0m \u001b[39m# Check if the request was successful\u001b[39;00m\n",
      "File \u001b[0;32m~/spiced-academy/data_science_capstone/convergence_oracle/.venv/lib/python3.11/site-packages/requests/api.py:73\u001b[0m, in \u001b[0;36mget\u001b[0;34m(url, params, **kwargs)\u001b[0m\n\u001b[1;32m     62\u001b[0m \u001b[39mdef\u001b[39;00m \u001b[39mget\u001b[39m(url, params\u001b[39m=\u001b[39m\u001b[39mNone\u001b[39;00m, \u001b[39m*\u001b[39m\u001b[39m*\u001b[39mkwargs):\n\u001b[1;32m     63\u001b[0m \u001b[39m    \u001b[39m\u001b[39mr\u001b[39m\u001b[39m\"\"\"Sends a GET request.\u001b[39;00m\n\u001b[1;32m     64\u001b[0m \n\u001b[1;32m     65\u001b[0m \u001b[39m    :param url: URL for the new :class:`Request` object.\u001b[39;00m\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m     70\u001b[0m \u001b[39m    :rtype: requests.Response\u001b[39;00m\n\u001b[1;32m     71\u001b[0m \u001b[39m    \"\"\"\u001b[39;00m\n\u001b[0;32m---> 73\u001b[0m     \u001b[39mreturn\u001b[39;00m request(\u001b[39m\"\u001b[39;49m\u001b[39mget\u001b[39;49m\u001b[39m\"\u001b[39;49m, url, params\u001b[39m=\u001b[39;49mparams, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mkwargs)\n",
      "File \u001b[0;32m~/spiced-academy/data_science_capstone/convergence_oracle/.venv/lib/python3.11/site-packages/requests/api.py:59\u001b[0m, in \u001b[0;36mrequest\u001b[0;34m(method, url, **kwargs)\u001b[0m\n\u001b[1;32m     55\u001b[0m \u001b[39m# By using the 'with' statement we are sure the session is closed, thus we\u001b[39;00m\n\u001b[1;32m     56\u001b[0m \u001b[39m# avoid leaving sockets open which can trigger a ResourceWarning in some\u001b[39;00m\n\u001b[1;32m     57\u001b[0m \u001b[39m# cases, and look like a memory leak in others.\u001b[39;00m\n\u001b[1;32m     58\u001b[0m \u001b[39mwith\u001b[39;00m sessions\u001b[39m.\u001b[39mSession() \u001b[39mas\u001b[39;00m session:\n\u001b[0;32m---> 59\u001b[0m     \u001b[39mreturn\u001b[39;00m session\u001b[39m.\u001b[39;49mrequest(method\u001b[39m=\u001b[39;49mmethod, url\u001b[39m=\u001b[39;49murl, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mkwargs)\n",
      "File \u001b[0;32m~/spiced-academy/data_science_capstone/convergence_oracle/.venv/lib/python3.11/site-packages/requests/sessions.py:589\u001b[0m, in \u001b[0;36mSession.request\u001b[0;34m(self, method, url, params, data, headers, cookies, files, auth, timeout, allow_redirects, proxies, hooks, stream, verify, cert, json)\u001b[0m\n\u001b[1;32m    584\u001b[0m send_kwargs \u001b[39m=\u001b[39m {\n\u001b[1;32m    585\u001b[0m     \u001b[39m\"\u001b[39m\u001b[39mtimeout\u001b[39m\u001b[39m\"\u001b[39m: timeout,\n\u001b[1;32m    586\u001b[0m     \u001b[39m\"\u001b[39m\u001b[39mallow_redirects\u001b[39m\u001b[39m\"\u001b[39m: allow_redirects,\n\u001b[1;32m    587\u001b[0m }\n\u001b[1;32m    588\u001b[0m send_kwargs\u001b[39m.\u001b[39mupdate(settings)\n\u001b[0;32m--> 589\u001b[0m resp \u001b[39m=\u001b[39m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49msend(prep, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49msend_kwargs)\n\u001b[1;32m    591\u001b[0m \u001b[39mreturn\u001b[39;00m resp\n",
      "File \u001b[0;32m~/spiced-academy/data_science_capstone/convergence_oracle/.venv/lib/python3.11/site-packages/requests/sessions.py:703\u001b[0m, in \u001b[0;36mSession.send\u001b[0;34m(self, request, **kwargs)\u001b[0m\n\u001b[1;32m    700\u001b[0m start \u001b[39m=\u001b[39m preferred_clock()\n\u001b[1;32m    702\u001b[0m \u001b[39m# Send the request\u001b[39;00m\n\u001b[0;32m--> 703\u001b[0m r \u001b[39m=\u001b[39m adapter\u001b[39m.\u001b[39;49msend(request, \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mkwargs)\n\u001b[1;32m    705\u001b[0m \u001b[39m# Total elapsed time of the request (approximately)\u001b[39;00m\n\u001b[1;32m    706\u001b[0m elapsed \u001b[39m=\u001b[39m preferred_clock() \u001b[39m-\u001b[39m start\n",
      "File \u001b[0;32m~/spiced-academy/data_science_capstone/convergence_oracle/.venv/lib/python3.11/site-packages/requests/adapters.py:486\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[0;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[1;32m    483\u001b[0m     timeout \u001b[39m=\u001b[39m TimeoutSauce(connect\u001b[39m=\u001b[39mtimeout, read\u001b[39m=\u001b[39mtimeout)\n\u001b[1;32m    485\u001b[0m \u001b[39mtry\u001b[39;00m:\n\u001b[0;32m--> 486\u001b[0m     resp \u001b[39m=\u001b[39m conn\u001b[39m.\u001b[39;49murlopen(\n\u001b[1;32m    487\u001b[0m         method\u001b[39m=\u001b[39;49mrequest\u001b[39m.\u001b[39;49mmethod,\n\u001b[1;32m    488\u001b[0m         url\u001b[39m=\u001b[39;49murl,\n\u001b[1;32m    489\u001b[0m         body\u001b[39m=\u001b[39;49mrequest\u001b[39m.\u001b[39;49mbody,\n\u001b[1;32m    490\u001b[0m         headers\u001b[39m=\u001b[39;49mrequest\u001b[39m.\u001b[39;49mheaders,\n\u001b[1;32m    491\u001b[0m         redirect\u001b[39m=\u001b[39;49m\u001b[39mFalse\u001b[39;49;00m,\n\u001b[1;32m    492\u001b[0m         assert_same_host\u001b[39m=\u001b[39;49m\u001b[39mFalse\u001b[39;49;00m,\n\u001b[1;32m    493\u001b[0m         preload_content\u001b[39m=\u001b[39;49m\u001b[39mFalse\u001b[39;49;00m,\n\u001b[1;32m    494\u001b[0m         decode_content\u001b[39m=\u001b[39;49m\u001b[39mFalse\u001b[39;49;00m,\n\u001b[1;32m    495\u001b[0m         retries\u001b[39m=\u001b[39;49m\u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mmax_retries,\n\u001b[1;32m    496\u001b[0m         timeout\u001b[39m=\u001b[39;49mtimeout,\n\u001b[1;32m    497\u001b[0m         chunked\u001b[39m=\u001b[39;49mchunked,\n\u001b[1;32m    498\u001b[0m     )\n\u001b[1;32m    500\u001b[0m \u001b[39mexcept\u001b[39;00m (ProtocolError, \u001b[39mOSError\u001b[39;00m) \u001b[39mas\u001b[39;00m err:\n\u001b[1;32m    501\u001b[0m     \u001b[39mraise\u001b[39;00m \u001b[39mConnectionError\u001b[39;00m(err, request\u001b[39m=\u001b[39mrequest)\n",
      "File \u001b[0;32m~/spiced-academy/data_science_capstone/convergence_oracle/.venv/lib/python3.11/site-packages/urllib3/connectionpool.py:790\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[0;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, preload_content, decode_content, **response_kw)\u001b[0m\n\u001b[1;32m    787\u001b[0m response_conn \u001b[39m=\u001b[39m conn \u001b[39mif\u001b[39;00m \u001b[39mnot\u001b[39;00m release_conn \u001b[39melse\u001b[39;00m \u001b[39mNone\u001b[39;00m\n\u001b[1;32m    789\u001b[0m \u001b[39m# Make the request on the HTTPConnection object\u001b[39;00m\n\u001b[0;32m--> 790\u001b[0m response \u001b[39m=\u001b[39m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_make_request(\n\u001b[1;32m    791\u001b[0m     conn,\n\u001b[1;32m    792\u001b[0m     method,\n\u001b[1;32m    793\u001b[0m     url,\n\u001b[1;32m    794\u001b[0m     timeout\u001b[39m=\u001b[39;49mtimeout_obj,\n\u001b[1;32m    795\u001b[0m     body\u001b[39m=\u001b[39;49mbody,\n\u001b[1;32m    796\u001b[0m     headers\u001b[39m=\u001b[39;49mheaders,\n\u001b[1;32m    797\u001b[0m     chunked\u001b[39m=\u001b[39;49mchunked,\n\u001b[1;32m    798\u001b[0m     retries\u001b[39m=\u001b[39;49mretries,\n\u001b[1;32m    799\u001b[0m     response_conn\u001b[39m=\u001b[39;49mresponse_conn,\n\u001b[1;32m    800\u001b[0m     preload_content\u001b[39m=\u001b[39;49mpreload_content,\n\u001b[1;32m    801\u001b[0m     decode_content\u001b[39m=\u001b[39;49mdecode_content,\n\u001b[1;32m    802\u001b[0m     \u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mresponse_kw,\n\u001b[1;32m    803\u001b[0m )\n\u001b[1;32m    805\u001b[0m \u001b[39m# Everything went great!\u001b[39;00m\n\u001b[1;32m    806\u001b[0m clean_exit \u001b[39m=\u001b[39m \u001b[39mTrue\u001b[39;00m\n",
      "File \u001b[0;32m~/spiced-academy/data_science_capstone/convergence_oracle/.venv/lib/python3.11/site-packages/urllib3/connectionpool.py:536\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[0;34m(self, conn, method, url, body, headers, retries, timeout, chunked, response_conn, preload_content, decode_content, enforce_content_length)\u001b[0m\n\u001b[1;32m    534\u001b[0m \u001b[39m# Receive the response from the server\u001b[39;00m\n\u001b[1;32m    535\u001b[0m \u001b[39mtry\u001b[39;00m:\n\u001b[0;32m--> 536\u001b[0m     response \u001b[39m=\u001b[39m conn\u001b[39m.\u001b[39;49mgetresponse()\n\u001b[1;32m    537\u001b[0m \u001b[39mexcept\u001b[39;00m (BaseSSLError, \u001b[39mOSError\u001b[39;00m) \u001b[39mas\u001b[39;00m e:\n\u001b[1;32m    538\u001b[0m     \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_raise_timeout(err\u001b[39m=\u001b[39me, url\u001b[39m=\u001b[39murl, timeout_value\u001b[39m=\u001b[39mread_timeout)\n",
      "File \u001b[0;32m~/spiced-academy/data_science_capstone/convergence_oracle/.venv/lib/python3.11/site-packages/urllib3/connection.py:461\u001b[0m, in \u001b[0;36mHTTPConnection.getresponse\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    458\u001b[0m \u001b[39mfrom\u001b[39;00m \u001b[39m.\u001b[39;00m\u001b[39mresponse\u001b[39;00m \u001b[39mimport\u001b[39;00m HTTPResponse\n\u001b[1;32m    460\u001b[0m \u001b[39m# Get the response from http.client.HTTPConnection\u001b[39;00m\n\u001b[0;32m--> 461\u001b[0m httplib_response \u001b[39m=\u001b[39m \u001b[39msuper\u001b[39;49m()\u001b[39m.\u001b[39;49mgetresponse()\n\u001b[1;32m    463\u001b[0m \u001b[39mtry\u001b[39;00m:\n\u001b[1;32m    464\u001b[0m     assert_header_parsing(httplib_response\u001b[39m.\u001b[39mmsg)\n",
      "File \u001b[0;32m~/.pyenv/versions/3.11.3/lib/python3.11/http/client.py:1375\u001b[0m, in \u001b[0;36mHTTPConnection.getresponse\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m   1373\u001b[0m \u001b[39mtry\u001b[39;00m:\n\u001b[1;32m   1374\u001b[0m     \u001b[39mtry\u001b[39;00m:\n\u001b[0;32m-> 1375\u001b[0m         response\u001b[39m.\u001b[39;49mbegin()\n\u001b[1;32m   1376\u001b[0m     \u001b[39mexcept\u001b[39;00m \u001b[39mConnectionError\u001b[39;00m:\n\u001b[1;32m   1377\u001b[0m         \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mclose()\n",
      "File \u001b[0;32m~/.pyenv/versions/3.11.3/lib/python3.11/http/client.py:318\u001b[0m, in \u001b[0;36mHTTPResponse.begin\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    316\u001b[0m \u001b[39m# read until we get a non-100 response\u001b[39;00m\n\u001b[1;32m    317\u001b[0m \u001b[39mwhile\u001b[39;00m \u001b[39mTrue\u001b[39;00m:\n\u001b[0;32m--> 318\u001b[0m     version, status, reason \u001b[39m=\u001b[39m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_read_status()\n\u001b[1;32m    319\u001b[0m     \u001b[39mif\u001b[39;00m status \u001b[39m!=\u001b[39m CONTINUE:\n\u001b[1;32m    320\u001b[0m         \u001b[39mbreak\u001b[39;00m\n",
      "File \u001b[0;32m~/.pyenv/versions/3.11.3/lib/python3.11/http/client.py:279\u001b[0m, in \u001b[0;36mHTTPResponse._read_status\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    278\u001b[0m \u001b[39mdef\u001b[39;00m \u001b[39m_read_status\u001b[39m(\u001b[39mself\u001b[39m):\n\u001b[0;32m--> 279\u001b[0m     line \u001b[39m=\u001b[39m \u001b[39mstr\u001b[39m(\u001b[39mself\u001b[39m\u001b[39m.\u001b[39mfp\u001b[39m.\u001b[39mreadline(_MAXLINE \u001b[39m+\u001b[39m \u001b[39m1\u001b[39m), \u001b[39m\"\u001b[39m\u001b[39miso-8859-1\u001b[39m\u001b[39m\"\u001b[39m)\n\u001b[1;32m    280\u001b[0m     \u001b[39mif\u001b[39;00m \u001b[39mlen\u001b[39m(line) \u001b[39m>\u001b[39m _MAXLINE:\n\u001b[1;32m    281\u001b[0m         \u001b[39mraise\u001b[39;00m LineTooLong(\u001b[39m\"\u001b[39m\u001b[39mstatus line\u001b[39m\u001b[39m\"\u001b[39m)\n",
      "File \u001b[0;32m~/.pyenv/versions/3.11.3/lib/python3.11/socket.py:706\u001b[0m, in \u001b[0;36mSocketIO.readinto\u001b[0;34m(self, b)\u001b[0m\n\u001b[1;32m    704\u001b[0m \u001b[39mwhile\u001b[39;00m \u001b[39mTrue\u001b[39;00m:\n\u001b[1;32m    705\u001b[0m     \u001b[39mtry\u001b[39;00m:\n\u001b[0;32m--> 706\u001b[0m         \u001b[39mreturn\u001b[39;00m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_sock\u001b[39m.\u001b[39;49mrecv_into(b)\n\u001b[1;32m    707\u001b[0m     \u001b[39mexcept\u001b[39;00m timeout:\n\u001b[1;32m    708\u001b[0m         \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_timeout_occurred \u001b[39m=\u001b[39m \u001b[39mTrue\u001b[39;00m\n",
      "File \u001b[0;32m~/.pyenv/versions/3.11.3/lib/python3.11/ssl.py:1278\u001b[0m, in \u001b[0;36mSSLSocket.recv_into\u001b[0;34m(self, buffer, nbytes, flags)\u001b[0m\n\u001b[1;32m   1274\u001b[0m     \u001b[39mif\u001b[39;00m flags \u001b[39m!=\u001b[39m \u001b[39m0\u001b[39m:\n\u001b[1;32m   1275\u001b[0m         \u001b[39mraise\u001b[39;00m \u001b[39mValueError\u001b[39;00m(\n\u001b[1;32m   1276\u001b[0m           \u001b[39m\"\u001b[39m\u001b[39mnon-zero flags not allowed in calls to recv_into() on \u001b[39m\u001b[39m%s\u001b[39;00m\u001b[39m\"\u001b[39m \u001b[39m%\u001b[39m\n\u001b[1;32m   1277\u001b[0m           \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m\u001b[39m__class__\u001b[39m)\n\u001b[0;32m-> 1278\u001b[0m     \u001b[39mreturn\u001b[39;00m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mread(nbytes, buffer)\n\u001b[1;32m   1279\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[1;32m   1280\u001b[0m     \u001b[39mreturn\u001b[39;00m \u001b[39msuper\u001b[39m()\u001b[39m.\u001b[39mrecv_into(buffer, nbytes, flags)\n",
      "File \u001b[0;32m~/.pyenv/versions/3.11.3/lib/python3.11/ssl.py:1134\u001b[0m, in \u001b[0;36mSSLSocket.read\u001b[0;34m(self, len, buffer)\u001b[0m\n\u001b[1;32m   1132\u001b[0m \u001b[39mtry\u001b[39;00m:\n\u001b[1;32m   1133\u001b[0m     \u001b[39mif\u001b[39;00m buffer \u001b[39mis\u001b[39;00m \u001b[39mnot\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[0;32m-> 1134\u001b[0m         \u001b[39mreturn\u001b[39;00m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_sslobj\u001b[39m.\u001b[39;49mread(\u001b[39mlen\u001b[39;49m, buffer)\n\u001b[1;32m   1135\u001b[0m     \u001b[39melse\u001b[39;00m:\n\u001b[1;32m   1136\u001b[0m         \u001b[39mreturn\u001b[39;00m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39m_sslobj\u001b[39m.\u001b[39mread(\u001b[39mlen\u001b[39m)\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "# Fetching Data\n",
    "\n",
    "for k,letter in enumerate(alphabet_letters):\n",
    "    # First request to get the number of total papers to limit the loop\n",
    "    params = {\n",
    "        'query': alphabet_letters[k],  # Use the first research field from the list\n",
    "        'venue': journal[0],\n",
    "        'year': year_of_publication,\n",
    "        'limit': 1,  # Number of results per request\n",
    "        'offset': 0,  # Offset for pagination\n",
    "        'fields': 'title,authors,abstract,citationCount,year',\n",
    "    }\n",
    "\n",
    "    # Make the GET request\n",
    "    response = requests.get(api_url, params=params)\n",
    "\n",
    "    # Check if the request was successful\n",
    "    if response.status_code == 200:\n",
    "        data = json.loads(response.text)\n",
    "    total = data['total']\n",
    "\n",
    "    # calculate the batches required to get every publication for that topic\n",
    "    num_batches, remainder = divmod(data['total'], number_of_publications_per_request)\n",
    "\n",
    "    batches = [number_of_publications_per_request] * num_batches\n",
    "    print(\"For the letter: \", letter ,\" batches: \",batches)\n",
    "    if remainder > 0:\n",
    "        batches.append(remainder)\n",
    "\n",
    "    for i,batch in enumerate(batches):\n",
    "        offset = initial_offset + number_of_publications_per_request * i  # Calculate the current offset\n",
    "        retries = 0\n",
    "\n",
    "        while retries < max_retries:\n",
    "            # Define query parameters\n",
    "            params = {\n",
    "                'query': alphabet_letters[k],  # Use the first research field from the list\n",
    "                'venue': journal[0],\n",
    "                'year': year_of_publication,\n",
    "                'limit': batch,  # Number of results per request\n",
    "                'offset': offset,  # Offset for pagination\n",
    "                'fields': 'title,authors,abstract,citationCount,year',\n",
    "            }\n",
    "\n",
    "            # Make the GET request\n",
    "            response = requests.get(api_url, params=params)\n",
    "            print(\"total amount\",response.json())\n",
    "\n",
    "            # Check if the request was successful\n",
    "            if response.status_code == 200:\n",
    "                data = json.loads(response.text)\n",
    "\n",
    "                # Create a DataFrame for unfiltered results\n",
    "                df = pd.DataFrame(data['data'])\n",
    "\n",
    "                break  # Successful request, exit the retry loop\n",
    "            else:\n",
    "                print(f\"Error (Attempt {retries + 1}):\", response.status_code)\n",
    "                retries += 1\n",
    "                if retries < max_retries:\n",
    "                    print(\"Retrying after sleep...\")\n",
    "                    time.sleep(sleep_duration)\n",
    "        print(\"total amount\",response.json())\n",
    "        length_df += len(df)  # Update the total length of unfiltered data\n",
    "\n",
    "        # Save each iteration into a separate file - to keep some of the data in case of an error\n",
    "        # The filename format includes the research field, offset, and is converted to lowercase\n",
    "        df.to_csv('./data/' + alphabet_letters[k].replace(\" \", \"_\").lower() + \"_\" + str(offset) + '.csv', index=False)\n",
    "\n",
    "        # Sleep for a specified duration between requests\n",
    "        print(\"Research field\", alphabet_letters[k], \"Iteration: \", i)\n",
    "        time.sleep(sleep_duration)\n",
    "\n",
    "# Print the total length of unfiltered data after all iterations\n",
    "print(\"Length of the unfiltered data\", length_df)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>paperId</th>\n",
       "      <td>5db4916130c2001927bbaccd5a3afb3872d8e6ac</td>\n",
       "      <td>765fc7d33627119d652113e6d065d047f8f6f9bf</td>\n",
       "      <td>fc975f2cf0473cdffeef8a8093f61e212f8edf6d</td>\n",
       "      <td>fc5e34c8d420cc464beb769eab57cf6baabb11af</td>\n",
       "      <td>a4d4027465ecdae907ed897d733609e0f49cd980</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>title</th>\n",
       "      <td>Authors’ 2015 additions to the IOC consensus s...</td>\n",
       "      <td>High Order Exponential Integrators for Nonline...</td>\n",
       "      <td>Classification of 4-dimensional homogeneous we...</td>\n",
       "      <td>Marginal and Irrelevant Disorder in Einstein-M...</td>\n",
       "      <td>Five-parameter class of solutions to the vacuu...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>abstract</th>\n",
       "      <td>In April 2014, the International Olympic Commi...</td>\n",
       "      <td>This article deals with the numerical integrat...</td>\n",
       "      <td>None</td>\n",
       "      <td>A. M. G. thanks Hong Liu and Elias Kiritsis fo...</td>\n",
       "      <td>We present a new five-parameter class of Ricci...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>year</th>\n",
       "      <td>2015</td>\n",
       "      <td>2015</td>\n",
       "      <td>2015</td>\n",
       "      <td>2015</td>\n",
       "      <td>2015</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>citationCount</th>\n",
       "      <td>59</td>\n",
       "      <td>31</td>\n",
       "      <td>15</td>\n",
       "      <td>13</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>authors</th>\n",
       "      <td>[{'authorId': '3888843', 'name': 'M. Mountjoy'...</td>\n",
       "      <td>[{'authorId': '1869826', 'name': 'C. Besse'}, ...</td>\n",
       "      <td>[{'authorId': '1405423205', 'name': 'T. Arias-...</td>\n",
       "      <td>[{'authorId': '1397392420', 'name': 'Antonio M...</td>\n",
       "      <td>[{'authorId': '2144838481', 'name': 'Yu Chen'}...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                               0   \n",
       "paperId                 5db4916130c2001927bbaccd5a3afb3872d8e6ac  \\\n",
       "title          Authors’ 2015 additions to the IOC consensus s...   \n",
       "abstract       In April 2014, the International Olympic Commi...   \n",
       "year                                                        2015   \n",
       "citationCount                                                 59   \n",
       "authors        [{'authorId': '3888843', 'name': 'M. Mountjoy'...   \n",
       "\n",
       "                                                               1   \n",
       "paperId                 765fc7d33627119d652113e6d065d047f8f6f9bf  \\\n",
       "title          High Order Exponential Integrators for Nonline...   \n",
       "abstract       This article deals with the numerical integrat...   \n",
       "year                                                        2015   \n",
       "citationCount                                                 31   \n",
       "authors        [{'authorId': '1869826', 'name': 'C. Besse'}, ...   \n",
       "\n",
       "                                                               2   \n",
       "paperId                 fc975f2cf0473cdffeef8a8093f61e212f8edf6d  \\\n",
       "title          Classification of 4-dimensional homogeneous we...   \n",
       "abstract                                                    None   \n",
       "year                                                        2015   \n",
       "citationCount                                                 15   \n",
       "authors        [{'authorId': '1405423205', 'name': 'T. Arias-...   \n",
       "\n",
       "                                                               3   \n",
       "paperId                 fc5e34c8d420cc464beb769eab57cf6baabb11af  \\\n",
       "title          Marginal and Irrelevant Disorder in Einstein-M...   \n",
       "abstract       A. M. G. thanks Hong Liu and Elias Kiritsis fo...   \n",
       "year                                                        2015   \n",
       "citationCount                                                 13   \n",
       "authors        [{'authorId': '1397392420', 'name': 'Antonio M...   \n",
       "\n",
       "                                                               4  \n",
       "paperId                 a4d4027465ecdae907ed897d733609e0f49cd980  \n",
       "title          Five-parameter class of solutions to the vacuu...  \n",
       "abstract       We present a new five-parameter class of Ricci...  \n",
       "year                                                        2015  \n",
       "citationCount                                                 11  \n",
       "authors        [{'authorId': '2144838481', 'name': 'Yu Chen'}...  "
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.head().T"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Single Query - Using the Batch Endpoint"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "api_url = 'https://api.semanticscholar.org/graph/v1/paper/search'\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Rest of the code is not required but it provides example how to filter the data before saving"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# # Fetching and filtering at the same time if required\n",
    "# # Fetching Data\n",
    "# length_filtered_df = 0\n",
    "# length_unfiltered_df = 0\n",
    "\n",
    "# for i in range(int(amount_of_data/100)):\n",
    "#     offset = initial_offset + 100 * i\n",
    "#     retries = 0\n",
    "    \n",
    "#     while retries < max_retries:\n",
    "#         # Define query parameters\n",
    "#         params = {\n",
    "#             'query': research_field,\n",
    "#             'year': year_of_publication,\n",
    "#             'limit': 100,\n",
    "#             'offset': offset,\n",
    "#             'fields': 'title,authors,abstract,citationCount,year',\n",
    "#         }\n",
    "\n",
    "#         # Make the GET request\n",
    "#         response = requests.get(api_url, params=params)\n",
    "\n",
    "#         # Check if the request was successful\n",
    "#         if response.status_code == 200:\n",
    "#             data = json.loads(response.text)\n",
    "\n",
    "#             # Create a DataFrame for unfiltered results\n",
    "#             df_unfiltered = pd.DataFrame(data['data'])\n",
    "\n",
    "#             # Filter the results based on 'citationCount' > 10\n",
    "#             filtered_results = [paper for paper in data['data'] if paper.get('citationCount', 0) > 10]\n",
    "\n",
    "#             # Create a DataFrame for filtered results\n",
    "#             df_filtered = pd.DataFrame(filtered_results)\n",
    "\n",
    "\n",
    "#             break  # Successful request, exit the retry loop\n",
    "#         else:\n",
    "#             print(f\"Error (Attempt {retries + 1}):\", response.status_code)\n",
    "#             retries += 1\n",
    "#             if retries < max_retries:\n",
    "#                 print(\"Retrying after sleep...\")\n",
    "#                 time.sleep(sleep_duration)\n",
    "    \n",
    "#     # length_filtered_df = len(df_filtered)\n",
    "#     length_unfiltered_df += len(df_unfiltered)\n",
    "\n",
    "#     # Save each iteration into separate file - to keep some of the data in case of an error\n",
    "#     df_unfiltered.to_csv('./data/'+research_field.replace(\" \",\"_\").lower() + \"_\" + str(offset) + '.csv', index=False)\n",
    "#     df_filtered.to_csv('./data/df_filtered' + str(offset) + '.csv', index=False)\n",
    "\n",
    "#     # Sleep for a specified duration between requests\n",
    "#     print(\"Research field\",research_field,\"Iteration: \", i)\n",
    "#     time.sleep(sleep_duration)\n",
    "# print(\"Length of the filtered data\",length_unfiltered_df)\n",
    "# print(\"Length of the unfiltered data\",length_filtered_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# https://api.semanticscholar.org/graph/v1/paper/search?query=computer+science&year=2015&fields=title,year,authors,citationCount&limit=50"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.3"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
